<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
    
    <title>Vishvas&#39;s notes  | 2 Encounters with Neanderthals</title>
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">

    <meta name="viewport" content="width=device-width,minimum-scale=1">
    <meta name="generator" content="Hugo 0.143.0">
    
    
      <META NAME="ROBOTS" CONTENT="INDEX, FOLLOW">
    

    <link href="/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/" rel="alternate" type="application/rss+xml" title="Vishvas&#39;s notes" />
    <link href="/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/" rel="feed" type="application/rss+xml" title="Vishvas&#39;s notes" />

    <meta property="og:url" content="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/">
  <meta property="og:site_name" content="Vishvas&#39;s notes">
  <meta property="og:title" content="2 Encounters with Neanderthals">
  <meta property="og:description" content="The Meeting of Neanderthals and Modern Humans
Today, the particular subgroup of humans to which we belong‚Äîmodern humans‚Äîis alone on our planet. We outcompeted or exterminated other humans, mostly during the period after around fifty thousand years ago when modern humans expanded throughout Eurasia and when major movements of humans likely happened within Africa too. Today, our closest living relatives are the African apes: the chimpanzees, bonobos, and gorillas, all incapable of making sophisticated tools or using conceptual language. But until around forty thousand years ago, the world was inhabited by multiple groups of archaic humans who differed from us physically but walked upright and shared many of our capabilities. The question that the archaeological record cannot answer‚Äîbut the DNA record can‚Äîis how those archaic people were related to us.">
  <meta property="og:locale" content="en_us">
  <meta property="og:type" content="article">
    <meta property="article:section" content="sapiens">

  <meta itemprop="name" content="2 Encounters with Neanderthals">
  <meta itemprop="description" content="The Meeting of Neanderthals and Modern Humans
Today, the particular subgroup of humans to which we belong‚Äîmodern humans‚Äîis alone on our planet. We outcompeted or exterminated other humans, mostly during the period after around fifty thousand years ago when modern humans expanded throughout Eurasia and when major movements of humans likely happened within Africa too. Today, our closest living relatives are the African apes: the chimpanzees, bonobos, and gorillas, all incapable of making sophisticated tools or using conceptual language. But until around forty thousand years ago, the world was inhabited by multiple groups of archaic humans who differed from us physically but walked upright and shared many of our capabilities. The question that the archaeological record cannot answer‚Äîbut the DNA record can‚Äîis how those archaic people were related to us.">
  <meta itemprop="wordCount" content="8431">
  <meta name="twitter:card" content="summary">
  <meta name="twitter:title" content="2 Encounters with Neanderthals">
  <meta name="twitter:description" content="The Meeting of Neanderthals and Modern Humans
Today, the particular subgroup of humans to which we belong‚Äîmodern humans‚Äîis alone on our planet. We outcompeted or exterminated other humans, mostly during the period after around fifty thousand years ago when modern humans expanded throughout Eurasia and when major movements of humans likely happened within Africa too. Today, our closest living relatives are the African apes: the chimpanzees, bonobos, and gorillas, all incapable of making sophisticated tools or using conceptual language. But until around forty thousand years ago, the world was inhabited by multiple groups of archaic humans who differed from us physically but walked upright and shared many of our capabilities. The question that the archaeological record cannot answer‚Äîbut the DNA record can‚Äîis how those archaic people were related to us.">
<script type="text/javascript">
    
    let baseURL = "https:\/\/vishvAsa.github.io\/notes\/";
    let basePath = "/" + baseURL.split("/").slice(3).join("/");
    let siteParams = JSON.parse("{\u0022build_img\u0022:\u0022https:\/\/github.com\/vishvAsa\/notes\/actions\/workflows\/build.yml\/badge.svg\u0022,\u0022build_url\u0022:\u0022https:\/\/github.com\/vishvAsa\/notes\/actions\/workflows\/build.yml\u0022,\u0022contactlink\u0022:\u0022https:\/\/github.com\/vishvAsa\/notes\/issues\/new\u0022,\u0022env\u0022:\u0022production\u0022,\u0022githubeditmepathbase\u0022:\u0022https:\/\/github.com\/vishvAsa\/notes\/edit\/content\/\u0022,\u0022js_extras\u0022:[\u0022mathjax\u0022]}");
    let pageDefaultsList = JSON.parse("[{\u0022scope\u0022:{\u0022pathPrefix\u0022:\u0022\u0022},\u0022values\u0022:{\u0022comments\u0022:true,\u0022layout\u0022:\u0022page\u0022,\u0022search\u0022:true,\u0022sidebar\u0022:\u0022home_sidebar\u0022}}]");
    let sidebarsData = JSON.parse("{\u0022home_sidebar\u0022:{\u0022contents\u0022:[{\u0022url\u0022:\u0022recdir:\/\/\u0022},{\u0022contents\u0022:[{\u0022title\u0022:\u0022‡§ú‡•ç‡§Ø‡•å‡§§‡§ø‡§∑‡§Æ‡•ç\u0022,\u0022url\u0022:\u0022..\/jyotiSham\/\u0022},{\u0022title\u0022:\u0022‡§∏‡§Ç‡§∏‡•ç‡§ï‡•É‡§§‡§Æ‡•ç\u0022,\u0022url\u0022:\u0022..\/sanskrit\/\u0022},{\u0022title\u0022:\u0022‡§¶‡•á‡§µ‡§É\u0022,\u0022url\u0022:\u0022..\/devaH\/\u0022},{\u0022title\u0022:\u0022‡§Ü‡§ó‡§Æ‡§É\u0022,\u0022url\u0022:\u0022..\/AgamaH\/\u0022},{\u0022title\u0022:\u0022‡§Ü‡§ó‡§Æ‡§É ‡§∂‡•à‡§µ‡§É\u0022,\u0022url\u0022:\u0022..\/AgamaH_shaivaH\/\u0022},{\u0022title\u0022:\u0022‡§Ü‡§ó‡§Æ‡§É ‡§µ‡•à‡§∑‡•ç‡§£‡§µ‡§É\u0022,\u0022url\u0022:\u0022..\/AgamaH_vaiShNavaH\/\u0022},{\u0022title\u0022:\u0022‡§∞‡§æ‡§Æ‡§æ‡§®‡•Å‡§ú‡•Ä‡§Ø‡§Æ‡•ç\u0022,\u0022url\u0022:\u0022..\/rAmAnujIyam\/\u0022},{\u0022title\u0022:\u0022‡§Æ‡§æ‡§ß‡•ç‡§µ‡§Æ‡•ç\u0022,\u0022url\u0022:\u0022..\/mAdhvam\/\u0022},{\u0022title\u0022:\u0022‡§Ü‡§ó‡§Æ‡§É ‡§¨‡•ç‡§∞‡§æ‡§π‡•ç‡§Æ‡§É\u0022,\u0022url\u0022:\u0022..\/AgamaH_brAhmaH\/\u0022},{\u0022title\u0022:\u0022‡§ï‡§≤‡•ç‡§™‡§æ‡§®‡•ç‡§§‡§∞‡§Æ‡•ç\u0022,\u0022url\u0022:\u0022..\/kalpAntaram\/\u0022},{\u0022title\u0022:\u0022Notes Home\u0022,\u0022url\u0022:\u0022..\/notes\/\u0022},{\u0022title\u0022:\u0022‡§µ‡•á‡§¶‡§æ‡§É\u0022,\u0022url\u0022:\u0022..\/vedAH\/\u0022},{\u0022title\u0022:\u0022‡§µ‡•á‡§¶‡§æ‡§É - ‡§ã‡§ï‡•ç\u0022,\u0022url\u0022:\u0022..\/vedAH_Rk\/\u0022},{\u0022title\u0022:\u0022‡§µ‡•á‡§¶‡§æ‡§É - ‡§Ø‡§ú‡•Å‡§É\u0022,\u0022url\u0022:\u0022..\/vedAH_yajuH\/\u0022},{\u0022title\u0022:\u0022‡§µ‡•á‡§¶‡§æ‡§É - ‡§∏‡§æ‡§Æ\u0022,\u0022url\u0022:\u0022..\/vedAH_sAma\/\u0022},{\u0022title\u0022:\u0022‡§™‡•Å‡§∞‡§æ‡§£‡§Æ‡•ç\u0022,\u0022url\u0022:\u0022..\/purANam\u0022},{\u0022title\u0022:\u0022‡§™‡•Å‡§∞‡§æ‡§£‡§Æ‡•ç ‡§µ‡•à‡§∑‡•ç‡§£‡§µ‡§Æ‡•ç\u0022,\u0022url\u0022:\u0022..\/purANam_vaiShNavam\/\u0022},{\u0022title\u0022:\u0022‡§Æ‡§π‡§æ‡§≠‡§æ‡§∞‡§§‡§Æ‡•ç\u0022,\u0022url\u0022:\u0022..\/mahAbhAratam\/\u0022},{\u0022title\u0022:\u0022‡§∞‡§æ‡§Æ‡§æ‡§Ø‡§£‡§Æ‡•ç\u0022,\u0022url\u0022:\u0022..\/rAmAyaNam\/\u0022},{\u0022title\u0022:\u0022‡§ï‡§æ‡§µ‡•ç‡§Ø‡§Æ‡•ç\u0022,\u0022url\u0022:\u0022..\/kAvyam\/\u0022},{\u0022title\u0022:\u0022‡§Æ‡•Ä‡§Æ‡§æ‡§Ç‡§∏‡§æ\u0022,\u0022url\u0022:\u0022..\/mImAMsA\/\u0022},{\u0022title\u0022:\u0022‡≤ï‡≤®‡≥ç‡≤®‡≤°\u0022,\u0022url\u0022:\u0022..\/kannaDa\/\u0022},{\u0022title\u0022:\u0022‡§§‡•ç‡§∞‡§ø‡§™‡§ø‡§ü‡§ï‡§Æ‡•ç\u0022,\u0022url\u0022:\u0022..\/tipiTaka\/\u0022},{\u0022title\u0022:\u0022‡§™‡§æ‡§≥‡§Ø‡§É\u0022,\u0022url\u0022:\u0022..\/pALi\/\u0022},{\u0022title\u0022:\u0022‡§≠‡§æ‡§∑‡§æ‡§®‡•ç‡§§‡§∞‡§Æ‡•ç\u0022,\u0022url\u0022:\u0022..\/bhAShAntaram\/\u0022},{\u0022title\u0022:\u0022English Lit\u0022,\u0022url\u0022:\u0022..\/english\/\u0022},{\u0022title\u0022:\u0022Book pub\u0022,\u0022url\u0022:\u0022..\/book-pub\u0022},{\u0022title\u0022:\u0022Vishvas\u0027s home page\u0022,\u0022url\u0022:\u0022..\/\u0022}],\u0022title\u0022:\u0022‡§∏‡§ô‡•ç‡§ó‡•ç‡§∞‡§π‡§æ‡§®‡•ç‡§§‡§∞‡§Æ‡•ç\u0022},{\u0022title\u0022:\u0022\\u003ci class=\\\u0022fas fa-hand-holding-heart\\\u0022\\u003e\\u003c\/i\\u003eDonate Via Vishvas\u0022,\u0022url\u0022:\u0022https:\/\/vishvAsa.github.io\/interests\/dharma-via-vishvas\/\u0022}],\u0022title\u0022:\u0022Parts\u0022}}");
    
    var pageRelUrlTree = {};
</script>

    
    <script type="text/javascript">
  let pageVars = {};
  pageVars.pageUrlMinusBasePath = "\/notes\/sapiens\/branches\/articles\/reich_who-we-are\/1_The_Deep_History_of_Our_Species\/2_Encounters_with_Neanderthals\/".replace(basePath, "/");
  pageVars.pageParams = {};
  pageVars.pageSource = "sapiens\/branches\/articles\/reich_who-we-are\/1_The_Deep_History_of_Our_Species\/2_Encounters_with_Neanderthals.md";
  console.log(pageVars.pageSource);
  var pageDefaults;
  for (let possiblePageDefaults of pageDefaultsList) {
    if (pageVars.pageSource.startsWith(possiblePageDefaults.scope.pathPrefix)) {
      pageDefaults = possiblePageDefaults.values
    }
  }
    
</script>


    <script src="/notes/webpack_dist/uiLib-bundle.js"></script>


<script>


MathJax = {
  tex: {
    inlineMath: [ ["@#", "#@"], ["\\(", "\\)"] ],
    displayMath: [ ["$$","$$"], ["\\[", "\\]"] ],
    processEscapes: true,  
  },
  svg: {
    fontCache: 'global'
  },
  options: {
    ignoreHtmlClass: "tex2jax_ignore|dno"
  }
};
</script>
<script type="text/javascript" id="MathJax-script" async
        src="/notes/non_webpack_js/mathjax/tex-svg.js">
</script>






    <link rel="stylesheet" href="/notes/webpack_dist/uiLib.css">
<link rel="stylesheet" href="/notes/css/uiLibPrint.css" media="print" >
<link rel="stylesheet" href="/notes/css/fonts.css" >
<link rel="stylesheet" href="/notes/css/@fortawesome/fontawesome-free/css/solid.min.css">

<link rel="stylesheet" href="/notes/css/@fortawesome/fontawesome-free/css/fontawesome.min.css">


<link rel="alternate" hreflang="sa-Deva" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/" />
<link rel="alternate" hreflang="sa-Deva" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=devanagari" />

<link rel="alternate" hreflang="sa-Knda" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=kannada" />
<link rel="alternate" hreflang="sa-Mlym" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=malayalam" />
<link rel="alternate" hreflang="sa-Telu" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=telugu" />
<link rel="alternate" hreflang="sa-Taml-t-sa-Taml-m0-superscript" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=tamil_superscripted" />
<link rel="alternate" hreflang="sa-Taml" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=tamil" />
<link rel="alternate" hreflang="sa-Gran" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=grantha" />

<link rel="alternate" hreflang="sa-Gujr" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=gujarati" />
<link rel="alternate" hreflang="sa-Guru" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=gurmukhi" />
<link rel="alternate" hreflang="sa-Shar" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=sharada" />
<link rel="alternate" hreflang="sa-Dogr" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=dogra" />
<link rel="alternate" hreflang="sa-Phag" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=phags_pa" />
<link rel="alternate" hreflang="sa-Tibt" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=tibetan" />
<link rel="alternate" hreflang="sa-Sidd" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=siddham" />
<link rel="alternate" hreflang="sa-Newa" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=newa" />
<link rel="alternate" hreflang="sa-Modi" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=modi" />
<link rel="alternate" hreflang="sa-Tirh" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=tirhuta_maithili" />
<link rel="alternate" hreflang="sa-Gong" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=gondi_gujala" />
<link rel="alternate" hreflang="sa-Gonm" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=gondi_masaram" />


<link rel="alternate" hreflang="sa-Orya" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=oriya" />
<link rel="alternate" hreflang="sa-Beng-t-sa-Beng-m0-assamese" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=assamese" />
<link rel="alternate" hreflang="sa-Beng" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=bengali" />
<link rel="alternate" hreflang="sa-Mtei" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=manipuri" />
<link rel="alternate" hreflang="sa-Sinh" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=sinhala" />
<link rel="alternate" hreflang="sa-Limb" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=limbu" />
<link rel="alternate" hreflang="sa-Lepc" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=lepcha" />


<link rel="alternate" hreflang="sa-Cyrl" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=cyrillic" />
<link rel="alternate" hreflang="sa-Latn-t-sa-Zyyy-m0-iso" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=iso" />
<link rel="alternate" hreflang="sa-Latn-t-sa-Zyyy-m0-iast" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=iast" />
<link rel="alternate" hreflang="sa-Aran" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=urdu" />
<link rel="alternate" hreflang="sa-Avst" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=avestan" />

<link rel="alternate" hreflang="sa-Brah" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=brahmi" />
<link rel="alternate" hreflang="sa-Khar" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=kharoshthi" />
<link rel="alternate" hreflang="sa-Zanb" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=zanbazar_square" />
<link rel="alternate" hreflang="sa-Bhks" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=bhaiksuki" />

<link rel="alternate" hreflang="sa-Bali" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=balinese" />
<link rel="alternate" hreflang="sa-Java" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=javanese" />
<link rel="alternate" hreflang="sa-Laoo" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=lao" />
<link rel="alternate" hreflang="sa-Laoo-t-sa-Laoo-m0-Laopali" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=lao_pali" />
<link rel="alternate" hreflang="sa-Mymr" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=burmese" />
<link rel="alternate" hreflang="sa-Mymr-t-sa-Mymr-m0-Mon" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=mon" />
<link rel="alternate" hreflang="sa-Cham" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=cham" />
<link rel="alternate" hreflang="sa-Khmr" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=khmer" />
<link rel="alternate" hreflang="sa-Thai" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/?transliteration_target=thai" />

  </head>

  <body class="ma0 bg-near-white">
    
\(
% groupings of objects.
\newcommand{\set}[1]{\left\{ #1 \right\}}
\newcommand{\seq}[1]{\left(#1\right)}
\newcommand{\ang}[1]{\langle#1\rangle}
\newcommand{\tuple}[1]{\left(#1\right)}
\newcommand{\size}[1]{\left| #1\right|}

\newcommand{\comp}{\circ}


% numerical shortcuts.
\newcommand{\abs}[1]{\left| #1\right|}
\newcommand{\floor}[1]{\left\lfloor #1 \right\rfloor}
\newcommand{\ceil}[1]{\left\lceil #1 \right\rceil}

% linear algebra shortcuts.
\newcommand{\change}{\Delta}
\newcommand{\norm}[1]{\left\| #1\right\|}
\newcommand{\dprod}[1]{\langle#1\rangle}
\newcommand{\linspan}[1]{\langle#1\rangle}
\newcommand{\conj}[1]{\overline{#1}}
\newcommand{\der}[1]{\frac{d#1}{dx}}
\newcommand{\lap}{\Delta}
\newcommand{\kron}{\otimes}
\newcommand{\nperp}{\nvdash}

\newcommand{\mat}[1]{\left[ \begin{smallmatrix}#1 \end{smallmatrix} \right]}

% derivatives and limits
\newcommand{\partder}[2]{\frac{\partial #1}{\partial #2}}
\newcommand{\partdern}[3]{\frac{\partial^{#3 #1}}{\partial #2^{#3}}}
\newcommand{\gradient}{\nabla}
\newcommand{\subdifferential}{\partial}

% Arrows
\newcommand{\diverge}{\nearrow}
\newcommand{\notto}{\nrightarrow}
\newcommand{\up}{\uparrow}
\newcommand{\down}{\downarrow}
% gets and gives are defined!

% ordering operators
\newcommand{\oleq}{\preceq}
\newcommand{\ogeq}{\succeq}

% programming and logic operators
\newcommand{\dfn}{:=}
\newcommand{\assign}{:=}
\newcommand{\co}{\ co\ }
\newcommand{\en}{\ en\ }


% logic operators
\newcommand{\xor}{\oplus}
\newcommand{\Land}{\bigwedge}
\newcommand{\Lor}{\bigvee}
\newcommand{\finish}{\Box}
\newcommand{\contra}{\Rightarrow \Leftarrow}
\newcommand{\iseq}{\stackrel{_?{=}}}


% Set theory
\newcommand{\symdiff}{\Delta}
\newcommand{\setdiff}{\backslash}
\newcommand{\union}{\cup}
\newcommand{\inters}{\cap}
\newcommand{\Union}{\bigcup}
\newcommand{\Inters}{\bigcap}
\newcommand{\nullSet}{\phi}


% graph theory
\newcommand{\nbd}{\Gamma}

% Script alphabets
% For reals, use \Re

% greek letters
\newcommand{\eps}{\epsilon}
\newcommand{\del}{\delta}
\newcommand{\ga}{\alpha}
\newcommand{\gb}{\beta}
\newcommand{\gd}{\del}
\newcommand{\gp}{\pi}
\newcommand{\gf}{\phi}
\newcommand{\gh}{\eta}
\newcommand{\gF}{\Phi}
\newcommand{\gl}{\lambda}
\newcommand{\gm}{\mu}
\newcommand{\gn}{\nu}
\newcommand{\gr}{\rho}
\newcommand{\gs}{\sigma}
\newcommand{\gth}{\theta}
\newcommand{\gx}{\xi}
\newcommand{\gw}{\omega}

\newcommand{\sw}{\sigma}
\newcommand{\SW}{\Sigma}
\newcommand{\ew}{\lambda}
\newcommand{\EW}{\Lambda}

\newcommand{\Del}{\Delta}
\newcommand{\gD}{\Delta}
\newcommand{\gG}{\Gamma}
\newcommand{\gW}{\Omega}
\newcommand{\gS}{\Sigma}
\newcommand{\gTh}{\Theta}

% Bold english letters.
\newcommand{\bA}{\mathbf{A}}
\newcommand{\bB}{\mathbf{B}}
\newcommand{\bC}{\mathbf{C}}
\newcommand{\bD}{\mathbf{D}}
\newcommand{\bE}{\mathbf{E}}
\newcommand{\bF}{\mathbf{F}}
\newcommand{\bG}{\mathbf{G}}
\newcommand{\bH}{\mathbf{H}}
\newcommand{\bI}{\mathbf{I}}
\newcommand{\bJ}{\mathbf{J}}
\newcommand{\bK}{\mathbf{K}}
\newcommand{\bL}{\mathbf{L}}
\newcommand{\bM}{\mathbf{M}}
\newcommand{\bN}{\mathbf{N}}
\newcommand{\bO}{\mathbf{O}}
\newcommand{\bP}{\mathbf{P}}
\newcommand{\bQ}{\mathbf{Q}}
\newcommand{\bR}{\mathbf{R}}
\newcommand{\bS}{\mathbf{S}}
\newcommand{\bT}{\mathbf{T}}
\newcommand{\bU}{\mathbf{U}}
\newcommand{\bV}{\mathbf{V}}
\newcommand{\bW}{\mathbf{W}}
\newcommand{\bX}{\mathbf{X}}
\newcommand{\bY}{\mathbf{Y}}
\newcommand{\bZ}{\mathbf{Z}}

\newcommand{\bba}{\mathbf{a}}
\newcommand{\bbb}{\mathbf{b}}
\newcommand{\bbc}{\mathbf{c}}
\newcommand{\bbd}{\mathbf{d}}
\newcommand{\bbe}{\mathbf{e}}
\newcommand{\bbf}{\mathbf{f}}
\newcommand{\bbg}{\mathbf{g}}
\newcommand{\bbh}{\mathbf{h}}
\newcommand{\bbk}{\mathbf{k}}
\newcommand{\bbl}{\mathbf{l}}
\newcommand{\bbm}{\mathbf{m}}
\newcommand{\bbn}{\mathbf{n}}
\newcommand{\bbp}{\mathbf{p}}
\newcommand{\bbq}{\mathbf{q}}
\newcommand{\bbr}{\mathbf{r}}
\newcommand{\bbs}{\mathbf{s}}
\newcommand{\bbt}{\mathbf{t}}
\newcommand{\bbu}{\mathbf{u}}
\newcommand{\bbv}{\mathbf{v}}
\newcommand{\bbw}{\mathbf{w}}
\newcommand{\bbx}{\mathbf{x}}
\newcommand{\bby}{\mathbf{y}}
\newcommand{\bbz}{\mathbf{z}}
\newcommand{\0}{\mathbf{0}}
\newcommand{\1}{\mathbf{1}}

% Caligraphic english alphabet
\newcommand{\cA}{\mathcal{A}}
\newcommand{\cB}{\mathcal{B}}
\newcommand{\cC}{\mathcal{C}}
\newcommand{\cD}{\mathcal{D}}
\newcommand{\cE}{\mathcal{E}}
\newcommand{\cF}{\mathcal{F}}
\newcommand{\cG}{\mathcal{G}}
\newcommand{\cH}{\mathcal{H}}
\newcommand{\cI}{\mathcal{I}}
\newcommand{\cJ}{\mathcal{J}}
\newcommand{\cK}{\mathcal{K}}
\newcommand{\cL}{\mathcal{L}}
\newcommand{\cM}{\mathcal{M}}
\newcommand{\cN}{\mathcal{N}}
\newcommand{\cO}{\mathcal{O}}
\newcommand{\cP}{\mathcal{P}}
\newcommand{\cQ}{\mathcal{Q}}
\newcommand{\cR}{\mathcal{R}}
\newcommand{\cS}{\mathcal{S}}
\newcommand{\cT}{\mathcal{T}}
\newcommand{\cU}{\mathcal{U}}
\newcommand{\cV}{\mathcal{V}}
\newcommand{\cW}{\mathcal{W}}
\newcommand{\cX}{\mathcal{X}}
\newcommand{\cY}{\mathcal{Y}}
\newcommand{\cZ}{\mathcal{Z}}


% Formatting shortcuts
\newcommand{\red}[1]{\textcolor{red}{#1}}
\newcommand{\blue}[1]{\textcolor{blue}{#1}}
\newcommand{\htext}[2]{\texorpdfstring{#1}{#2}}

% Statistics
\newcommand{\distr}{\sim}
\newcommand{\stddev}{\sigma}
\newcommand{\covmatrix}{\Sigma}
\newcommand{\mean}{\mu}
\newcommand{\param}{\theta}
\newcommand{\gthEst}{\hat{\theta}}
\newcommand{\ftr}{\phi}
\newcommand{\est}[1]{\hat{#1}}

% General utility
\newcommand{\todo}[1]{\textbf{[TODO]}] \footnote{TODO: #1}}
\newcommand{\tbc}{[\textbf{Incomplete}]}
\newcommand{\chk}{[\textbf{Check}]}
\newcommand{\why}{[\textbf{Find proof}]}
\newcommand{\opt}[1]{\textit{#1}}

\newcommand{\experience}[1]{[\textbf{Personal Experience}]: #1 \blacktriangle}
\newcommand{\pf}[1]{[\textbf{Proof}]: #1 \Box}
\newcommand{\core}[1]{\textbf{Core Idea}: #1 \Arrowvert}
\newcommand{\example}[1]{\textbf{Example}: #1 \blacktriangle}
\newcommand{\error}[1]{\textbf{Error alert}: #1 \triangle}
\newcommand{\oprob}{[\textbf{OP}]: }


\renewcommand{\~}{\htext{$\sim$}{~}}

\DeclareMathOperator*{\argmin}{arg\,min}
\DeclareMathOperator*{\argmax}{arg\,max}


\)



    
<header class="p-1 bg-yellow noPrint">
    <nav role="navigation" class="col">
      <div id="div_top_bar" class="row">
        <div class="col-md-3 justify-content-start">
          <a  href="/" class="btn col border">
            <i class="fas fa-gopuram ">Vishvas&#39;s notes <br/> 2 Encounters with Neanderthals</i>
          </a>
        </div>
        <div id="div_top_bar_right" class="col-md-auto d-flex justify-content-center">
          <button class="btn btn-light"  href="#transliterationDropdown" data-toggle="collapse">
            ‚úç
          </button>
          <select id="transliterationDropdown" size="1" class="collapse" onchange="module_uiLib.default.content.updateTransliteration()">
            <optgroup label="üåç">
              <option value="iso">ƒÅISO</option>
              <option value="iso_vedic">ƒÅi-ƒÅu</option>
              <option value="cyrillic">–ø—É</option>
              <option value="urdu">ŸÜÿ≥</option>
              <option value="??">üåç??</option>
            </optgroup>
            <optgroup label="üõïüß≠S">
              <option value="kannada">‡≤Ö</option>
              <option value="malayalam">‡¥Ö</option>
              <option value="telugu">‡∞ï</option>
              <option value="tamil_superscripted">‡Æï¬≤</option>
              <option value="tamil_extended">‡Æï</option>
              <option value="grantha">ëåÖ</option>
            </optgroup>
            <optgroup label="üõïüß≠N">
              <option value="devanagari" selected="">‡§∏</option>
              <option value="gujarati">‡™Ö</option>
              <option value="gurmukhi">‡®Ö</option>
            </optgroup>
            <optgroup label="üõïüß≠E">
              <option value="oriya">‡¨Ö</option>
              <option value="assamese">‡¶Ö‡¶∏</option>
              <option value="bengali">‡¶Ö</option>
            </optgroup>
            <optgroup label="üõïüè∫üìú">
              <option value="sharada">ëÜëëáÄëÜ∞</option>
              <option value="brahmi">ëÄÖ</option>
            </optgroup>
            <optgroup label="üõïüåã">
              <option value="balinese">·¨©·¨Æ·¨∂</option>
            </optgroup>
          </select>
          <div>
          <button class="btn btn-outline-secondary btn-sm"  href="#transliterationDropdownAlt" data-toggle="collapse">
          (‚úç)
          </button>
          <select id="transliterationDropdownAlt" size="1"  style="font-size:12px" class="bg-light-yellow collapse" onchange="module_uiLib.default.content.updateTransliteration()">
            <optgroup label="üåç">
              <option value="iso">ƒÅISO</option>
              <option value="iso_vedic">ƒÅi-ƒÅu</option>
              <option value="cyrillic">–ø—É</option>
              <option value="urdu">ŸÜÿ≥</option>
              <option value="??">üåç??</option>
            </optgroup>
            <optgroup label="üõïüß≠S">
              <option value="kannada">‡≤Ö</option>
              <option value="malayalam">‡¥Ö</option>
              <option value="telugu">‡∞ï</option>
              <option value="tamil_superscripted">‡Æï¬≤</option>
              <option value="tamil_extended">‡Æï</option>
              <option value="grantha">ëåÖ</option>
            </optgroup>
            <optgroup label="üõïüß≠N">
              <option value="devanagari" selected="">‡§∏</option>
              <option value="gujarati">‡™Ö</option>
              <option value="gurmukhi">‡®Ö</option>
            </optgroup>
            <optgroup label="üõïüß≠E">
              <option value="oriya">‡¨Ö</option>
              <option value="assamese">‡¶Ö‡¶∏</option>
              <option value="bengali">‡¶Ö</option>
            </optgroup>
            <optgroup label="üõïüè∫üìú">
              <option value="sharada">ëÜëëáÄëÜ∞</option>
              <option value="brahmi">ëÄÖ</option>
            </optgroup>
            <optgroup label="üõïüåã">
              <option value="balinese">·¨©·¨Æ·¨∂</option>
            </optgroup>
          </select>
          </div>
          <div class="row col d-flex  dropdown">
            <button class="btn btn-secondary-outline btn-sm dropdown-toggle" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
              A¬±
            </button>
            <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
            <a class="btn btn-secondary btn-sm" onclick="module_uiLib.changeTextSize(2)">A+</a>
            <a class="btn btn-secondary btn-sm" onclick="module_uiLib.changeTextSize(-2)">A-</a>
            </div>
          </div>
          <button class="btn btn-light btn-sm"  href="#commentStyleDropdown" data-toggle="collapse">
            üó®
          </button>
          <select id="commentStyleDropdown" class="collapse" size="1" onchange="module_uiLib.default.content.updateCommentStyleFromDropdown()">
            <option value="on">‚úî</option>
            <option value="hidden">‚úó</option>
          </select>
          <div><a id="expandAllButton" class="btn btn-secondary" onclick="module_uiLib.default.query.setParamsAndGo({'expandDetails': '.*'})">‚ÜïÔ∏èÔ∏è</a></div>
          <div id="printStyleDiv" class="row col d-flex  dropdown">
          <button class="btn btn-light btn-sm dropdown-toggle"  href="#printStyleDropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
          üñ®
          </button>
            <div class="dropdown-menu col" aria-labelledby="printStyleDropdown">
              <div class="row justify-content-center">
                <p>Cols:</p>
                <input type="number" min="1" max="10" value="2" step="1" id="tbColumns" style="width: 3ch;"  accesskey="c"/>
              </div>
              <div class="row justify-content-center">
                <p>A¬±:</p> <input type="text" id="tbBodyFontSize" value="0.4cm" style="width: 6ch;"  accesskey="f"></input>
              </div>
              <div class="row justify-content-center">
                <p>Incl:</p> <input checked type="checkbox" id="tbIncludeStyle" value="on"></input>
                <button id="btnPrintStyle" onclick="module_uiLib.default.content.updatePrintStyle()">üñ®
                </button>
              </div>
            </div>
        </div>
        </div>
        <div id="audioDiv" class="row col d-flex  dropdown">
          <button class="btn btn-secondary-outline btn-sm dropdown-toggle" type="button" id="audioDropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
            üéß
          </button>
          <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
          <a id="speakerButton" class="btn btn-outline-secondary btn-sm" onclick="module_uiLib.default.content.handleSpeakToggle()">‚èØ</a>
          <button id="speakerPauseButton" class="btn btn-outline-secondary btn-sm">‚è∏</button>
            <div id="audioOptionsDiv" class="row justify-content-center" >
              <div>
                <p class="lead">Vol</p>
                <input type="range" min="0" max="1" value="1" step="0.1" id="volume" />
                <span id="volume-label" class="ms-2">1</span>
              </div>
              <div class="mx-5">
                <p class="lead">Rate</p>
                <input type="range" min="0.1" max="10" value="1" id="rate" step="0.1" />
                <span id="rate-label" class="ms-2">1</span>
              </div>
              <div>
                <p class="lead">Pitch</p>
                <input type="range" min="0" max="2" value="1" step="0.1" id="pitch" />
                <span id="pitch-label" class="ms-2">1</span>
              </div>
            </div>
          </div>
        </div>
        <a href="https://vishvAsa.github.io/notes/search"  class="btn btn-light btn-sm">ü•Ωüî¶</a>
        <div class="row col  d-flex justify-content-center">
          <div><a id="previousPage" href="" class="btn btn-secondary">###<i class="fas fa-caret-left"></i></a></div>
          <div ><a id="nextPage" href="" class="btn btn-secondary"><i class="fas fa-caret-right"></i> ### </a></div>
        </div>
        <ul id="top-bar-right-custom" class="list-group list-group-horizontal">
        </ul>
      </div>
    </nav>
</header>

    
    <div class="row" name="contentRow">
      
      <aside class="col-md-3 card border noPrint " id="sidebar">
          <div id="fastSearch">
              <datalist id="pageDataList"></datalist>
              <input id="searchInput" placeholder="‡§∂‡•Ä‡§∞‡•ç‡§∑‡§ø‡§ï‡§æ‡§®‡•ç‡§µ‡§ø‡§∑‡•ç‡§Ø‡§§‡§æ‡§Æ‡•ç" list="pageDataList" onchange="module_uiLib.default.navigation.pageLoader()">
          </div>
          <details open  class="bg-light-gray" >
              <summary id="sidebarToggleLink" onclick="module_uiLib.default.navigation.sidebarToggleHandler()">
            Menu
              </summary>
              <ul id="sidebarBody" class="list pl2 p-0 bg-yellow">
              </ul>
          </details>
      </aside>
      <main class="col p-3" role="main" data-pagefind-body>
        
<header class='border d-flex justify-content-between'>
    <h1 id="2 Encounters with Neanderthals">2 Encounters with Neanderthals</h1>
    
    <a id="editLink" class="btn btn-primary noPrint"  href="https://github.com/vishvAsa/notes/edit/content/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals.md"><i class="fas fa-edit"></i></a>
    
</header>
<article>
  <aside id="toc_card" class="card border noPrint ">
    <details id="toc_header">
        <summary>What's in this page?</summary>
      <div id="toc_body" class="p-0">
        
        <ul id="toc_ul" class="list p-0">
        </ul>
      </div>
    </details>
    <div id="progressBarDiv" class="row d-flex justify-content-center">
      üêéüí™<progress id="progressLoading" value="0" max="100"> 32% </progress>
    </div>
  </aside>
  <div id="post_content">
  <figure class="">

    <a href="../images/000031.jpg" target="_blank" rel="noopener">
        <img src="../images/000031.jpg" alt="The Age of Neanderthals" title="">
    </a>

    

</figure>
<p>The Meeting of Neanderthals and Modern Humans</p>
<p>Today, the particular subgroup of humans to which we belong‚Äîmodern humans‚Äîis alone on our planet. We outcompeted or exterminated other humans, mostly during the period after around fifty thousand years ago when modern humans expanded throughout Eurasia and when major movements of humans likely happened within Africa too. Today, our closest living relatives are the African apes: the chimpanzees, bonobos, and gorillas, all incapable of making sophisticated tools or using conceptual language. But until around forty thousand years ago, the world was inhabited by multiple groups of archaic humans who differed from us physically but walked upright and shared many of our capabilities. The question that the archaeological record cannot answer‚Äîbut the DNA record can‚Äîis how those archaic people were related to us.</p>
<p>For no archaic group has the answer to this question seemed more urgent than for the Neanderthals. In Europe after four hundred thousand years ago, the landscape was dominated by these large-bodied people with brains slightly bigger on average than those of modern humans. The specimen that gave its name to Neanderthals was found in 1856 by miners in a limestone quarry in the Neander Valley (the German word for valley is <em>Thal</em> or <em>Tal</em>). For years, debate raged over whether these remains came from a deformed human, a human ancestor, or a human lineage that is extremely divergent from our own. Neanderthals became the first archaic humans to be recognized by science. In <em>The Descent of Man,</em> published in 1871, Charles Darwin argued that humans are like other animals in that they are also the products of evolution.1 Although Darwin didn‚Äôt himself appreciate their significance, Neanderthals were eventually acknowledged to be from a population more closely related to modern humans than to living apes, providing evidence for Darwin‚Äôs theory that such populations must have existed in the past.</p>
<p>Over the next century and a half there were discoveries of many additional Neanderthal skeletons. These studies revealed that Neanderthals had evolved in Europe from even more archaic humans. In popular culture, they garnered a reputation as beastly‚Äîmuch more different from us than they in fact were. The primitive reputation of Neanderthals was fueled in large part by a slouched reconstruction of the Neanderthal skeleton from La Chapelle-aux-Saints, France, made in 1911. But from all the evidence we have, before about one hundred thousand years ago, Neanderthals were behaviorally just as sophisticated as our own ancestors‚Äîanatomically modern humans.</p>
<p>Both Neanderthals and anatomically modern humans made stone tools using a technique that has become known as Levallois, which requires as much cognitive skill and dexterity as the Upper Paleolithic and Later Stone Age toolmaking techniques that emerged among modern humans after around fifty thousand years ago. In this technique, flakes are struck off carefully prepared rock cores that have little resemblance to the resulting tools, so that craftspeople must hold in their minds an image of what the finished tool will look like and execute the complex steps by which the stone must be worked to achieve that goal.</p>
<p>Other signs of the cognitive sophistication of Neanderthals include the evidence that they cared for their sick and elderly. An excavation at Shanidar Cave in Iraq has revealed nine skeletons, all apparently deliberately buried, one of which was a half-blind elderly man with a withered arm, suggesting that the only way he could have survived is if friends and family had lovingly cared for him.2 The Neanderthals also had an appreciation of symbolism, as revealed by jewelry made of eagle talons found at Krapina Cave in Croatia and dating to about 130,000 years ago,3 and stone circles built deep inside Bruniquel Cave in France and dating to around 180,000 years ago.4</p>
<p>Yet despite similarities between Neanderthals and modern humans, profound differences are evident. An article written in the 1950s claimed that a Neanderthal on the New York City subway would attract no attention, ‚Äúprovided that he were bathed, shaved, and dressed in modern clothing.‚Äù5 But in truth, his or her strangely projecting brow and impressively muscular body would be giveaways. Neanderthals were much more different from any human population today than present-day populations are from each other.</p>
<p>The encounter of Neanderthals and modern humans has also captured the imagination of novelists. In William Golding‚Äôs 1955 <em>The Inheritors,</em> a band of Neanderthals is killed by modern humans, who adopt a surviving Neanderthal child.6 In Jean Auel‚Äôs 1980 <em>The Clan of the Cave Bear,</em> a modern human woman is brought up by Neanderthals, and the conceit of the book is a dramatization of what close interaction of these two sophisticated groups of humans, so alien to each other and yet so similar, might have been like.7</p>
<p>There is hard scientific evidence that modern humans and Neanderthals met. The most direct is from western Europe, where Neanderthals disappeared around thirty-nine thousand years ago.8 The arrival of modern humans in western Europe was at least a few thousand years earlier, as is evident at Fumane in southern Italy where around forty-four thousand years ago, Neanderthal-type stone tools gave way to tools typical of modern humans. In southwestern Europe, tools typical of modern humans, made in a style called Ch√¢telperronian, have been found amidst Neanderthal remains that date to between forty-four thousand and thirty-nine thousand years ago, suggesting that Neanderthals may have imitated modern human toolmaking, or that the two groups traded tools or materials. Not all archaeologists accept this interpretation, though, and there is ongoing debate about whether Ch√¢telperronian artifacts were made by Neanderthals or by modern humans.9</p>
<p>Meetings between Neanderthals and modern humans took place not only in Europe but almost certainly in the Near East as well. After around seventy thousand years ago, a strong and successful Neanderthal population expanded from Europe into central Asia as far as the Altai Mountains, and into the Near East. The Near East had already been inhabited by modern humans, as attested by remains at Skhul Cave on the Carmel Ridge in Israel and Qafzeh Cave in the Lower Galilee dating to between about 130,000 and 100,000 years ago.10 Later, Neanderthals moved into the region, with one skeleton at Kebara Cave on the Carmel Ridge dating to between sixty and forty-eight thousand years ago.11 Reversing the expectation we might have that modern humans displaced Neanderthals at every encounter, Neanderthals were advancing from their homeland (Europe) even as modern humans retreated. Sometime after sixty thousand years ago, though, modern humans began to predominate in the Near East. Now the Neanderthals were the losers in the encounter, and they went extinct not only in the Near East but eventually elsewhere in Eurasia as well. So it was that in the Near East there were at least two opportunities for encounters between Neanderthals and modern humans: when early modern humans first peopled the region before around one hundred thousand years ago and established a population that met the expanding Neanderthals, and when modern humans returned and displaced the Neanderthals there sometime around sixty or fifty thousand years ago.</p>
<figure class="">

    <a href="../images/000035.jpg" target="_blank" rel="noopener">
        <img src="../images/000035.jpg" alt="Figure 6" title="">
    </a>

    

</figure>
<p>Figure 6. After around 400,000 years ago, Neanderthals were the dominant humans in western Eurasia, eventually extending as far east as the Altai Mountains. They survived an initial influx of modern humans at least by 120,000 years ago. Then, after 60,000 years ago, modern humans made a second push out of Africa into Eurasia. Before long, the Neanderthals went extinct.</p>
<p>Did the two populations interbreed? Are the Neanderthals among the direct ancestors of any present-day humans? There is some skeletal evidence for hybridization. Erik Trinkaus identified remains such as those from Oase Cave in Romania that he argued were intermediate between modern humans and Neanderthals.12 However, shared skeletal features sometimes reflect adaptation to the same environmental pressures, not shared ancestry. This is why archaeological and skeletal records cannot determine the relatedness of Neanderthals to us. Studies of the genome can.</p>
<p>Neanderthal DNA</p>
<p>Early on, scientists studying ancient DNA focused almost exclusively on mitochondrial DNA, for two reasons. First, there are about one thousand copies of mitochondrial DNA in each cell, compared to two copies of most of the rest of the genome, increasing the chance of successful extraction. Second, mitochondrial DNA is information-dense: there are many more differences for a given number of DNA letters than in most other places in the genome, making it possible to obtain a more precise measurement of genetic separation time for every letter of DNA that is successfully analyzed. Mitochondrial data analysis confirmed that Neanderthals shared maternal-line ancestors with modern humans more recently than previously thought13‚Äîthe best current estimate is 470,000 to 360,000 years ago.14 Mitochondrial DNA analysis also confirmed that the Neanderthals were highly distinctive. Their DNA type was outside the range of present-day variation in humans, sharing a common ancestor with us at a date several times more ancient than the time when ‚ÄúMitochondrial Eve‚Äù lived.15</p>
<p>Neanderthal mitochondrial DNA provided no support for the theory that Neanderthals and modern humans interbred when they encountered each other, but at the same time the mitochondrial DNA evidence could not exclude up to around a 25 percent contribution of Neanderthals to the DNA of present-day non-Africans.16 There is a reason why we have so little power to make statements about the Neanderthal contribution to modern humans based only on mitochondrial DNA. Even if modern humans outside Africa today do have substantial Neanderthal ancestry, there are only one or few women who lived at that time and were lucky enough to pass down their mitochondrial DNA to present-day people, and if most of those women were modern humans, the patterns we see today would not be surprising. So the mitochondrial data were not conclusive, but nevertheless the view that Neanderthals and modern humans did not mix remained the scientific orthodoxy until Svante P√§√§bo‚Äôs team extracted DNA from the whole genome of a Neanderthal, making it possible to examine the history of all its ancestors, not just the exclusively maternal line.</p>
<p>The advance to sequencing the whole Neanderthal genome was made possible by a huge leap in the efficiency of the technology for studying ancient DNA in the decade after the sequencing of Neanderthal mitochondrial DNA.</p>
<p>The mainstay of ancient DNA research prior to 2010 was a technique called polymerase chain reaction (PCR). This involved selecting a stretch of DNA to be targeted, and then synthesizing approximately twenty-letter-long fragments of DNA that match the genome on each side of the targeted segment. These unique fragments pick out the targeted part of the genome, which is then duplicated many times over by enzymes. The effect is to take a tiny fraction of all the DNA in the sample and make it the dominant sequence. This method throws away the vast majority of DNA (the part that is not targeted). Nevertheless, it can extract at least some DNA that is of interest.</p>
<p>The new approach for extracting ancient DNA was radically different. It relied on sequencing all of the DNA in the sample, regardless of the part of the genome it comes from, and without preselecting the DNA based on targeting sequences. It took advantage of the brute power of new machines, which from 2006 to 2010 reduced the cost of sequencing by at least about ten thousandfold. The data could be analyzed by a computer to piece together most of a genome, or alternatively to pick out a gene of interest.</p>
<p>To make the new approach work, P√§√§bo‚Äôs team needed to overcome several challenges. First, they needed to find a bone from which they could extract enough DNA. Anthropologists often work with fossils‚Äîbones completely mineralized into rocks. But it is impossible to get any DNA from a true fossil. P√§√§bo was therefore looking for bones that were not completely mineralized but contained organic material, including stretches of well-preserved DNA. Second, supposing the team could find a ‚Äúgolden sample‚Äù with well-preserved DNA, they still had to overcome the problem of contamination of the sample by microbial DNA, which comes from the bacteria and fungi that embed themselves in bone after an individual‚Äôs death. These contribute the overwhelming majority of DNA in most ancient samples. Finally, the team had to consider the likelihood of contamination by the researchers‚Äîarchaeologists or molecular biologists‚Äîwho handled the samples and chemicals and may have left traces of their own DNA on them.</p>
<p>Contamination is a huge danger for studies of ancient human DNA. Contaminated sequences can mislead analysts because the modern humans handling the bone are related, even if very distantly, to the individual being sequenced. A typical Neanderthal ancient DNA fragment from a well-preserved sample is only about forty letters long, while the rate of differences between modern humans and Neanderthals is about one per six hundred letters, so it is sometimes impossible to tell whether a particular stretch of DNA comes from the bone or from someone who handled it. Contamination has bedeviled ancient DNA researchers time and again. For example, in 2006 P√§√§bo‚Äôs group sequenced about a million letters of DNA from Neanderthals as a trial run prior to whole-genome sequencing.17 A high fraction of the sequences were modern human contaminants, compromising interpretation of the data.18</p>
<p>Modern measures to minimize the possibility of contamination in ancient DNA analysis, which had already begun to be implemented in the 2006 study and which became even more elaborate afterward, involve an obsessive set of precautions. For the 2010 study in which P√§√§bo and his team successfully sequenced an uncontaminated Neanderthal genome, they took each of the bones they screened into a ‚Äúclean room,‚Äù which they adapted from the blueprints of the clean spaces used in microchip fabrication facilities in the computer industry. There was an overhead ultraviolet (UV) light of the same type used in surgical operating suites that was turned on whenever researchers were not present, in order to convert contaminating DNA into a form that cannot be sequenced (the light also destroys ancient DNA on the outside of samples, but researchers drill beneath the surface and so are able to access DNA that is not destroyed). The air was ultra-filtered to remove tiny dust particles‚Äîanything more than one thousand times smaller than the width of a human hair‚Äîthat might contain DNA. The suite was pressurized so that air flowed from inside to outside, to protect the samples from any contaminating DNA wafting in from outside the lab.</p>
<p>There were three separate rooms in the suite. In the first, the researchers donned full-body clean suits, gloves, and face masks. In the second, they placed the bones chosen for sampling into a chamber where they were exposed to high-energy UV radiation, again with the goal of converting the contaminating DNA that might be lying on the surface into a form that cannot be sequenced. The researchers then cored the bones using a sterilized dental drill, collected tens or hundreds of milligrams of powder onto UV-irradiated aluminum foil, and deposited this powder into a UV-irradiated tube. In the third chamber, they immersed the powder into chemical solutions that removed bone minerals and protein, and ran the solution over pure sand (silicon dioxide), which under the right conditions binds the DNA while removing the compounds that poison the chemical reactions used for sequencing.</p>
<p>The researchers then transformed the resulting DNA fragments into a form that could be sequenced. First, they chemically removed the ragged ends of the DNA fragments that had been degraded after tens of thousands of years buried under the ground. In an extra measure to remove contamination beyond what had been done in the 2006 study, P√§√§bo and his team attached an artificially synthesized sequence of letters, a chemical ‚Äúbarcode,‚Äù to the ends of the DNA fragments. Any contaminating sequences that entered the experiment after the attachment of the barcode could thus be distinguished from the DNA of the ancient sample. The final step was to attach molecular adapters at either end that allowed the DNA fragment to be sequenced in one of the new machines that had made sequencing tens of thousands of times cheaper than the previous technology.</p>
<p>The best-preserved Neanderthal samples turned out to be three approximately forty-thousand-year-old arm and leg bones from Vindija Cave in the highlands of Croatia. After sequencing from these bones, P√§√§bo‚Äôs team found that the great majority of DNA fragments they obtained were from bacteria and fungi that had colonized the bones. But by comparing the millions of fragments to the present-day human and chimpanzee genome sequences, they found gold amidst the dross. These reference genomes were like the picture on a jigsaw puzzle box, providing the key to aligning the tiny fragments of DNA they had sequenced. The bones contained as much as 4 percent archaic human DNA.</p>
<p>Once P√§√§bo realized in 2007 that he would be able to sequence almost the entire Neanderthal genome, he assembled an international team of experts with the goal of ensuring that the analysis would do justice to the data. This is how I got involved, together with my chief scientific partner, the applied mathematician Nick Patterson. P√§√§bo reached out to us because over the previous five years we had established ourselves as innovators in the area of studying population mixture. Over the course of many trips to Germany, I played an important role in the analyses that proved interbreeding between Neanderthals and some modern humans.</p>
<p>Affinities Between Neanderthals and Non-Africans</p>
<p>The Neanderthal genome sequences we were working with were unfortunately full of errors. We could see as much because the data suggested that several times more mutations had occurred on the Neanderthal lineage than on the modern human lineage after the two sequences separated from their common ancestors. Most of these apparent mutations could not be real, since mutations occur at an approximately constant rate over time, and as the Neanderthal bones were ancient, they were actually closer in time to the common ancestor than are present-day human genomes, and so should have accumulated fewer mutations. Based on the degree of excess mutations on the Neanderthal lineage, we estimated that the Neanderthal sequences we were working with had a mistake approximately every two hundred DNA letters. While this might sound small, it is actually much higher than the rate of true differences between Neanderthals and present-day humans, so most of the differences we found between the Neanderthal sequence and present-day human sequences were errors created by the measurement process and not genuine differences between the Neanderthal and present-day human genomes. To deal with the problem, we restricted our study to positions in the genome that are known to be variable among present-day humans. At these positions, an error rate of about 0.5 percent was too low to confuse the interpretation. Based on these positions, we designed a mathematical test for measuring whether Neanderthals were more closely related to some present-day humans than to others.</p>
<p>The test we developed is now called the ‚ÄúFour Population Test,‚Äù and it has become a workhorse for comparing populations. The test takes as its input the DNA letters seen at the same position in four genomes: for example, two modern human genomes, the Neanderthal, and a chimpanzee. It examines whether, at positions where there is a mutation distinguishing the two modern human genomes that is also observed in the Neanderthal genome‚Äîwhich must reflect a mutation that occurred prior to the final separation of Neanderthals and modern humans‚Äîthe Neanderthal matches the second human population at a different rate from the first. If the two modern humans descend from a common ancestral population that separated earlier from the ancestors of Neanderthals, there is no reason why the mutation is more likely to have been passed down one modern human line than another, and thus the rate of matching of each of the two modern human genomes to Neanderthal is expected to be equal. In contrast, if Neanderthals and some modern humans interbred, the modern human population descended from the interbreeding will share more mutations with Neanderthals.</p>
<figure class="">

    <a href="../images/000039.jpg" target="_blank" rel="noopener">
        <img src="../images/000039.jpg" alt="Figure 7" title="">
    </a>

    

</figure>
<p>Figure 7. We can evaluate whether two populations are consistent with descending from a common ancestral population through the ‚ÄúFour Population Test.‚Äù For example, consider a mutation that occurred in the ancestors of the Neanderthal (letter T, above) that is not seen in chimpanzee DNA. There are about 9 percent more of these mutations shared with Europeans than with African genomes, reflecting a history of Neanderthal interbreeding into the ancestors of Europeans.</p>
<p>When we tested diverse present-day human populations, we found Neanderthals to be about equally close to Europeans, East Asians, and New Guineans, but closer to all non-Africans than to all sub-Saharan Africans, including populations as different as West Africans and San hunter-gatherers from southern Africa. The difference was slight, but the probability of these findings happening by chance was less than one in a quadrillion. We reached this conclusion however we analyzed the data. This was the pattern that would be expected if Neanderthals had interbred with the ancestors of non-Africans but not Africans.</p>
<p>Trying to Make the Evidence Go Away</p>
<p>We were skeptical about this conclusion because it went against the scientific consensus of the time‚Äîa consensus that had been strongly impressed on many members of our team. P√§√§bo had done his postdoctoral training in the laboratory that in 1987 had discovered that the most deeply splitting human mitochondrial DNA lineages are found today in Africa, providing strong evidence in favor of an African origin for all modern humans. P√§√§bo‚Äôs own 1997 work strengthened the evidence for a purely African origin by showing that Neanderthal mitochondrial DNA fell far outside all modern human variation.19</p>
<p>I too came into the Neanderthal genome project with a strong bias against the possibility of Neanderthal interbreeding with modern humans. My Ph.D. supervisor, David Goldstein, was a student of Luca Cavalli-Sforza, who had made a fully out-of-Africa model a centerpiece of his models of human evolution, and I was steeped in this paradigm. The genetic data I knew about supported the out-of-Africa picture so consistently that from my perspective the strictest possible version of the out-of-Africa hypothesis, in which there was no interbreeding between the ancestors of present-day humans and Neanderthals, seemed like a good bet.</p>
<p>Coming from this background, we were deeply suspicious of the evidence we were finding for interbreeding with Neanderthals, and so we applied a particularly stringent series of tests in order to find some problem with our evidence. We tested whether the result was dependent on the genome sequencing technology that we used, but we obtained the same result from two very different technologies. We considered the possibility that the finding might be an artifact of a high rate of error in ancient DNA, which is known to affect particular DNA letters much more than others. However, we obtained the same result regardless of the type of mutation we analyzed. We wondered if our finding resulted from contamination of the Neanderthal sample by present-day humans. This could perhaps have tainted the data despite the measures that P√§√§bo‚Äôs team had taken to guard against it in the lab, and despite the tests we had performed on the data to measure the degree of modern human contamination, which had suggested that any contamination that was present was too small to produce the patterns we observed. However, even if there had been contamination from present-day humans, the patterns we observed looked nothing like what would be expected from it. If there had been contamination, it would most likely have come from a European, since almost all the Neanderthal bones we analyzed were excavated and handled by Europeans. Yet the Neanderthal sequence we had was no closer to Europeans than to East Asians or to New Guineans‚Äîthree very different populations.</p>
<p>We remained skeptical, wondering if something we had not thought of could explain the patterns. Then, in June 2009, I attended a conference at the University of Michigan where I met Rasmus Nielsen, who had been scanning through the genomes of diverse humans from around the world. In most parts of the genome, Africans are more genetically diverse than non-Africans and carry the most deeply diverging lineages, as is the case with mitochondrial DNA. But Nielsen was identifying rare places in the genome where the genetic diversity among non-Africans was greater than in Africans because of lineages that split off the tree of present-day human sequences early and were present only in non-Africans. These sequences just might be derived from archaic humans who had interbred with non-Africans. Nielsen joined our collaboration and compared the regions he and his colleagues identified to the data. When he compared twelve of his special regions to the Neanderthal genome sequence, he found that in ten of them there was a close match to the Neanderthal. This was far too high a fraction to happen by chance. Most of Nielsen‚Äôs highly divergent bits of DNA had to be Neanderthal in origin.</p>
<p>Next, we obtained a date for when the Neanderthal-related genetic material entered the ancestors of non-Africans. To do this, we took advantage of recombination‚Äîthe process that occurs during the production of a person‚Äôs sperm or eggs that swaps large segments of parental DNA to produce novel spliced chromosomes that are passed to the offspring. For example, consider a woman who is a first-generation mixture of a Neanderthal mother and a modern human father. In her cells, each pair of her chromosomes consists of one unbroken Neanderthal chromosome and one unbroken modern human chromosome. However, her eggs contain twenty-three mixed chromosomes. One chromosome in an egg of hers might have its first half of Neanderthal origin and its other half of modern human origin. Suppose she mates with a modern human, and mixture continues down the generations with more modern humans. Over the generations, the segments of Neanderthal DNA get chopped into smaller and smaller bits, with recombination operating like the whirring blade of a food processor, splicing the parental DNA at random positions along the chromosome in each generation. By measuring the typical sizes of the stretches of Neanderthal-related DNA in present humans, evident from the size of sequences that match the Neanderthal genome more than they do sub-Saharan African genomes, we can learn how many generations have passed since the Neanderthal DNA entered a modern person‚Äôs ancestors.</p>
<figure class="">

    <a href="../images/000043.jpg" target="_blank" rel="noopener">
        <img src="../images/000043.jpg" alt="The chopping up of chromosomes every generation" title="">
    </a>

    

</figure>
<figure class="">

    <a href="../images/000000.jpg" target="_blank" rel="noopener">
        <img src="../images/000000.jpg" alt="provides a clock for dating mixture events" title="">
    </a>

    

</figure>
<p>Figure 8. When a person produces a sperm or an egg, he or she passes down to the next generation only one chromosome from each of the twenty-three pairs he or she carries. The transmitted chromosomes are spliced-together versions of the ones inherited from the mother and father (facing page). This means that the sizes of the bits of Neanderthal DNA in modern human genomes became smaller as the time since mixture increased (above, real data from chromosome 12).</p>
<p>With this approach, we found that at least some Neanderthal-related genetic material came into the ancestors of present-day non-Africans eighty-six thousand to thirty-seven thousand years ago.20 We have since refined this date by analyzing ancient DNA from a modern human from Siberia who, radiocarbon dating studies show, lived around forty-five thousand years ago. The stretches of Neanderthal-derived DNA in this individual are on average seven times larger than the stretches of Neanderthal-derived DNA in modern humans today, confirming that he lived much closer to the time of Neanderthal mixture. His proximity in time to the mixing event makes it possible to obtain a more accurate date of fifty-four thousand to forty-nine thousand years ago.21</p>
<p>But in 2012 we hadn‚Äôt yet proven that the interbreeding we had detected was with Neanderthals themselves. The most serious questioning came from Graham Coop, who was convinced that we had detected interbreeding with archaic humans, but pointed out that it was possible that the interbreeding hadn‚Äôt actually been with Neanderthals.22 Instead, the patterns could be the result of interbreeding with an as yet unknown archaic human in turn distantly related to Neanderthals.</p>
<p>A year later we were able to rule out Coop‚Äôs scenario after P√§√§bo‚Äôs laboratory sequenced a high-quality Neanderthal genome from a toe bone found in southern Siberia dating to at least fifty thousand years ago (if a sample is older than about fifty thousand years, radiocarbon dating can only provide a minimum date, so it actually could be substantially older).23 For this genome, we were able to gather about forty times more data than from the Croatian Neanderthal. With so much data, we could cross-check the sequence and edit away the errors. The resulting sequence was freer of errors than most genomes that are generated from living humans. The high-quality sequence allowed us to determine how closely related modern humans and Neanderthals are to each other based on the number of mutations that have occurred on the lineages since they separated. We found few or no segments where the Siberian Neanderthal shared common ancestors with present-day sub-Saharan Africans within the last half million years. However, there were shared segments with non-Africans roughly within the past one hundred thousand years. These dates fell within the time frame when Neanderthals were fully established in West Eurasia. This meant that the interbreeding was with true Neanderthals, not some distantly related groups.</p>
<p>Mixing in the Near East</p>
<p>So how much Neanderthal ancestry do people outside of Africa carry today? We found that non-African genomes today are around 1.5 to 2.1 percent Neanderthal in origin,24 with the higher numbers in East Asians and the lower numbers in Europeans, despite the fact that Europe was the homeland of the Neanderthals.25 We now know that at least part of the explanation is dilution. Ancient DNA from Europeans who lived before nine thousand years ago shows that pre-farming Europeans had just as much Neanderthal ancestry as East Asians do today.26 The reduction in Neanderthal ancestry in present-day Europeans is due to the fact that they harbor some of their ancestry from a group of people who separated from all other non-Africans prior to the mixture with Neanderthals (the story of this early-splitting group revealed by ancient DNA is told in part II of this book). The spread of farmers with this inheritance diluted the Neanderthal ancestry in Europe, but not in East Asia.27</p>
<p>Based on archaeological evidence alone, it would seem a natural guess that Neanderthals interbred with modern humans in Europe, the place where Neanderthals originated. But is that the place where the main interbreeding that left its mark in people today occurred? The genetic data cannot tell us for sure. Genetic data can show how people are related, but humans are capable of migrating thousands of kilometers in a lifetime even on foot, so genetic patterns need not reflect events that occurred near the locations where the people who carry the DNA live. If the ancient DNA studies of the last few years have shown anything clearly, it is that the geographic distribution of people living today is often misleading about the dwelling places of their ancestors.</p>
<p>However, we can make plausible conjectures about geographic origin. Evidence of interbreeding is detected today not just in Europeans but also in East Asians and New Guineans. Europe is a cul-de-sac of sorts within Eurasia, and would not have been a likely detour for modern humans expanding eastward. So where could Neanderthals and modern humans have met and mixed to give rise to a population that expanded not only to Europe but also to East Asia and New Guinea? Archaeologists have shown how in the Near East, Neanderthals and modern humans traded places as the dominant human population at least twice between 130,000 and 50,000 years ago, and it is reasonable to guess that they might have met during this period. So interbreeding in the Near East provides a plausible explanation for the Neanderthal ancestry that is shared by Europeans and East Asians.</p>
<p>Did interbreeding happen in Europe at all? In 2014, P√§√§bo‚Äôs group sequenced DNA from a skeleton from Oase Cave in Romania, the same skeleton that Erik Trinkaus had interpreted as a hybrid of Neanderthals with modern humans, based on features of its skull that were similar to both.28 Our analysis of the data showed that the Oase individual, who radiocarbon dating studies had shown lived about forty thousand years ago, had around 6 to 9 percent Neanderthal ancestry, far more than the approximately 2 percent that we measure in present-day non-Africans.29 Some stretches of Neanderthal DNA extend a third of the length of his chromosomes‚Äîa span so large and unbroken by recombination that we can be sure that the Oase individual had an actual Neanderthal no more than six generations back in his family tree. Contamination cannot explain these findings, as it would dilute the Neanderthal ancestry in the Oase individual, not increase it. It would also generate random matching to Neanderthals throughout the genome, not large stretches of Neanderthal DNA that could be readily identified by eye when we simply plotted along the genome the positions of mutations that match the Neanderthal genome sequence more closely than they match modern humans. This evidence of Neanderthal interbreeding didn‚Äôt need statistics. The proof was in the picture.</p>
<p>The discoveries about the interbreeding in the recent family tree of the Oase individual suggested that modern humans and Neanderthals also hybridized in Europe, the homeland of the Neanderthals. But the population of which Oase was a part‚Äîand which carried this clear imprint of interbreeding with European Neanderthals‚Äîmay not have left any descendants among people living today. When we analyzed the genome of Oase, we found no evidence that he was more closely related to Europeans than to East Asians. This means that he had to have been part of a population that was an evolutionary dead end‚Äîa pioneer modern human population that arrived early in Europe, flourished there briefly and interbred with local Neanderthals, and then went extinct. Thus, while the Oase individual provides powerful evidence that interbreeding between Neanderthals and modern humans occurred in Europe, he does not provide any evidence that Neanderthal ancestry in non-Africans today is derived from European Neanderthals. It remains the case that the most likely source of Neanderthal ancestry in non-Africans is Near Eastern Neanderthals.</p>
<p>The finding that Oase was from a dead-end population accords with the archaeological record of the first modern humans of Europe. The stone tools these humans made came in a variety of styles, but like the population of Oase himself, most were dead ends in the sense that they disappeared from the archaeological record after a few thousand years. However, one style known as the Protoaurignacian‚Äîthought to derive from the earlier Ahmarian of the Near East‚Äîpersisted after thirty-nine thousand years ago and likely developed into the Aurignacian, the first widespread modern human culture in Europe.30 These patterns could be explained if the makers of Aurignacian tools derived from a different migration into Europe compared to other early modern humans like Oase. This scenario could explain how it could be that Oase‚Äôs population interbred heavily with local European Neanderthals, and yet the Neanderthal ancestry in Europeans today is not from Europe.</p>
<p>Two Groups at the Edge of Compatibility</p>
<p>The low fertility of hybrids may also have reduced Neanderthal ancestry in the DNA of people living today. This possibility was first advanced by Laurent Excoffier, who knew from studies of animals and plants that when one population moves into a region occupied by another population with which it can interbreed, even a small rate of interbreeding is enough to produce high proportions of mixture in the descendants‚Äîfar more than the approximately 2 percent Neanderthal ancestry seen in non-Africans today. Excoffier argued that the only way that the modern human genome could have ended up with so little Neanderthal ancestry was if expanding modern humans had offspring with other modern humans at least fifty times more often than they did with the Neanderthals living in their midst.31 He thought that the most likely explanation for this was that Neanderthals and modern human offspring were much less fertile than the offspring of matings between pairs of modern humans.</p>
<p>I wasn‚Äôt convinced by this argument. Rather than low hybrid fertility, I favored the explanation that there simply wasn‚Äôt much interbreeding for social reasons. Even today, many groups of modern humans keep largely to themselves because of cultural, religious, or caste barriers. Why should it have been any different for modern humans and Neanderthals when they encountered one another?</p>
<p>But Excoffier got something important right. This became evident when we and others analyzed the bits of Neanderthal DNA that entered into the modern human population and mapped their positions in the genome. To do this, Sriram Sankararaman in my laboratory searched for mutations that were present in the sequenced Neanderthals but were rare or absent in sub-Saharan Africans. By studying stretches of such mutations, we were able to find a substantial fraction of all the Neanderthal ancestry fragments in each non-African. Looking at where in the genome these Neanderthal ancestry fragments occurred, it became clear that the impact of Neanderthal interbreeding varied dramatically across the genome of non-African people today. The average proportion of Neanderthal ancestry in non-African populations is around 2 percent, but it is not spread evenly. In more than half the genome, no Neanderthal ancestry has been detected in anyone. But in some unusual places in the genome, more than 50 percent of DNA sequences are from Neanderthals.32</p>
<p>A critical clue that helped us to understand how this pattern had formed came from studying the places in non-African genomes where Neanderthal ancestry is rare. In any one stretch of DNA, an absence of Neanderthal ancestry in the population can happen by chance, as we think is the case for mitochondrial DNA. However, it is improbable that a substantial subset of the genome with particular biological functions will be systematically depleted of Neanderthal ancestry unless natural selection systematically worked to remove it.</p>
<p>But evidence of systematic removal of Neanderthal ancestry is exactly what we found‚Äîand, remarkably, we found a particularly intense depletion of Neanderthal ancestry by natural selection in two parts of the genome known to be relevant to the fertility of hybrids.</p>
<p>The first place of reduced Neanderthal ancestry was on chromosome X, one of the two sex chromosomes. This reminded me of a pattern that Nick Patterson and I had run into in our work on the separation of human and chimpanzee ancestors in a study we had carried out together and published years before.33 There are only three copies of chromosome X in any population for every four other chromosomes (because females carry two copies and males only one, in contrast to two copies in each sex for most of the rest of the chromosomes). This means that in any one generation, the probability that any two X chromosomes share a common parent is four-thirds the probability that any two of one of the other chromosomes share a common parent. It follows that the expected time since any pair of X chromosome sequences descend from a common ancestral sequence is about four-thirds of that in the rest of the genome. In fact, though, the real data suggest a number that is around half or even less.34 In our study of the common ancestral population of humans and chimpanzees, we had not been able to identify any history that could explain this pattern, such as a lower rate of females moving among groups than males, or a more variable number of children in females than in males, or population expansion or contraction. However, the patterns could be explained by a history in which the ancestors of humans and chimpanzees initially separated, then came together to form either human or chimpanzee ancestors before the final separation of the two lineages.</p>
<p>How is it that hybridization can lead to so much less genetic variation on chromosome X than on the rest of the genome? From studies of a variety of species across the animal kingdom, it is known that when two populations are separated for long enough, hybrid offspring have reduced fertility. In mammals like us, reduced fertility is much more common in males, and the genetic factors contributing to this reduced fertility are concentrated on chromosome X.35 So when two populations are so separated that their offspring have reduced fertility, but nevertheless mix together to produce hybrids, it is expected that there will be intense natural selection to remove the factors contributing to reduced fertility. This process will be especially evident on chromosome X because of the concentration of genes contributing to infertility on it. As a result, there tends to be natural selection on chromosome X for stretches of DNA from the population that contributed most of the hybrid population‚Äôs ancestry. This causes the hybrid population to derive its chromosome X almost entirely from the majority population, leading to an anomalously low genetic divergence on chromosome X between the hybrid population and one of the hybridizing populations, consistent with the pattern seen in humans and chimpanzees.</p>
<p>This theoretical prediction might sound fanciful, but in fact it is borne out in hybrids of the western European and eastern European house mouse species in a band of territory that runs in a north-to-south direction through central Europe, roughly along the line of the former Cold War Iron Curtain. While the density of mutations separating the hybrid mice from western European mice is high in most of the genome because the hybrid mice carry DNA not just from western European mice but also from highly divergent eastern European mice, the density on the X chromosome is far less because the hybrid mice harbor very little DNA from the eastern European population whose X chromosomes are known to cause infertility in male hybrids.36</p>
<p>Since the publication of our paper in 2006 suggesting that either humans or chimpanzees may derive from an ancient major hybridization, the evidence for ancient major hybridization in the ancestry of humans and chimpanzees has, if anything, become even stronger. In 2012 Mikkel Schierup, Thomas Mailund, and colleagues developed a new method to estimate the suddenness of separation of the ancestors of two present-day species from genetic data, based on principles similar to the Li and Durbin approach described in chapter one.37 When they applied the method to study the separation time of common chimpanzees and their distant cousins, bonobos, they found evidence that the separation was very sudden, consistent with the hypothesis that the species were separated by a huge river (the Congo) that formed rather suddenly one to two million years ago. In contrast, when they applied the method to study humans and chimpanzees, they found evidence for an extended period of genetic interchange after population differentiation began, as expected for hybridization.38</p>
<p>An even more important piece of evidence came from a paper Schierup and Mailund published in 2015, when together with other colleagues, they showed that the regions that are denuded of Neanderthal mixture on chromosome X in non-Africans are to a large extent the same regions that are driving the low genetic divergence between humans and chimpanzees.39 This is what would be expected if mutations that contribute to reduced fertility when they occur in a hybrid individual tend to be concentrated not just on chromosome X, but in particular regions along chromosome X, causing the minority ancestry to be removed from the population by natural selection against the male hybrids who carry it. The evidence of selection to remove Neanderthal DNA from chromosome X was a tell-tale sign that male hybrids had reduced fertility.</p>
<figure class="">

    <a href="../images/000006.jpg" target="_blank" rel="noopener">
        <img src="../images/000006.jpg" alt="Figure 9: Neanderthal ancestry has been removed over time by natural selection" title="">
    </a>

    

</figure>
<p>We also found a second line of evidence for infertility in hybrids of Neanderthals and modern humans‚Äîa line of evidence that had nothing to do with the X chromosome. When reduced fertility is observed in hybrid males, the genes responsible tend to be highly active in the male reproductive tissue, causing malfunctions of sperm. So a prediction of the hypothesis of male hybrid infertility suggested to me by evolutionary biologist Daven Presgraves after I showed him the X chromosome evidence is that genes unusually active in the germ cells of a man‚Äôs testicles will have less Neanderthal ancestry on average than genes that are most active in other body tissues. When we looked in real data, Presgraves‚Äôs prediction was exactly borne out.40</p>
<p>The problems faced by modern humans with Neanderthal ancestry went beyond reduced fertility, as it turns out that Neanderthal ancestry is not just reduced on the X chromosome and around genes important in male reproduction, but is also reduced around the great majority of genes (there is far more Neanderthal ancestry in ‚Äújunk‚Äù parts of the genome with few biological functions). The clearest evidence for this came from a study in 2016, in which we published a genome-wide ancient DNA dataset from more than fifty Eurasians spread over the last forty-five thousand years.41 We showed that Neanderthal ancestry decreased continually from 3 to 6 percent in most of the samples we analyzed from earlier times to its present-day value of around 2 percent at later times and that this was driven by widespread natural selection against Neanderthal DNA.</p>
<p>A large part of the Neanderthal range was in a region where ice ages caused periodic collapses of the animal and plant populations that Neanderthals depended on, a problem that may not have afflicted modern human ancestors in tropical Africa to the same extent. There is genetic confirmation for smaller Neanderthal than modern human population sizes from the fact that the diversity of their genomes was about four times smaller. A history of small size is problematic for the genetic health of a population, because the fluctuations in mutation frequency that occur every generation are substantial enough to allow some mutations to spread through the population even in the face of the prevailing wind of natural selection that tends to reduce their frequencies.42 So in the half million years since Neanderthals and modern humans separated, Neanderthal genomes accumulated mutations that would prove detrimental when later, Neanderthal/modern human interbreeding occurred.</p>
<p>The problematic mutations in the Neanderthal genome form a sharp contrast with more recent mixtures of divergent modern human populations where there is no evidence for such effects. For example, among African Americans, in studies of about thirty thousand people, we have found no evidence for natural selection against African or European ancestry.43 One explanation for this is that when Neanderthals and modern humans mixed they had been separated for about ten times longer than had West Africans and Europeans, giving that much more time for biological incompatibilities to develop. A second explanation relates to the observation, from studies of many species, that when infertility arises between populations, it is often due to interactions between two genes in different parts of the genome. Since two changes are required to produce such an incompatibility, the rate of infertility increases with the square of population separation time, so a ten-times-larger population separation translates to one hundred times more genetic incompatibility. In light of this the lack of infertility in hybrids of present-day humans may no longer seem so surprising.</p>
<p>Thesis, Antithesis, Synthesis</p>
<p>An important strand in continental European philosophy beginning in the eighteenth century was that the march of ideas proceeds in a ‚Äúdialectic‚Äù: a clash of opposed perspectives that leads to a synthesis.44 The dialectic begins with a ‚Äúthesis,‚Äù followed by an ‚Äúantithesis.‚Äù Progress is achieved through a resolution, or ‚Äúsynthesis,‚Äù which transcends the two-sided debate that engendered it.</p>
<p>So it has been with our understanding of modern human origins. For a long time, many anthropologists favored multiregionalism, the theory that modern humans in any given place in the world descend substantially from archaic humans who lived in the same geographical region. Thus Europeans were thought to derive large proportions of their ancestry from Neanderthals, East Asians from humans who dispersed to eastern Eurasia more than a million years ago, and Africans from African archaic forms. The biological differences among modern human populations would then have extremely deep roots.</p>
<p>Multiregionalism soon encountered its antithesis, the out-of-Africa theory. In this theory, modern humans did not evolve in each location in the world separately from local archaic forms. Instead, modern humans everywhere derive from a relatively recent migration from Africa and the Near East beginning around fifty thousand years ago. The recent date of ‚ÄúMitochondrial Eve‚Äù compared with the deep divergence of Neanderthal mitochondrial DNA provided some of the best evidence for this theory. In opposition to the multiregional hypothesis, the out-of-Africa theory emphasizes the recent origin of the differences among present-day human populations, relative to the multimillion-year time depth of the human skeletal record.</p>
<p>Yet the out-of-Africa argument is not entirely right either. We now have a synthesis, driven by the finding of gene flow between Neanderthals and modern humans based on ancient DNA. This affirms a ‚Äúmostly out-of-Africa‚Äù theory, and also reveals something profound about the culture of those modern humans who must have known Neanderthals intimately. While it is clear from the genetic data that modern humans outside of Africa descend from the expansion of an African-origin group that swept around the world, we now know that some interbreeding occurred. This must make us think differently about our ancestors and the archaic humans they encountered. The Neanderthals were more like us than we had imagined, perhaps capable of many behaviors that we typically associate with modern humans. There must have been cultural exchange that accompanied the mixture‚Äîthe novels by William Golding and Jean Auel were right to dramatize these encounters. We also know that there has been a biological legacy bequeathed by Neanderthals to non-Africans, including genes for adapting to different Eurasian environments, a topic to which I will return in the next chapter.</p>
<p>At the conclusion of the Neanderthal genome project, I am still amazed by the surprises we encountered. Having found the first evidence of interbreeding between Neanderthals and modern humans, I continue to have nightmares that the finding is some kind of mistake. But the data are sternly consistent: the evidence for Neanderthal interbreeding turns out to be everywhere. As we continue to do genetic work, we keep encountering more and more patterns that reflect the extraordinary impact this interbreeding has had on the genomes of people living today.</p>
<p>So the genetic record has forced our hand. Instead of confirming scientists‚Äô expectations, it has produced surprises. We now know that Neanderthal/modern human hybrid populations were living in Europe and across Eurasia, and that while many hybrid populations eventually died out, some survived and gave rise to large numbers of people today. We now know approximately when the modern human and Neanderthal lineages separated. We now also know that when these lineages reencountered each other, they had evolved to such an extent that they were at the very limit of biological compatibility. This raises a question: Were the Neanderthals the only archaic humans who interbred with our ancestors? Or were there other major hybridizations in our past?</p>
<figure class="">

    <a href="../images/000011.jpg" target="_blank" rel="noopener">
        <img src="../images/000011.jpg" alt="A Multiplicity of Archaic Human Lineages" title="">
    </a>

    

</figure>

  </div>
</article>


        

        



    <aside class="card border noPrint" id="section-tree-item-sapiens_branches_articles_reich_who-we-are_1_The_Deep_History_of_Our_Species_2_Encounters_with_Neanderthals_md">
        <div class="card-title bg-light-gray border d-flex justify-content-between">
            <a href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/2_Encounters_with_Neanderthals/">2 Encounters with Neanderthals </a>
            <a data-toggle="collapse" href="#section-tree-item-body-sapiens_branches_articles_reich_who-we-are_1_The_Deep_History_of_Our_Species_2_Encounters_with_Neanderthals_md" role="button" aria-expanded="false" aria-controls="section-tree-item-body-sapiens_branches_articles_reich_who-we-are_1_The_Deep_History_of_Our_Species_2_Encounters_with_Neanderthals_md" >‚Ä¶<i class="fas fa-caret-down" class="collapsed"></i> </a>
        </div>
        <nav id="section-tree-item-body-sapiens_branches_articles_reich_who-we-are_1_The_Deep_History_of_Our_Species_2_Encounters_with_Neanderthals_md" class="card-body p-0 collapse">
            
            <li>draft: false</li>
            
            <li>iscjklanguage: false</li>
            
            <li>title: 2 Encounters with Neanderthals</li>
            
        </nav>
    </aside>


      </main>
    </div>
    <footer class="bg-yellow  p-1 noPrint" role="contentinfo">
  

  <div id="div_foot_nav_bar" class="row">
    <a href="https://vishvAsa.github.io/notes/search"  class="btn btn-light btn-sm">ü•Ωüî¶</a>
    <div class="row col  d-flex justify-content-center">
      <div><a id="previousPageFoot" href="" class="btn btn-secondary">###<i class="fas fa-caret-left"></i></a></div>
      <div ><a id="nextPageFoot" href="" class="btn btn-secondary"><i class="fas fa-caret-right"></i> ### </a></div>
    </div>
      <ul id="footer-bar-right-custom" class="list-group list-group-horizontal">
      </ul>
  </div>
  <div id="div_footer_bar" class="container-fluid d-flex justify-content-between">
    <a class="btn btn-secondary" href="https://github.com/vishvAsa/notes/issues/new" >
      ‚úâÔ∏èContact
    </a>
    <a class="btn btn-secondary" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/80x15.png" /></a>
    <div class="btn small border">
      
      <a href="https://github.com/vishvAsa/notes/actions/workflows/build.yml"><img src="https://github.com/vishvAsa/notes/actions/workflows/build.yml/badge.svg"> Built </a> on 2026 Feb 12 01:34:45 UTC. (<a href="http://google.com/search?q=01%3a34%3a45%20UTC to IST">IST</a>)
    </div>
  </div>
</footer>



<script async src="https://www.googletagmanager.com/gtag/js?id=G-33H08JSLQR"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-33H08JSLQR');
</script>

  </body>
  <script type='text/javascript'>
    
    
    
    module_uiLib.onDocumentReadyTasks();
  </script>
  
</html>
