<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
    
    <title>Vishvas&#39;s notes  | 3 Ancient DNA Opens the Floodgates</title>
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">

    <meta name="viewport" content="width=device-width,minimum-scale=1">
    <meta name="generator" content="Hugo 0.143.0">
    
    
      <META NAME="ROBOTS" CONTENT="INDEX, FOLLOW">
    

    <link href="/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/" rel="alternate" type="application/rss+xml" title="Vishvas&#39;s notes" />
    <link href="/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/" rel="feed" type="application/rss+xml" title="Vishvas&#39;s notes" />

    <meta property="og:url" content="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/">
  <meta property="og:site_name" content="Vishvas&#39;s notes">
  <meta property="og:title" content="3 Ancient DNA Opens the Floodgates">
  <meta property="og:description" content="A Surprise from the East
In 2008, Russian archaeologists dug up a pinky bone at Denisova Cave in the Altai Mountains of southern Siberia, named after an eighteenth-century Russian hermit named Denis who had made his home there. The bone’s growth plates were not fused, showing that the bone came from a child. Its date was uncertain, as it was too small to be dated by radiocarbon analysis, and it was found in a mixed-up soil layer of the cave that contained artifacts dating to both less than thirty thousand and more than fifty thousand years ago. The leader of the excavation, Anatoly Derevianko, reasoned that the bone’s owner could have been a modern human, and the sample was so labeled. Alternatively, could the bone’s owner have been a Neanderthal, as Neanderthal remains were also found near the cave?1 Derevianko sent part of the bone to Svante Pääbo in Germany.">
  <meta property="og:locale" content="en_us">
  <meta property="og:type" content="article">
    <meta property="article:section" content="sapiens">

  <meta itemprop="name" content="3 Ancient DNA Opens the Floodgates">
  <meta itemprop="description" content="A Surprise from the East
In 2008, Russian archaeologists dug up a pinky bone at Denisova Cave in the Altai Mountains of southern Siberia, named after an eighteenth-century Russian hermit named Denis who had made his home there. The bone’s growth plates were not fused, showing that the bone came from a child. Its date was uncertain, as it was too small to be dated by radiocarbon analysis, and it was found in a mixed-up soil layer of the cave that contained artifacts dating to both less than thirty thousand and more than fifty thousand years ago. The leader of the excavation, Anatoly Derevianko, reasoned that the bone’s owner could have been a modern human, and the sample was so labeled. Alternatively, could the bone’s owner have been a Neanderthal, as Neanderthal remains were also found near the cave?1 Derevianko sent part of the bone to Svante Pääbo in Germany.">
  <meta itemprop="wordCount" content="6891">
  <meta name="twitter:card" content="summary">
  <meta name="twitter:title" content="3 Ancient DNA Opens the Floodgates">
  <meta name="twitter:description" content="A Surprise from the East
In 2008, Russian archaeologists dug up a pinky bone at Denisova Cave in the Altai Mountains of southern Siberia, named after an eighteenth-century Russian hermit named Denis who had made his home there. The bone’s growth plates were not fused, showing that the bone came from a child. Its date was uncertain, as it was too small to be dated by radiocarbon analysis, and it was found in a mixed-up soil layer of the cave that contained artifacts dating to both less than thirty thousand and more than fifty thousand years ago. The leader of the excavation, Anatoly Derevianko, reasoned that the bone’s owner could have been a modern human, and the sample was so labeled. Alternatively, could the bone’s owner have been a Neanderthal, as Neanderthal remains were also found near the cave?1 Derevianko sent part of the bone to Svante Pääbo in Germany.">
<script type="text/javascript">
    
    let baseURL = "https:\/\/vishvAsa.github.io\/notes\/";
    let basePath = "/" + baseURL.split("/").slice(3).join("/");
    let siteParams = JSON.parse("{\u0022build_img\u0022:\u0022https:\/\/github.com\/vishvAsa\/notes\/actions\/workflows\/build.yml\/badge.svg\u0022,\u0022build_url\u0022:\u0022https:\/\/github.com\/vishvAsa\/notes\/actions\/workflows\/build.yml\u0022,\u0022contactlink\u0022:\u0022https:\/\/github.com\/vishvAsa\/notes\/issues\/new\u0022,\u0022env\u0022:\u0022production\u0022,\u0022githubeditmepathbase\u0022:\u0022https:\/\/github.com\/vishvAsa\/notes\/edit\/content\/\u0022,\u0022js_extras\u0022:[\u0022mathjax\u0022]}");
    let pageDefaultsList = JSON.parse("[{\u0022scope\u0022:{\u0022pathPrefix\u0022:\u0022\u0022},\u0022values\u0022:{\u0022comments\u0022:true,\u0022layout\u0022:\u0022page\u0022,\u0022search\u0022:true,\u0022sidebar\u0022:\u0022home_sidebar\u0022}}]");
    let sidebarsData = JSON.parse("{\u0022home_sidebar\u0022:{\u0022contents\u0022:[{\u0022url\u0022:\u0022recdir:\/\/\u0022},{\u0022contents\u0022:[{\u0022title\u0022:\u0022ज्यौतिषम्\u0022,\u0022url\u0022:\u0022..\/jyotiSham\/\u0022},{\u0022title\u0022:\u0022संस्कृतम्\u0022,\u0022url\u0022:\u0022..\/sanskrit\/\u0022},{\u0022title\u0022:\u0022देवः\u0022,\u0022url\u0022:\u0022..\/devaH\/\u0022},{\u0022title\u0022:\u0022आगमः\u0022,\u0022url\u0022:\u0022..\/AgamaH\/\u0022},{\u0022title\u0022:\u0022आगमः शैवः\u0022,\u0022url\u0022:\u0022..\/AgamaH_shaivaH\/\u0022},{\u0022title\u0022:\u0022आगमः वैष्णवः\u0022,\u0022url\u0022:\u0022..\/AgamaH_vaiShNavaH\/\u0022},{\u0022title\u0022:\u0022रामानुजीयम्\u0022,\u0022url\u0022:\u0022..\/rAmAnujIyam\/\u0022},{\u0022title\u0022:\u0022माध्वम्\u0022,\u0022url\u0022:\u0022..\/mAdhvam\/\u0022},{\u0022title\u0022:\u0022आगमः ब्राह्मः\u0022,\u0022url\u0022:\u0022..\/AgamaH_brAhmaH\/\u0022},{\u0022title\u0022:\u0022कल्पान्तरम्\u0022,\u0022url\u0022:\u0022..\/kalpAntaram\/\u0022},{\u0022title\u0022:\u0022Notes Home\u0022,\u0022url\u0022:\u0022..\/notes\/\u0022},{\u0022title\u0022:\u0022वेदाः\u0022,\u0022url\u0022:\u0022..\/vedAH\/\u0022},{\u0022title\u0022:\u0022वेदाः - ऋक्\u0022,\u0022url\u0022:\u0022..\/vedAH_Rk\/\u0022},{\u0022title\u0022:\u0022वेदाः - यजुः\u0022,\u0022url\u0022:\u0022..\/vedAH_yajuH\/\u0022},{\u0022title\u0022:\u0022वेदाः - साम\u0022,\u0022url\u0022:\u0022..\/vedAH_sAma\/\u0022},{\u0022title\u0022:\u0022पुराणम्\u0022,\u0022url\u0022:\u0022..\/purANam\u0022},{\u0022title\u0022:\u0022पुराणम् वैष्णवम्\u0022,\u0022url\u0022:\u0022..\/purANam_vaiShNavam\/\u0022},{\u0022title\u0022:\u0022महाभारतम्\u0022,\u0022url\u0022:\u0022..\/mahAbhAratam\/\u0022},{\u0022title\u0022:\u0022रामायणम्\u0022,\u0022url\u0022:\u0022..\/rAmAyaNam\/\u0022},{\u0022title\u0022:\u0022काव्यम्\u0022,\u0022url\u0022:\u0022..\/kAvyam\/\u0022},{\u0022title\u0022:\u0022मीमांसा\u0022,\u0022url\u0022:\u0022..\/mImAMsA\/\u0022},{\u0022title\u0022:\u0022ಕನ್ನಡ\u0022,\u0022url\u0022:\u0022..\/kannaDa\/\u0022},{\u0022title\u0022:\u0022त्रिपिटकम्\u0022,\u0022url\u0022:\u0022..\/tipiTaka\/\u0022},{\u0022title\u0022:\u0022पाळयः\u0022,\u0022url\u0022:\u0022..\/pALi\/\u0022},{\u0022title\u0022:\u0022भाषान्तरम्\u0022,\u0022url\u0022:\u0022..\/bhAShAntaram\/\u0022},{\u0022title\u0022:\u0022English Lit\u0022,\u0022url\u0022:\u0022..\/english\/\u0022},{\u0022title\u0022:\u0022Book pub\u0022,\u0022url\u0022:\u0022..\/book-pub\u0022},{\u0022title\u0022:\u0022Vishvas\u0027s home page\u0022,\u0022url\u0022:\u0022..\/\u0022}],\u0022title\u0022:\u0022सङ्ग्रहान्तरम्\u0022},{\u0022title\u0022:\u0022\\u003ci class=\\\u0022fas fa-hand-holding-heart\\\u0022\\u003e\\u003c\/i\\u003eDonate Via Vishvas\u0022,\u0022url\u0022:\u0022https:\/\/vishvAsa.github.io\/interests\/dharma-via-vishvas\/\u0022}],\u0022title\u0022:\u0022Parts\u0022}}");
    
    var pageRelUrlTree = {};
</script>

    
    <script type="text/javascript">
  let pageVars = {};
  pageVars.pageUrlMinusBasePath = "\/notes\/sapiens\/branches\/articles\/reich_who-we-are\/1_The_Deep_History_of_Our_Species\/3_Ancient_DNA_Opens_the_Floodgates\/".replace(basePath, "/");
  pageVars.pageParams = {};
  pageVars.pageSource = "sapiens\/branches\/articles\/reich_who-we-are\/1_The_Deep_History_of_Our_Species\/3_Ancient_DNA_Opens_the_Floodgates.md";
  console.log(pageVars.pageSource);
  var pageDefaults;
  for (let possiblePageDefaults of pageDefaultsList) {
    if (pageVars.pageSource.startsWith(possiblePageDefaults.scope.pathPrefix)) {
      pageDefaults = possiblePageDefaults.values
    }
  }
    
</script>


    <script src="/notes/webpack_dist/uiLib-bundle.js"></script>


<script>


MathJax = {
  tex: {
    inlineMath: [ ["@#", "#@"], ["\\(", "\\)"] ],
    displayMath: [ ["$$","$$"], ["\\[", "\\]"] ],
    processEscapes: true,  
  },
  svg: {
    fontCache: 'global'
  },
  options: {
    ignoreHtmlClass: "tex2jax_ignore|dno"
  }
};
</script>
<script type="text/javascript" id="MathJax-script" async
        src="/notes/non_webpack_js/mathjax/tex-svg.js">
</script>






    <link rel="stylesheet" href="/notes/webpack_dist/uiLib.css">
<link rel="stylesheet" href="/notes/css/uiLibPrint.css" media="print" >
<link rel="stylesheet" href="/notes/css/fonts.css" >
<link rel="stylesheet" href="/notes/css/@fortawesome/fontawesome-free/css/solid.min.css">

<link rel="stylesheet" href="/notes/css/@fortawesome/fontawesome-free/css/fontawesome.min.css">


<link rel="alternate" hreflang="sa-Deva" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/" />
<link rel="alternate" hreflang="sa-Deva" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=devanagari" />

<link rel="alternate" hreflang="sa-Knda" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=kannada" />
<link rel="alternate" hreflang="sa-Mlym" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=malayalam" />
<link rel="alternate" hreflang="sa-Telu" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=telugu" />
<link rel="alternate" hreflang="sa-Taml-t-sa-Taml-m0-superscript" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=tamil_superscripted" />
<link rel="alternate" hreflang="sa-Taml" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=tamil" />
<link rel="alternate" hreflang="sa-Gran" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=grantha" />

<link rel="alternate" hreflang="sa-Gujr" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=gujarati" />
<link rel="alternate" hreflang="sa-Guru" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=gurmukhi" />
<link rel="alternate" hreflang="sa-Shar" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=sharada" />
<link rel="alternate" hreflang="sa-Dogr" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=dogra" />
<link rel="alternate" hreflang="sa-Phag" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=phags_pa" />
<link rel="alternate" hreflang="sa-Tibt" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=tibetan" />
<link rel="alternate" hreflang="sa-Sidd" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=siddham" />
<link rel="alternate" hreflang="sa-Newa" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=newa" />
<link rel="alternate" hreflang="sa-Modi" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=modi" />
<link rel="alternate" hreflang="sa-Tirh" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=tirhuta_maithili" />
<link rel="alternate" hreflang="sa-Gong" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=gondi_gujala" />
<link rel="alternate" hreflang="sa-Gonm" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=gondi_masaram" />


<link rel="alternate" hreflang="sa-Orya" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=oriya" />
<link rel="alternate" hreflang="sa-Beng-t-sa-Beng-m0-assamese" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=assamese" />
<link rel="alternate" hreflang="sa-Beng" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=bengali" />
<link rel="alternate" hreflang="sa-Mtei" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=manipuri" />
<link rel="alternate" hreflang="sa-Sinh" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=sinhala" />
<link rel="alternate" hreflang="sa-Limb" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=limbu" />
<link rel="alternate" hreflang="sa-Lepc" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=lepcha" />


<link rel="alternate" hreflang="sa-Cyrl" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=cyrillic" />
<link rel="alternate" hreflang="sa-Latn-t-sa-Zyyy-m0-iso" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=iso" />
<link rel="alternate" hreflang="sa-Latn-t-sa-Zyyy-m0-iast" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=iast" />
<link rel="alternate" hreflang="sa-Aran" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=urdu" />
<link rel="alternate" hreflang="sa-Avst" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=avestan" />

<link rel="alternate" hreflang="sa-Brah" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=brahmi" />
<link rel="alternate" hreflang="sa-Khar" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=kharoshthi" />
<link rel="alternate" hreflang="sa-Zanb" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=zanbazar_square" />
<link rel="alternate" hreflang="sa-Bhks" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=bhaiksuki" />

<link rel="alternate" hreflang="sa-Bali" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=balinese" />
<link rel="alternate" hreflang="sa-Java" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=javanese" />
<link rel="alternate" hreflang="sa-Laoo" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=lao" />
<link rel="alternate" hreflang="sa-Laoo-t-sa-Laoo-m0-Laopali" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=lao_pali" />
<link rel="alternate" hreflang="sa-Mymr" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=burmese" />
<link rel="alternate" hreflang="sa-Mymr-t-sa-Mymr-m0-Mon" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=mon" />
<link rel="alternate" hreflang="sa-Cham" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=cham" />
<link rel="alternate" hreflang="sa-Khmr" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=khmer" />
<link rel="alternate" hreflang="sa-Thai" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/?transliteration_target=thai" />

  </head>

  <body class="ma0 bg-near-white">
    
\(
% groupings of objects.
\newcommand{\set}[1]{\left\{ #1 \right\}}
\newcommand{\seq}[1]{\left(#1\right)}
\newcommand{\ang}[1]{\langle#1\rangle}
\newcommand{\tuple}[1]{\left(#1\right)}
\newcommand{\size}[1]{\left| #1\right|}

\newcommand{\comp}{\circ}


% numerical shortcuts.
\newcommand{\abs}[1]{\left| #1\right|}
\newcommand{\floor}[1]{\left\lfloor #1 \right\rfloor}
\newcommand{\ceil}[1]{\left\lceil #1 \right\rceil}

% linear algebra shortcuts.
\newcommand{\change}{\Delta}
\newcommand{\norm}[1]{\left\| #1\right\|}
\newcommand{\dprod}[1]{\langle#1\rangle}
\newcommand{\linspan}[1]{\langle#1\rangle}
\newcommand{\conj}[1]{\overline{#1}}
\newcommand{\der}[1]{\frac{d#1}{dx}}
\newcommand{\lap}{\Delta}
\newcommand{\kron}{\otimes}
\newcommand{\nperp}{\nvdash}

\newcommand{\mat}[1]{\left[ \begin{smallmatrix}#1 \end{smallmatrix} \right]}

% derivatives and limits
\newcommand{\partder}[2]{\frac{\partial #1}{\partial #2}}
\newcommand{\partdern}[3]{\frac{\partial^{#3 #1}}{\partial #2^{#3}}}
\newcommand{\gradient}{\nabla}
\newcommand{\subdifferential}{\partial}

% Arrows
\newcommand{\diverge}{\nearrow}
\newcommand{\notto}{\nrightarrow}
\newcommand{\up}{\uparrow}
\newcommand{\down}{\downarrow}
% gets and gives are defined!

% ordering operators
\newcommand{\oleq}{\preceq}
\newcommand{\ogeq}{\succeq}

% programming and logic operators
\newcommand{\dfn}{:=}
\newcommand{\assign}{:=}
\newcommand{\co}{\ co\ }
\newcommand{\en}{\ en\ }


% logic operators
\newcommand{\xor}{\oplus}
\newcommand{\Land}{\bigwedge}
\newcommand{\Lor}{\bigvee}
\newcommand{\finish}{\Box}
\newcommand{\contra}{\Rightarrow \Leftarrow}
\newcommand{\iseq}{\stackrel{_?{=}}}


% Set theory
\newcommand{\symdiff}{\Delta}
\newcommand{\setdiff}{\backslash}
\newcommand{\union}{\cup}
\newcommand{\inters}{\cap}
\newcommand{\Union}{\bigcup}
\newcommand{\Inters}{\bigcap}
\newcommand{\nullSet}{\phi}


% graph theory
\newcommand{\nbd}{\Gamma}

% Script alphabets
% For reals, use \Re

% greek letters
\newcommand{\eps}{\epsilon}
\newcommand{\del}{\delta}
\newcommand{\ga}{\alpha}
\newcommand{\gb}{\beta}
\newcommand{\gd}{\del}
\newcommand{\gp}{\pi}
\newcommand{\gf}{\phi}
\newcommand{\gh}{\eta}
\newcommand{\gF}{\Phi}
\newcommand{\gl}{\lambda}
\newcommand{\gm}{\mu}
\newcommand{\gn}{\nu}
\newcommand{\gr}{\rho}
\newcommand{\gs}{\sigma}
\newcommand{\gth}{\theta}
\newcommand{\gx}{\xi}
\newcommand{\gw}{\omega}

\newcommand{\sw}{\sigma}
\newcommand{\SW}{\Sigma}
\newcommand{\ew}{\lambda}
\newcommand{\EW}{\Lambda}

\newcommand{\Del}{\Delta}
\newcommand{\gD}{\Delta}
\newcommand{\gG}{\Gamma}
\newcommand{\gW}{\Omega}
\newcommand{\gS}{\Sigma}
\newcommand{\gTh}{\Theta}

% Bold english letters.
\newcommand{\bA}{\mathbf{A}}
\newcommand{\bB}{\mathbf{B}}
\newcommand{\bC}{\mathbf{C}}
\newcommand{\bD}{\mathbf{D}}
\newcommand{\bE}{\mathbf{E}}
\newcommand{\bF}{\mathbf{F}}
\newcommand{\bG}{\mathbf{G}}
\newcommand{\bH}{\mathbf{H}}
\newcommand{\bI}{\mathbf{I}}
\newcommand{\bJ}{\mathbf{J}}
\newcommand{\bK}{\mathbf{K}}
\newcommand{\bL}{\mathbf{L}}
\newcommand{\bM}{\mathbf{M}}
\newcommand{\bN}{\mathbf{N}}
\newcommand{\bO}{\mathbf{O}}
\newcommand{\bP}{\mathbf{P}}
\newcommand{\bQ}{\mathbf{Q}}
\newcommand{\bR}{\mathbf{R}}
\newcommand{\bS}{\mathbf{S}}
\newcommand{\bT}{\mathbf{T}}
\newcommand{\bU}{\mathbf{U}}
\newcommand{\bV}{\mathbf{V}}
\newcommand{\bW}{\mathbf{W}}
\newcommand{\bX}{\mathbf{X}}
\newcommand{\bY}{\mathbf{Y}}
\newcommand{\bZ}{\mathbf{Z}}

\newcommand{\bba}{\mathbf{a}}
\newcommand{\bbb}{\mathbf{b}}
\newcommand{\bbc}{\mathbf{c}}
\newcommand{\bbd}{\mathbf{d}}
\newcommand{\bbe}{\mathbf{e}}
\newcommand{\bbf}{\mathbf{f}}
\newcommand{\bbg}{\mathbf{g}}
\newcommand{\bbh}{\mathbf{h}}
\newcommand{\bbk}{\mathbf{k}}
\newcommand{\bbl}{\mathbf{l}}
\newcommand{\bbm}{\mathbf{m}}
\newcommand{\bbn}{\mathbf{n}}
\newcommand{\bbp}{\mathbf{p}}
\newcommand{\bbq}{\mathbf{q}}
\newcommand{\bbr}{\mathbf{r}}
\newcommand{\bbs}{\mathbf{s}}
\newcommand{\bbt}{\mathbf{t}}
\newcommand{\bbu}{\mathbf{u}}
\newcommand{\bbv}{\mathbf{v}}
\newcommand{\bbw}{\mathbf{w}}
\newcommand{\bbx}{\mathbf{x}}
\newcommand{\bby}{\mathbf{y}}
\newcommand{\bbz}{\mathbf{z}}
\newcommand{\0}{\mathbf{0}}
\newcommand{\1}{\mathbf{1}}

% Caligraphic english alphabet
\newcommand{\cA}{\mathcal{A}}
\newcommand{\cB}{\mathcal{B}}
\newcommand{\cC}{\mathcal{C}}
\newcommand{\cD}{\mathcal{D}}
\newcommand{\cE}{\mathcal{E}}
\newcommand{\cF}{\mathcal{F}}
\newcommand{\cG}{\mathcal{G}}
\newcommand{\cH}{\mathcal{H}}
\newcommand{\cI}{\mathcal{I}}
\newcommand{\cJ}{\mathcal{J}}
\newcommand{\cK}{\mathcal{K}}
\newcommand{\cL}{\mathcal{L}}
\newcommand{\cM}{\mathcal{M}}
\newcommand{\cN}{\mathcal{N}}
\newcommand{\cO}{\mathcal{O}}
\newcommand{\cP}{\mathcal{P}}
\newcommand{\cQ}{\mathcal{Q}}
\newcommand{\cR}{\mathcal{R}}
\newcommand{\cS}{\mathcal{S}}
\newcommand{\cT}{\mathcal{T}}
\newcommand{\cU}{\mathcal{U}}
\newcommand{\cV}{\mathcal{V}}
\newcommand{\cW}{\mathcal{W}}
\newcommand{\cX}{\mathcal{X}}
\newcommand{\cY}{\mathcal{Y}}
\newcommand{\cZ}{\mathcal{Z}}


% Formatting shortcuts
\newcommand{\red}[1]{\textcolor{red}{#1}}
\newcommand{\blue}[1]{\textcolor{blue}{#1}}
\newcommand{\htext}[2]{\texorpdfstring{#1}{#2}}

% Statistics
\newcommand{\distr}{\sim}
\newcommand{\stddev}{\sigma}
\newcommand{\covmatrix}{\Sigma}
\newcommand{\mean}{\mu}
\newcommand{\param}{\theta}
\newcommand{\gthEst}{\hat{\theta}}
\newcommand{\ftr}{\phi}
\newcommand{\est}[1]{\hat{#1}}

% General utility
\newcommand{\todo}[1]{\textbf{[TODO]}] \footnote{TODO: #1}}
\newcommand{\tbc}{[\textbf{Incomplete}]}
\newcommand{\chk}{[\textbf{Check}]}
\newcommand{\why}{[\textbf{Find proof}]}
\newcommand{\opt}[1]{\textit{#1}}

\newcommand{\experience}[1]{[\textbf{Personal Experience}]: #1 \blacktriangle}
\newcommand{\pf}[1]{[\textbf{Proof}]: #1 \Box}
\newcommand{\core}[1]{\textbf{Core Idea}: #1 \Arrowvert}
\newcommand{\example}[1]{\textbf{Example}: #1 \blacktriangle}
\newcommand{\error}[1]{\textbf{Error alert}: #1 \triangle}
\newcommand{\oprob}{[\textbf{OP}]: }


\renewcommand{\~}{\htext{$\sim$}{~}}

\DeclareMathOperator*{\argmin}{arg\,min}
\DeclareMathOperator*{\argmax}{arg\,max}


\)



    
<header class="p-1 bg-yellow noPrint">
    <nav role="navigation" class="col">
      <div id="div_top_bar" class="row">
        <div class="col-md-3 justify-content-start">
          <a  href="/" class="btn col border">
            <i class="fas fa-gopuram ">Vishvas&#39;s notes <br/> 3 Ancient DNA Opens the Floodgates</i>
          </a>
        </div>
        <div id="div_top_bar_right" class="col-md-auto d-flex justify-content-center">
          <button class="btn btn-light"  href="#transliterationDropdown" data-toggle="collapse">
            ✍
          </button>
          <select id="transliterationDropdown" size="1" class="collapse" onchange="module_uiLib.default.content.updateTransliteration()">
            <optgroup label="🌍">
              <option value="iso">āISO</option>
              <option value="iso_vedic">āi-āu</option>
              <option value="cyrillic">пу</option>
              <option value="urdu">نس</option>
              <option value="??">🌍??</option>
            </optgroup>
            <optgroup label="🛕🧭S">
              <option value="kannada">ಅ</option>
              <option value="malayalam">അ</option>
              <option value="telugu">క</option>
              <option value="tamil_superscripted">க²</option>
              <option value="tamil_extended">க</option>
              <option value="grantha">𑌅</option>
            </optgroup>
            <optgroup label="🛕🧭N">
              <option value="devanagari" selected="">स</option>
              <option value="gujarati">અ</option>
              <option value="gurmukhi">ਅ</option>
            </optgroup>
            <optgroup label="🛕🧭E">
              <option value="oriya">ଅ</option>
              <option value="assamese">অস</option>
              <option value="bengali">অ</option>
            </optgroup>
            <optgroup label="🛕🏺📜">
              <option value="sharada">𑆑𑇀𑆰</option>
              <option value="brahmi">𑀅</option>
            </optgroup>
            <optgroup label="🛕🌋">
              <option value="balinese">ᬩᬮᬶ</option>
            </optgroup>
          </select>
          <div>
          <button class="btn btn-outline-secondary btn-sm"  href="#transliterationDropdownAlt" data-toggle="collapse">
          (✍)
          </button>
          <select id="transliterationDropdownAlt" size="1"  style="font-size:12px" class="bg-light-yellow collapse" onchange="module_uiLib.default.content.updateTransliteration()">
            <optgroup label="🌍">
              <option value="iso">āISO</option>
              <option value="iso_vedic">āi-āu</option>
              <option value="cyrillic">пу</option>
              <option value="urdu">نس</option>
              <option value="??">🌍??</option>
            </optgroup>
            <optgroup label="🛕🧭S">
              <option value="kannada">ಅ</option>
              <option value="malayalam">അ</option>
              <option value="telugu">క</option>
              <option value="tamil_superscripted">க²</option>
              <option value="tamil_extended">க</option>
              <option value="grantha">𑌅</option>
            </optgroup>
            <optgroup label="🛕🧭N">
              <option value="devanagari" selected="">स</option>
              <option value="gujarati">અ</option>
              <option value="gurmukhi">ਅ</option>
            </optgroup>
            <optgroup label="🛕🧭E">
              <option value="oriya">ଅ</option>
              <option value="assamese">অস</option>
              <option value="bengali">অ</option>
            </optgroup>
            <optgroup label="🛕🏺📜">
              <option value="sharada">𑆑𑇀𑆰</option>
              <option value="brahmi">𑀅</option>
            </optgroup>
            <optgroup label="🛕🌋">
              <option value="balinese">ᬩᬮᬶ</option>
            </optgroup>
          </select>
          </div>
          <div class="row col d-flex  dropdown">
            <button class="btn btn-secondary-outline btn-sm dropdown-toggle" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
              A±
            </button>
            <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
            <a class="btn btn-secondary btn-sm" onclick="module_uiLib.changeTextSize(2)">A+</a>
            <a class="btn btn-secondary btn-sm" onclick="module_uiLib.changeTextSize(-2)">A-</a>
            </div>
          </div>
          <button class="btn btn-light btn-sm"  href="#commentStyleDropdown" data-toggle="collapse">
            🗨
          </button>
          <select id="commentStyleDropdown" class="collapse" size="1" onchange="module_uiLib.default.content.updateCommentStyleFromDropdown()">
            <option value="on">✔</option>
            <option value="hidden">✗</option>
          </select>
          <div><a id="expandAllButton" class="btn btn-secondary" onclick="module_uiLib.default.query.setParamsAndGo({'expandDetails': '.*'})">↕️️</a></div>
          <div id="printStyleDiv" class="row col d-flex  dropdown">
          <button class="btn btn-light btn-sm dropdown-toggle"  href="#printStyleDropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
          🖨
          </button>
            <div class="dropdown-menu col" aria-labelledby="printStyleDropdown">
              <div class="row justify-content-center">
                <p>Cols:</p>
                <input type="number" min="1" max="10" value="2" step="1" id="tbColumns" style="width: 3ch;"  accesskey="c"/>
              </div>
              <div class="row justify-content-center">
                <p>A±:</p> <input type="text" id="tbBodyFontSize" value="0.4cm" style="width: 6ch;"  accesskey="f"></input>
              </div>
              <div class="row justify-content-center">
                <p>Incl:</p> <input checked type="checkbox" id="tbIncludeStyle" value="on"></input>
                <button id="btnPrintStyle" onclick="module_uiLib.default.content.updatePrintStyle()">🖨
                </button>
              </div>
            </div>
        </div>
        </div>
        <div id="audioDiv" class="row col d-flex  dropdown">
          <button class="btn btn-secondary-outline btn-sm dropdown-toggle" type="button" id="audioDropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
            🎧
          </button>
          <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
          <a id="speakerButton" class="btn btn-outline-secondary btn-sm" onclick="module_uiLib.default.content.handleSpeakToggle()">⏯</a>
          <button id="speakerPauseButton" class="btn btn-outline-secondary btn-sm">⏸</button>
            <div id="audioOptionsDiv" class="row justify-content-center" >
              <div>
                <p class="lead">Vol</p>
                <input type="range" min="0" max="1" value="1" step="0.1" id="volume" />
                <span id="volume-label" class="ms-2">1</span>
              </div>
              <div class="mx-5">
                <p class="lead">Rate</p>
                <input type="range" min="0.1" max="10" value="1" id="rate" step="0.1" />
                <span id="rate-label" class="ms-2">1</span>
              </div>
              <div>
                <p class="lead">Pitch</p>
                <input type="range" min="0" max="2" value="1" step="0.1" id="pitch" />
                <span id="pitch-label" class="ms-2">1</span>
              </div>
            </div>
          </div>
        </div>
        <a href="https://vishvAsa.github.io/notes/search"  class="btn btn-light btn-sm">🥽🔦</a>
        <div class="row col  d-flex justify-content-center">
          <div><a id="previousPage" href="" class="btn btn-secondary">###<i class="fas fa-caret-left"></i></a></div>
          <div ><a id="nextPage" href="" class="btn btn-secondary"><i class="fas fa-caret-right"></i> ### </a></div>
        </div>
        <ul id="top-bar-right-custom" class="list-group list-group-horizontal">
        </ul>
      </div>
    </nav>
</header>

    
    <div class="row" name="contentRow">
      
      <aside class="col-md-3 card border noPrint " id="sidebar">
          <div id="fastSearch">
              <datalist id="pageDataList"></datalist>
              <input id="searchInput" placeholder="शीर्षिकान्विष्यताम्" list="pageDataList" onchange="module_uiLib.default.navigation.pageLoader()">
          </div>
          <details open  class="bg-light-gray" >
              <summary id="sidebarToggleLink" onclick="module_uiLib.default.navigation.sidebarToggleHandler()">
            Menu
              </summary>
              <ul id="sidebarBody" class="list pl2 p-0 bg-yellow">
              </ul>
          </details>
      </aside>
      <main class="col p-3" role="main" data-pagefind-body>
        
<header class='border d-flex justify-content-between'>
    <h1 id="3 Ancient DNA Opens the Floodgates">3 Ancient DNA Opens the Floodgates</h1>
    
    <a id="editLink" class="btn btn-primary noPrint"  href="https://github.com/vishvAsa/notes/edit/content/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates.md"><i class="fas fa-edit"></i></a>
    
</header>
<article>
  <aside id="toc_card" class="card border noPrint ">
    <details id="toc_header">
        <summary>What's in this page?</summary>
      <div id="toc_body" class="p-0">
        
        <ul id="toc_ul" class="list p-0">
        </ul>
      </div>
    </details>
    <div id="progressBarDiv" class="row d-flex justify-content-center">
      🐎💪<progress id="progressLoading" value="0" max="100"> 32% </progress>
    </div>
  </aside>
  <div id="post_content">
  <p>A Surprise from the East</p>
<p>In 2008, Russian archaeologists dug up a pinky bone at Denisova Cave in the Altai Mountains of southern Siberia, named after an eighteenth-century Russian hermit named Denis who had made his home there. The bone’s growth plates were not fused, showing that the bone came from a child. Its date was uncertain, as it was too small to be dated by radiocarbon analysis, and it was found in a mixed-up soil layer of the cave that contained artifacts dating to both less than thirty thousand and more than fifty thousand years ago. The leader of the excavation, Anatoly Derevianko, reasoned that the bone’s owner could have been a modern human, and the sample was so labeled. Alternatively, could the bone’s owner have been a Neanderthal, as Neanderthal remains were also found near the cave?1 Derevianko sent part of the bone to Svante Pääbo in Germany.</p>
<p>Pääbo’s team, led by Johannes Krause, was successful in extracting mitochondrial DNA from the Denisova Cave bone.2 Its sequence was of a type that had never before been observed in more than ten thousand modern human and seven Neanderthal sequences. There are around two hundred mutational differences separating the mitochondrial DNA of people living today from that of Neanderthals. The new mitochondrial DNA from the Denisova finger bone featured nearly four hundred differences from the mitochondrial DNA of both present-day humans and Neanderthals. Based on the rate at which mutations accumulate, mitochondrial DNA sequences from present-day humans and Neanderthals are estimated to have separated from each other 470,000 to 360,000 years ago.3 The number of mutational differences found in the mitochondrial DNA from the Denisova finger bone suggested a separation time of roughly eight hundred thousand to one million years ago. This suggested that the finger bone might belong to a member of a never-before-sampled group of archaic humans.4</p>
<p>The identity of the population, however, was unclear. No skeletons or toolmaking styles existed to give a hint, as they had in the case of the Neanderthals. For Neanderthals, archaeological discoveries had motivated the sequencing of a genome. For this new archaic group, the genetic data came first.</p>
<p>A Genome in Search of a Fossil</p>
<p>I first found out about this previously unknown archaic human population in early 2010, while visiting Svante Pääbo’s laboratory in Leipzig, Germany. I was there on one of the thrice-yearly trips I had been making since joining the consortium that Pääbo put together in 2007 to analyze the Neanderthal genome. One evening, Pääbo took me out to a beer garden and told me about the new mitochondrial sequence they had come across. Miraculously, the Denisova finger bone had provided one of the best-preserved samples of ancient DNA ever found. While Pääbo had screened dozens of Neanderthal samples to find a few with up to 4 percent primate DNA, this finger bone had about 70 percent. Pääbo and his team had already been able to obtain more data on the whole genome (not just mitochondrial DNA) from this small bone than they had previously obtained from Neanderthals. He asked if I’d be interested in helping to analyze the data. The invitation to analyze the Denisovan genome was the greatest piece of good fortune I have had in my scientific career.</p>
<p>The mitochondrial genome suggested that the Denisova finger bone came from an individual who was part of a human population that split from the ancestors of modern humans and Neanderthals before they separated from each other. But mitochondrial DNA only records information on the entirely female line, a tiny fraction of the many tens of thousands of lineages that have contributed to any person’s genome. To understand what really happened in an individual’s history, it is incomparably more valuable to examine all ancestral lineages together. For the Denisova finger bone, the whole genome painted a very different picture from what was recorded in the mitochondrial DNA.</p>
<p>The first revelation from the whole genome was that Neanderthals and the new humans from Denisova Cave were more closely related to each other than either was to modern humans—a different pattern from what was observed in mitochondrial DNA.5 We eventually estimated the separation between the Neanderthal and Denisovan ancestral populations to have occurred 470,000 to 380,000 years ago, and the separation between the common ancestral populations of both of these archaic groups and modern humans to have occurred 770,000 to 550,000 years ago.6 The different pattern of relatedness for mitochondrial DNA and the consensus of the rest of the genome were not necessarily a contradiction, as the time in the past when two individuals share a common ancestor at any section of their DNA is always at least as old as the time when their ancestors separated into populations, and can sometimes be far older. However, by studying the whole genome we can learn when the populations split, recognizing that the whole genome encompasses a whole multitude of ancestors so that we can search for short segments of the genome with a relatively low density of mutations reflecting a shared ancestor who lived just before the population separation. Our findings meant that the Denisovans were cousins of Neanderthals, but were also very different, having separated from Neanderthal ancestors before many Neanderthal traits appeared in the fossil record.</p>
<p>We had a heated debate about what to call the new population, and decided to use a generic non-Latin name, “Denisovans,” after the cave where they were first discovered, in the same way that Neanderthals are named after the Neander Valley in Germany. This decision distressed some of our colleagues, who lobbied for a new species name—perhaps <em>Homo altaiensis,</em> after the mountains where Denisova Cave is located. <em>Homo altaiensis</em> is now used in a museum exhibit in Novosibirsk in Russia that describes the discovery at Denisova. We geneticists, however, were reluctant to use a species name. There has long been contention as to whether Neanderthals constitute a species separate from modern humans, with some experts designating Neanderthals as a distinct species of the genus <em>Homo</em> (<em>Homo neanderthalensis</em>), and others as a subgroup of modern humans (<em>Homo sapiens neanderthalensis</em>). The designation of two living groups as distinct species is often based on the supposition that the two do not in practice interbreed.7 But we now know Neanderthals interbred successfully with modern humans and in fact did so on multiple occasions seems to undermine the argument that they are distinct species. Our data showed that Denisovans were cousins of Neanderthals, and thus if we are uncertain about whether Neanderthals are a species, we need to be uncertain about whether Denisovans are a species as well. Decisions about whether extinct populations are distinct enough to merit designation as different species are traditionally made based on the shapes of skeletons, and for Denisovans there are very few physical remains, providing even more reason to be cautious.</p>
<p>The few remains that we do have are intriguing. Derevianko and his colleagues sent Pääbo a couple of molar teeth from Denisova Cave that contained mitochondrial DNA closely related to the finger bone. These teeth were enormous, beyond the range of nearly all teeth previously reported in the genus <em>Homo.</em> Large molars are thought to be biological adaptations to a diet that includes lots of tough uncooked plants. Prior to the Denisovans, the humans closest to us who were known to have had teeth of this size were the primarily plant-eating <em>australopithecenes,</em> like the famous “Lucy,” whose skeleton, dating to more than three million years ago, was found in the Awash Valley of Ethiopia. “Lucy” did not use tools and had a brain only slightly larger than chimpanzees’ after correcting for her smaller body size, but she walked upright. Thus the little skeletal information we had confirmed the idea that Denisovans were very distinctive compared to both Neanderthals and modern humans.</p>
<p>The Hybridization Principle</p>
<p>Armed with a whole-genome sequence, we tested whether the Denisovans were more closely related to some present-day populations than others. This led to a huge surprise.</p>
<p>Denisovans were genetically a little closer to New Guineans than they were to any population from mainland Eurasia, suggesting that New Guinean ancestors had interbred with Denisovans. Yet the distance from Denisova Cave to New Guinea is around nine thousand kilometers, and New Guinea is, of course, separated by sea from the Asian mainland. The climate in New Guinea is also largely tropical, which could not be more different from Siberia’s bitter winters, and this makes it unlikely that archaic humans adapted to one environment would have flourished in the other.</p>
<p>Skeptical of our findings, we cast around for alternative explanations. Had the ancestors of modern humans been divided into several populations hundreds of thousands of years ago, one of which was more closely related to Denisovans and contributed more to New Guineans’ ancestry than it contributed to the ancestry of most other present-day populations? However, this scenario would suggest that the genetic affinity to Denisovans in present-day New Guineans would be due to segments of DNA that entered the New Guinean lineage many hundreds of thousands of years ago. In New Guinean genomes today we were able to measure the size of intact archaic ancestry segments, and found that the ones related to Denisovans were about 12 percent longer than the ones related to Neanderthals, implying that the Denisovan-related segments had been introduced that much more recently on average.8</p>
<p>As soon as archaic populations mix with modern ones, the DNA segments contributed by archaic humans are chopped up by the process of recombination, spliced together with modern human segments at the rate of one or two splices per chromosome per generation. As discussed in chapter two, the length of Neanderthal ancestry segments corresponds to mixture between fifty-four and forty-nine thousand years ago.9 Based on how much longer the Denisovan segments were than the Neanderthal segments in New Guineans, we could conclude that the interbreeding between Denisovan and New Guinean ancestors occurred fifty-nine to forty-four thousand years ago.10</p>
<figure class="">

    <a href="../images/000015.jpg" target="_blank" rel="noopener">
        <img src="../images/000015.jpg" alt="Figure 10" title="">
    </a>

    

</figure>
<p>Figure 10. Approximate proportions of Neanderthal (left) and Denisovan ancestry (right) in representative present-day human populations as a fraction of the maximum detected in any group today. Today, Denisovan ancestry is concentrated east of Huxley’s Line, a deep-sea trench that has always divided mainland Asia from Australia and New Guinea even in the ice ages when sea levels were lower.</p>
<p>What percentage of New Guinean genomes today derives from Denisovans? By measuring how much stronger the genetic evidence of archaic ancestry is in New Guineans compared to other non-Africans, we estimated that about 3 to 6 percent of New Guinean ancestry derives from Denisovans. That is above and beyond the approximately 2 percent from Neanderthals. Thus in total, 5 to 8 percent of New Guinean ancestry comes from archaic humans. This is the largest known contribution of archaic humans to any present-day human population.</p>
<p>The Denisova discovery proved that interbreeding between archaic and modern humans during the migration of modern humans from Africa and the Near East was not a freak event. So far, DNA from two archaic human populations—Neanderthals and Denisovans—has been sequenced, and in both cases, the data made it possible to detect hybridization between modern and archaic humans that had been previously unknown. I would not be surprised if DNA sequenced from the next newly discovered archaic population will also point to a previously unknown hybridization event.</p>
<p>Breaching Huxley’s Line</p>
<p>Where, given the vast distance between Siberia and New Guinea, did interbreeding between Denisovans and the ancestors of New Guineans occur?</p>
<p>Our first guess was mainland Asia, perhaps India or central Asia, on a plausible human migratory path from Africa to New Guinea. If this had been the case, the lack of much Denisovan-related ancestry in mainland East or South Asia could be explained by later waves of expansion on the part of modern humans without Denisovan-related ancestry, who replaced populations having Denisovan-related ancestry. That these later migrations did not contribute much to the DNA of present-day New Guineans might account for the relatively high proportion of Denisovan-related ancestry in New Guinean populations today.</p>
<p>A first glance at the geographic distribution of Denisovan-related ancestry in present-day people seemed to support this idea. We collected DNA from present-day humans from the islands of the Southwest Pacific and from East Asia, South Asia, and Australia, and estimated how much Denisovan-related ancestry each of them had. We found the largest amounts of ancestry in indigenous populations in the islands off Southeast Asia and especially in the Philippines and the very large islands of New Guinea and Australia (by the word “indigenous” I refer to people who were established prior to the population movements associated with the spread of farming).11 The populations in question are largely east of Huxley’s Line, a natural boundary that separates New Guinea, Australia, and the Philippines from the western parts of Indonesia and the Asian mainland. This line was described by the nineteenth-century British naturalist Alfred Russel Wallace, and adapted by his contemporary the biologist Thomas Henry Huxley to highlight differences in the animals living on either side, for example, it roughly forms the boundary between placental mammals to the west and marsupials to the east. It corresponds to deep ocean trenches that have formed geographical barriers to the crossing of animals and plants, even in ice ages when sea levels were up to one hundred meters lower. It is remarkable that modern humans after fifty thousand years ago made it across this barrier. These pioneers did manage to cross, but it must have been difficult. Modern humans with Denisovan-related ancestry living east of Huxley’s Line—the ancestors of New Guineans, Australians, and Philippine populations who we found are the groups with the largest proportions of Denisovan ancestry today—are likely to have been protected by the same barrier from further migrations from Asia, just like the animals with whom they share their landscape.</p>
<p>But a deeper look suggests that population mixture in the heart of Asia is not as easy an explanation as it might at first seem. Although some populations east of Huxley’s Line have large amounts of Denisovan-related ancestry, the situation is very different to the west. Most notably, the indigenous hunter-gatherers of the Andaman Island chain off the coasts of India and Sumatra, and also the indigenous hunter-gatherers of the Malay Peninsula of mainland Southeast Asia, descend from lineages just as divergent as those in indigenous New Guineans and Australians, and yet they do not have much Denisovan-related ancestry. There is also no evidence of elevated Denisovan-related ancestry in genome-wide data from the approximately forty-thousand-year-old human of Tianyuan Cave near Beijing in China, which was sequenced several years later by Pääbo and his laboratory.12 Had the interbreeding occurred in mainland Asia, and modern humans carrying Denisovan-related ancestry then spread all over, multiple populations of the region as well as ancient humans from East Asia would be expected to carry Denisovan-related ancestry in amounts comparable to what is seen in New Guineans. But this is not what we observe.</p>
<p>The simplest explanation for the large fractions of Denisovan-related ancestry on the islands off the southeastern tip of Asia and in New Guinea and Australia would be the occurrence of interbreeding near the islands—on the islands themselves or in mainland Southeast Asia—but in either case in a tropical region very far from Denisova Cave. However, the anthropologist Yousuke Kaifu pointed out in a talk I attended in 2011 that the hypothesis of interbreeding near the islands is difficult to square with an absence of archaeological artifacts in the region that could plausibly reflect the presence of a big-brained cousin of Neanderthals and modern humans. Kaifu also pointed out that no big-skulled skeletons from this time in this region have so far been found. This makes me think that it is more likely that interbreeding occurred in southern China or mainland Southeast Asia. There are archaic human remains from Dali in Shaanxi province in north-central China, from Jinniushan in Liaoning in northeastern China, and from Maba in Guangdong in southeastern China, all dating to around two hundred thousand years ago, all of which are more plausible skeletal matches for the Denisovans. An archaic human from Narmada in central India may date to around seventy-five thousand years ago. Chinese and Indian government rules complicate the export of skeletal material, but world-class ancient DNA labs have now been established in China and are beginning to be built in India. DNA from these samples could lead to extraordinary insights.</p>
<p>Meet the Australo-Denisovans</p>
<p>While the interbreeding Neanderthals were close relatives of those we obtained samples from and sequenced, the archaic people who interbred with the ancestors of New Guineans were not close relatives of the Siberian Denisovans. When we examined the genomes of present-day New Guineans and Australians, and counted the number of DNA letter differences between them and the Siberian Denisovans to estimate when their ancestors separated from a common parent population, we discovered that everywhere in the genome, the number of differences was at least what would be expected for a population split that occurred 400,000 to 280,000 years ago.13 This meant that the ancestors of the Siberian Denisovans separated from the Denisovan lineage that contributed ancestry to New Guineans two-thirds of the way back to the separation of the ancestors of Denisovans from Neanderthals.</p>
<p>In light of the remote relationship, the two groups probably had different adaptations, which would explain how they were able to thrive in such different climates. Given the extraordinary diversity of Denisovans—with much more time separation among their populations than exists among present-day groups—it makes sense to think of them as a broad category of humans, one branch of which became the ancestors of the archaic population that interbred with New Guineans and another that became Siberian Denisovans. Most likely there are other Denisovan populations as well that we haven’t sampled at all. Maybe we should even consider Neanderthals as part of this broad Denisovan family.</p>
<p>We never assigned a special name to the Denisovan-related population that interbred with modern humans who migrated to the islands off Southeast Asia, but I like to call them “Australo-Denisovans” to highlight their likely southern geographical distribution. Anthropologist Chris Stringer prefers “Sunda Denisovans” after the landmass that joined most of the Indonesian islands to the Southeast Asian mainland.14 But this would not be an accurate name if the interbreeding occurred in what is now mainland Southeast Asia, China, or India.</p>
<p>It is tempting to think that the Australo-Denisovans, Denisovans, and Neanderthals descend from the first <em>Homo erectus</em> populations that expanded out of Africa, and that modern humans descend from the <em>Homo</em> <em>erectus</em> populations that stayed in Africa, but that would be wrong. The oldest <em>Homo</em> <em>erectus</em> skeletons outside of Africa have been found at the site of Dmanisi in Georgia dating to around 1.8 million years ago, and on the island of Java in Indonesia dating to almost the same time. If <em>Homo erectus</em> from the first radiation out of Africa was ancestral to the Denisovans and Neanderthals, then the split of these populations from modern humans would be at least as old as the dispersal to Eurasia—far too old to be consistent with the genetic observations. The genetic data give a split date of 770,000 to 550,000 years ago, too recent to be consistent with a 1.8-million-year-old population separation.</p>
<p>There is, however, a candidate in the fossil record for an ancestor in the right period, dating to long after the <em>Homo erectus</em> out-of-Africa migration but after the <em>Homo sapiens</em> one. A big-skulled skeleton found near Heidelberg in Germany in 1907 and dated to around six hundred thousand years ago15 was plausibly from a species that was ancestral to modern humans and Neanderthals,16 and by implication, Denisovans too. <em>Homo heidelbergensis</em> is often viewed as both a West Eurasian and an African species, but not an East Eurasian species. However, the genetic evidence from the Australo-Denisovans shows that the <em>Homo heidelbergensis</em> lineage may have been established very anciently in East Eurasia too. One of the profound implications of the Denisovan discovery was that East Eurasia is a central stage of human evolution and not a sideshow as westerners often assume.</p>
<p>So we now have access to genome-wide data from four highly divergent human populations that all likely had big brains, and that were all still living more recently than seventy thousand years ago. These populations are modern humans, Neanderthals, Siberian Denisovans, and Australo-Denisovans. To these we need to add the tiny humans of Flores island in present-day Indonesia—the “hobbits” who likely descend from early <em>Homo erectus</em> whose descendants arrived at Flores island before seven hundred thousand years ago and became isolated there by deep waters.17 These five groups of humans and probably more groups still undiscovered who lived at that time were each separated by hundreds of thousands of years of evolution. This is greater than the separation times of the most distantly related human lineages today—for example, the one highly represented in San hunter-gatherers from southern Africa and everyone else. Seventy thousand years ago, the world was populated by very diverse human forms, and we have genomes from an increasing number of them, allowing us to peer back to a time when humanity was much more variable than it is today.</p>
<p>How Archaic Encounters Helped Modern Humans</p>
<p>What is the biological legacy of the interbreeding between modern humans and Denisovans? The highest proportion of Denisovan-related ancestry in any present-day population is found in New Guineans and Australians and the people to whom they contributed ancestry.18 However, once we obtained better data and used more sensitive techniques, we found that there is also some Denisovan-related ancestry, albeit far less, even in mainland Asia,19 and it is from the mainland that we have a clue about its biological effects.</p>
<p>The Denisovan-related ancestry in East Asians is about a twenty-fifth of that seen in New Guineans—it comprises about 0.2 percent of East Asians’ genomes, rising to up to 0.3–0.6 percent in parts of South Asia.20 We have not yet been able to determine if the Denisovan-related ancestry in mainland Asia and the islands off Southeast Asia comes from the same archaic population or from different ones. If the ancestry comes from very different sources, we would be detecting yet another instance of archaic human interbreeding with modern humans. But whatever its origin, the Denisovan interbreeding was biologically significant.</p>
<p>One of the most striking genomic discoveries of the past few years is a mutation in a gene that is active in red blood cells and that allows people who live in high-altitude Tibet to thrive in their oxygen-poor environment. Rasmus Nielsen and colleagues have shown that the segment of DNA on which this mutation occurs matches much more closely to the Siberian Denisovan genome than to DNA from Neanderthals or present-day Africans.21 This suggests that some Denisovan relatives in mainland Asia may have harbored an adaptation to high altitude, which the ancestors of Tibetans inherited through Denisovan interbreeding. Archaeological evidence shows that the first inhabitants of the Tibetan high plateau began living there seasonally after eleven thousand years ago, and that permanent occupation based on agriculture began around thirty-six hundred years ago.22 It is likely that the mutation increased rapidly in frequency only after these dates, a prediction that will be possible to test directly through DNA studies of ancient Tibetans.</p>
<p>Interbreeding with Neanderthals helped modern humans to adapt to new environments just as interbreeding with Denisovans did. We and others showed that at genes associated with the biology of keratin proteins, present-day Europeans and East Asians have inherited much more Neanderthal ancestry on average than is the case for most other groups of genes.23 This suggests that versions of keratin biology genes carried by Neanderthals were preserved in non-Africans by the pressures of natural selection, perhaps because keratin is an essential ingredient of skin and hair, which are important for providing protection from the elements in cold environments such as the ones that modern humans were moving into and to which Neanderthals were already adapted.</p>
<p>Superarchaic Humans</p>
<p>Given that Denisovans and Neanderthals are genetically closer to each other than either is to modern humans, it would be reasonable to expect them to be equidistantly related to present-day populations that have not received genetic input from either of these archaic populations—that is, to sub-Saharan Africans. Yet we found sub-Saharan Africans to be slightly more closely related to Neanderthals than to Denisovans.24 This must reflect another example of interbreeding we didn’t know about. The pattern we observed could only be explained by Denisovan interbreeding with a deeply divergent, still unknown archaic population—one from which Africans and Neanderthals have little or no DNA, and which separated from the common ancestors of modern humans, Neanderthals, and Denisovans well before their separation from each other.</p>
<p>The evidence for an unknown archaic contribution to Denisovans is that at locations in the genome where all Africans share a mutation, the mutation is more often seen in Neanderthals than in Denisovans. Because these are mutations that all Africans carry, we know that they occurred long ago, as it typically takes around a million years or more in humans for a new mutation not under natural selection to spread throughout a population and achieve 100 percent frequency. The only way to explain the fact that Denisovans do not also share these mutations is if the ancestors of the Denisovans interbred with a population that diverged from Denisovans, Neanderthals, and modern humans so long ago that nearly all modern humans carry the new mutation.</p>
<p>By examining mutations that occur at 100 percent frequency in present-day Africans, and measuring the excess rate at which they matched the Neanderthal over the Denisovan genome, we estimated that the unknown archaic population that interbred into Denisovans first split off from the lineage leading to modern humans 1.4 to 0.9 million years ago and that this unknown archaic population contributed at least 3 to 6 percent of Denisovan-related ancestry. The date is shaky, as knowledge of the human mutation rate is poor. However, even with the uncertainty about the mutation rate, we can estimate relative dates reasonably well, and we can be confident that this previously unsampled human population split off at about twice the separation time of Denisovans, Neanderthals, and modern humans. I think of this group as “superarchaic” humans, as they represent a more deeply splitting lineage than Denisovans. They are what I call a “ghost” population, a population we do not have data from in unmixed form, but whose past existence can be detected from its genetic contributions to later people.</p>
<p>Eurasia as a Hothouse of Human Evolution</p>
<p>From a combination of archaeological and genetic data, we can be confident of at least four major population separations involving modern and archaic human lineages over the last two million years.</p>
<p>The skeletal evidence shows that the first important spread of humans to Eurasia occurred at least 1.8 million years ago, bringing <em>Homo erectus</em> from Africa. The genetic evidence suggests that a second lineage split from the one leading to modern humans around 1.4 to 0.9 million years ago, giving rise to the superarchaic group that we have evidence of through its mixture with the ancestors of Denisovans and that plausibly contributed the highly divergent Denisovan mitochondrial DNA sequence that shares a common ancestor with both Neanderthals and modern humans in this time frame. Genetics also suggests a third major split 770,000 to 550,000 years ago when the ancestors of modern humans separated from Denisovans and Neanderthals, followed by Denisovans and Neanderthals from each other 470,000 to 380,000 years ago.</p>
<p>These genetic dates depend on estimates of the mutation rate and will change as those estimates become more exact. It is easy to get ensnared in trying to establish neat correlations between genetic dates and the archaeological record, only to have dates shift when a new genetic estimate of the rate of occurrence of new mutations comes along, causing the whole intellectual edifice to come tumbling down. However, the order of these splits and the distinctness of the populations can be determined well from genetics.</p>
<p>The usual assumption is that all four of these splits correspond to ancestral populations in Africa expanding into Eurasia. But does this really have to be the case?</p>
<p>The argument that modern humans radiated from Africa comes from the observation that the most deeply divergent branches among present-day humans are most strongly represented in African hunter-gatherers (such as San from southern Africa and central African Pygmies). The oldest remains of humans with anatomically modern features are also found in Africa and date to up to around three hundred thousand years ago. However, the genetic comparisons of present-day populations that point to an origin in Africa can only probe the population structure that has arisen in the last couple of hundred thousand years, the time frame of the diversification of the ancestors of present-day populations. With ancient DNA data in hand, we are confronted with the observation that of the four deepest human lineages from which we have DNA data, the three most deeply branching ones are represented only in human specimens excavated from Eurasia: the Neanderthals, the Denisovans, and the “superarchaic” population that left traces among the Siberian Denisovans.</p>
<p>Part of the reason we detect the oldest splitting lineages in Eurasians may be what scientists call “ascertainment bias”: the fact that almost all ancient DNA work has been done in Eurasia rather than in Africa, and so naturally that is where new lineages have been discovered. Perhaps if we had as many archaic ancient DNA sequences from Africa as we do from Eurasia, we would find lineages there that split from modern humans and Neanderthals even more deeply in time than the superarchaic.</p>
<p>But another possibility suggests itself, which is that the ancestral population of modern humans, Neanderthals, and Denisovans actually lived in Eurasia, descending from the original <em>Homo erectus</em> spread out of Africa. In this scenario, there was later migration back from Eurasia to Africa, providing the primary founders of the population that later evolved into modern humans. The attraction of this theory is its economy: it requires one less major population movement between Africa and Eurasia to explain the data. The superarchaic population and the ancestral population of modern humans, Denisovans, and Neanderthals could both have arisen within Eurasia, without requiring two further out-of-Africa migrations, as long as there was just one later migration back into Africa to establish shared ancestry with modern humans there.</p>
<figure class="">

    <a href="../images/000019.jpg" target="_blank" rel="noopener">
        <img src="../images/000019.jpg" alt="Figure 11" title="">
    </a>

    

</figure>
<p>Figure 11. Can the modern human lineage have sojourned for hundreds of thousands of years outside Africa? Conventional models have the human lineage evolving in Africa at all times. To explain current skeletal and genetic data, a minimum of four out-of-Africa migrations are required. However, if our ancestors lived outside Africa from before 1.8 million years ago until up to three hundred thousand years ago, as few as three major migrations would be required.</p>
<p>An argument from economy is not a proof. But the bigger point is that the evidence for many lineages and admixtures should have the effect of shaking our confidence in what to many people is now an unquestioned assumption that Africa has been the epicenter of all major events in human evolution. Based on the skeletal record, it is certain that Africa played a central role in the evolution of our lineage prior to two million years ago, as we have known ever since the discovery of the upright walking apes who lived in Africa millions of years before <em>Homo</em>. We know too that Africa has played a central role in the origin of anatomically modern humans, based on the skeletons of humans with anatomically modern features there up to around three hundred thousand years ago, and the genetic evidence for a dispersal in the last fifty thousand years out of Africa and the Near East. But what of the intervening period between two million years ago and about three hundred thousand years ago? In a large part of this time, the human skeletons we have from Africa are not obviously more closely related to modern humans than are the human skeletons of Eurasia.25 Over the last couple of decades, there has been a pendulum swing toward the view that because our lineage was in Africa before two million years ago and after three hundred thousand years ago, our ancestors must always have been there. But Eurasia is a rich and varied supercontinent, and there is no fundamental reason that the lineage leading to modern humans cannot have sojourned there for an important period before returning to Africa.</p>
<p>The genetic evidence that the ancestors of modern humans may have spent a substantial part of their evolutionary history in Eurasia is in fact consistent with a theory advanced by María Martinón-Torres and Robin Dennell.26 Theirs is a minority viewpoint within the fields of archaeology and anthropology, but a respected one. They argue that humans they call <em>Homo antecessor,</em> found in Atapuerca, Spain, and dating to around one million years ago, show a mix of traits indicating that they are from a population ancestral to modern humans and Neanderthals. This is a very ancient date for a modern human/Neanderthal ancestral population to exist in Eurasia. Many who think that Neanderthals in Europe descend from an out-of-Africa radiation of an ancestral population would assume that the ancestors of both populations were still in Africa at that time. Combining this evidence with archaeological analysis of stone tool types, Martinón-Torres and Dennell argue for the possibility of continuous Eurasian habitation from at least 1.4 million years ago until the most recent common ancestor of humans and Neanderthals after eight hundred thousand years ago, at which point one lineage migrated back to Africa to become the lineage that evolved into modern humans.27 The Martinón-Torres and Dennell theory becomes more plausible in light of the new genetic evidence.</p>
<p>Part of the “out of Africa” allure is the simplifying idea that Africa—and especially East Africa—has always been the cradle of human diversity and the place where innovation occurred, and that the rest of the world is an evolutionarily inert receptacle. But is there really such a strong case that all the key events in human evolution happened in the same region of the world? The genetic data show that many groups of archaic humans populated Eurasia and that some of these interbred with modern humans. This forces us to question why the direction of migration would have always been out of Africa and into Eurasia, and whether it could sometimes have been the other way around.</p>
<p>The Most Ancient DNA Yet</p>
<p>At the beginning of 2014, Matthias Meyer, Svante Pääbo, and their colleagues in Leipzig extended by a factor of around four the record for the oldest human DNA obtained, sequencing mitochondrial DNA from a more than four-hundred-thousand-year-old <em>Homo heidelbergensis</em> individual from the Sima de los Huesos cave system in Spain where twenty-eight ancient humans were found at the bottom of a thirteen-meter shaft.28 The Sima skeletons have early Neanderthal-like traits, and the archaeologists who excavated them have interpreted them as being on the lineage leading to Neanderthals after the separation from the ancestors of modern humans. Two years after Meyer and Pääbo published mitochondrial DNA data from Sima de los Huesos, they published genome-wide data.29 Their analysis not only confirmed that the Sima humans were on the Neanderthal lineage, but went further in showing that the Sima humans were more closely related to Neanderthals than they are to Denisovans. These results provided direct evidence that Neanderthal ancestors were already evolving in Europe at least four hundred thousand years ago, and that the separation of the Neanderthal and Denisovan lineages had already begun by that time.</p>
<p>But the Sima data were also perplexing: Sima’s mitochondrial genome was more closely related to Denisovans than to Neanderthals, at odds with the genome-wide pattern of it being most closely related to Neanderthals.30 If there were only one discrepancy between the average relationship measured by the whole genome and the relationship seen in mitochondrial DNA, it might just be possible to believe that this was a statistical fluctuation. But there are two discrepancies in the genetic relationships: the fact that the Sima de los Huesos individual has Denisovan-type mitochondrial DNA despite being closer to Neanderthals in the rest of the genome, and the fact that the Siberian Denisovan individual has mitochondrial DNA twice as divergent from modern humans and Neanderthals as they were from each other despite being closer to Neanderthals in the rest of the genome.31 The coincidence of these two observations is so improbable that it seems more likely that there is a deeper story to unravel.</p>
<p>Perhaps the superarchaic humans—the ones who interbred with Denisovans—were a much more important part of Eurasian human population history than we initially imagined. Maybe, after separating from the lineage leading to modern humans around 1.4 to 0.9 million years ago, these superarchaic humans spread across Eurasia and began to evolve the ancient mitochondrial lineage found in the Denisovans and Sima humans. At roughly half this time, another group may have split off the lineage leading to modern humans and then spread throughout Eurasia. This group may have mixed into the superarchaic population, contributing the largest proportion of ancestry to populations in the west that evolved into Neanderthals, and a smaller but still substantial proportion of ancestry to populations in the east that became the ancestors of Denisovans. This scenario would explain the findings of two anciently divergent mitochondrial DNA types in the different groups. It could also explain an odd unpublished observation I have: that in studying the variation in the time since the common genetic ancestor of modern human genomes with both Denisovan and Neanderthal genomes, I have not been able to find evidence for a superarchaic population that contributed to Denisovans but not to Neanderthals. Instead the patterns suggest that Denisovans and Neanderthals both had ancestry from the same superarchaic population, with just a larger proportion present in the Denisovans.</p>
<p>Johannes Krause and colleagues have suggested an alternative theory. Krause’s idea is that several hundred thousand years ago, an early modern human population migrated out of Africa and mixed with groups like the one that lived in Sima de los Huesos, replacing their mitochondrial DNA along with a bit of the rest of their genomes and creating a mixed population that evolved into true Neanderthals.32 The idea might seem complicated, but in fact it could explain multiple disparate observations beyond the fact that Neanderthals had a mitochondrial sequence much more similar to modern humans than it did to either the Sima de los Huesos individual or the Siberian Denisovan. It could account for the fact that the estimated date of the common ancestor of humans and Neanderthals in mitochondrial DNA (470,000 to 360,000 years ago)33 is paradoxically more recent than the estimated date of separation of the ancestors of these two populations based on the analysis of the whole genome (770,000 to 550,000 years ago).34 It could also explain how it was that Neanderthals and modern humans both used complex Middle Stone Age methods of manufacturing stone tools, even though the earliest evidence for this tool type is hundreds of thousands of years after the genetically estimated separation of the Neanderthals and modern human lineages.35 The theory finally becomes more plausible in light of a study led by Sergi Castellano and Adam Siepel that suggested up to 2 percent interbreeding into the ancestors of Neanderthals from an early modern human lineage.36 If Krause’s theory is right, this could have been the lineage that spread the mitochondrial DNA found in all Neanderthals.</p>
<p>Whatever explains these patterns, it is clear that we have much more to learn. The period before fifty thousand years ago was a busy time in Eurasia, with multiple human populations arriving from Africa beginning at least 1.8 million years ago. These populations split into sister groups, diverged, and mixed again with each other and with new arrivals. Most of those groups have since gone extinct, at least in their “pure” forms. We have known for a while, from skeletons and archaeology, that there was some impressive human diversity prior to the migration of modern humans out of Africa. However, we did not know before ancient DNA was extracted and studied that Eurasia was a locus of human evolution that rivaled Africa. Against this background, the fierce debates about whether modern humans and Neanderthals interbred when they met in western Eurasia—which have been definitively resolved in favor of interbreeding events that made a contribution to billions of people living today—seem merely anticipatory. Europe is a peninsula, a modest-sized tip of Eurasia. Given the wide diversity of Denisovans and Neanderthals—already represented in DNA sequences from at least three populations separated from each other by hundreds of thousands of years, namely Siberian Denisovans, Australo-Denisovans, and Neanderthals—the right way to view these populations is as members of a loosely related family of highly evolved archaic humans who inhabited a vast region of Eurasia.</p>
<p>Ancient DNA has allowed us to peer deep into time, and forced us to question our understanding of the past. If the first Neanderthal genome published in 2010 opened a sluice in the dam of knowledge about the deep past, the Denisova genome and subsequent ancient DNA discoveries opened the floodgates, producing a torrent of findings that have disrupted many of the comfortable understandings we had before. And that was only the beginning.</p>

  </div>
</article>


        

        



    <aside class="card border noPrint" id="section-tree-item-sapiens_branches_articles_reich_who-we-are_1_The_Deep_History_of_Our_Species_3_Ancient_DNA_Opens_the_Floodgates_md">
        <div class="card-title bg-light-gray border d-flex justify-content-between">
            <a href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/3_Ancient_DNA_Opens_the_Floodgates/">3 Ancient DNA Opens the Floodgates </a>
            <a data-toggle="collapse" href="#section-tree-item-body-sapiens_branches_articles_reich_who-we-are_1_The_Deep_History_of_Our_Species_3_Ancient_DNA_Opens_the_Floodgates_md" role="button" aria-expanded="false" aria-controls="section-tree-item-body-sapiens_branches_articles_reich_who-we-are_1_The_Deep_History_of_Our_Species_3_Ancient_DNA_Opens_the_Floodgates_md" >…<i class="fas fa-caret-down" class="collapsed"></i> </a>
        </div>
        <nav id="section-tree-item-body-sapiens_branches_articles_reich_who-we-are_1_The_Deep_History_of_Our_Species_3_Ancient_DNA_Opens_the_Floodgates_md" class="card-body p-0 collapse">
            
            <li>draft: false</li>
            
            <li>iscjklanguage: false</li>
            
            <li>title: 3 Ancient DNA Opens the Floodgates</li>
            
        </nav>
    </aside>


      </main>
    </div>
    <footer class="bg-yellow  p-1 noPrint" role="contentinfo">
  

  <div id="div_foot_nav_bar" class="row">
    <a href="https://vishvAsa.github.io/notes/search"  class="btn btn-light btn-sm">🥽🔦</a>
    <div class="row col  d-flex justify-content-center">
      <div><a id="previousPageFoot" href="" class="btn btn-secondary">###<i class="fas fa-caret-left"></i></a></div>
      <div ><a id="nextPageFoot" href="" class="btn btn-secondary"><i class="fas fa-caret-right"></i> ### </a></div>
    </div>
      <ul id="footer-bar-right-custom" class="list-group list-group-horizontal">
      </ul>
  </div>
  <div id="div_footer_bar" class="container-fluid d-flex justify-content-between">
    <a class="btn btn-secondary" href="https://github.com/vishvAsa/notes/issues/new" >
      ✉️Contact
    </a>
    <a class="btn btn-secondary" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/80x15.png" /></a>
    <div class="btn small border">
      
      <a href="https://github.com/vishvAsa/notes/actions/workflows/build.yml"><img src="https://github.com/vishvAsa/notes/actions/workflows/build.yml/badge.svg"> Built </a> on 2025 Oct 31 15:25:51 UTC. (<a href="http://google.com/search?q=15%3a25%3a51%20UTC to IST">IST</a>)
    </div>
  </div>
</footer>



<script async src="https://www.googletagmanager.com/gtag/js?id=G-33H08JSLQR"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-33H08JSLQR');
</script>

  </body>
  <script type='text/javascript'>
    
    
    
    module_uiLib.onDocumentReadyTasks();
  </script>
  
</html>
