---
title: Remote Ancestry दूर-पितरः
---


## Migratory hypotheses

- Out of Africa routes
    - NatGeo [2011](http://voices.nationalgeographic.com/2011/11/03/modern-humans-wandered-out-of-africa-via-arabia/).

## Genetics
### R1a1a* (Paternal)
![](../r1a1a_map.png)
{caption="R1A1A distribution" class="thumbnail"}

#### Descent (23andme 2017)
- A
    - 275,000 Years Ago
- F-M89
    - 76,000 Years Ago
- K-M9
    - 53,000 Years Ago
- R-M207
    - 35,000 Years Ago
- R-M420 (R1a)
    - 25,000 Years Ago
- R-SRY1532.2 (R1a1)
- R-M17/M198 (R1a1a)
    - R-M512 snp

<details><summary>On R-M512 (द्रष्टुं नोद्यम्)</summary>

> Today, the men who share your haplogroup are most common in Eastern Europe, Russia and Ukraine. The lineage is also quite common in Poland, but decreases in frequency toward the Mediterranean countries. Farther to the west, about one-third of Norwegian men and a quarter of men from the far northern British Isles carry R-M512. Their ancestors arrived with various groups over the past 2,000 years, including with the Anglo-Saxons from central Europe in the 5th century and the Vikings who came from Scandinavia beginning about 800 CE. Additionally, the haplogroup is still relatively common in the Middle East, as well as in Central and South Asia where it reaches levels of up to 60% among the Kyrgyz and the Tajiks.
</details>



### R30a (Maternal line) Haplogroup: R, a subgroup of N
- Distribution
    - [![](http://i.imgur.com/dtJzFQL.png)](http://i.imgur.com/dtJzFQL.png)        
- Haplogroup R
    - R arose in the Near East about 60,000 years ago, not long after humans first spread out of Africa into southwestern Asia.

### Modeling

Observations:

- Cluster with draviDian brAhmaNa-s.
- Higher Iran component than average South Indian. Only some Telugus have such high Iran in the South, and these people have 2-3% less steppe.
- Roughly same level (8%) of  steppe component as dravidian v4s.

<details><summary>qpAdm models (द्रष्टुं नोद्यम्)</summary>

- Euclidean distance in 25-d space between the real sample (you) and the best possible simulation based on source populations: 1.1341% / 0.01134109 (Via Aryamsha - "Steppe ancestry is a bit underestimated on G25 simulations")
  - 56.8 IRN_Shahr_I_Sokhta_BA2
  - 15.0 Aryan
  - 14.6 AASI
  - 10.2 TKM_Gonur2_BA
  - 3.4 CHN_Upper_Yellow_River_LN
- 52.1 IVC_I8726 (90% IranN), 35.8 Onge, 12.2 Sintashta with p=0.2
- 48.7% pure IranN, 43.5% pure Onge and 7.8% pure Sintashta with p=0.74 when modelled distally
- 45.6% IVC (70% IranN, 25% AASI and 5% neolithic steppe), 45.7% Irula, 8.6% Sintashta. 
  - Steppe rich ancestors admixed with Steppe poor ancestors around 300 AD. (algorithm DATES developed by Moorjani. date was not accurate. Unfortunately for getting decent admixture dates one needs many samples.)
</details>


<details><summary>Modelling FAQ (द्रष्टुं नोद्यम्)</summary>

- IVC = Indus Valley Civilization, Irula = A tribe acting as a placeholder for early Indian hunter gatherers
- p>0.05 means an acceptable model.
- Modelled distally means modelled with sources who are far removed from the actual admixture event. So in this case one removes Ancient South Indians and IVC as the two likely sources with Sintashta and used Iran and Onge instead to get a better idea of the absolute genetic contribution.
- The fundamental issue is this: IVC samples we have so far DON'T form a cluster (IVC people vary a lot in AASI from 15% to 60%!). But they are very different from each other - this is one main reason why Razib, MT, etc believe that jAti stratification predated the Arya-s. For the sake of simplicity all of them are combined in literature as IVC-periphery.
</details>

### G25 coordinates
![](../images/gedmatch_g25-sim-coords-in-pca_v.png)
{caption="" class="thumbnail"}

<details><summary>Sim data (द्रष्टुं नोद्यम्)</summary>

```
sample_simulated_g25_scaled,0.052518853080000205,-0.063301992006,-0.14566985323960002,0.10784362091799965,-0.07612815976539998,0.0565666492762,-0.0009902706709999654,0.010137610529000085,0.027979721893999776,0.013410032445999975,-0.005207541857356,-0.001137532665999988,-0.00009183506799999297,-0.001645104876999648,0.0010020213650001053,0.003043852208000075,-0.001736520640999939,0.0006642111010000673,0.0009010056189999865,-0.004519488418000001,-0.0006131895440000257,-0.0018655687889999606,0.0026530372280000186,-0.00029511439999993505,0.0014945043999997631
```
</details>

<details><summary>Getting coordinates (द्रष्टुं नोद्यम्)</summary>

- create an account and upload raw data on GEDMATCH
- Go to Free tools > Admixture calculators > Eurogenes. 
- There enter your gedmatch kit code, select K36 and click on enter. Wait.
- Copy values you get by selecting with your cursor and copy in this link.
- Generate simulated coordinates from [allelocator](https://allelocator.ovh/K36vertical.html).g
</details>


## गोत्रादिकम्
- भार्गव-च्यावन-आप्नवान-और्व-जामदग्न्य-प्रवरः।
  - [भार्गवविषयोऽत्र](/notes/history/homo/sapiens/Aryan/satem/indo-iranian/indo-aryan/v1/persons/sage-bloodlines/bhRguH/dvitIyajanmani_bhRguH/chyavanaH/ApnavAna/aurvaH/jamadagniH/) ।
- ಹೆಬ್ಬಾರ್ ಅಯ್ಯಂಗಾರ್ ಎಮ್ಬ [ಸಮುದಾಯವು](/notes/history/homo/sapiens/Aryan/satem/indo-iranian/indo-aryan/v1/hebbAr-iyengAr/). 

## Projects

- Some genes affecting human intelligence here: [MT](https://manasataramgini.wordpress.com/2007/10/31/genes-affecting-human-intelligence/) and [23andme1](https://www.23andme.com/you/community/thread/717/).
- Genetic haplotypes marking gotra-kartR-s.