<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
    
    <title>Vishvas&#39;s notes  | Acknowledgments</title>
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">

    <meta name="viewport" content="width=device-width,minimum-scale=1">
    <meta name="generator" content="Hugo 0.143.0">
    
    
      <META NAME="ROBOTS" CONTENT="INDEX, FOLLOW">
    

    <link href="/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/" rel="alternate" type="application/rss+xml" title="Vishvas&#39;s notes" />
    <link href="/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/" rel="feed" type="application/rss+xml" title="Vishvas&#39;s notes" />

    <meta property="og:url" content="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/">
  <meta property="og:site_name" content="Vishvas&#39;s notes">
  <meta property="og:title" content="Acknowledgments">
  <meta property="og:description" content="First thing first. This book emerged out of a year of intense collaboration with my wife, Eugenie Reich. We researched the book together, prepared the first drafts of the chapters together, and talked about the book incessantly as it matured. This book would not have come into being without her.
I am grateful to Bridget Alex, Peter Bellwood, Samuel Fenton-Whittet, Henry Louis Gates Jr., Yonatan Grad, Iosif Lazaridis, Daniel Lieberman, Shop Mallick, Erroll McDonald, Latha Menon, Nick Patterson, Molly Przeworski, Juliet Samuel, Clifford Tabin, Daniel Reich, Tova Reich, Walter Reich, Robert Weinberg, and Matthew Spriggs for close critical readings of the entire book.">
  <meta property="og:locale" content="en_us">
  <meta property="og:type" content="article">
    <meta property="article:section" content="sapiens">

  <meta itemprop="name" content="Acknowledgments">
  <meta itemprop="description" content="First thing first. This book emerged out of a year of intense collaboration with my wife, Eugenie Reich. We researched the book together, prepared the first drafts of the chapters together, and talked about the book incessantly as it matured. This book would not have come into being without her.
I am grateful to Bridget Alex, Peter Bellwood, Samuel Fenton-Whittet, Henry Louis Gates Jr., Yonatan Grad, Iosif Lazaridis, Daniel Lieberman, Shop Mallick, Erroll McDonald, Latha Menon, Nick Patterson, Molly Przeworski, Juliet Samuel, Clifford Tabin, Daniel Reich, Tova Reich, Walter Reich, Robert Weinberg, and Matthew Spriggs for close critical readings of the entire book.">
  <meta itemprop="wordCount" content="5440">
  <meta name="twitter:card" content="summary">
  <meta name="twitter:title" content="Acknowledgments">
  <meta name="twitter:description" content="First thing first. This book emerged out of a year of intense collaboration with my wife, Eugenie Reich. We researched the book together, prepared the first drafts of the chapters together, and talked about the book incessantly as it matured. This book would not have come into being without her.
I am grateful to Bridget Alex, Peter Bellwood, Samuel Fenton-Whittet, Henry Louis Gates Jr., Yonatan Grad, Iosif Lazaridis, Daniel Lieberman, Shop Mallick, Erroll McDonald, Latha Menon, Nick Patterson, Molly Przeworski, Juliet Samuel, Clifford Tabin, Daniel Reich, Tova Reich, Walter Reich, Robert Weinberg, and Matthew Spriggs for close critical readings of the entire book.">
<script type="text/javascript">
    
    let baseURL = "https:\/\/vishvAsa.github.io\/notes\/";
    let basePath = "/" + baseURL.split("/").slice(3).join("/");
    let siteParams = JSON.parse("{\u0022build_img\u0022:\u0022https:\/\/github.com\/vishvAsa\/notes\/actions\/workflows\/build.yml\/badge.svg\u0022,\u0022build_url\u0022:\u0022https:\/\/github.com\/vishvAsa\/notes\/actions\/workflows\/build.yml\u0022,\u0022contactlink\u0022:\u0022https:\/\/github.com\/vishvAsa\/notes\/issues\/new\u0022,\u0022env\u0022:\u0022production\u0022,\u0022githubeditmepathbase\u0022:\u0022https:\/\/github.com\/vishvAsa\/notes\/edit\/content\/\u0022,\u0022js_extras\u0022:[\u0022mathjax\u0022]}");
    let pageDefaultsList = JSON.parse("[{\u0022scope\u0022:{\u0022pathPrefix\u0022:\u0022\u0022},\u0022values\u0022:{\u0022comments\u0022:true,\u0022layout\u0022:\u0022page\u0022,\u0022search\u0022:true,\u0022sidebar\u0022:\u0022home_sidebar\u0022}}]");
    let sidebarsData = JSON.parse("{\u0022home_sidebar\u0022:{\u0022contents\u0022:[{\u0022url\u0022:\u0022recdir:\/\/\u0022},{\u0022contents\u0022:[{\u0022title\u0022:\u0022ज्यौतिषम्\u0022,\u0022url\u0022:\u0022..\/jyotiSham\/\u0022},{\u0022title\u0022:\u0022संस्कृतम्\u0022,\u0022url\u0022:\u0022..\/sanskrit\/\u0022},{\u0022title\u0022:\u0022देवः\u0022,\u0022url\u0022:\u0022..\/devaH\/\u0022},{\u0022title\u0022:\u0022आगमः\u0022,\u0022url\u0022:\u0022..\/AgamaH\/\u0022},{\u0022title\u0022:\u0022आगमः शैवः\u0022,\u0022url\u0022:\u0022..\/AgamaH_shaivaH\/\u0022},{\u0022title\u0022:\u0022आगमः वैष्णवः\u0022,\u0022url\u0022:\u0022..\/AgamaH_vaiShNavaH\/\u0022},{\u0022title\u0022:\u0022रामानुजीयम्\u0022,\u0022url\u0022:\u0022..\/rAmAnujIyam\/\u0022},{\u0022title\u0022:\u0022माध्वम्\u0022,\u0022url\u0022:\u0022..\/mAdhvam\/\u0022},{\u0022title\u0022:\u0022आगमः ब्राह्मः\u0022,\u0022url\u0022:\u0022..\/AgamaH_brAhmaH\/\u0022},{\u0022title\u0022:\u0022कल्पान्तरम्\u0022,\u0022url\u0022:\u0022..\/kalpAntaram\/\u0022},{\u0022title\u0022:\u0022Notes Home\u0022,\u0022url\u0022:\u0022..\/notes\/\u0022},{\u0022title\u0022:\u0022वेदाः\u0022,\u0022url\u0022:\u0022..\/vedAH\/\u0022},{\u0022title\u0022:\u0022वेदाः - ऋक्\u0022,\u0022url\u0022:\u0022..\/vedAH_Rk\/\u0022},{\u0022title\u0022:\u0022वेदाः - यजुः\u0022,\u0022url\u0022:\u0022..\/vedAH_yajuH\/\u0022},{\u0022title\u0022:\u0022वेदाः - साम\u0022,\u0022url\u0022:\u0022..\/vedAH_sAma\/\u0022},{\u0022title\u0022:\u0022पुराणम्\u0022,\u0022url\u0022:\u0022..\/purANam\u0022},{\u0022title\u0022:\u0022पुराणम् वैष्णवम्\u0022,\u0022url\u0022:\u0022..\/purANam_vaiShNavam\/\u0022},{\u0022title\u0022:\u0022महाभारतम्\u0022,\u0022url\u0022:\u0022..\/mahAbhAratam\/\u0022},{\u0022title\u0022:\u0022रामायणम्\u0022,\u0022url\u0022:\u0022..\/rAmAyaNam\/\u0022},{\u0022title\u0022:\u0022काव्यम्\u0022,\u0022url\u0022:\u0022..\/kAvyam\/\u0022},{\u0022title\u0022:\u0022मीमांसा\u0022,\u0022url\u0022:\u0022..\/mImAMsA\/\u0022},{\u0022title\u0022:\u0022ಕನ್ನಡ\u0022,\u0022url\u0022:\u0022..\/kannaDa\/\u0022},{\u0022title\u0022:\u0022त्रिपिटकम्\u0022,\u0022url\u0022:\u0022..\/tipiTaka\/\u0022},{\u0022title\u0022:\u0022पाळयः\u0022,\u0022url\u0022:\u0022..\/pALi\/\u0022},{\u0022title\u0022:\u0022भाषान्तरम्\u0022,\u0022url\u0022:\u0022..\/bhAShAntaram\/\u0022},{\u0022title\u0022:\u0022English Lit\u0022,\u0022url\u0022:\u0022..\/english\/\u0022},{\u0022title\u0022:\u0022Book pub\u0022,\u0022url\u0022:\u0022..\/book-pub\u0022},{\u0022title\u0022:\u0022Vishvas\u0027s home page\u0022,\u0022url\u0022:\u0022..\/\u0022}],\u0022title\u0022:\u0022सङ्ग्रहान्तरम्\u0022},{\u0022title\u0022:\u0022\\u003ci class=\\\u0022fas fa-hand-holding-heart\\\u0022\\u003e\\u003c\/i\\u003eDonate Via Vishvas\u0022,\u0022url\u0022:\u0022https:\/\/vishvAsa.github.io\/interests\/dharma-via-vishvas\/\u0022}],\u0022title\u0022:\u0022Parts\u0022}}");
    
    var pageRelUrlTree = {};
</script>

    
    <script type="text/javascript">
  let pageVars = {};
  pageVars.pageUrlMinusBasePath = "\/notes\/sapiens\/branches\/articles\/reich_who-we-are\/0\/Acknowledgments\/".replace(basePath, "/");
  pageVars.pageParams = {};
  pageVars.pageSource = "sapiens\/branches\/articles\/reich_who-we-are\/0\/Acknowledgments.md";
  console.log(pageVars.pageSource);
  var pageDefaults;
  for (let possiblePageDefaults of pageDefaultsList) {
    if (pageVars.pageSource.startsWith(possiblePageDefaults.scope.pathPrefix)) {
      pageDefaults = possiblePageDefaults.values
    }
  }
    
</script>


    <script src="/notes/webpack_dist/uiLib-bundle.js"></script>


<script>


MathJax = {
  tex: {
    inlineMath: [ ["@#", "#@"], ["\\(", "\\)"] ],
    displayMath: [ ["$$","$$"], ["\\[", "\\]"] ],
    processEscapes: true,  
  },
  svg: {
    fontCache: 'global'
  },
  options: {
    ignoreHtmlClass: "tex2jax_ignore|dno"
  }
};
</script>
<script type="text/javascript" id="MathJax-script" async
        src="/notes/non_webpack_js/mathjax/tex-svg.js">
</script>






    <link rel="stylesheet" href="/notes/webpack_dist/uiLib.css">
<link rel="stylesheet" href="/notes/css/uiLibPrint.css" media="print" >
<link rel="stylesheet" href="/notes/css/fonts.css" >
<link rel="stylesheet" href="/notes/css/@fortawesome/fontawesome-free/css/solid.min.css">

<link rel="stylesheet" href="/notes/css/@fortawesome/fontawesome-free/css/fontawesome.min.css">


<link rel="alternate" hreflang="sa-Deva" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/" />
<link rel="alternate" hreflang="sa-Deva" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=devanagari" />

<link rel="alternate" hreflang="sa-Knda" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=kannada" />
<link rel="alternate" hreflang="sa-Mlym" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=malayalam" />
<link rel="alternate" hreflang="sa-Telu" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=telugu" />
<link rel="alternate" hreflang="sa-Taml-t-sa-Taml-m0-superscript" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=tamil_superscripted" />
<link rel="alternate" hreflang="sa-Taml" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=tamil" />
<link rel="alternate" hreflang="sa-Gran" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=grantha" />

<link rel="alternate" hreflang="sa-Gujr" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=gujarati" />
<link rel="alternate" hreflang="sa-Guru" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=gurmukhi" />
<link rel="alternate" hreflang="sa-Shar" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=sharada" />
<link rel="alternate" hreflang="sa-Dogr" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=dogra" />
<link rel="alternate" hreflang="sa-Phag" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=phags_pa" />
<link rel="alternate" hreflang="sa-Tibt" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=tibetan" />
<link rel="alternate" hreflang="sa-Sidd" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=siddham" />
<link rel="alternate" hreflang="sa-Newa" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=newa" />
<link rel="alternate" hreflang="sa-Modi" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=modi" />
<link rel="alternate" hreflang="sa-Tirh" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=tirhuta_maithili" />
<link rel="alternate" hreflang="sa-Gong" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=gondi_gujala" />
<link rel="alternate" hreflang="sa-Gonm" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=gondi_masaram" />


<link rel="alternate" hreflang="sa-Orya" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=oriya" />
<link rel="alternate" hreflang="sa-Beng-t-sa-Beng-m0-assamese" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=assamese" />
<link rel="alternate" hreflang="sa-Beng" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=bengali" />
<link rel="alternate" hreflang="sa-Mtei" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=manipuri" />
<link rel="alternate" hreflang="sa-Sinh" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=sinhala" />
<link rel="alternate" hreflang="sa-Limb" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=limbu" />
<link rel="alternate" hreflang="sa-Lepc" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=lepcha" />


<link rel="alternate" hreflang="sa-Cyrl" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=cyrillic" />
<link rel="alternate" hreflang="sa-Latn-t-sa-Zyyy-m0-iso" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=iso" />
<link rel="alternate" hreflang="sa-Latn-t-sa-Zyyy-m0-iast" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=iast" />
<link rel="alternate" hreflang="sa-Aran" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=urdu" />
<link rel="alternate" hreflang="sa-Avst" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=avestan" />

<link rel="alternate" hreflang="sa-Brah" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=brahmi" />
<link rel="alternate" hreflang="sa-Khar" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=kharoshthi" />
<link rel="alternate" hreflang="sa-Zanb" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=zanbazar_square" />
<link rel="alternate" hreflang="sa-Bhks" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=bhaiksuki" />

<link rel="alternate" hreflang="sa-Bali" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=balinese" />
<link rel="alternate" hreflang="sa-Java" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=javanese" />
<link rel="alternate" hreflang="sa-Laoo" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=lao" />
<link rel="alternate" hreflang="sa-Laoo-t-sa-Laoo-m0-Laopali" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=lao_pali" />
<link rel="alternate" hreflang="sa-Mymr" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=burmese" />
<link rel="alternate" hreflang="sa-Mymr-t-sa-Mymr-m0-Mon" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=mon" />
<link rel="alternate" hreflang="sa-Cham" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=cham" />
<link rel="alternate" hreflang="sa-Khmr" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=khmer" />
<link rel="alternate" hreflang="sa-Thai" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/?transliteration_target=thai" />

  </head>

  <body class="ma0 bg-near-white">
    
\(
% groupings of objects.
\newcommand{\set}[1]{\left\{ #1 \right\}}
\newcommand{\seq}[1]{\left(#1\right)}
\newcommand{\ang}[1]{\langle#1\rangle}
\newcommand{\tuple}[1]{\left(#1\right)}
\newcommand{\size}[1]{\left| #1\right|}

\newcommand{\comp}{\circ}


% numerical shortcuts.
\newcommand{\abs}[1]{\left| #1\right|}
\newcommand{\floor}[1]{\left\lfloor #1 \right\rfloor}
\newcommand{\ceil}[1]{\left\lceil #1 \right\rceil}

% linear algebra shortcuts.
\newcommand{\change}{\Delta}
\newcommand{\norm}[1]{\left\| #1\right\|}
\newcommand{\dprod}[1]{\langle#1\rangle}
\newcommand{\linspan}[1]{\langle#1\rangle}
\newcommand{\conj}[1]{\overline{#1}}
\newcommand{\der}[1]{\frac{d#1}{dx}}
\newcommand{\lap}{\Delta}
\newcommand{\kron}{\otimes}
\newcommand{\nperp}{\nvdash}

\newcommand{\mat}[1]{\left[ \begin{smallmatrix}#1 \end{smallmatrix} \right]}

% derivatives and limits
\newcommand{\partder}[2]{\frac{\partial #1}{\partial #2}}
\newcommand{\partdern}[3]{\frac{\partial^{#3 #1}}{\partial #2^{#3}}}
\newcommand{\gradient}{\nabla}
\newcommand{\subdifferential}{\partial}

% Arrows
\newcommand{\diverge}{\nearrow}
\newcommand{\notto}{\nrightarrow}
\newcommand{\up}{\uparrow}
\newcommand{\down}{\downarrow}
% gets and gives are defined!

% ordering operators
\newcommand{\oleq}{\preceq}
\newcommand{\ogeq}{\succeq}

% programming and logic operators
\newcommand{\dfn}{:=}
\newcommand{\assign}{:=}
\newcommand{\co}{\ co\ }
\newcommand{\en}{\ en\ }


% logic operators
\newcommand{\xor}{\oplus}
\newcommand{\Land}{\bigwedge}
\newcommand{\Lor}{\bigvee}
\newcommand{\finish}{\Box}
\newcommand{\contra}{\Rightarrow \Leftarrow}
\newcommand{\iseq}{\stackrel{_?{=}}}


% Set theory
\newcommand{\symdiff}{\Delta}
\newcommand{\setdiff}{\backslash}
\newcommand{\union}{\cup}
\newcommand{\inters}{\cap}
\newcommand{\Union}{\bigcup}
\newcommand{\Inters}{\bigcap}
\newcommand{\nullSet}{\phi}


% graph theory
\newcommand{\nbd}{\Gamma}

% Script alphabets
% For reals, use \Re

% greek letters
\newcommand{\eps}{\epsilon}
\newcommand{\del}{\delta}
\newcommand{\ga}{\alpha}
\newcommand{\gb}{\beta}
\newcommand{\gd}{\del}
\newcommand{\gp}{\pi}
\newcommand{\gf}{\phi}
\newcommand{\gh}{\eta}
\newcommand{\gF}{\Phi}
\newcommand{\gl}{\lambda}
\newcommand{\gm}{\mu}
\newcommand{\gn}{\nu}
\newcommand{\gr}{\rho}
\newcommand{\gs}{\sigma}
\newcommand{\gth}{\theta}
\newcommand{\gx}{\xi}
\newcommand{\gw}{\omega}

\newcommand{\sw}{\sigma}
\newcommand{\SW}{\Sigma}
\newcommand{\ew}{\lambda}
\newcommand{\EW}{\Lambda}

\newcommand{\Del}{\Delta}
\newcommand{\gD}{\Delta}
\newcommand{\gG}{\Gamma}
\newcommand{\gW}{\Omega}
\newcommand{\gS}{\Sigma}
\newcommand{\gTh}{\Theta}

% Bold english letters.
\newcommand{\bA}{\mathbf{A}}
\newcommand{\bB}{\mathbf{B}}
\newcommand{\bC}{\mathbf{C}}
\newcommand{\bD}{\mathbf{D}}
\newcommand{\bE}{\mathbf{E}}
\newcommand{\bF}{\mathbf{F}}
\newcommand{\bG}{\mathbf{G}}
\newcommand{\bH}{\mathbf{H}}
\newcommand{\bI}{\mathbf{I}}
\newcommand{\bJ}{\mathbf{J}}
\newcommand{\bK}{\mathbf{K}}
\newcommand{\bL}{\mathbf{L}}
\newcommand{\bM}{\mathbf{M}}
\newcommand{\bN}{\mathbf{N}}
\newcommand{\bO}{\mathbf{O}}
\newcommand{\bP}{\mathbf{P}}
\newcommand{\bQ}{\mathbf{Q}}
\newcommand{\bR}{\mathbf{R}}
\newcommand{\bS}{\mathbf{S}}
\newcommand{\bT}{\mathbf{T}}
\newcommand{\bU}{\mathbf{U}}
\newcommand{\bV}{\mathbf{V}}
\newcommand{\bW}{\mathbf{W}}
\newcommand{\bX}{\mathbf{X}}
\newcommand{\bY}{\mathbf{Y}}
\newcommand{\bZ}{\mathbf{Z}}

\newcommand{\bba}{\mathbf{a}}
\newcommand{\bbb}{\mathbf{b}}
\newcommand{\bbc}{\mathbf{c}}
\newcommand{\bbd}{\mathbf{d}}
\newcommand{\bbe}{\mathbf{e}}
\newcommand{\bbf}{\mathbf{f}}
\newcommand{\bbg}{\mathbf{g}}
\newcommand{\bbh}{\mathbf{h}}
\newcommand{\bbk}{\mathbf{k}}
\newcommand{\bbl}{\mathbf{l}}
\newcommand{\bbm}{\mathbf{m}}
\newcommand{\bbn}{\mathbf{n}}
\newcommand{\bbp}{\mathbf{p}}
\newcommand{\bbq}{\mathbf{q}}
\newcommand{\bbr}{\mathbf{r}}
\newcommand{\bbs}{\mathbf{s}}
\newcommand{\bbt}{\mathbf{t}}
\newcommand{\bbu}{\mathbf{u}}
\newcommand{\bbv}{\mathbf{v}}
\newcommand{\bbw}{\mathbf{w}}
\newcommand{\bbx}{\mathbf{x}}
\newcommand{\bby}{\mathbf{y}}
\newcommand{\bbz}{\mathbf{z}}
\newcommand{\0}{\mathbf{0}}
\newcommand{\1}{\mathbf{1}}

% Caligraphic english alphabet
\newcommand{\cA}{\mathcal{A}}
\newcommand{\cB}{\mathcal{B}}
\newcommand{\cC}{\mathcal{C}}
\newcommand{\cD}{\mathcal{D}}
\newcommand{\cE}{\mathcal{E}}
\newcommand{\cF}{\mathcal{F}}
\newcommand{\cG}{\mathcal{G}}
\newcommand{\cH}{\mathcal{H}}
\newcommand{\cI}{\mathcal{I}}
\newcommand{\cJ}{\mathcal{J}}
\newcommand{\cK}{\mathcal{K}}
\newcommand{\cL}{\mathcal{L}}
\newcommand{\cM}{\mathcal{M}}
\newcommand{\cN}{\mathcal{N}}
\newcommand{\cO}{\mathcal{O}}
\newcommand{\cP}{\mathcal{P}}
\newcommand{\cQ}{\mathcal{Q}}
\newcommand{\cR}{\mathcal{R}}
\newcommand{\cS}{\mathcal{S}}
\newcommand{\cT}{\mathcal{T}}
\newcommand{\cU}{\mathcal{U}}
\newcommand{\cV}{\mathcal{V}}
\newcommand{\cW}{\mathcal{W}}
\newcommand{\cX}{\mathcal{X}}
\newcommand{\cY}{\mathcal{Y}}
\newcommand{\cZ}{\mathcal{Z}}


% Formatting shortcuts
\newcommand{\red}[1]{\textcolor{red}{#1}}
\newcommand{\blue}[1]{\textcolor{blue}{#1}}
\newcommand{\htext}[2]{\texorpdfstring{#1}{#2}}

% Statistics
\newcommand{\distr}{\sim}
\newcommand{\stddev}{\sigma}
\newcommand{\covmatrix}{\Sigma}
\newcommand{\mean}{\mu}
\newcommand{\param}{\theta}
\newcommand{\gthEst}{\hat{\theta}}
\newcommand{\ftr}{\phi}
\newcommand{\est}[1]{\hat{#1}}

% General utility
\newcommand{\todo}[1]{\textbf{[TODO]}] \footnote{TODO: #1}}
\newcommand{\tbc}{[\textbf{Incomplete}]}
\newcommand{\chk}{[\textbf{Check}]}
\newcommand{\why}{[\textbf{Find proof}]}
\newcommand{\opt}[1]{\textit{#1}}

\newcommand{\experience}[1]{[\textbf{Personal Experience}]: #1 \blacktriangle}
\newcommand{\pf}[1]{[\textbf{Proof}]: #1 \Box}
\newcommand{\core}[1]{\textbf{Core Idea}: #1 \Arrowvert}
\newcommand{\example}[1]{\textbf{Example}: #1 \blacktriangle}
\newcommand{\error}[1]{\textbf{Error alert}: #1 \triangle}
\newcommand{\oprob}{[\textbf{OP}]: }


\renewcommand{\~}{\htext{$\sim$}{~}}

\DeclareMathOperator*{\argmin}{arg\,min}
\DeclareMathOperator*{\argmax}{arg\,max}


\)



    
<header class="p-1 bg-yellow noPrint">
    <nav role="navigation" class="col">
      <div id="div_top_bar" class="row">
        <div class="col-md-3 justify-content-start">
          <a  href="/" class="btn col border">
            <i class="fas fa-gopuram ">Vishvas&#39;s notes <br/> Acknowledgments</i>
          </a>
        </div>
        <div id="div_top_bar_right" class="col-md-auto d-flex justify-content-center">
          <button class="btn btn-light"  href="#transliterationDropdown" data-toggle="collapse">
            ✍
          </button>
          <select id="transliterationDropdown" size="1" class="collapse" onchange="module_uiLib.default.content.updateTransliteration()">
            <optgroup label="🌍">
              <option value="iso">āISO</option>
              <option value="iso_vedic">āi-āu</option>
              <option value="cyrillic">пу</option>
              <option value="urdu">نس</option>
              <option value="??">🌍??</option>
            </optgroup>
            <optgroup label="🛕🧭S">
              <option value="kannada">ಅ</option>
              <option value="malayalam">അ</option>
              <option value="telugu">క</option>
              <option value="tamil_superscripted">க²</option>
              <option value="tamil_extended">க</option>
              <option value="grantha">𑌅</option>
            </optgroup>
            <optgroup label="🛕🧭N">
              <option value="devanagari" selected="">स</option>
              <option value="gujarati">અ</option>
              <option value="gurmukhi">ਅ</option>
            </optgroup>
            <optgroup label="🛕🧭E">
              <option value="oriya">ଅ</option>
              <option value="assamese">অস</option>
              <option value="bengali">অ</option>
            </optgroup>
            <optgroup label="🛕🏺📜">
              <option value="sharada">𑆑𑇀𑆰</option>
              <option value="brahmi">𑀅</option>
            </optgroup>
            <optgroup label="🛕🌋">
              <option value="balinese">ᬩᬮᬶ</option>
            </optgroup>
          </select>
          <div>
          <button class="btn btn-outline-secondary btn-sm"  href="#transliterationDropdownAlt" data-toggle="collapse">
          (✍)
          </button>
          <select id="transliterationDropdownAlt" size="1"  style="font-size:12px" class="bg-light-yellow collapse" onchange="module_uiLib.default.content.updateTransliteration()">
            <optgroup label="🌍">
              <option value="iso">āISO</option>
              <option value="iso_vedic">āi-āu</option>
              <option value="cyrillic">пу</option>
              <option value="urdu">نس</option>
              <option value="??">🌍??</option>
            </optgroup>
            <optgroup label="🛕🧭S">
              <option value="kannada">ಅ</option>
              <option value="malayalam">അ</option>
              <option value="telugu">క</option>
              <option value="tamil_superscripted">க²</option>
              <option value="tamil_extended">க</option>
              <option value="grantha">𑌅</option>
            </optgroup>
            <optgroup label="🛕🧭N">
              <option value="devanagari" selected="">स</option>
              <option value="gujarati">અ</option>
              <option value="gurmukhi">ਅ</option>
            </optgroup>
            <optgroup label="🛕🧭E">
              <option value="oriya">ଅ</option>
              <option value="assamese">অস</option>
              <option value="bengali">অ</option>
            </optgroup>
            <optgroup label="🛕🏺📜">
              <option value="sharada">𑆑𑇀𑆰</option>
              <option value="brahmi">𑀅</option>
            </optgroup>
            <optgroup label="🛕🌋">
              <option value="balinese">ᬩᬮᬶ</option>
            </optgroup>
          </select>
          </div>
          <div class="row col d-flex  dropdown">
            <button class="btn btn-secondary-outline btn-sm dropdown-toggle" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
              A±
            </button>
            <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
            <a class="btn btn-secondary btn-sm" onclick="module_uiLib.changeTextSize(2)">A+</a>
            <a class="btn btn-secondary btn-sm" onclick="module_uiLib.changeTextSize(-2)">A-</a>
            </div>
          </div>
          <button class="btn btn-light btn-sm"  href="#commentStyleDropdown" data-toggle="collapse">
            🗨
          </button>
          <select id="commentStyleDropdown" class="collapse" size="1" onchange="module_uiLib.default.content.updateCommentStyleFromDropdown()">
            <option value="on">✔</option>
            <option value="hidden">✗</option>
          </select>
          <div><a id="expandAllButton" class="btn btn-secondary" onclick="module_uiLib.default.query.setParamsAndGo({'expandDetails': '.*'})">↕️️</a></div>
          <div id="printStyleDiv" class="row col d-flex  dropdown">
          <button class="btn btn-light btn-sm dropdown-toggle"  href="#printStyleDropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
          🖨
          </button>
            <div class="dropdown-menu col" aria-labelledby="printStyleDropdown">
              <div class="row justify-content-center">
                <p>Cols:</p>
                <input type="number" min="1" max="10" value="2" step="1" id="tbColumns" style="width: 3ch;"  accesskey="c"/>
              </div>
              <div class="row justify-content-center">
                <p>A±:</p> <input type="text" id="tbBodyFontSize" value="0.4cm" style="width: 6ch;"  accesskey="f"></input>
              </div>
              <div class="row justify-content-center">
                <p>Incl:</p> <input checked type="checkbox" id="tbIncludeStyle" value="on"></input>
                <button id="btnPrintStyle" onclick="module_uiLib.default.content.updatePrintStyle()">🖨
                </button>
              </div>
            </div>
        </div>
        </div>
        <div id="audioDiv" class="row col d-flex  dropdown">
          <button class="btn btn-secondary-outline btn-sm dropdown-toggle" type="button" id="audioDropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
            🎧
          </button>
          <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
          <a id="speakerButton" class="btn btn-outline-secondary btn-sm" onclick="module_uiLib.default.content.handleSpeakToggle()">⏯</a>
          <button id="speakerPauseButton" class="btn btn-outline-secondary btn-sm">⏸</button>
            <div id="audioOptionsDiv" class="row justify-content-center" >
              <div>
                <p class="lead">Vol</p>
                <input type="range" min="0" max="1" value="1" step="0.1" id="volume" />
                <span id="volume-label" class="ms-2">1</span>
              </div>
              <div class="mx-5">
                <p class="lead">Rate</p>
                <input type="range" min="0.1" max="10" value="1" id="rate" step="0.1" />
                <span id="rate-label" class="ms-2">1</span>
              </div>
              <div>
                <p class="lead">Pitch</p>
                <input type="range" min="0" max="2" value="1" step="0.1" id="pitch" />
                <span id="pitch-label" class="ms-2">1</span>
              </div>
            </div>
          </div>
        </div>
        <a href="https://vishvAsa.github.io/notes/search"  class="btn btn-light btn-sm">🥽🔦</a>
        <div class="row col  d-flex justify-content-center">
          <div><a id="previousPage" href="" class="btn btn-secondary">###<i class="fas fa-caret-left"></i></a></div>
          <div ><a id="nextPage" href="" class="btn btn-secondary"><i class="fas fa-caret-right"></i> ### </a></div>
        </div>
        <ul id="top-bar-right-custom" class="list-group list-group-horizontal">
        </ul>
      </div>
    </nav>
</header>

    
    <div class="row" name="contentRow">
      
      <aside class="col-md-3 card border noPrint " id="sidebar">
          <div id="fastSearch">
              <datalist id="pageDataList"></datalist>
              <input id="searchInput" placeholder="शीर्षिकान्विष्यताम्" list="pageDataList" onchange="module_uiLib.default.navigation.pageLoader()">
          </div>
          <details open  class="bg-light-gray" >
              <summary id="sidebarToggleLink" onclick="module_uiLib.default.navigation.sidebarToggleHandler()">
            Menu
              </summary>
              <ul id="sidebarBody" class="list pl2 p-0 bg-yellow">
              </ul>
          </details>
      </aside>
      <main class="col p-3" role="main" data-pagefind-body>
        
<header class='border d-flex justify-content-between'>
    <h1 id="Acknowledgments">Acknowledgments</h1>
    
    <a id="editLink" class="btn btn-primary noPrint"  href="https://github.com/vishvAsa/notes/edit/content/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments.md"><i class="fas fa-edit"></i></a>
    
</header>
<article>
  <aside id="toc_card" class="card border noPrint ">
    <details id="toc_header">
        <summary>What's in this page?</summary>
      <div id="toc_body" class="p-0">
        
        <ul id="toc_ul" class="list p-0">
        </ul>
      </div>
    </details>
    <div id="progressBarDiv" class="row d-flex justify-content-center">
      🐎💪<progress id="progressLoading" value="0" max="100"> 32% </progress>
    </div>
  </aside>
  <div id="post_content">
  <p>First thing first. This book emerged out of a year of intense collaboration with my wife, Eugenie Reich. We researched the book together, prepared the first drafts of the chapters together, and talked about the book incessantly as it matured. This book would not have come into being without her.</p>
<p>I am grateful to Bridget Alex, Peter Bellwood, Samuel Fenton-Whittet, Henry Louis Gates Jr., Yonatan Grad, Iosif Lazaridis, Daniel Lieberman, Shop Mallick, Erroll McDonald, Latha Menon, Nick Patterson, Molly Przeworski, Juliet Samuel, Clifford Tabin, Daniel Reich, Tova Reich, Walter Reich, Robert Weinberg, and Matthew Spriggs for close critical readings of the entire book.</p>
<p>I thank David Anthony, Ofer Bar-Yosef, Caroline Bearsted, Deborah Bolnick, Dorcas Brown, Katherine Brunson, Qiaomei Fu, David Goldstein, Alexander Kim, Carles Lalueza-Fox, Iain Mathieson, Eric Lander, Mark Lipson, Scott MacEachern, Richard Meadow, David Meltzer, Priya Moorjani, John Novembre, Svante Pääbo, Pier Palamara, Eleftheria Palkopoulou, Mary Prendergast, Rebecca Reich, Colin Renfrew, Nadin Rohland, Daniel Rozas, Pontus Skoglund, Chuanchao Wang, and Michael Witzel for critiques of individual chapters. I also thank Stanley Ambrose, Graham Coop, Dorian Fuller, Éadaion Harney, Linda Heywood, Yousuke Kaifu, Kristian Kristiansen, Michelle Lee, Daniel Lieberman, Michael McCormick, Michael Petraglia, Joseph Pickrell, Stephen Schiffels, Beth Shapiro, and Bence Viola for reviewing sections of the book for accuracy.</p>
<p>I am grateful to Harvard Medical School, the Howard Hughes Medical Institute, and the National Science Foundation, all of which generously supported my science while I was working on this project, and viewed it as complementary to my primary research.</p>
<p>I finally thank several people who repeatedly encouraged me to write this book. I resisted the idea for years because I did not want to distract myself from my science, and because for geneticists papers are the currency, not books. But my mind changed as my colleagues grew to include archaeologists, anthropologists, historians, linguists, and others eager to come to grips with the ancient DNA revolution. There are many papers I did not write, and many analyses I did not complete, because of the time I needed to write this book. I hope that those who read the book will emerge with a new perspective on who we are.</p>
<p>Introduction</p>
<p>This book is inspired by a visionary, Luca Cavalli-Sforza, the founder of genetic studies of our past. I was trained by one of his students, and so it is that I am part of his school, inspired by his vision of the genome as a prism for understanding the history of our species.</p>
<p>The high-water mark of Cavalli-Sforza’s career came in 1994 when he published <em>The History and Geography of Human Genes,</em> which synthesized what was then known from archaeology, linguistics, history, and genetics to tell a grand story about how the world’s peoples got to be the way they are today.1 The book offered an overview of the deep past. But it was based on what was known at the time and was therefore handicapped by the paucity of genetic data then available, which were so limited as to be nearly useless compared to the far more extensive information from archaeology and linguistics. The genetic data of the time could sometimes reveal patterns consistent with what was already known, but the information they provided were not rich enough to demonstrate anything truly new. In fact, the few major new claims that Cavalli-Sforza did make have essentially all been proven wrong. Two decades ago, everyone, from Cavalli-Sforza to beginning graduate students such as myself, was working in the dark ages of DNA.</p>
<p>Cavalli-Sforza made a grand bet in 1960 that would drive his entire career. He bet that it would be possible to reconstruct the great migrations of the past based entirely on the genetic differences among present-day peoples.2</p>
<p>Through study after study over the subsequent five decades, Cavalli-Sforza seemed to be well on the path to making good on his bet. When he started his work, the technology for studying human variation was so poor that the only possibility was to measure proteins in the blood, using variations like the A, B, and O blood types that are tested by physicians to match blood donors to recipients. By the 1990s, he and his colleagues had assembled data from more than one hundred such variations in diverse populations. Using these data they were able to reliably cluster individuals by continent based on how often they matched each other at these variations: for example, Europeans have a high rate of matching to other Europeans, East Asians to East Asians, and Africans to Africans. In the 1990s and 2000s, they brought their work to a new level by moving beyond protein variation and directly examining DNA, our genetic code. They analyzed a total of about one thousand individuals from around fifty populations spread across the planet, examining variation at more than three hundred positions in the genome.3 When they told their computer—which had no knowledge of the population labels—to cluster the individuals into five groups, the results corresponded uncannily well to commonly held intuitions about the deep ancestral divisions among humans (West Eurasians, East Asians, Native Americans, New Guineans, and Africans).</p>
<p>Cavalli-Sforza was especially interested in interpreting the genetic clusters among present-day people in terms of population history. He and his colleagues analyzed their blood group data by using a technique that identifies combinations of biological variations that are most efficient at summarizing differences across individuals. Plotting these combinations of blood group types onto a map of West Eurasia, they found that the one summarizing the most variation across individuals reached its extreme value in the Near East, and declined along a southeast-to-northwest gradient into Europe.4 They interpreted this as a genetic footprint of the migration of farmers into Europe from the Near East, known from archaeology to have occurred after nine thousand years ago. The declining intensity suggested to them that after arriving in Europe, the first farmers mixed with local hunter-gatherers, accumulating more hunter-gatherer ancestry as they expanded, a process they called “demic diffusion.”5 Until recently, many archaeologists viewed the demic diffusion model as an exemplary merging of insights from archaeology and genetics.</p>
<p>The model that Cavalli-Sforza and colleagues proposed to describe the data was intellectually attractive, but it was wrong. Its flaws became apparent beginning in 2008, when John Novembre and colleagues demonstrated that gradients like those observed in Europe can arise without migration.6 They then showed that a Near Eastern farming expansion into Europe might counter-intuitively cause the mathematical technique that Cavalli-Sforza used to produce a gradient perpendicular to the direction of migration, not parallel to it as had been seen in the real data.7</p>
<p>It took the revolution wrought by the ability to extract DNA from ancient bones—the “ancient DNA revolution”—to drive a nail into the coffin of the demic diffusion model. The ancient DNA revolution documented that the first farmers even in the most remote reaches of Europe—Britain, Scandinavia, and Iberia—had very little hunter-gatherer-related ancestry. In fact, they had less hunter-gatherer ancestry than is present in diverse European populations today. The highest proportion of early farmer ancestry in Europe is today not in Southeast Europe, the place where Cavalli-Sforza thought it was most common based on the blood group data, but instead is in the Mediterranean island of Sardinia to the west of Italy.8</p>
<p>The example of Cavalli-Sforza’s maps shows why his Sforza’s grand bet went sour. He was correct in his assumption that the present-day genetic structure of populations echoes some of the great events in the human past. For example, the lower genetic diversity of non-Africans compared to Africans reflects the reduced diversity of the modern human population that expanded out of Africa and the Near East after around fifty thousand years ago. But the present-day structure of human populations cannot recover the fine details of ancient events. The problem is not just that people have mixed with their neighbors, blurring the genetic signatures of past events. It is actually far more difficult, in that we now know, from ancient DNA, that the people who live in a particular place today almost never exclusively descend from the people who lived in the same place far in the past.9 Under these circumstances, the power of any study that attempts to reconstruct past population movements from present-day populations is limited. In <em>The History and Geography of Human Genes,</em> Cavalli-Sforza wrote that he was excluding from his analysis populations known to be the product of major migrations, such as those of European and African ancestry in the Americas that owe their origin to transatlantic migrations in the last five hundred years, or European minorities such as Roma and Jews. His bet was that the past was a much simpler place than the present, and that by focusing on populations today that are not affected by major migrations in their recorded history, he might be studying direct descendants of people who lived in the same places long before. But what the study of ancient DNA has now shown is that the past was no less complicated than the present. Human populations have repeatedly turned over.</p>
<figure class="">

    <a href="../images/000046.jpg" target="_blank" rel="noopener">
        <img src="../images/000046.jpg" alt="Figure 1a" title="">
    </a>

    

</figure>
<p>Figure 1a. A contour plot made by Luca Cavalli-Sforza in 1993 (adapted above) suggested that the movement of farmers from the east could be reconstructed from the patterns of blood group variation among people living today, with the highest proportions of such ancestry in the southeast near Anatolia.</p>
<p>Cavalli-Sforza’s transformative contribution to the field of genetic studies of human prehistory recalls the story of Moses, a visionary leader whose achievement was greater than that of anyone who followed him and who created a new template for seeing the world. The Bible says, “No prophet ever arose again in Israel like Moses,” but also tells how Moses was not allowed to reach the promised land. After leading his people for forty years through the wilderness, Moses climbed the mountain of Nebo and looked west over the Jordan River to see the land his people had been promised. But he was not allowed to enter that land. That privilege had been reserved for his successors.</p>
<figure class="">

    <a href="../images/000003.jpg" target="_blank" rel="noopener">
        <img src="../images/000003.jpg" alt="Figure 1b" title="">
    </a>

    

</figure>
<p>Figure 1b. Modern genome-wide data shows that the primary gradient of farmer ancestry in Europe does not flow southeast-to-northwest but instead in an almost perpendicular direction, a result of a major migration of pastoralists from the east that displaced much of the ancestry of the first farmers.</p>
<p>So it is with genetic studies of the past. Cavalli-Sforza saw before anyone else the full potential of genetics for revealing the human past, but his vision predated the technology needed to fulfill it. Today, however, things are very different. We have several hundred thousand times more data, and in addition we have access to the rich lode of information contained in ancient DNA, which has become a more definitive source of information about past population movements than the traditional tools of archaeology and linguistics.</p>
<p>The first five ancient human genomes were published in 2010: a few archaic Neanderthal genomes,10 the archaic Denisova genome,11 and an approximately four-thousand-year-old individual from Greenland.12 The next few years saw the publication of genome-wide data from five additional humans, followed by a burst of data from thirty-eight individuals in 2014. But in 2015, whole-genome analysis of ancient DNA went into hyperdrive. Three papers added genome-wide datasets from another sixty-six,13 then one hundred,14 and then eighty-three samples.15 By August 2017, my laboratory alone had generated genome-wide data for more than three thousand ancient samples. We are now producing data so fast that the time lag between data production and publication is longer than the time it takes to double the data in the field.</p>
<figure class="">

    <a href="../images/000008.jpg" target="_blank" rel="noopener">
        <img src="../images/000008.jpg" alt="Figure 2" title="">
    </a>

    

</figure>
<p>Figure 2. Ancient DNA labs are now producing data so fast that the time lag between data production and publication is longer than the time it takes to double the data in the field.</p>
<p>Much of the technology for the genome-wide ancient DNA revolution was invented by Svante Pääbo and his colleagues at the Max Planck Institute for Evolutionary Anthropology in Leipzig, Germany, who developed it to study extremely old samples such as archaic Neanderthals and Denisovans. My contribution has been to scale up the methods to study large numbers of relatively more recent samples, albeit still many thousands of years old. The traditional length of an apprenticeship is seven years, and I began mine in 2007 when I started working with Pääbo on the Neanderthal and Denisova genome projects. In 2013, Pääbo helped me to establish my own ancient DNA laboratory—the first in the United States focused on studying whole-genome ancient human DNA. My partner in this effort has been Nadin Rohland, who did her own seven-year apprenticeship in Pääbo’s laboratory before she came to mine. Our idea was to make ancient DNA industrial—to build an American-style genomics factory out of the techniques developed in Europe to study individual samples.</p>
<p>Rohland and I realized that a technique developed by Matthias Meyer and Qiaomei Fu in Pääbo’s laboratory could be the key to the industrial-scale study of ancient DNA. Meyer and Fu’s invention was born of necessity: the need to extract DNA from an approximately forty-thousand-year-old early modern human from Tianyuan Cave in China.16 When Meyer and Fu extracted DNA from Tianyuan’s leg bones, they found that only about 0.02 percent of it was from the man himself. The rest came from microbes that had colonized his bones after he died. This made direct sequencing too expensive, even using the hundred-thousand-times cheaper technology that had become available after around 2006. To get around this challenge, Meyer and Fu borrowed a page from the playbook of methods developed by medical geneticists. Just as medical geneticists had developed methods to isolate DNA from the 2 percent of the genome that is most interesting and to discard the other 98 percent, Meyer and Fu isolated a tiny fraction of sequences from the Tianyuan bone that were human and discarded the rest.</p>
<p>The method of DNA isolation that Meyer and Fu developed has been central to the success of the ancient DNA revolution. In the 1990s, molecular biologists learned how to adapt laser-etching techniques invented for printing electronic circuits to attach millions of DNA sequences of their choice to silicon or glass wafers. These sequences could then be cut off the wafers using molecular scissors (enzymes) and released into a watery mix. Meyer and Fu took advantage of this method to synthesize fifty-two-letter-long sequences of DNA that, overlapping like shingles on a roof, covered much of human chromosome 21. Exploiting DNA’s tendency to bind to highly similar sequences, they “fished” out the DNA sequences from Tianyuan that they were interested in by using as “bait” the sequences they had artificially synthesized. They found that a large fraction of the DNA they obtained was from Tianyuan’s genome. Not only that, but it was from the parts of Tianyuan’s genome that they wanted to study. They analyzed the data to show that Tianyuan was an early modern human, part of the lineage leading to present-day East Asians. He did not have a particularly large amount of ancestry from archaic human lineages that were diverged by hundreds of thousands of years from modern human lineages, contradicting earlier claims based on the shape of his skeleton.17</p>
<p>Rohland and I adapted this technique to study the whole genome. We worked with our colleagues in Germany to synthesize fifty-two-letter-long DNA sequences covering more than a million positions at which people are known to vary. We used these bait sequences to enrich for human compared to microbial DNA, which in some cases increased the fraction of DNA that was of interest to us by more than a hundredfold. We gained another approximately tenfold jump in efficiency because we only targeted informative positions in the genome. We automated the whole approach, processing the DNA using robots that allowed a single person to study more than ninety samples at once in the span of a few days. We hired a team of technicians to grind powder out of ancient remains, to extract DNA from the powder, and then to turn the extracted DNA into a form that we could sequence. The laboratory work was only the beginning. An equally intricate task was sorting the billions of DNA sequences into the individuals to whom they belonged, analyzing the data and weeding out samples with evidence of contamination, and creating an easily accessible dataset. Shop Mallick, a physicist who had joined my laboratory six years before, set up our computers to do all of this, and continually updated our strategy for processing the data as the nature of the data evolved and its volume increased.</p>
<p>The results were even better than we had hoped. The cost of producing genome-wide data dropped to less than five hundred dollars per sample. This was many dozens of times cheaper than brute-force whole-genome sequencing. Even better, our method made it possible to get genome-wide data out of around half of the skeletal samples we screened, although the success rate of course varied depending on the degree to which the skeletons we examined had been preserved. For example, we have obtained about 75 percent success rates for ancient samples from the cold climate of Russia, but only around 30 percent for samples from the hot Near East.</p>
<p>These advances mean that whole-genome study of ancient DNA no longer requires screening large numbers of skeletal remains before it is possible to find a few individuals whose DNA can be analyzed. Instead, a substantial fraction of screened samples dating to the last ten thousand years can now be converted to working genome-wide data. The new methods have made it possible to analyze hundreds of samples in a single study. With such data, it is possible to reconstruct population changes in exquisite detail, transforming our understanding of the past.</p>
<p>By the end of 2015, my ancient DNA laboratory at Harvard had published more than half of the world’s genome-wide human ancient DNA. We discovered that the population of northern Europe was largely replaced by a mass migration from the eastern European steppe after five thousand years ago18; that farming developed in the Near East more than ten thousand years ago among multiple highly differentiated human populations that then expanded in all directions and mixed with each other along with the spread of agriculture19; and that the first human migrants into the remote Pacific islands beginning around three thousand years ago were not the sole ancestors of the present-day inhabitants.20 In parallel, I initiated a project to survey the diversity of the world’s present-day populations, using a microchip for analyzing human variation that my collaborators and I designed specifically for the purpose of studying the human past. We used the chip to study more than ten thousand individuals from more than a thousand populations worldwide—a dataset that has become a mainstay of studies of human variation not just in my laboratory but also in other laboratories around the world.21</p>
<p>The resolution with which this revolution has allowed us to reconstruct events in the human past is stunning. I remember a dinner at the end of graduate school with my Ph.D. supervisor, David Goldstein, and his wife, Kavita Nayar, both of whom had been students of Cavalli-Sforza. It was 1999, a decade before the advent of genome-wide ancient DNA, and we daydreamed together, wondering how accurately events of the past could be reconstructed by traces left behind. After a grenade explosion in a room, could the exact position of each object prior to the explosion be reconstructed by piecing together the scattered remains and studying the shrapnel in the wall? Could languages long extinct be recalled by unsealing a cave still reverberating with the echoes of words spoken there thousands of years ago? Today, ancient DNA is enabling this kind of detailed reconstruction of deep relationships among ancient human populations.</p>
<p>These days, human genome variation has surpassed the traditional toolkit of archaeology—the study of the artifacts left behind by past societies—in what it can reveal of changes in human populations in the deep past.22 This has come as a surprise to nearly everyone. Carl Zimmer, a science journalist at <em>The</em> <em>New York Times</em> who has written frequently about this new field, told me that when he was assigned by his newspaper to cover the study of ancient DNA, he agreed to do it as a service to the science team, thinking it would be a minor sideshow to his main focus on evolution and human physiology. He imagined writing an article about the field every six months or so, and that the rush of discoveries would end after a year or two. Instead, Zimmer now finds himself dealing with a major new scientific paper every few weeks, even as developments are accelerating and the revolution intensifies.</p>
<p>This book is about the genome revolution in the study of the human past. This revolution consists of the avalanche of discoveries based on data taken from the whole genome—meaning, the entire genome analyzed at once instead of just small stretches of it such as mitochondrial DNA. The revolution has been made far more powerful by the new technologies for extracting whole genomes’ worth of DNA from ancient humans. I make no attempt to trace the history of the field of genetic studies of the past—the decades of scientific analysis of human variation that began with studies of skeletal variation and continued with studies of genetic variation in tiny snippets of the human genome. These efforts provided insights into population relationships and migrations, but those insights pale when compared to the dazzling information provided by the extraordinary tranches of data that began to be available after 2009. Before and after that year, studies of one or a few locations in the genome were occasionally the basis for important discoveries, providing evidence in favor of some scenarios over others. Yet genetic evidence before around 2009 was mostly incidental to studies of the human past in other fields, a poor handmaiden to the main business of archaeology. Since 2009, though, whole-genome data have begun to challenge long-held views in archaeology, history, anthropology, and even linguistics—and to resolve controversies in those fields.</p>
<p>The ancient DNA revolution is rapidly disrupting our assumptions about the past. Yet there is at present no book by a working geneticist that lays out the impact of the new science and explains how it can be used to establish compelling new facts. The findings needed to grasp the scope of the ancient DNA revolution are scattered among hard-to-read, jargon-filled scientific papers, sometimes supplemented by hundreds of pages of dense notes on methodology. In <em>Who We Are and How We Got Here,</em> I aim to offer readers a clear view through this extraordinary window into the past—to provide a book about the ancient DNA revolution intended for lay reader and specialist alike. My goal is not to present a synthesis—the field is moving too quickly. By the time this book reaches readers, some advances that it describes will have been superseded or even contradicted. In the three years since I began writing, many fresh findings have emerged, so that most of what I describe here is based on results obtained after I started. I hope that readers will take the topics I discuss as examples of the disruptive power of whole-genome studies, not as a definitive summary of the state of the science.</p>
<p>My approach is to take readers through the process of discovery, with each chapter serving as an argument that has as its goal to bring readers, who may have come with one perspective when they started, to another place when they finish. I try to make a virtue of my laboratory’s central role in the ancient DNA revolution by telling the story of my own work where it is relevant—as this is a subject on which I can speak with great authority—while also discussing work in which I was not involved when it is critical to the story. Because I take this approach, the book disproportionately highlights the work from my laboratory. I apologize that I have been able to mention by name only a tiny fraction of the people who made equally important contributions. My priority has been to convey the excitement and surprise of the genome revolution, and to take readers on a compelling narrative path through it, not to write a scientific review.</p>
<p>I also highlight some of the great themes that are emerging, especially the finding that mixture between highly differentiated populations is a recurrent process in the human past. Today, many people assume that humans can be grouped biologically into “primeval” groups, corresponding to our notion of “races,” whose origins are populations that separated tens of thousands of years ago. But this long-held view about “race” has just in the last few years been proven wrong—and the critique of concepts of race that the new data provide is very different from the classic one that has been developed by anthropologists over the last hundred years. A great surprise that emerges from the genome revolution is that in the relatively recent past, human populations were just as different from each other as they are today, but that the fault lines across populations were almost unrecognizably different from today. DNA extracted from remains of people who lived, say, ten thousand years ago shows that the structure of human populations at that time was qualitatively different. Present-day populations are blends of past populations, which were blends themselves. The African American and Latino populations of the Americas are only the latest in a long line of major population mixtures.</p>
<p><em>Who We Are and How We Got Here</em> is divided into three parts. Part I, “The Deep History of Our Species,” describes how the human genome not only provides all the information that a fertilized human egg needs to develop, but also contains within it the history of our species. Chapter 1, “How the Genome Explains Who We Are,” argues that the genome revolution has taught us about who we are as humans not by revealing the distinctive features of our biology compared to other animals but by uncovering the history of migrations and population mixtures that formed us. Chapter 2, “Encounters with Neanderthals,” reveals how the breakthrough technology of ancient DNA provided data from Neanderthals, our big-brained cousins, and showed how they interbred with the ancestors of all modern humans living outside of Africa. The chapter also explains how genetic data can be used to prove that ancient mixture between populations occurred. Chapter 3, “Ancient DNA Opens the Floodgates,” highlights how ancient DNA can reveal features of the past that no one had anticipated, starting with the discovery of the Denisovans, a previously unknown archaic population that had not been predicted by archaeologists and that mixed with the ancestors of present-day New Guineans. The sequencing of the Denisovan genome unleashed a cavalcade of discoveries of additional archaic populations and mixtures, and demonstrated unequivocally that population mixture is central to human nature.</p>
<p>Part II, “How We Got to Where We Are Today,” is about how the genome revolution and ancient DNA have transformed our understanding of our own particular lineage of modern humans, and it takes readers on a tour around the world with population mixture as a unifying theme. Chapter 4, “Humanity’s Ghosts,” introduces the idea that we can reconstruct populations that no longer exist in unmixed form based on the bits of genetic material they have left behind in present-day people. Chapter 5, “The Making of Modern Europe,” explains how Europeans today descend from three highly divergent populations, which came together over the last nine thousand years in a way that archaeologists never anticipated before ancient DNA became available. Chapter 6, “The Collision That Formed India,” explains how the formation of South Asian populations parallels that of Europeans. In both cases, a mass migration of farmers from the Near East after nine thousand years ago mixed with previously established hunter-gatherers, and then a second mass migration from the Eurasian steppe after five thousand years ago brought a different kind of ancestry and probably Indo-European languages as well. Chapter 7, “In Search of Native American Ancestors,” shows how the analysis of modern and ancient DNA has demonstrated that Native American populations prior to the arrival of Europeans derive ancestry from multiple major pulses of migration from Asia. Chapter 8, “The Genomic Origins of East Asians,” describes how much of East Asian ancestry derives from major expansions of populations from the Chinese agricultural heartland. Chapter 9, “Rejoining Africa to the Human Story,” highlights how ancient DNA studies are beginning to peel back the veil on the deep history of the African continent drawn by the great expansions of farmers in the last few thousand years that overran or mixed with previously resident populations.</p>
<p>Part III, “The Disruptive Genome,” focuses on the implications of the genome revolution for society. It offers some suggestions for how to conceive of our personal place in the world, our connection to the more than seven billion people who live on earth with us, and the even larger numbers of people who inhabit our past and future. Chapter 10, “The Genomics of Inequality,” shows how ancient DNA studies have revealed the deep history of inequality in social power among populations, between the sexes, and among individuals within a population, based on how that inequality determined success or failure of reproduction. Chapter 11, “The Genomics of Race and Identity,” argues that the orthodoxy that has emerged over the last century—the idea that human populations are all too closely related to each other for there to be substantial average biological differences among them—is no longer sustainable, while also showing that racist pictures of the world that have long been offered as alternatives are even more in conflict with the lessons of the genetic data. The chapter suggests a new way of conceiving the differences among human populations—a way informed by the genome revolution. Chapter 12, “The Future of Ancient DNA,” is a discussion of what comes next in the genome revolution. It argues that the genome revolution, with the help of ancient DNA, has realized Luca Cavalli-Sforza’s dream, emerging as a tool for investigating past populations that is no less useful than the traditional tools of archaeology and historical linguistics. Ancient DNA and the genome revolution can now answer a previously unresolvable question about the deep past: the question of <em>what happened</em>—how ancient peoples related to each other and how migrations contributed to the changes evident in the archaeological record. Ancient DNA should be liberating to archaeologists because with answers to these questions in reach, archaeologists can get on with investigating what they have always been at least as interested in, which is <em>why</em> the changes occurred.</p>
<p>Before diving into the book, I will recount something that happened during a guest lecture I gave at the Massachusetts Institute of Technology in 2009. Mine was one of the last lectures of the term, meant to add spice to a course aimed at introducing students to computer-aided research into genomes with the goal of finding cures for disease. As I addressed Indian population history, an undergraduate sitting at the center of the front row stared me down. When I concluded, she asked me, with a grin, “How do you get funded to do this stuff?”</p>
<p>I mumbled something about how the human past shapes genetic variation, and about how, in order to identify risk factors for disease, it is important to understand that past. I gave the example of how among the thousands of distinctive human populations of India, there are high rates of disease because mutations that happened to be carried by the founders increased in frequency as the groups expanded. I make arguments along these lines in my applications to the U.S. National Institutes of Health, in which I propose to find disease risk factors that occur at different frequencies across populations. Grants of this type have funded much of my work since I started my laboratory in 2003.</p>
<p>True as these arguments are, I wish I had responded differently. We scientists are conditioned by the system of research funding to justify what we do in terms of practical application to health or technology. But shouldn’t intrinsic curiosity be valued for itself? Shouldn’t fundamental inquiry into who we are be the pinnacle of what we as a species hope to achieve? Isn’t an attribute of an enlightened society that it values intellectual activity that may not have immediate economic or other practical impact? The study of the human past—as of art, music, literature, or cosmology—is vital because it makes us aware of aspects of our common condition that are profoundly important and that we heretofore never imagined.</p>

  </div>
</article>


        

        



    <aside class="card border noPrint" id="section-tree-item-sapiens_branches_articles_reich_who-we-are_0_Acknowledgments_md">
        <div class="card-title bg-light-gray border d-flex justify-content-between">
            <a href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/0/Acknowledgments/">Acknowledgments </a>
            <a data-toggle="collapse" href="#section-tree-item-body-sapiens_branches_articles_reich_who-we-are_0_Acknowledgments_md" role="button" aria-expanded="false" aria-controls="section-tree-item-body-sapiens_branches_articles_reich_who-we-are_0_Acknowledgments_md" >…<i class="fas fa-caret-down" class="collapsed"></i> </a>
        </div>
        <nav id="section-tree-item-body-sapiens_branches_articles_reich_who-we-are_0_Acknowledgments_md" class="card-body p-0 collapse">
            
            <li>draft: false</li>
            
            <li>iscjklanguage: false</li>
            
            <li>title: Acknowledgments</li>
            
        </nav>
    </aside>


      </main>
    </div>
    <footer class="bg-yellow  p-1 noPrint" role="contentinfo">
  

  <div id="div_foot_nav_bar" class="row">
    <a href="https://vishvAsa.github.io/notes/search"  class="btn btn-light btn-sm">🥽🔦</a>
    <div class="row col  d-flex justify-content-center">
      <div><a id="previousPageFoot" href="" class="btn btn-secondary">###<i class="fas fa-caret-left"></i></a></div>
      <div ><a id="nextPageFoot" href="" class="btn btn-secondary"><i class="fas fa-caret-right"></i> ### </a></div>
    </div>
      <ul id="footer-bar-right-custom" class="list-group list-group-horizontal">
      </ul>
  </div>
  <div id="div_footer_bar" class="container-fluid d-flex justify-content-between">
    <a class="btn btn-secondary" href="https://github.com/vishvAsa/notes/issues/new" >
      ✉️Contact
    </a>
    <a class="btn btn-secondary" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/80x15.png" /></a>
    <div class="btn small border">
      
      <a href="https://github.com/vishvAsa/notes/actions/workflows/build.yml"><img src="https://github.com/vishvAsa/notes/actions/workflows/build.yml/badge.svg"> Built </a> on 2025 Oct 31 15:25:51 UTC. (<a href="http://google.com/search?q=15%3a25%3a51%20UTC to IST">IST</a>)
    </div>
  </div>
</footer>



<script async src="https://www.googletagmanager.com/gtag/js?id=G-33H08JSLQR"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-33H08JSLQR');
</script>

  </body>
  <script type='text/javascript'>
    
    
    
    module_uiLib.onDocumentReadyTasks();
  </script>
  
</html>
