---
title: Genetics
---

## Proper pramANa-s

[![](https://i.imgur.com/B8QB5zS.png)](https://i.imgur.com/B8QB5zS.png)

The above is from \[[Nature16](http://www.nature.com/nature/journal/v541/n7637/full/nature21347.html)\].  

- "First, a paper from 2006 is wildly out of date for this field. The methods used have been superseded. Instead of using one locus, the Y chromosome, and focusing on microsatellites (which have upsides and downsides), researchers now look at the whole genome." [RK15](http://www.unz.com/gnxp/the-aryan-invasion-was-not-fantasy/#comments)
- "Dr. Underhill says there is no comparison between the kind of data available in 2010 and now. “Then, it was like looking into a darkened room from the outside through a keyhole with a little torch in hand; you could see some corners but not all, and not the whole picture. With whole genome sequencing, we can now see nearly the entire room, in clearer light.”" [H17](http://www.thehindu.com/sci-tech/science/how-genetics-is-settling-the-aryan-migration-debate/article19090301.ece).

## Gene-distribution in jAti-s

- ANI ancestry among various castes: [RK15](http://www.unz.com/gnxp/the-aryan-invasion-was-not-fantasy/#comments).
- Caste-gene heatmap \[[MT5](https://manasataramgini.wordpress.com/2005/12/20/socio-genetic-stratification-of-india/)\].
    - [![](http://i.imgur.com/nC8AgJh.jpg)](http://i.imgur.com/nC8AgJh.jpg)
- -"Not shown in this diagram is the O3e marker that is predominantly seen in the Tibeto-Burman speaking eastern tribals (who also have some O2a), and nearly absent elsewhere."\[[MT5](https://manasataramgini.wordpress.com/2005/12/20/socio-genetic-stratification-of-india/)\]
- -"The O2a, H1, F* and H2 haplogroups are predominantly seen in the mainland tribal population and decline in the middle-lower castes and are low or almost absent in upper castes." \[[MT5](https://manasataramgini.wordpress.com/2005/12/20/socio-genetic-stratification-of-india/)\]
    - O2a
        - " The O2a marker is seen in all tribal groups but hardly in any caste groups, suggesting that it was indeed one of the markers of the earliest population. In contrast to O2a, H1 while highest in the tribal populations is also seen in caste population in a declining gradient. It was in this early population that probably Austro-Asiatic, Nahali, Kusunda and possibly Indo-Pacific languages arose. Of these the Austro-Asiatic and Dravidian languages showed an expansion. .. Both the frequency and variance plots of the O2a haplogroup show an increasing Eastern component. This would imply that its expansion was probably associated with an Eastern population expansion and emigration. The expansion probably corresponded with the Austro-Asiatic expansion."\[[MT5](https://manasataramgini.wordpress.com/2005/12/20/socio-genetic-stratification-of-india/)\]
    - H1
        - "H1 in contrast is hardly present in Sino-Tibetan and Eastern populations. Its centre gravity in terms of frequency is the central part of the country from where it appears to have expanded outwards. We explain H1 expansion as an expansion of agricultural indigenous population associated with Central and South Indian archaeological cultures like Jorwe and Daimabad and these agricultural groups managed ascend the ladder of the Indo-Aryan hierarchy coming in from the North."\[[MT5](https://manasataramgini.wordpress.com/2005/12/20/socio-genetic-stratification-of-india/)\]
        - "Around 6300 YBP a man in India belonging to the H1-M52 lineage left a huge impact on the genetic landscape of India. He hailed from a patriline, which was already prominent and had expanded, albeit more modestly, around 7300 YBP. .. His imprint is the dominant one in those jāti-s occupying the middle and lower rungs of the Indian jāti-stratification. H1-M52 is prominently present in the currently or formerly socially powerful middle jāti-s like the Marāṭhā, Vellālar, the Vanniyar, the Kammā and the Reḍḍī. His imprint is less prominent in the saṃskṛta brāhmaṇa and kṣatriya jāti-s but not absent. This distribution of H1-M52 together with the timing of the expansions clearly suggest that the father of this lineage was a very prominent figure in pre-Indo-Aryan India. The prestige of his lineage was so high that even after the conquest of India by the ārya-s his descendants were able to enter and establish themselves to degree within the highest rungs of the ārya system as brāhmaṇa-s and kṣatriya-s. Even those who did not were still socially prominent, often attaining de facto kṣatriya status and spreading their genes extensively into the social strata who lay below them." \[[M16](https://manasataramgini.wordpress.com/2016/05/08/the-giants-among-the-lilliputs/)\]
- Influx or radiation of R1a1, R2, J2a G2 (G2-P15).
    - See R1 clade: Background information below.
    - -"The R1a1, R2, J2a G2 (G2-P15) are clearly high in upper caste, and decline in middle-lower castes and almost vanish in the lower castes." \[[MT5](https://manasataramgini.wordpress.com/2005/12/20/socio-genetic-stratification-of-india/)\]
        - 30-70% of brAhmaNa jAti-s have R1a1 Y chromosome.
    - "The R1a1 marker is elsewhere seen in the Central Asians and Eastern Europeans; J2a and G2-P15 appear to peak in the Middle East, Anatolia and the Caucusus with some presence in Europe. G2-P15 is interestingly found only in Iyer and Iyengars in this set at a high 10-12 %. Another study interestingly showed a similar rare haplotype N1a to be present in similar percentages in the Havik Brahmins of Karnataka. This marker is found elsewhere in the regions bordering the Eurasian steppe zone and to some extant in Iran. It was also found in the early European settlers belonging to the Linearbandkeramik (Haak et al in Science Magazine) who are supposed to have spread agriculture to Central and Northern Europe."\[[MT5](https://manasataramgini.wordpress.com/2005/12/20/socio-genetic-stratification-of-india/)\]
        - Anecdote: "bhrgu angirasa show high steppe component"
    - "The fascinating Lambadi tribe of Southern India is a nomadic tribe moving around with horses and cattle. I had witnessed a few bands of these nomads as a youth and was struck by their oral tradition. 40% of the Lambadi males bear the M74 mutation that marks them of Northern Asian affinities." \[[M0512](https://manasataramgini.wordpress.com/2005/12/25/chenchus-lambadis-and-the-l1-haplogroup/)\]
    - "The Chenchus are a most remarkable tribe in Andhra. They are completely adapted to the forest and live entirely as hunter-gatherers. They have extraordinary stamina and eat only 1 time a day. They have special position amongst the tribes of India: the caste Hindus consider them completely pure for accepting food. They are also considered eaters and makers of bhojya and bhakShya food, so a brahmin can eat from a Chenchu. Chenchus have special privileges to perform rites in the Shrisailam, Ahobilam and Bhramaramba temples and enter the garbha gR^iha normally reserved for brahmins only. They have their own priests who invoke the standard Hindu deities of whom they have good mythological knowledge with elaborate unique rituals. Interestingly, they are the only tribal population with a particular ratio of H1, R1a1 and L1 haplogroups resembling several middle caste populations of India. It is possible that they represent an ancient varNa population that degenerated to the tribal condition. Alternatively their special symbiotic association with the caste populations may have resulted in gene flow into them."\[[M0512](https://manasataramgini.wordpress.com/2005/12/25/chenchus-lambadis-and-the-l1-haplogroup/)\]
    - Also see R1 clade background information below.
    - Possible absorption and expansion of R2 - "kulin type bengali v1 group chatterjees are kashyapa of r2 haplogroups which originates in india.".
- L1
    - "L1 marker is predominant in Tamils and to some extant in Marathas, with the most prevalent presence in Tamil middle castes like Vellalars, Vanniyars, Ambalavasis and the like. It is low or absent in most of the Dravidian tribal groups outside Tamil Nad, though it is present in Brahmins in the Tamil region. It is also peculiarly found in Balochistan and Makran but low in the Brahui speakers. " \[[MT5](https://manasataramgini.wordpress.com/2005/12/20/socio-genetic-stratification-of-india/)\]
        - "Kallars, a middle level warrior Tamil caste, has up to 50% L1 markers. L1 haplogroup is also found in the Yadava caste, a group of cowherds from from the Andhra country (the clan of the famous cowherd hero kATaM rAju) show about 19%, Kammas of Andhra 17%, so also the Sinhalese show about 17% of it. " \[[M0512](https://manasataramgini.wordpress.com/2005/12/25/chenchus-lambadis-and-the-l1-haplogroup/)\]
    - "L1 which is restricted to Tamil population but seen across caste boundaries albeit with the greatest presence in the middle-low castes is interpreted as not being spread by the Dravidians. As most Dravidian tribals have it lower levels and it is absent in Dravidians outside Tamilnad we specifically see at the haplogroup expanded by the proto-Tamil cultural rennaisance, probably archaeologically corresponding to the expansion of the Megalithic (neDukkal/vira-kal) culture. This appears to have expanded from the middle of the peninsular India somewhat Northwards to coastal Pakistan and Makran and of course its main center was the Tamil country. This expansion possibly also bore with it the H1 marker, further driving it up the caste hierarchy." \[[MT5](https://manasataramgini.wordpress.com/2005/12/20/socio-genetic-stratification-of-india/)\]
    - "We suspect that the Tamil upper castes absorbed native priests as kurukals and the like that resulted in the influx of L1 into their ranks."\[[MT5](https://manasataramgini.wordpress.com/2005/12/20/socio-genetic-stratification-of-india/)\]
    - "The later point is illustrated by the fact that in Brahmins of Tamil Nad, Andhra and Maharashtra (the Konkanastha Brahmins) the L1 haplogroup is seen in 10-13% frequency. Most interestingly, the Saurashtrans who are an immigrant group into Tamil Nad from the Gujarat-Maharastra zone show upto 25% frequency of the L1 marker. This is in fact in contrast to the overall level of L1 in Gujarat being around 10% (Albeit a relatively small sample of 29 males), and in general in Maharashtran upper and middle castes around 10-11%. Thus, despite retaining their unique language of the Indo-Aryan family, the Saurashtrans have acquired a considerable component from the Tamil gene pool (lower bound of 15%). This suggests that even the rather insulated Saurashtrian jAti was genetically porous to a certain degree. Probably the Saurashtran jAti’s tight connection with certain professional guilds strongly preserved its language while letting people in." \[[M0512](https://manasataramgini.wordpress.com/2005/12/25/chenchus-lambadis-and-the-l1-haplogroup/)\]

## R1 clade: Background information

- Haplogroup R
    - 23andme: "Haplogroup R is a widespread and diverse branch of the Y-chromosometree that is extremely common in Europe, where it spread after the end of the Ice Age about 12,000 years ago. The haplogroup appears to have originated in southwestern Asia about 30,000 years ago. It then split into two main branches. R1 ultimately spread widely across Eurasia, from Iceland to Japan, whereas R2 mostly remained near its region of origin. Today it can be found in southwestern Asia and India."
- Haplogroup R1
    - 23andme: "R1 is the dominant haplogroup in Europe today, accounting for well over half of all men. After being confined to the continent's southern fringes during the Ice Age, it expanded rapidly in the wake of the receding glaciers about 12,000 years ago. "
- Haplogroup R1a and R1b
    - "R1a and perhaps R2 expanded in the Central/Western Asian zone close to the boundary of the Eurasian steppes somewhere between 10-15,000 years BP. These two seem to have then seeped into India at a gradual rate seeding the indigenous tribal groups with R1a1 males at some low rate. The main bulk of the less diverse R1a1 appears to have come from Western/Central Eurasia along with the other above-mentioned Western Eurasian markers where it had already differentiated into eastern and western branches. The time of divergence with the central Asian R1a1 comes to around 6000-10000yrs bp."\[[MT5](https://manasataramgini.wordpress.com/2005/12/20/socio-genetic-stratification-of-india/)\]
    - "Around approximately 5-6000 YBP, there were two males who each left behind over 100 million descendants. One bore a R1a Y-haplogroup while the other bore a R1b-L11 haplogroup. Both these men came from a related chain of cultures and likely spoke Indo-European languages. Among the Indo-Europeans there is the persistent belief of the great male common ancestor. We ārya-s call him father Manu Vaivasvata. Among our cousins, the Iranians, a similar figure appear as Yima Vivaṅhat, who like our Manu is ancient ancestor and law-giver."  \[[M16](https://manasataramgini.wordpress.com/2016/05/08/the-giants-among-the-lilliputs/)\] "In Western Europe, related lineages within R1b-L11 expanded ~4.8–5.9 kya (Supplementary Fig. 14e), most markedly around 4.8 and 5.5 kya. The earlier of these times, 5.5 kya, is associated with the origin of the Bronze Age Yamnaya culture." \[[N16](http://www.nature.com.ucsf.idm.oclc.org/ng/journal/vaop/ncurrent/full/ng.3559.html) [T](http://www.telegraph.co.uk/science/2016/04/25/half-of-british-men-descended-from-one-bronze-age-king/)16 \]
- Haplogroup R1a1 (R-M17 or R-M198)
    - 23andme: "Haplogroup R1a1 appears to have arisen in the Near East or present-day Pakistan during the peak of the Ice Age about 18,000 years ago. Until the Ice Age began to wane about 15,000 years ago, it may have been limited to the area around the Black Sea, a region that remained relatively ice-free and hospitable while much of Eurasia was covered by glaciers and tundra."
- Haplogroup R1a1a
    - "The fact is that the Y chromosomal phylogeny of R1a1a the world over exhibits the hallmarks of massive population expansion ~5,000 years ago. No one knows where the R1 lineages are truly from in a deep time sense, as it seems likely they were low frequency variants before their explosion."[RK15](http://www.unz.com/gnxp/the-aryan-invasion-was-not-fantasy/#comments)
    - Eastern Europe (probable locus of origin)
        - "After the Ice Age, northward migrations carried R1a1a into the gradually greening territory surrendered by the receding glaciers. The abundance of R1a1a in Ukraine, where it reaches levels of about 50%, and the haplogroup's high genetic diversity among Ukrainians, have led researchers to suggest that it expanded from the relatively ice-free region around the Black Sea about 12,000 years ago." (23andme) 
        - "The first is that 99% of modern R1a descends from R1a1a (M417), a subclade that clearly expanded from the Bronze Age onwards, not from the early Neolithic. Its phylogeny also points at an Eastern European origin." ([Eupedia](http://www.eupedia.com/europe/Haplogroup_R1a_Y-DNA.shtml))
    - R-Z93 (R1a1a1b2)
        - "The most striking were expansions within R1a-Z93, occurring ~4.0–4.5 kya. This time predates by a few centuries the collapse of the Indus Valley Civilization, associated by some with the historical migration of Indo-European speakers from the Western Steppe into the Indian subcontinent. " \[[N16](http://www.nature.com.ucsf.idm.oclc.org/ng/journal/vaop/ncurrent/full/ng.3559.html) \]
    - Near East and Asia
        
        - Haplogroup R1a1a expanded less dramatically after the Ice Age in the Near East, parts of central Asia and northern India. It is found at levels of about 7% in Iraq and Syria, and at similar levels in Turkey. The higher genetic diversity of R1a1a in eastern Turkey and the decrease in frequency from east to west, indicates that R1a1a entered Turkey from the Iranian plateau, probably about 4,000 years ago.  (23andme)
            
        - R1a1a is quite common among some central Asian populations, reaching levels of up to 60% among the Kyrgyz and the Tajiks. It is possible that the haplogroup expanded through central Asia and into northwestern India shortly before the arrival of agriculture, which arrived in the region shortly after 10,000 years ago.  (23andme)

## African background

- "The great father in Africa belonged to the the E1b-M180 clade. His descendants in thre1e great spurts from around 5300-4800 YBP expanded among the speakers of Niger-Congo languages and spread across the vast African continent. .. It is likely that the E1b-M180 lineage arose there and then penetrated deep into sub-Saharan Africa as the Niger-Congo languages diversified. Its initial expansions could have been triggered by the events of the emergence of civilization in the Egyptian cultural zone, with its bearers subsequently acquiring high status in the sub-Saharan milieu east and west. " \[[M16](https://manasataramgini.wordpress.com/2016/05/08/the-giants-among-the-lilliputs/)\]

