<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
    
    <title>Vishvas&#39;s notes  | 1 How the Genome Explains Who We Are</title>
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">

    <meta name="viewport" content="width=device-width,minimum-scale=1">
    <meta name="generator" content="Hugo 0.143.0">
    
    
      <META NAME="ROBOTS" CONTENT="INDEX, FOLLOW">
    

    <link href="/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/" rel="alternate" type="application/rss+xml" title="Vishvas&#39;s notes" />
    <link href="/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/" rel="feed" type="application/rss+xml" title="Vishvas&#39;s notes" />

    <meta property="og:url" content="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/">
  <meta property="og:site_name" content="Vishvas&#39;s notes">
  <meta property="og:title" content="1 How the Genome Explains Who We Are">
  <meta property="og:description" content="The Master Chronicle of Human Variation To understand why genetics is able to shed light on the human past, it is necessary to understand how the genome—defined as the full set of genetic code each of us inherits from our parents—records information. Francis Crick, Rosalind Franklin, James Watson, and Maurice Wilkins showed in 1953 that the genome is written out in twin chains of about three billion chemical building blocks (six billion in all) that can be thought of as the letters of an alphabet: A (adenine), C (cytosine), G (guanine), and T (thymine).1 What we call a “gene” consists of tiny fragments of these chains, typically around one thousand letters long, which are used as templates to assemble the proteins that do most of the work in cells. In between the genes is noncoding DNA, sometimes referred to as “junk” DNA. The order of the letters can be read by machines that perform chemical reactions on fragments of DNA, releasing flashes of light as the reactions pass along the length of the DNA sequence.&#43;&#43;&#43;(5)&#43;&#43;&#43; The reactions emit a different color for each of the letters A, C, G, and T, so that the sequence of letters can be scanned into a computer by a camera.">
  <meta property="og:locale" content="en_us">
  <meta property="og:type" content="article">
    <meta property="article:section" content="sapiens">

  <meta itemprop="name" content="1 How the Genome Explains Who We Are">
  <meta itemprop="description" content="The Master Chronicle of Human Variation To understand why genetics is able to shed light on the human past, it is necessary to understand how the genome—defined as the full set of genetic code each of us inherits from our parents—records information. Francis Crick, Rosalind Franklin, James Watson, and Maurice Wilkins showed in 1953 that the genome is written out in twin chains of about three billion chemical building blocks (six billion in all) that can be thought of as the letters of an alphabet: A (adenine), C (cytosine), G (guanine), and T (thymine).1 What we call a “gene” consists of tiny fragments of these chains, typically around one thousand letters long, which are used as templates to assemble the proteins that do most of the work in cells. In between the genes is noncoding DNA, sometimes referred to as “junk” DNA. The order of the letters can be read by machines that perform chemical reactions on fragments of DNA, releasing flashes of light as the reactions pass along the length of the DNA sequence.&#43;&#43;&#43;(5)&#43;&#43;&#43; The reactions emit a different color for each of the letters A, C, G, and T, so that the sequence of letters can be scanned into a computer by a camera.">
  <meta itemprop="wordCount" content="7254">
  <meta name="twitter:card" content="summary">
  <meta name="twitter:title" content="1 How the Genome Explains Who We Are">
  <meta name="twitter:description" content="The Master Chronicle of Human Variation To understand why genetics is able to shed light on the human past, it is necessary to understand how the genome—defined as the full set of genetic code each of us inherits from our parents—records information. Francis Crick, Rosalind Franklin, James Watson, and Maurice Wilkins showed in 1953 that the genome is written out in twin chains of about three billion chemical building blocks (six billion in all) that can be thought of as the letters of an alphabet: A (adenine), C (cytosine), G (guanine), and T (thymine).1 What we call a “gene” consists of tiny fragments of these chains, typically around one thousand letters long, which are used as templates to assemble the proteins that do most of the work in cells. In between the genes is noncoding DNA, sometimes referred to as “junk” DNA. The order of the letters can be read by machines that perform chemical reactions on fragments of DNA, releasing flashes of light as the reactions pass along the length of the DNA sequence.&#43;&#43;&#43;(5)&#43;&#43;&#43; The reactions emit a different color for each of the letters A, C, G, and T, so that the sequence of letters can be scanned into a computer by a camera.">
<script type="text/javascript">
    
    let baseURL = "https:\/\/vishvAsa.github.io\/notes\/";
    let basePath = "/" + baseURL.split("/").slice(3).join("/");
    let siteParams = JSON.parse("{\u0022build_img\u0022:\u0022https:\/\/github.com\/vishvAsa\/notes\/actions\/workflows\/build.yml\/badge.svg\u0022,\u0022build_url\u0022:\u0022https:\/\/github.com\/vishvAsa\/notes\/actions\/workflows\/build.yml\u0022,\u0022contactlink\u0022:\u0022https:\/\/github.com\/vishvAsa\/notes\/issues\/new\u0022,\u0022env\u0022:\u0022production\u0022,\u0022githubeditmepathbase\u0022:\u0022https:\/\/github.com\/vishvAsa\/notes\/edit\/content\/\u0022,\u0022js_extras\u0022:[\u0022mathjax\u0022]}");
    let pageDefaultsList = JSON.parse("[{\u0022scope\u0022:{\u0022pathPrefix\u0022:\u0022\u0022},\u0022values\u0022:{\u0022comments\u0022:true,\u0022layout\u0022:\u0022page\u0022,\u0022search\u0022:true,\u0022sidebar\u0022:\u0022home_sidebar\u0022}}]");
    let sidebarsData = JSON.parse("{\u0022home_sidebar\u0022:{\u0022contents\u0022:[{\u0022url\u0022:\u0022recdir:\/\/\u0022},{\u0022contents\u0022:[{\u0022title\u0022:\u0022ज्यौतिषम्\u0022,\u0022url\u0022:\u0022..\/jyotiSham\/\u0022},{\u0022title\u0022:\u0022संस्कृतम्\u0022,\u0022url\u0022:\u0022..\/sanskrit\/\u0022},{\u0022title\u0022:\u0022देवः\u0022,\u0022url\u0022:\u0022..\/devaH\/\u0022},{\u0022title\u0022:\u0022आगमः\u0022,\u0022url\u0022:\u0022..\/AgamaH\/\u0022},{\u0022title\u0022:\u0022आगमः शैवः\u0022,\u0022url\u0022:\u0022..\/AgamaH_shaivaH\/\u0022},{\u0022title\u0022:\u0022आगमः वैष्णवः\u0022,\u0022url\u0022:\u0022..\/AgamaH_vaiShNavaH\/\u0022},{\u0022title\u0022:\u0022रामानुजीयम्\u0022,\u0022url\u0022:\u0022..\/rAmAnujIyam\/\u0022},{\u0022title\u0022:\u0022माध्वम्\u0022,\u0022url\u0022:\u0022..\/mAdhvam\/\u0022},{\u0022title\u0022:\u0022आगमः ब्राह्मः\u0022,\u0022url\u0022:\u0022..\/AgamaH_brAhmaH\/\u0022},{\u0022title\u0022:\u0022कल्पान्तरम्\u0022,\u0022url\u0022:\u0022..\/kalpAntaram\/\u0022},{\u0022title\u0022:\u0022Notes Home\u0022,\u0022url\u0022:\u0022..\/notes\/\u0022},{\u0022title\u0022:\u0022वेदाः\u0022,\u0022url\u0022:\u0022..\/vedAH\/\u0022},{\u0022title\u0022:\u0022वेदाः - ऋक्\u0022,\u0022url\u0022:\u0022..\/vedAH_Rk\/\u0022},{\u0022title\u0022:\u0022वेदाः - यजुः\u0022,\u0022url\u0022:\u0022..\/vedAH_yajuH\/\u0022},{\u0022title\u0022:\u0022वेदाः - साम\u0022,\u0022url\u0022:\u0022..\/vedAH_sAma\/\u0022},{\u0022title\u0022:\u0022पुराणम्\u0022,\u0022url\u0022:\u0022..\/purANam\u0022},{\u0022title\u0022:\u0022पुराणम् वैष्णवम्\u0022,\u0022url\u0022:\u0022..\/purANam_vaiShNavam\/\u0022},{\u0022title\u0022:\u0022महाभारतम्\u0022,\u0022url\u0022:\u0022..\/mahAbhAratam\/\u0022},{\u0022title\u0022:\u0022रामायणम्\u0022,\u0022url\u0022:\u0022..\/rAmAyaNam\/\u0022},{\u0022title\u0022:\u0022काव्यम्\u0022,\u0022url\u0022:\u0022..\/kAvyam\/\u0022},{\u0022title\u0022:\u0022मीमांसा\u0022,\u0022url\u0022:\u0022..\/mImAMsA\/\u0022},{\u0022title\u0022:\u0022ಕನ್ನಡ\u0022,\u0022url\u0022:\u0022..\/kannaDa\/\u0022},{\u0022title\u0022:\u0022त्रिपिटकम्\u0022,\u0022url\u0022:\u0022..\/tipiTaka\/\u0022},{\u0022title\u0022:\u0022पाळयः\u0022,\u0022url\u0022:\u0022..\/pALi\/\u0022},{\u0022title\u0022:\u0022भाषान्तरम्\u0022,\u0022url\u0022:\u0022..\/bhAShAntaram\/\u0022},{\u0022title\u0022:\u0022English Lit\u0022,\u0022url\u0022:\u0022..\/english\/\u0022},{\u0022title\u0022:\u0022Book pub\u0022,\u0022url\u0022:\u0022..\/book-pub\u0022},{\u0022title\u0022:\u0022Vishvas\u0027s home page\u0022,\u0022url\u0022:\u0022..\/\u0022}],\u0022title\u0022:\u0022सङ्ग्रहान्तरम्\u0022},{\u0022title\u0022:\u0022\\u003ci class=\\\u0022fas fa-hand-holding-heart\\\u0022\\u003e\\u003c\/i\\u003eDonate Via Vishvas\u0022,\u0022url\u0022:\u0022https:\/\/vishvAsa.github.io\/interests\/dharma-via-vishvas\/\u0022}],\u0022title\u0022:\u0022Parts\u0022}}");
    
    var pageRelUrlTree = {};
</script>

    
    <script type="text/javascript">
  let pageVars = {};
  pageVars.pageUrlMinusBasePath = "\/notes\/sapiens\/branches\/articles\/reich_who-we-are\/1_The_Deep_History_of_Our_Species\/1_How_the_Genome_Explains_Who_We_Are\/".replace(basePath, "/");
  pageVars.pageParams = {};
  pageVars.pageSource = "sapiens\/branches\/articles\/reich_who-we-are\/1_The_Deep_History_of_Our_Species\/1_How_the_Genome_Explains_Who_We_Are.md";
  console.log(pageVars.pageSource);
  var pageDefaults;
  for (let possiblePageDefaults of pageDefaultsList) {
    if (pageVars.pageSource.startsWith(possiblePageDefaults.scope.pathPrefix)) {
      pageDefaults = possiblePageDefaults.values
    }
  }
    
</script>


    <script src="/notes/webpack_dist/uiLib-bundle.js"></script>


<script>


MathJax = {
  tex: {
    inlineMath: [ ["@#", "#@"], ["\\(", "\\)"] ],
    displayMath: [ ["$$","$$"], ["\\[", "\\]"] ],
    processEscapes: true,  
  },
  svg: {
    fontCache: 'global'
  },
  options: {
    ignoreHtmlClass: "tex2jax_ignore|dno"
  }
};
</script>
<script type="text/javascript" id="MathJax-script" async
        src="/notes/non_webpack_js/mathjax/tex-svg.js">
</script>






    <link rel="stylesheet" href="/notes/webpack_dist/uiLib.css">
<link rel="stylesheet" href="/notes/css/uiLibPrint.css" media="print" >
<link rel="stylesheet" href="/notes/css/fonts.css" >
<link rel="stylesheet" href="/notes/css/@fortawesome/fontawesome-free/css/solid.min.css">

<link rel="stylesheet" href="/notes/css/@fortawesome/fontawesome-free/css/fontawesome.min.css">


<link rel="alternate" hreflang="sa-Deva" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/" />
<link rel="alternate" hreflang="sa-Deva" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=devanagari" />

<link rel="alternate" hreflang="sa-Knda" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=kannada" />
<link rel="alternate" hreflang="sa-Mlym" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=malayalam" />
<link rel="alternate" hreflang="sa-Telu" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=telugu" />
<link rel="alternate" hreflang="sa-Taml-t-sa-Taml-m0-superscript" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=tamil_superscripted" />
<link rel="alternate" hreflang="sa-Taml" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=tamil" />
<link rel="alternate" hreflang="sa-Gran" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=grantha" />

<link rel="alternate" hreflang="sa-Gujr" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=gujarati" />
<link rel="alternate" hreflang="sa-Guru" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=gurmukhi" />
<link rel="alternate" hreflang="sa-Shar" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=sharada" />
<link rel="alternate" hreflang="sa-Dogr" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=dogra" />
<link rel="alternate" hreflang="sa-Phag" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=phags_pa" />
<link rel="alternate" hreflang="sa-Tibt" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=tibetan" />
<link rel="alternate" hreflang="sa-Sidd" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=siddham" />
<link rel="alternate" hreflang="sa-Newa" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=newa" />
<link rel="alternate" hreflang="sa-Modi" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=modi" />
<link rel="alternate" hreflang="sa-Tirh" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=tirhuta_maithili" />
<link rel="alternate" hreflang="sa-Gong" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=gondi_gujala" />
<link rel="alternate" hreflang="sa-Gonm" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=gondi_masaram" />


<link rel="alternate" hreflang="sa-Orya" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=oriya" />
<link rel="alternate" hreflang="sa-Beng-t-sa-Beng-m0-assamese" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=assamese" />
<link rel="alternate" hreflang="sa-Beng" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=bengali" />
<link rel="alternate" hreflang="sa-Mtei" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=manipuri" />
<link rel="alternate" hreflang="sa-Sinh" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=sinhala" />
<link rel="alternate" hreflang="sa-Limb" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=limbu" />
<link rel="alternate" hreflang="sa-Lepc" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=lepcha" />


<link rel="alternate" hreflang="sa-Cyrl" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=cyrillic" />
<link rel="alternate" hreflang="sa-Latn-t-sa-Zyyy-m0-iso" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=iso" />
<link rel="alternate" hreflang="sa-Latn-t-sa-Zyyy-m0-iast" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=iast" />
<link rel="alternate" hreflang="sa-Aran" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=urdu" />
<link rel="alternate" hreflang="sa-Avst" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=avestan" />

<link rel="alternate" hreflang="sa-Brah" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=brahmi" />
<link rel="alternate" hreflang="sa-Khar" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=kharoshthi" />
<link rel="alternate" hreflang="sa-Zanb" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=zanbazar_square" />
<link rel="alternate" hreflang="sa-Bhks" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=bhaiksuki" />

<link rel="alternate" hreflang="sa-Bali" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=balinese" />
<link rel="alternate" hreflang="sa-Java" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=javanese" />
<link rel="alternate" hreflang="sa-Laoo" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=lao" />
<link rel="alternate" hreflang="sa-Laoo-t-sa-Laoo-m0-Laopali" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=lao_pali" />
<link rel="alternate" hreflang="sa-Mymr" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=burmese" />
<link rel="alternate" hreflang="sa-Mymr-t-sa-Mymr-m0-Mon" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=mon" />
<link rel="alternate" hreflang="sa-Cham" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=cham" />
<link rel="alternate" hreflang="sa-Khmr" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=khmer" />
<link rel="alternate" hreflang="sa-Thai" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/?transliteration_target=thai" />

  </head>

  <body class="ma0 bg-near-white">
    
\(
% groupings of objects.
\newcommand{\set}[1]{\left\{ #1 \right\}}
\newcommand{\seq}[1]{\left(#1\right)}
\newcommand{\ang}[1]{\langle#1\rangle}
\newcommand{\tuple}[1]{\left(#1\right)}
\newcommand{\size}[1]{\left| #1\right|}

\newcommand{\comp}{\circ}


% numerical shortcuts.
\newcommand{\abs}[1]{\left| #1\right|}
\newcommand{\floor}[1]{\left\lfloor #1 \right\rfloor}
\newcommand{\ceil}[1]{\left\lceil #1 \right\rceil}

% linear algebra shortcuts.
\newcommand{\change}{\Delta}
\newcommand{\norm}[1]{\left\| #1\right\|}
\newcommand{\dprod}[1]{\langle#1\rangle}
\newcommand{\linspan}[1]{\langle#1\rangle}
\newcommand{\conj}[1]{\overline{#1}}
\newcommand{\der}[1]{\frac{d#1}{dx}}
\newcommand{\lap}{\Delta}
\newcommand{\kron}{\otimes}
\newcommand{\nperp}{\nvdash}

\newcommand{\mat}[1]{\left[ \begin{smallmatrix}#1 \end{smallmatrix} \right]}

% derivatives and limits
\newcommand{\partder}[2]{\frac{\partial #1}{\partial #2}}
\newcommand{\partdern}[3]{\frac{\partial^{#3 #1}}{\partial #2^{#3}}}
\newcommand{\gradient}{\nabla}
\newcommand{\subdifferential}{\partial}

% Arrows
\newcommand{\diverge}{\nearrow}
\newcommand{\notto}{\nrightarrow}
\newcommand{\up}{\uparrow}
\newcommand{\down}{\downarrow}
% gets and gives are defined!

% ordering operators
\newcommand{\oleq}{\preceq}
\newcommand{\ogeq}{\succeq}

% programming and logic operators
\newcommand{\dfn}{:=}
\newcommand{\assign}{:=}
\newcommand{\co}{\ co\ }
\newcommand{\en}{\ en\ }


% logic operators
\newcommand{\xor}{\oplus}
\newcommand{\Land}{\bigwedge}
\newcommand{\Lor}{\bigvee}
\newcommand{\finish}{\Box}
\newcommand{\contra}{\Rightarrow \Leftarrow}
\newcommand{\iseq}{\stackrel{_?{=}}}


% Set theory
\newcommand{\symdiff}{\Delta}
\newcommand{\setdiff}{\backslash}
\newcommand{\union}{\cup}
\newcommand{\inters}{\cap}
\newcommand{\Union}{\bigcup}
\newcommand{\Inters}{\bigcap}
\newcommand{\nullSet}{\phi}


% graph theory
\newcommand{\nbd}{\Gamma}

% Script alphabets
% For reals, use \Re

% greek letters
\newcommand{\eps}{\epsilon}
\newcommand{\del}{\delta}
\newcommand{\ga}{\alpha}
\newcommand{\gb}{\beta}
\newcommand{\gd}{\del}
\newcommand{\gp}{\pi}
\newcommand{\gf}{\phi}
\newcommand{\gh}{\eta}
\newcommand{\gF}{\Phi}
\newcommand{\gl}{\lambda}
\newcommand{\gm}{\mu}
\newcommand{\gn}{\nu}
\newcommand{\gr}{\rho}
\newcommand{\gs}{\sigma}
\newcommand{\gth}{\theta}
\newcommand{\gx}{\xi}
\newcommand{\gw}{\omega}

\newcommand{\sw}{\sigma}
\newcommand{\SW}{\Sigma}
\newcommand{\ew}{\lambda}
\newcommand{\EW}{\Lambda}

\newcommand{\Del}{\Delta}
\newcommand{\gD}{\Delta}
\newcommand{\gG}{\Gamma}
\newcommand{\gW}{\Omega}
\newcommand{\gS}{\Sigma}
\newcommand{\gTh}{\Theta}

% Bold english letters.
\newcommand{\bA}{\mathbf{A}}
\newcommand{\bB}{\mathbf{B}}
\newcommand{\bC}{\mathbf{C}}
\newcommand{\bD}{\mathbf{D}}
\newcommand{\bE}{\mathbf{E}}
\newcommand{\bF}{\mathbf{F}}
\newcommand{\bG}{\mathbf{G}}
\newcommand{\bH}{\mathbf{H}}
\newcommand{\bI}{\mathbf{I}}
\newcommand{\bJ}{\mathbf{J}}
\newcommand{\bK}{\mathbf{K}}
\newcommand{\bL}{\mathbf{L}}
\newcommand{\bM}{\mathbf{M}}
\newcommand{\bN}{\mathbf{N}}
\newcommand{\bO}{\mathbf{O}}
\newcommand{\bP}{\mathbf{P}}
\newcommand{\bQ}{\mathbf{Q}}
\newcommand{\bR}{\mathbf{R}}
\newcommand{\bS}{\mathbf{S}}
\newcommand{\bT}{\mathbf{T}}
\newcommand{\bU}{\mathbf{U}}
\newcommand{\bV}{\mathbf{V}}
\newcommand{\bW}{\mathbf{W}}
\newcommand{\bX}{\mathbf{X}}
\newcommand{\bY}{\mathbf{Y}}
\newcommand{\bZ}{\mathbf{Z}}

\newcommand{\bba}{\mathbf{a}}
\newcommand{\bbb}{\mathbf{b}}
\newcommand{\bbc}{\mathbf{c}}
\newcommand{\bbd}{\mathbf{d}}
\newcommand{\bbe}{\mathbf{e}}
\newcommand{\bbf}{\mathbf{f}}
\newcommand{\bbg}{\mathbf{g}}
\newcommand{\bbh}{\mathbf{h}}
\newcommand{\bbk}{\mathbf{k}}
\newcommand{\bbl}{\mathbf{l}}
\newcommand{\bbm}{\mathbf{m}}
\newcommand{\bbn}{\mathbf{n}}
\newcommand{\bbp}{\mathbf{p}}
\newcommand{\bbq}{\mathbf{q}}
\newcommand{\bbr}{\mathbf{r}}
\newcommand{\bbs}{\mathbf{s}}
\newcommand{\bbt}{\mathbf{t}}
\newcommand{\bbu}{\mathbf{u}}
\newcommand{\bbv}{\mathbf{v}}
\newcommand{\bbw}{\mathbf{w}}
\newcommand{\bbx}{\mathbf{x}}
\newcommand{\bby}{\mathbf{y}}
\newcommand{\bbz}{\mathbf{z}}
\newcommand{\0}{\mathbf{0}}
\newcommand{\1}{\mathbf{1}}

% Caligraphic english alphabet
\newcommand{\cA}{\mathcal{A}}
\newcommand{\cB}{\mathcal{B}}
\newcommand{\cC}{\mathcal{C}}
\newcommand{\cD}{\mathcal{D}}
\newcommand{\cE}{\mathcal{E}}
\newcommand{\cF}{\mathcal{F}}
\newcommand{\cG}{\mathcal{G}}
\newcommand{\cH}{\mathcal{H}}
\newcommand{\cI}{\mathcal{I}}
\newcommand{\cJ}{\mathcal{J}}
\newcommand{\cK}{\mathcal{K}}
\newcommand{\cL}{\mathcal{L}}
\newcommand{\cM}{\mathcal{M}}
\newcommand{\cN}{\mathcal{N}}
\newcommand{\cO}{\mathcal{O}}
\newcommand{\cP}{\mathcal{P}}
\newcommand{\cQ}{\mathcal{Q}}
\newcommand{\cR}{\mathcal{R}}
\newcommand{\cS}{\mathcal{S}}
\newcommand{\cT}{\mathcal{T}}
\newcommand{\cU}{\mathcal{U}}
\newcommand{\cV}{\mathcal{V}}
\newcommand{\cW}{\mathcal{W}}
\newcommand{\cX}{\mathcal{X}}
\newcommand{\cY}{\mathcal{Y}}
\newcommand{\cZ}{\mathcal{Z}}


% Formatting shortcuts
\newcommand{\red}[1]{\textcolor{red}{#1}}
\newcommand{\blue}[1]{\textcolor{blue}{#1}}
\newcommand{\htext}[2]{\texorpdfstring{#1}{#2}}

% Statistics
\newcommand{\distr}{\sim}
\newcommand{\stddev}{\sigma}
\newcommand{\covmatrix}{\Sigma}
\newcommand{\mean}{\mu}
\newcommand{\param}{\theta}
\newcommand{\gthEst}{\hat{\theta}}
\newcommand{\ftr}{\phi}
\newcommand{\est}[1]{\hat{#1}}

% General utility
\newcommand{\todo}[1]{\textbf{[TODO]}] \footnote{TODO: #1}}
\newcommand{\tbc}{[\textbf{Incomplete}]}
\newcommand{\chk}{[\textbf{Check}]}
\newcommand{\why}{[\textbf{Find proof}]}
\newcommand{\opt}[1]{\textit{#1}}

\newcommand{\experience}[1]{[\textbf{Personal Experience}]: #1 \blacktriangle}
\newcommand{\pf}[1]{[\textbf{Proof}]: #1 \Box}
\newcommand{\core}[1]{\textbf{Core Idea}: #1 \Arrowvert}
\newcommand{\example}[1]{\textbf{Example}: #1 \blacktriangle}
\newcommand{\error}[1]{\textbf{Error alert}: #1 \triangle}
\newcommand{\oprob}{[\textbf{OP}]: }


\renewcommand{\~}{\htext{$\sim$}{~}}

\DeclareMathOperator*{\argmin}{arg\,min}
\DeclareMathOperator*{\argmax}{arg\,max}


\)



    
<header class="p-1 bg-yellow noPrint">
    <nav role="navigation" class="col">
      <div id="div_top_bar" class="row">
        <div class="col-md-3 justify-content-start">
          <a  href="/" class="btn col border">
            <i class="fas fa-gopuram ">Vishvas&#39;s notes <br/> 1 How the Genome Explains Who We Are</i>
          </a>
        </div>
        <div id="div_top_bar_right" class="col-md-auto d-flex justify-content-center">
          <button class="btn btn-light"  href="#transliterationDropdown" data-toggle="collapse">
            ✍
          </button>
          <select id="transliterationDropdown" size="1" class="collapse" onchange="module_uiLib.default.content.updateTransliteration()">
            <optgroup label="🌍">
              <option value="iso">āISO</option>
              <option value="iso_vedic">āi-āu</option>
              <option value="cyrillic">пу</option>
              <option value="urdu">نس</option>
              <option value="??">🌍??</option>
            </optgroup>
            <optgroup label="🛕🧭S">
              <option value="kannada">ಅ</option>
              <option value="malayalam">അ</option>
              <option value="telugu">క</option>
              <option value="tamil_superscripted">க²</option>
              <option value="tamil_extended">க</option>
              <option value="grantha">𑌅</option>
            </optgroup>
            <optgroup label="🛕🧭N">
              <option value="devanagari" selected="">स</option>
              <option value="gujarati">અ</option>
              <option value="gurmukhi">ਅ</option>
            </optgroup>
            <optgroup label="🛕🧭E">
              <option value="oriya">ଅ</option>
              <option value="assamese">অস</option>
              <option value="bengali">অ</option>
            </optgroup>
            <optgroup label="🛕🏺📜">
              <option value="sharada">𑆑𑇀𑆰</option>
              <option value="brahmi">𑀅</option>
            </optgroup>
            <optgroup label="🛕🌋">
              <option value="balinese">ᬩᬮᬶ</option>
            </optgroup>
          </select>
          <div>
          <button class="btn btn-outline-secondary btn-sm"  href="#transliterationDropdownAlt" data-toggle="collapse">
          (✍)
          </button>
          <select id="transliterationDropdownAlt" size="1"  style="font-size:12px" class="bg-light-yellow collapse" onchange="module_uiLib.default.content.updateTransliteration()">
            <optgroup label="🌍">
              <option value="iso">āISO</option>
              <option value="iso_vedic">āi-āu</option>
              <option value="cyrillic">пу</option>
              <option value="urdu">نس</option>
              <option value="??">🌍??</option>
            </optgroup>
            <optgroup label="🛕🧭S">
              <option value="kannada">ಅ</option>
              <option value="malayalam">അ</option>
              <option value="telugu">క</option>
              <option value="tamil_superscripted">க²</option>
              <option value="tamil_extended">க</option>
              <option value="grantha">𑌅</option>
            </optgroup>
            <optgroup label="🛕🧭N">
              <option value="devanagari" selected="">स</option>
              <option value="gujarati">અ</option>
              <option value="gurmukhi">ਅ</option>
            </optgroup>
            <optgroup label="🛕🧭E">
              <option value="oriya">ଅ</option>
              <option value="assamese">অস</option>
              <option value="bengali">অ</option>
            </optgroup>
            <optgroup label="🛕🏺📜">
              <option value="sharada">𑆑𑇀𑆰</option>
              <option value="brahmi">𑀅</option>
            </optgroup>
            <optgroup label="🛕🌋">
              <option value="balinese">ᬩᬮᬶ</option>
            </optgroup>
          </select>
          </div>
          <div class="row col d-flex  dropdown">
            <button class="btn btn-secondary-outline btn-sm dropdown-toggle" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
              A±
            </button>
            <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
            <a class="btn btn-secondary btn-sm" onclick="module_uiLib.changeTextSize(2)">A+</a>
            <a class="btn btn-secondary btn-sm" onclick="module_uiLib.changeTextSize(-2)">A-</a>
            </div>
          </div>
          <button class="btn btn-light btn-sm"  href="#commentStyleDropdown" data-toggle="collapse">
            🗨
          </button>
          <select id="commentStyleDropdown" class="collapse" size="1" onchange="module_uiLib.default.content.updateCommentStyleFromDropdown()">
            <option value="on">✔</option>
            <option value="hidden">✗</option>
          </select>
          <div><a id="expandAllButton" class="btn btn-secondary" onclick="module_uiLib.default.query.setParamsAndGo({'expandDetails': '.*'})">↕️️</a></div>
          <div id="printStyleDiv" class="row col d-flex  dropdown">
          <button class="btn btn-light btn-sm dropdown-toggle"  href="#printStyleDropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
          🖨
          </button>
            <div class="dropdown-menu col" aria-labelledby="printStyleDropdown">
              <div class="row justify-content-center">
                <p>Cols:</p>
                <input type="number" min="1" max="10" value="2" step="1" id="tbColumns" style="width: 3ch;"  accesskey="c"/>
              </div>
              <div class="row justify-content-center">
                <p>A±:</p> <input type="text" id="tbBodyFontSize" value="0.4cm" style="width: 6ch;"  accesskey="f"></input>
              </div>
              <div class="row justify-content-center">
                <p>Incl:</p> <input checked type="checkbox" id="tbIncludeStyle" value="on"></input>
                <button id="btnPrintStyle" onclick="module_uiLib.default.content.updatePrintStyle()">🖨
                </button>
              </div>
            </div>
        </div>
        </div>
        <div id="audioDiv" class="row col d-flex  dropdown">
          <button class="btn btn-secondary-outline btn-sm dropdown-toggle" type="button" id="audioDropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
            🎧
          </button>
          <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
          <a id="speakerButton" class="btn btn-outline-secondary btn-sm" onclick="module_uiLib.default.content.handleSpeakToggle()">⏯</a>
          <button id="speakerPauseButton" class="btn btn-outline-secondary btn-sm">⏸</button>
            <div id="audioOptionsDiv" class="row justify-content-center" >
              <div>
                <p class="lead">Vol</p>
                <input type="range" min="0" max="1" value="1" step="0.1" id="volume" />
                <span id="volume-label" class="ms-2">1</span>
              </div>
              <div class="mx-5">
                <p class="lead">Rate</p>
                <input type="range" min="0.1" max="10" value="1" id="rate" step="0.1" />
                <span id="rate-label" class="ms-2">1</span>
              </div>
              <div>
                <p class="lead">Pitch</p>
                <input type="range" min="0" max="2" value="1" step="0.1" id="pitch" />
                <span id="pitch-label" class="ms-2">1</span>
              </div>
            </div>
          </div>
        </div>
        <a href="https://vishvAsa.github.io/notes/search"  class="btn btn-light btn-sm">🥽🔦</a>
        <div class="row col  d-flex justify-content-center">
          <div><a id="previousPage" href="" class="btn btn-secondary">###<i class="fas fa-caret-left"></i></a></div>
          <div ><a id="nextPage" href="" class="btn btn-secondary"><i class="fas fa-caret-right"></i> ### </a></div>
        </div>
        <ul id="top-bar-right-custom" class="list-group list-group-horizontal">
        </ul>
      </div>
    </nav>
</header>

    
    <div class="row" name="contentRow">
      
      <aside class="col-md-3 card border noPrint " id="sidebar">
          <div id="fastSearch">
              <datalist id="pageDataList"></datalist>
              <input id="searchInput" placeholder="शीर्षिकान्विष्यताम्" list="pageDataList" onchange="module_uiLib.default.navigation.pageLoader()">
          </div>
          <details open  class="bg-light-gray" >
              <summary id="sidebarToggleLink" onclick="module_uiLib.default.navigation.sidebarToggleHandler()">
            Menu
              </summary>
              <ul id="sidebarBody" class="list pl2 p-0 bg-yellow">
              </ul>
          </details>
      </aside>
      <main class="col p-3" role="main" data-pagefind-body>
        
<header class='border d-flex justify-content-between'>
    <h1 id="1 How the Genome Explains Who We Are">1 How the Genome Explains Who We Are</h1>
    
    <a id="editLink" class="btn btn-primary noPrint"  href="https://github.com/vishvAsa/notes/edit/content/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are.md"><i class="fas fa-edit"></i></a>
    
</header>
<article>
  <aside id="toc_card" class="card border noPrint ">
    <details id="toc_header">
        <summary>What's in this page?</summary>
      <div id="toc_body" class="p-0">
        
        <ul id="toc_ul" class="list p-0">
        </ul>
      </div>
    </details>
    <div id="progressBarDiv" class="row d-flex justify-content-center">
      🐎💪<progress id="progressLoading" value="0" max="100"> 32% </progress>
    </div>
  </aside>
  <div id="post_content">
  <h2 id="the-master-chronicle-of-human-variation">The Master Chronicle of Human Variation</h2>
<p>To understand why genetics is able to shed light on the human past, it is necessary to understand how the genome—defined as the full set of genetic code each of us inherits from our parents—records information. Francis Crick, Rosalind Franklin, James Watson, and Maurice Wilkins showed in 1953 that the genome is written out in twin chains of about three billion chemical building blocks (six billion in all) that can be thought of as the letters of an alphabet: A (adenine), C (cytosine), G (guanine), and T (thymine).<sup id="fnref:1"><a href="#fn:1" class="footnote-ref" role="doc-noteref">1</a></sup> What we call a “gene” consists of tiny fragments of these chains, typically around one thousand letters long, which are used as templates to assemble the proteins that do most of the work in cells. In between the genes is noncoding DNA, sometimes referred to as “junk” DNA. The order of the letters can be read by machines that perform chemical reactions on fragments of DNA, releasing flashes of light as the reactions pass along the length of the DNA sequence.+++(5)+++ The reactions emit a different color for each of the letters A, C, G, and T, so that the sequence of letters can be scanned into a computer by a camera.</p>
<p>Although the great majority of scientists are focused on the biological information that is contained within the genes, there are also occasional differences between DNA sequences. These differences are due to random errors in copying of genomes (known as mutations) that occurred at some point in the past. It is these differences, occurring about one every thousand letters or so in both genes and in “junk,” that geneticists study to learn about the past. Over the approximately three billion letters, there are typically around three million differences between unrelated genomes. The higher the density of differences separating two genomes on any segment, the longer it has been since the segments shared a common ancestor as the mutations accumulate at a more or less constant rate over time. So the density of differences provides a biological stopwatch, a record of how long it has been since key events occurred in the past.+++(5)+++</p>
<figure class="">

    <a href="../images/1_3_genome-sequences.jpg" target="_blank" rel="noopener">
        <img src="../images/1_3_genome-sequences.jpg" alt="" title="">
    </a>

    

</figure>
<p>Figure 3. The genome contains about three billion nucleotides, which can be thought of as four letters in a biological alphabet: adenine (A), cytosine (C), guanine (G), and thymine (T). Around 99.9 percent of these letters are identical across two lined-up genomes, but in that last ~0.1 percent there are differences, reflecting mutations that accumulate over time. These mutations tell us how closely related two people are and record exquisitely precise information about the past.</p>
<p>The first startling application of genetics to the study of the past involved mitochondrial DNA. This is a tiny portion of the genome—only approximately 1/200,000th of it—which is passed down along the maternal line from mother to daughter to granddaughter. In 1987, Allan Wilson and his colleagues sequenced a few hundred letters of mitochondrial DNA from diverse people around the world. By comparing the mutations that were different among these sequences, he and his colleagues were able to reconstruct a family tree of maternal relationships. What they found is that the deepest branch of the tree—the branch that left the main trunk earliest—is found today only in people of sub-Saharan African ancestry, suggesting that the ancestors of modern humans lived in Africa.+++(5)+++ In contrast, all non-Africans today descend from a later branch of the tree.<sup id="fnref:2"><a href="#fn:2" class="footnote-ref" role="doc-noteref">2</a></sup> This finding became an important part of the triumphant synthesis of archaeological and genetic and skeletal evidence that emerged in the 1980s and 1990s for the theory that modern humans descend from ancestors who lived in the last hundred thousand years or so in Africa. Based on the rate at which mutations are known to accumulate, Wilson and his colleagues estimated that the most recent African ancestor of all the branches, “Mitochondrial Eve,” lived sometime after 200,000 years ago.<sup id="fnref:3"><a href="#fn:3" class="footnote-ref" role="doc-noteref">3</a></sup> The best current estimate is around 160,000 years ago, although it is important to realize that like most genetic dates, this one is imprecise because of uncertainty about the true rate at which human mutations occur.+++(4)+++<sup id="fnref:4"><a href="#fn:4" class="footnote-ref" role="doc-noteref">4</a></sup></p>
<p>The finding of such a recent common ancestor was exciting because it refuted the “multiregional hypothesis,” according to which present-day humans living in many parts of Africa and Eurasia descend substantially from an early dispersal (at least 1.8 million years ago) of <em>Homo erectus,</em> a species that made crude stone tools and had a brain about two-thirds the size of ours. The multiregional hypothesis implied that descendants of <em>Homo erectus</em> evolved in parallel across Africa and Eurasia to give rise to the populations that live in the same places today. The multiregional hypothesis would therefore predict that there would be mitochondrial DNA sequences among present-day people that are separated by close to two million years, the age of the dispersal of <em>Homo erectus</em>. However, the genetic data was impossible to reconcile with this prediction. The fact that all people today share a common mitochondrial DNA ancestor about ten times more recently showed that humans today largely descend from a much later expansion from Africa.</p>
<p>Anthropological evidence pointed to a likely scenario for what occurred. The earliest human skeletons with “anatomically modern” features—defined as falling within the range of variation of all humans today with regard to having a globular brain case and other traits—date up to two hundred to three hundred thousand years ago and are all from Africa.<sup id="fnref:5"><a href="#fn:5" class="footnote-ref" role="doc-noteref">5</a></sup> Outside of Africa and the Near East, though, there is no convincing evidence of anatomically modern humans older than a hundred thousand years and very limited evidence older than around fifty thousand years.<sup id="fnref:6"><a href="#fn:6" class="footnote-ref" role="doc-noteref">6</a></sup> Archaeological evidence of stone tool types also points to a great change after around fifty thousand years ago, a period known to archaeologists of West Eurasia as the Upper Paleolithic, and to archaeologists of Africa as the Later Stone Age.+++(5)+++ After this time, the technology for manufacturing stone tools became very different, and there were changes in style every few thousand years, compared to the glacial earlier pace of change. Humans in this period also began to leave behind far more artifacts that revealed their aesthetic and spiritual lives: beads made of ostrich eggshells, polished stone bracelets, body paint made from red iron oxide, and the world’s first representational art. The world’s earliest known figurine is a roughly forty-thousand-year-old “lion-man” carved from a woolly mammoth tusk, found in Hohlenstein-Stadel in Germany.<sup id="fnref:7"><a href="#fn:7" class="footnote-ref" role="doc-noteref">7</a></sup> The approximately thirty-thousand-year-old drawings of pre–ice age beasts, found on the walls of Chauvet Cave in France, even today are recognizable as transcendent art.</p>
<p>The dramatic acceleration of change in the archaeological record after around fifty thousand years ago was also reflected by evidence of population change.+++(4)+++ The Neanderthals, who had evolved in Europe by around four hundred thousand years ago and are considered “archaic” in the sense that their skeletal shape did not fall within present-day human variation, went extinct in their last holdout of western Europe between about forty-one thousand and thirty-nine thousand years ago, within a few thousand years of the arrival of modern humans.<sup id="fnref:8"><a href="#fn:8" class="footnote-ref" role="doc-noteref">8</a></sup> Population turnovers also occurred elsewhere in Eurasia, as well as in southern Africa, where there is evidence of abandonment of sites and the sudden appearance of Later Stone Age cultures.<sup id="fnref:9"><a href="#fn:9" class="footnote-ref" role="doc-noteref">9</a></sup></p>
<p>The natural explanation for all these changes was the spread of an anatomically modern human population whose ancestors included “Mitochondrial Eve,” who practiced a sophisticated new culture, and who largely replaced the people who lived in each place before.</p>
<h2 id="the-siren-call-of-the-genetic-switch">The Siren Call of the Genetic Switch</h2>
<p>The finding that genetics could help to distinguish between competing hypotheses of human origins led in the 1980s and 1990s to exuberance about the power of the discipline to provide simple explanations. Some even wondered if genetics might be able to do more than provide a supporting line of evidence for the spread of modern humans from Africa and the Near East after around fifty thousand years ago. Perhaps genes could also be the cause of that spread, offering an explanation as simple and beautiful as the four-letter code written in DNA for the quickening pace of change in the archaeological record.</p>
<p>The anthropologist best known for embracing the idea that a genetic change might explain how we came to be behaviorally distinct from our predecessors was Richard Klein. He put forward the idea that the Later Stone Age revolution of Africa and the Upper Paleolithic revolution of western Eurasia, when recognizably modern human behavior burst into full flower after about fifty thousand years ago, were driven by the rise in frequency of a single mutation of a gene affecting the biology of the brain, which permitted the manufacture of innovative tools and the development of complex behavior.</p>
<p>According to Klein’s theory, the rise in frequency of this mutation primed humans for some enabling trait, such as the ability to use conceptual language. Klein thought that prior to the occurrence of this mutation, humans were incapable of modern behaviors. Supporting his notion are examples among other species of a small number of genetic changes that have effected major adaptations, such as the five changes that are sufficient to turn the tiny ears of the Mexican wild grass teosinte into the huge cobs of corn that we buy in the supermarket today.+++(4)+++<sup id="fnref:10"><a href="#fn:10" class="footnote-ref" role="doc-noteref">10</a></sup></p>
<p>Klein’s hypothesis came under intense criticism almost as soon as he suggested it, most notably from the archaeologists Sally McBrearty and Alison Brooks, who showed that almost every trait that Klein considered to be a hallmark of distinctly modern human behavior was evident in the African and Near Eastern archaeological records tens of thousands of years before the Upper Paleolithic and Later Stone Age transitions.<sup id="fnref:11"><a href="#fn:11" class="footnote-ref" role="doc-noteref">11</a></sup> But even if no single behavior was new, Klein had put his finger on something important. The intensification of evidence for modern human behavior after fifty thousand years ago is undeniable, and raises the question of whether biological change contributed to it.</p>
<p>One geneticist who came of age at this time of exuberance about the power of genetics to provide simple explanations for great mysteries was Svante Pääbo, who arrived in Allan Wilson’s laboratory just after the “Mitochondrial Eve” discovery, and who would go on to invent much of the toolkit of the ancient DNA revolution and to sequence the Neanderthal genome. In 2002, Pääbo and his colleagues discovered two mutations in the gene <em>FOXP2</em> that seemed to be candidates for propelling the great changes that occurred after around fifty thousand years ago. The previous year, medical geneticists had identified <em>FOXP2</em> as a gene that, when mutated, produces an extraordinary syndrome whose sufferers have normal-range cognitive capabilities, but cannot use complex language, including most grammar.<sup id="fnref:12"><a href="#fn:12" class="footnote-ref" role="doc-noteref">12</a></sup> Pääbo and his colleagues showed that the protein produced by the <em>FOXP2</em> gene has remained almost identical during the more than hundred million years of evolution separating chimpanzees and mice. However, two changes to the protein occurred on just the human lineage since it branched out of the common ancestral population of humans and chimpanzees, showing that the gene had evolved much more rapidly on the human lineage.<sup id="fnref:13"><a href="#fn:13" class="footnote-ref" role="doc-noteref">13</a></sup> Later work by Pääbo and his colleagues found that engineered mice with the human versions of <em>FOXP2</em> are identical to regular mice in most respects, but squeak differently, consistent with the idea that these changes affect the formation of sounds.<sup id="fnref:14"><a href="#fn:14" class="footnote-ref" role="doc-noteref">14</a></sup> These two mutations at <em>FOXP2</em> cannot have contributed to the changes after fifty thousand years ago, since Neanderthals shared them,<sup id="fnref:15"><a href="#fn:15" class="footnote-ref" role="doc-noteref">15</a></sup> but Pääbo and his colleagues later identified a third mutation that is found in almost all present-day humans and that affects when and in what cells <em>FOXP2</em> gets turned into protein.+++(5)+++ This change is absent in Neanderthals, and thus is a candidate for contributing to the evolution of modern humans after their separation from Neanderthals hundreds of thousands of years ago.<sup id="fnref:16"><a href="#fn:16" class="footnote-ref" role="doc-noteref">16</a></sup></p>
<p>Regardless of how important <em>FOXP2</em> itself is in modern human biology, Pääbo cites the search for the genetic basis for modern human behavior as a justification for sequencing the genomes of archaic humans.<sup id="fnref:17"><a href="#fn:17" class="footnote-ref" role="doc-noteref">17</a></sup> Between 2010 and 2013, when he led a series of studies that published whole-genome sequences from archaic humans like Neanderthals, Pääbo’s papers highlighted an evolving list of about one hundred thousand places in the genome where nearly all present-day humans carry genetic changes that are absent in Neanderthals.<sup id="fnref:18"><a href="#fn:18" class="footnote-ref" role="doc-noteref">18</a></sup> There are surely biologically important changes hiding in the list, but we are still only at the very beginning of the process of determining what they are, reflecting a more general problem that we are like kindergartners in our ability to read the genome. While we have learned to decode the individual words—as we know how the sequence of DNA letters gets turned into proteins—we still can’t parse the sentences.</p>
<p>The sad truth is that it is possible to count on the fingers of two hands the examples like <em>FOXP2</em> of mutations that increased in frequency in human ancestors under the pressure of natural selection and whose functions we partly understand. In each of these cases, the insights only came from years of hand-to-hand combat with life’s secrets by graduate students or postdoctoral scientists making engineered mice or fish, suggesting that it will take an evolutionary Manhattan Project to understand the function of each mutation that we have and that Neanderthals do not. This Manhattan Project of human evolutionary biology is one to which we as a species should commit ourselves. But even when it is carried out, I expect that the findings will be so complicated—with so many individual genetic changes contributing to what makes humans distinctive—that few people will find the answer comprehensible. While the scientific question is profoundly important, I expect that no intellectually elegant and emotionally satisfying molecular explanation for behavioral modernity will ever be found.</p>
<p>But even if studying just a few locations in the genome will not provide a satisfying explanation for how modern human behavior evolved, the great surprise of the genome revolution is the explanations it is starting to provide from another perspective—that of history. By comprehending the entire genome—by going beyond the tiny slice of the past sampled by our mitochondrial DNA and Y chromosome and embracing the story of our past told by the multiplicity of our ancestors that is written in the record of our whole genome—we have already begun to sketch out a new picture of how we got to be the way we are.+++(4)+++ This explanation based on migrations and population mixture is the subject of this book.</p>
<h2 id="one-hundred-thousand-adams-and-eves">One Hundred Thousand Adams and Eves</h2>
<p>When the journalist Roger Lewin in 1987 dubbed the common maternal ancestor of all people living today “Mitochondrial Eve,” he evoked a creation story—that of a woman who was the mother of us all, and whose descendants dispersed throughout the earth.<sup id="fnref:19"><a href="#fn:19" class="footnote-ref" role="doc-noteref">19</a></sup> The name captured the collective imagination, and is still used not only by the public but also by many scientists to refer to this common maternal ancestor. But the name has been more misleading than helpful. It has fostered the mistaken impression that all of our DNA comes from precisely two ancestors and that to learn about our history it would be sufficient to simply track the purely maternal line represented by mitochondrial DNA, and the purely paternal line represented by the Y chromosome. Inspired by this possibility, the National Geographic Society’s “Genographic Project,” beginning in 2005, collected mitochondrial DNA and Y-chromosome data from close to a million people of diverse ethnic groups. But the project was outdated even before it began. It has been largely recreational, and has produced few interesting scientific results.+++(4)+++ From the outset, it was clear that most of the information about the human past present in mitochondrial DNA and Y-chromosome data had already been mined, and that far richer stories were buried in the whole genome.</p>
<p>The truth is that the genome contains the stories of many diverse ancestors—tens of thousands of independent genealogical lineages, not just the two whose stories can be traced with the Y chromosome and mitochondrial DNA. To understand this, one needs to realize that beyond mitochondrial DNA, the genome is not one continuous sequence from a single ancestor but is instead a mosaic. Forty-six of the mosaic tiles, as it were, are chromosomes—long stretches of DNA that are physically separated in the cell. A genome consists of twenty-three chromosomes, and because a person carries two genomes, one from each parent, the total number is forty-six.</p>
<p>But the chromosomes themselves are mosaics of even smaller tiles. For example, the first third of a chromosome a woman passes down to her egg might come from her father and the last two-thirds from her mother, the result of a splicing together of her father’s and mother’s copies of that chromosome in her ovaries. Females create an average of about forty-five new splices when producing eggs, while males create about twenty-six splices when producing sperm, for a total of about seventy-one new splices per generation.<sup id="fnref:20"><a href="#fn:20" class="footnote-ref" role="doc-noteref">20</a></sup>+++(5)+++ So it is that as we trace each generation back further into the past, a person’s genome is derived from an ever-increasing number of spliced-together ancestral fragments.</p>
<p>This means that our genomes hold within them a multitude of ancestors. Any person’s genome is derived from 47 stretches of DNA corresponding to the chromosomes transmitted by mother and father plus mitochondrial DNA. One generation back, a person’s genome is derived from about 118 (47 plus 71) stretches of DNA transmitted by his or her parents. Two generations back, the number of ancestral stretches of DNA grows to around 189 (47 plus 71 plus another 71) transmitted by four grandparents. Look even further back in time, and the additional increase in ancestral stretches of DNA every generation is rapidly overtaken by the doubling of ancestors. Ten generations back, for example, the number of ancestral stretches of DNA is around 757 but the number of ancestors is 1,024, guaranteeing that each person has several hundred ancestors from whom he or she has received no DNA whatsoever.+++(5)+++ Twenty generations in the past, the number of ancestors is almost a thousand times greater than the number of ancestral stretches of DNA in a person’s genome, so it is a certainty that each person has not inherited any DNA from the great majority of his or her actual ancestors.</p>
<p>These calculations mean that a person’s genealogy, as reconstructed from historical records, is not the same as his or her genetic inheritance. The Bible and the chronicles of royal families record who begat whom over dozens of generations. Yet even if the genealogies are accurate, Queen Elizabeth II of England almost certainly inherited no DNA from William of Normandy, who conquered England in 1066 and who is believed to be her ancestor twenty-four generations back in time.<sup id="fnref:21"><a href="#fn:21" class="footnote-ref" role="doc-noteref">21</a></sup> This does not mean that Queen Elizabeth II did not inherit DNA from ancestors that far back, just that it is expected that only about 1,751 of her 16,777,216 twenty-fourth-degree genealogical ancestors contributed any DNA to her.+++(5)+++ This is such a small fraction that the only way William could plausibly be her genetic ancestor is if he was her genealogical ancestor in thousands of different lineage paths, which seems unlikely even considering the high level of inbreeding in the British royal family.+++(5)+++</p>
<figure class="">

    <a href="../images/1_4_ancestor-contributions.jpg" target="_blank" rel="noopener">
        <img src="../images/1_4_ancestor-contributions.jpg" alt="" title="">
    </a>

    

</figure>
<p>Figure 4. The number of ancestors you have doubles every generation back in time. However, the number of stretches of DNA that contributed to you increases by only around seventy-one per generation. This means that if you go back eight or more generations, it is almost certain that you will have some ancestors whose DNA did not get passed down to you.+++(5)+++ Go back fifteen generations and the probability that any one ancestor contributed directly to your DNA becomes exceedingly small.</p>
<p>Going back deeper in time, a person’s genome gets scattered into more and more ancestral stretches of DNA spread over ever-larger numbers of ancestors. Tracing back fifty thousand years in the past, our genome is scattered into more than one hundred thousand ancestral stretches of DNA, greater than the number of people who lived in any population at that time, so we inherit DNA from nearly everyone in our ancestral population who had a substantial number of offspring at times that remote in the past.+++(5)+++</p>
<p>There is a limit, though, to the information that comparison of genome sequences provides about deep time. At each place in the genome, if we trace back our lineages far enough into the past, we reach a point where everyone descends from the same ancestor, beyond which it becomes impossible to obtain any information about deeper time from comparison of the DNA sequences of people living today. From this perspective, the common ancestor at each point in the genome is like a black hole in astrophysics, from which no information about deeper time can escape.+++(5)+++ For mitochondrial DNA this black hole occurs around 160,000 years ago, the date of “Mitochondrial Eve.” For the great majority of the rest of the genome the black hole occurs between five million and one million years ago, and thus the rest of the genome can provide information about far deeper time than is accessible through analysis of mitochondrial DNA.<sup id="fnref:22"><a href="#fn:22" class="footnote-ref" role="doc-noteref">22</a></sup>+++(5)+++ Beyond this, everything goes dark.</p>
<p>The power of tracing this multitude of lineages to reveal the past is extraordinary. In my mind’s eye, when I think of a genome, I view it not as a thing of the present, but as deeply rooted in time, a tapestry of threads consisting of lines of descent and DNA sequences copied from parent to child winding back into the distant past. Tracing back, the threads wind themselves through ever more ancestors, providing information about population size and substructure in each generation. When an African American person is said to have 80 percent West African and 20 percent European ancestry, for example, a statement is being made that about five hundred years ago, prior to the population migrations and mixtures precipitated by European colonialism, 80 percent of the person’s ancestral threads probably resided in West Africa and the remainder probably lived in Europe. But such statements are like still frames in a movie, capturing one point in the past. An equally valid perspective is that one hundred thousand years ago, the vast majority of lineages of African American ancestors, like those of everyone today, were in Africa.</p>
<h2 id="the-story-told-by-the-multitudes-in-our-genomes">The Story Told by the Multitudes in Our Genomes</h2>
<p>In 2001, the human genome was sequenced for the first time—which means that the great majority of its chemical letters were read. About 70 percent of the sequence came from a single individual, an African American,<sup id="fnref:23"><a href="#fn:23" class="footnote-ref" role="doc-noteref">23</a></sup> but some came from other people. By 2006, companies began selling robots that reduced the cost of reading DNA letters by more than ten thousandfold and soon by one hundred thousandfold, making it economical to map the genomes of many more people. It thus became possible to compare sequences not just from a few isolated locations, such as mitochondrial DNA, but from the whole genome. That made it possible to reconstruct each person’s tens of thousands of ancestral lines of descent. This revolutionized the study of the past. Scientists could gather orders of magnitude more data, and test whether the history of our species suggested by the whole genome was the same as that told by mitochondrial DNA and the Y chromosome.+++(4)+++</p>
<p>A 2011 paper by Heng Li and Richard Durbin showed that the idea that a single person’s genome contains information about a multitude of ancestors was not just a theoretical possibility, but a reality. To decipher the deep history of a population from a single person’s DNA, Li and Durbin leveraged the fact that any single person actually carries not one but two genomes: one from his or her father and one from his or her mother.<sup id="fnref:24"><a href="#fn:24" class="footnote-ref" role="doc-noteref">24</a></sup> Thus it is possible to count the number of mutations separating the genome a person receives from his or her mother and the genome the person receives from his or her father to determine when they shared a common ancestor at each location. By examining the range of dates when these ancestors lived—plotting the ages of one hundred thousand Adams and Eves—Li and Durbin established the size of the ancestral population at different times. In a small population, there is a substantial chance that two randomly chosen genome sequences derive from the same parent genome sequence, because the individuals who carry them share a parent. However, in a large population the chance is far lower. Thus, the times in the past when the population size was low can be identified based on the periods in the past when a disproportionate fraction of lineages have evidence of sharing common ancestors. Walt Whitman, in the poem “Song of Myself,” wrote, “Do I contradict myself? / Very well, then I contradict myself, / (I am large, I contain multitudes).” Whitman could just as well have been talking about the Li and Durbin experiment and its demonstration that a whole population history is contained within a single person as revealed by the multitude of ancestors whose histories are recorded within that person’s genome.</p>
<p>An unanticipated finding of the Li and Durbin study was its evidence that after the separation of non-African and African populations, there was an extended period in the shared history of non-Africans when populations were small, as reflected in evidence for many shared ancestors spread over tens of thousands of years.<sup id="fnref:25"><a href="#fn:25" class="footnote-ref" role="doc-noteref">25</a></sup>+++(5)+++ A shared “bottleneck event” among non-Africans—when a small number of ancestors gave rise to a large number of descendants today—was not a new finding. But prior to Li and Durbin’s work, there was no good information about the duration of this event, and it seemed plausible that it could have transpired over just a few generations—for example, a small band of people crossing the Sahara into North Africa, or from Africa into Asia. The Li and Durbin evidence of an extended period of small population size was also hard to square with the idea of an unstoppable expansion of modern humans both within and outside Africa around fifty thousand years ago. Our history may not be as simple as the story of a dominant group that was immediately successful wherever it went.</p>
<figure class="">

    <a href="../images/1_5_common-ancestor-period-estimation.jpg" target="_blank" rel="noopener">
        <img src="../images/1_5_common-ancestor-period-estimation.jpg" alt="Figure 5: How We Can Tell How Long It Has Been Since Our Genes Shared Common Ancestors" title="">
    </a>

    

</figure>
<h2 id="how-the-whole-genome-perspective-put-an-end-to-simple-explanations">How the Whole-Genome Perspective Put an End to Simple Explanations</h2>
<p>The newfound ability to take a whole-genome view of human biology, made possible by leaps in technology in the last decades, has allowed reconstruction of population history in far more detail than had been previously possible. In doing so it revealed that the simple picture from mitochondrial DNA, and the just-so stories about one or a few changes propelling the Later Stone Age and Upper Paleolithic transitions when recognizably modern human behavior became widespread as reflected in archaeological sites across Africa and Eurasia, are no longer tenable.</p>
<p>In 2016, my colleagues and I used an adaptation of the Li and Durbin method<sup id="fnref:26"><a href="#fn:26" class="footnote-ref" role="doc-noteref">26</a></sup> to compare populations from around the world to the earliest branching modern human lineage that has contributed a large proportion of the ancestry of a population living today: the one that contributed the lion’s share of ancestry to the San hunter-gatherers of southern Africa. Our study,<sup id="fnref:27"><a href="#fn:27" class="footnote-ref" role="doc-noteref">27</a></sup> like most others,<sup id="fnref:28"><a href="#fn:28" class="footnote-ref" role="doc-noteref">28</a></sup> found that the separation had begun by around two hundred thousand years ago and was mostly complete by more than one hundred thousand years ago. The evidence for this is that the density of mutations separating San genomes from non-San genomes is uniformly high, implying few shared ancestors between San and non-San in the last hundred thousand years. “Pygmy” groups from Central African forests harbor ancestry that is arguably just as distinctive. The extremely ancient isolation of some pairs of human populations from each other conflicts with the idea that a single mutation essential to distinctively modern human behavior occurred shortly before the Upper Paleolithic and Later Stone Age. A key change essential to modern human behavior in this time frame would be expected to be at high frequency in some human populations today—those that descend from the population in which the mutation occurred—and absent or very rare in others. But this seems hard to reconcile with the fact that all people today are capable of mastering conceptual language and innovating their culture in a way that is a hallmark of modern humans.</p>
<p>A second problem with the notion of a genetic switch became apparent when we applied the Li and Durbin method to search for places where all the genomes we analyzed shared a common ancestor in the period before the Upper Paleolithic and Later Stone Age. At *FOXP2—*the gene that seemed the best candidate for a switch based on previous studies—we found that the common ancestor of everyone living today (that is, the person in whom modern humanity’s shared copy of <em>FOXP2</em> last occurred), lived more than one million years ago.<sup id="fnref:29"><a href="#fn:29" class="footnote-ref" role="doc-noteref">29</a></sup></p>
<p>Expanding our analysis to the whole genome, we could not find any location—apart from mitochondrial DNA and the Y chromosome—where all people living today share a common ancestor less than about 320,000 years ago.+++(5)+++ This is a far longer time scale than the one required by Klein’s hypothesis. If Klein was right, it would be expected that there would be places in the genome, beyond mitochondrial DNA and the Y chromosome, where almost everyone shares a common ancestor within the last hundred thousand years. But these do not in fact seem to exist.</p>
<p>Our results do not completely rule out the hypothesis of a single critical genetic change. There is a small fraction of the genome that contains complicated sequences that are difficult to study and that was not included in our survey.+++(4)+++ But the key change, if it exists, is running out of places to hide. The time scale of human genetic innovation and population differentiation is also far longer than mitochondrial DNA and other genetic data suggested prior to the genome revolution. If we are going to try to search the genome for clues to what makes modern humans distinctive, it is likely that we cannot look to explanations involving one or a few changes.</p>
<p>The whole-genome approaches that became possible after the technological revolution of the 2000s also soon made it clear that natural selection was not likely to take the simple form of changes in a small number of genes, as Klein had imagined. When the first whole-genome datasets were published, many geneticists (myself included) developed methods that scoured the genome for mutations that were affected by natural selection.<sup id="fnref:30"><a href="#fn:30" class="footnote-ref" role="doc-noteref">30</a></sup> We were searching for the “low-hanging fruit”—instances in which natural selection had operated strongly on a few mutations. Examples of such low-hanging fruit include the mutations allowing people to digest cow’s milk into adulthood, or mutations that cause darkening or lightening of skin to adapt to local climates, or mutations that bequeath resistance to the infectious disease malaria. As a community, we have been successful in identifying selection on mutations like these because they have risen rapidly from low to high frequency, resulting in a large number of people today sharing a recent ancestor or striking differences in mutation frequency between two otherwise similar populations.+++(4)+++ Events like these leave great scars on patterns of genome variation that can be detected without too much trouble.</p>
<p>Excitement about this bonanza was tempered by work led by Molly Przeworski, who studied the types of patterns that natural selection is likely to leave on the genome as a whole. A 2006 study by Przeworski and her colleagues showed that genome scans of present-day human genetic variation will miss most instances of natural selection because they simply will not have the statistical power needed to detect it, and that scans of this type will also have more power to detect some types of selection than others.<sup id="fnref:31"><a href="#fn:31" class="footnote-ref" role="doc-noteref">31</a></sup> A study she led in 2011 then showed that only a small fraction of evolution in humans has likely involved intense natural selection for advantageous mutations that had not previously been present in the population.<sup id="fnref:32"><a href="#fn:32" class="footnote-ref" role="doc-noteref">32</a></sup> Thus, intense and easily detectable episodes of natural selection such as those that have facilitated the digestion of cow’s milk into adulthood are an exception.<sup id="fnref:33"><a href="#fn:33" class="footnote-ref" role="doc-noteref">33</a></sup></p>
<p>So what has been the dominant mode of natural selection in humans if not selection on newly arising single mutation changes that then rocket up to high frequency? An important clue comes from the study of height. In 2010, medical geneticists analyzed the genomes of around 180,000 people with measured heights, and found 180 independent genetic changes that are more common in shorter people. This means that these changes, or ones nearby on the genome, contribute directly to reduced height. In 2012, a second study showed that at the 180 changes, southern Europeans tend to have the versions that reduce height, and that this pattern is so pronounced that the only possible explanation is natural selection—likely for increased height in northern Europeans or decreased height in southern Europeans since the two lineages separated.<sup id="fnref:34"><a href="#fn:34" class="footnote-ref" role="doc-noteref">34</a></sup>+++(4)+++ In 2015, an ancient DNA study led by Iain Mathieson in my laboratory revealed more about this process. We assembled DNA data from the bones and teeth of 230 ancient Europeans and analyzed the data to suggest that these patterns reflected natural selection for mutations that decreased height in farmers in southern Europe after eight thousand years ago, or increased height in ancestors of northern Europeans who lived in the eastern European steppe lands before five thousand years ago.<sup id="fnref:35"><a href="#fn:35" class="footnote-ref" role="doc-noteref">35</a></sup> The advantages that accrued to shorter people in southern Europe, or to taller people in far eastern Europe, must have increased the number of their surviving children, which had the effect of systematically changing the frequencies of these mutations until a new average height was achieved.</p>
<p>Since the discoveries about height, other scientists have documented additional examples of natural selection on other complex human traits. A 2016 study analyzed the genomes of several thousand present-day Britons and found natural selection for increased height, blonder hair, bluer eyes, larger infant head size, larger female hip size, later growth spurt in males, and later age of puberty in females.<sup id="fnref:36"><a href="#fn:36" class="footnote-ref" role="doc-noteref">36</a></sup>+++(4)+++</p>
<p>These examples demonstrate that by leveraging the power of the whole genome to examine thousands of independent positions in the genome simultaneously, it is possible to get beyond the barrier that Molly Przeworski had identified—“Przeworski’s Limit”—by taking advantage of information that we now have about a large number of genetic variations at many locations in the genome that have similar biological effects.+++(4)+++ We have such information from “genome-wide association studies,” which since 2005 have collected data from more than one million people with a variety of measured traits, thereby identifying more than ten thousand individual mutations that occur at significantly elevated frequency in people with particular traits, including height.<sup id="fnref:37"><a href="#fn:37" class="footnote-ref" role="doc-noteref">37</a></sup> The value of genome-wide association studies for understanding human health and disease has been contentious because the specific mutation changes that these studies have identified typically have such small effects that their results are hardly useful for predicting who gets a disease and who does not.<sup id="fnref:38"><a href="#fn:38" class="footnote-ref" role="doc-noteref">38</a></sup> But what is often overlooked is that genome-wide association studies have provided a powerful resource for investigating human evolutionary change over time. By testing whether the mutations identified by genome-wide association studies as affecting particular biological traits have all tended to shift in frequency in the same direction, we can obtain evidence of natural selection for specific biological traits.</p>
<p>As genome-wide association studies proceed, they are beginning to investigate human variation in cognitive and behavioral traits,<sup id="fnref:39"><a href="#fn:39" class="footnote-ref" role="doc-noteref">39</a></sup> and studies like these—such as the ones for height—will make it possible to explore whether the shift to behavioral modernity among our ancestors was driven by natural selection. This means that there is new hope for providing genetic insight into the mystery that puzzled Klein—the great change in human behavior suggested by the archaeological records of the Upper Paleolithic and Later Stone Age.</p>
<p>But even if genetic changes—through coordinated natural selection on combinations of many mutations simultaneously—did enable new cognitive capacities, this is a very different scenario from Klein’s idea of a genetic switch. Genetic changes in this scenario are not a creative force abruptly enabling modern human behavior, but instead are responsive to nongenetic pressures imposed from the outside. In this scenario, it is not the case that the human population was unable to adapt because no one carried a mutation that allows a biological capability not previously present. Instead, the genetic formula that may have been necessary to drive the striking advances in human behavior and capacities that occurred during the Upper Paleolithic and Later Stone Ages is not particularly mysterious. The mutations necessary to facilitate modern human behavior were already in place, and many alternative combinations of these mutations could have increased in frequency together due to natural selection in response to changing needs imposed by the development of conceptual language or new environmental conditions.+++(4)+++ This in turn could have enabled further changes in lifestyle and innovation, in a self-reinforcing cycle. Thus, even if it is true that increases in the frequency of mutations were important in allowing modern humans to match their biology to new conditions during the Upper Paleolithic and Later Stone Age transition, what we now know about the nature of natural selection in humans and about the genetic encoding of many biological traits means it is unlikely that the first occurrence of these mutations triggered the great changes that followed. If we search for answers in a small number of mutations that arose shortly before the time of the Upper Paleolithic and Later Stone Age transitions, we are unlikely to find satisfying explanations of who we are.</p>
<h2 id="how-the-genome-can-explain-who-we-are">How the Genome Can Explain Who We Are</h2>
<p>It was molecular biologists who first focused the power of the genome onto the study of human evolution. Perhaps because of their background—and their track record of using reductionist approaches to solve great mysteries of life like the genetic code—molecular biologists were motivated by the hope that genetics would provide insights into the biological nature of how humans differ from other animals. Excitement about this prospect has also been shared by archaeologists and the public. But this research program, important as it is, is still at its very beginning because the answer is not going to be simple.</p>
<p>It is in the area of shedding light on human migrations—rather than in explaining human biology—that the genome revolution has already been a runaway success. In the last few years, the genome revolution—turbocharged by ancient DNA—has revealed that human populations are related to each other in ways that no one expected. The story that is emerging differs from the one we learned as children, or from popular culture. It is full of surprises: massive mixtures of differentiated populations; sweeping population replacements and expansions; and population divisions in prehistoric times that did not fall along the same lines as population differences that exist today. It is a story about how our interconnected human family was formed, in myriad ways never imagined.</p>
<div class="footnotes" role="doc-endnotes">
<hr>
<ol>
<li id="fn:1">
<p>J. D. Watson and F. H. Crick, “Molecular Structure of Nucleic Acids; a Structure for Deoxyribose Nucleic Acid,” <em>Nature</em> 171 (1953): 737–38.&#160;<a href="#fnref:1" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:2">
<p>R. L. Cann, M. Stoneking, and A. C. Wilson, “Mitochondrial DNA and Human Evolution,” <em>Nature</em> 325 (1987): 31–36.&#160;<a href="#fnref:2" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:3">
<p>Cann et al. “Mitochondrial DNA and Human Evolution.”&#160;<a href="#fnref:3" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:4">
<p>Q. Fu et al., “A Revised Timescale for Human Evolution Based on Ancient Mitochondrial Genomes,” <em>Current Biology</em> 23 (2013): 553–59.&#160;<a href="#fnref:4" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:5">
<p>D. E. Lieberman, B. M. McBratney, and G. Krovitz, “The Evolution and Development of Cranial Form in <em>Homo sapiens,</em>” <em>Proceedings of the National Academy of Sciences of the U.S.A.</em> 99 (2002):1134–39. Richter et al., “The Age of the Hominin Fossils from Jebel Irhoud, Morocco, and the Origins of the Middle Stone Age,” <em>Nature</em> 546 (2017): 293–96.&#160;<a href="#fnref:5" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:6">
<p>H. S. Groucutt et al., “Rethinking the Dispersal of <em>Homo sapiens</em> Out of Africa,” <em>Evolutionary Anthropology</em> 24 (2015): 149–64.&#160;<a href="#fnref:6" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:7">
<p>C.-J. Kind et al., “The Smile of the Lion Man: Recent Excavations in Stadel Cave (Baden-Württemberg, South-Western Germany) and the Restoration of the Famous Upper Palaeolithic Figurine,” <em>Quartär</em> 61 (2014): 129–45.&#160;<a href="#fnref:7" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:8">
<p>T. Higham et al., “The Timing and Spatiotemporal Patterning of Neanderthal Disappearance,” <em>Nature</em> 512 (2014): 306–9.&#160;<a href="#fnref:8" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:9">
<p>Richard G. Klein and Blake Edgar, <em>The Dawn of Human Culture</em> (New York: Wiley, 2002).&#160;<a href="#fnref:9" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:10">
<p>J. Doebley, “Mapping the Genes That Made Maize,” <em>Trends in Genetics</em> 8 (1992): 302–7.&#160;<a href="#fnref:10" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:11">
<p>S. McBrearty and A. S. Brooks, “The Revolution That Wasn’t: A New Interpretation of the Origin of Modern Human Behavior,” <em>Journal of Human Evolution</em> 39 (2000): 453–563.&#160;<a href="#fnref:11" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:12">
<p>C. S. L. Lai et al., “A Forkhead-Domain Gene Is Mutated in a Severe Speech and Language Disorder,” <em>Nature</em> 413 (2001): 519–23.&#160;<a href="#fnref:12" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:13">
<p>W. Enard et al., “Molecular Evolution of <em>FOXP2,</em> a Gene Involved in Speech and Language,” <em>Nature</em> 418 (2002): 869–72.&#160;<a href="#fnref:13" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:14">
<p>W. Enard et al., “A Humanized Version of <em>FOXP2</em> Affects Cortico-Basal Ganglia Circuits in Mice,” <em>Cell</em> 137 (2009): 961–71.&#160;<a href="#fnref:14" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:15">
<p>J. Krause et al., “The Derived <em>FOXP2</em> Variant of Modern Humans Was Shared with Neandertals,” <em>Current Biology</em> 17 (2007): 1908–12.&#160;<a href="#fnref:15" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:16">
<p>T. Maricic et al., “A Recent Evolutionary Change Affects a Regulatory Element in the Human <em>FOXP2</em> Gene,” <em>Molecular Biology and Evolution</em> 30 (2013): 844–52.&#160;<a href="#fnref:16" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:17">
<p>S. Pääbo, “The Human Condition—a Molecular Approach,” <em>Cell</em> 157 (2014): 216–26.&#160;<a href="#fnref:17" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:18">
<p>R. E. Green et al., “A Draft Sequence of the Neandertal Genome,” <em>Science</em> 328 (2010): 710–22; K. Prüfer et al., “The Complete Genome Sequence of a Neanderthal from the Altai Mountains,” <em>Nature</em> (2013): doi: 10.1038/nature 1288.&#160;<a href="#fnref:18" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:19">
<p>R. Lewin, “The Unmasking of Mitochondrial Eve,” <em>Science</em> 238 (1987): 24–26.&#160;<a href="#fnref:19" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:20">
<p>A. Kong et al., “A High-Resolution Recombination Map of the Human Genome,” <em>Nature Genetics</em> 31 (2002): 241–47.&#160;<a href="#fnref:20" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:21">
<p>“Descent of Elizabeth II from William I,” Familypedia, <a href="http://familypedia.wikia.com/wiki/Descent_of_Elizabeth_II_from_William_I#Shorter_line_of_descent">http://familypedia.wikia.com/wiki/Descent_of_Elizabeth_II_from_William_I#Shorter_line_of_descent</a>.&#160;<a href="#fnref:21" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:22">
<p>S. Mallick et al., “The Simons Genome Diversity Project: 300 Genomes from 142 Diverse Populations,” <em>Nature</em> 538 (2016): 201–6.&#160;<a href="#fnref:22" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:23">
<p>Green et al., “Draft Sequence.”&#160;<a href="#fnref:23" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:24">
<p>H. Li and R. Durbin, “Inference of Human Population History from Individual Whole-Genome Sequences,” <em>Nature</em> 475 (2011): 493–96.&#160;<a href="#fnref:24" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:25">
<p>Ibid.&#160;<a href="#fnref:25" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:26">
<p>S. Schiffels and R. Durbin, “Inferring Human Population Size and Separation History from Multiple Genome Sequences,” <em>Nature Genetics</em> 46 (2014): 919–25.&#160;<a href="#fnref:26" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:27">
<p>Mallick et al., “Simons Genome Diversity Project.”&#160;<a href="#fnref:27" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:28">
<p>I. Gronau et al., “Bayesian Inference of Ancient Human Demography from Individual Genome Sequences,” <em>Nature Genetics</em> 43 (2011): 1031–34.&#160;<a href="#fnref:28" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:29">
<p>Mallick et al., “Simons Genome Diversity Project.”&#160;<a href="#fnref:29" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:30">
<p>P. C. Sabeti et al., “Detecting Recent Positive Selection in the Human Genome from Haplotype Structure,” <em>Nature</em> 419 (2002): 832–37; B. F. Voight, S. Kudaravalli, X. Wen, and J. K. Pritchard, “A Map of Recent Positive Selection in the Human Genome,” <em>PLoS Biology</em> 4 (2006): e72.&#160;<a href="#fnref:30" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:31">
<p>K. M. Teshima, G. Coop, and M. Przeworski, “How Reliable Are Empirical Genomic Scans for Selective Sweeps?,” <em>Genome Research</em> 16 (2006): 702–12.&#160;<a href="#fnref:31" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:32">
<p>R. D. Hernandez et al., “Classic Selective Sweeps Were Rare in Recent Human Evolution,” <em>Science</em> 331 (2011): 920–24.&#160;<a href="#fnref:32" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:33">
<p>S. A. Tishkoff et al., “Convergent Adaptation of Human Lactase Persistence in Africa and Europe,” <em>Nature Genetics</em> 38 (2006): 31–40.&#160;<a href="#fnref:33" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:34">
<p>M. C. Turchin et al., “Evidence of Widespread Selection on Standing Variation in Europe at Height-Associated SNPs,” <em>Nature Genetics</em> 44 (2012): 1015–19.&#160;<a href="#fnref:34" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:35">
<p>I. Mathieson et al., “Genome-Wide Patterns of Selection in 230 Ancient Eurasians,” <em>Nature</em> 528 (2015): 499–503.&#160;<a href="#fnref:35" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:36">
<p>Y. Field et al., “Detection of Human Adaptation During the Past 2000 Years,” <em>Science</em> 354 (2016): 760–64.&#160;<a href="#fnref:36" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:37">
<p>D. Welter et al., “The NHGRI GWAS Catalog, a Curated Resource of SNP-Trait Associations,” <em>Nucleic Acids Research</em> 42 (2014): D1001–6.&#160;<a href="#fnref:37" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:38">
<p>D. B. Goldstein, “Common Genetic Variation and Human Traits,” <em>New England Journal of Medicine</em> 360 (2009): 1696–98.&#160;<a href="#fnref:38" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:39">
<p>A. Okbay et al., “Genome-Wide Association Study Identifies 74 Loci Associated with Educational Attainment,” <em>Nature</em> 533 (2016): 539–42; M. T. Lo et al., “Genome-Wide Analyses for Personality Traits Identify Six Genomic Loci and Show Correlations with Psychiatric Disorders,” <em>Nature Genetics</em> 49 (2017): 152–56; G. Davies et al., “Genome-Wide Association Study of Cognitive Functions and Educational Attainment in UK Biobank (N=112 151),” <em>Molecular Psychiatry</em> 21 (2016): 758–67.&#160;<a href="#fnref:39" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
</ol>
</div>

  </div>
</article>


        

        



    <aside class="card border noPrint" id="section-tree-item-sapiens_branches_articles_reich_who-we-are_1_The_Deep_History_of_Our_Species_1_How_the_Genome_Explains_Who_We_Are_md">
        <div class="card-title bg-light-gray border d-flex justify-content-between">
            <a href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/1_The_Deep_History_of_Our_Species/1_How_the_Genome_Explains_Who_We_Are/">1 How the Genome Explains Who We Are </a>
            <a data-toggle="collapse" href="#section-tree-item-body-sapiens_branches_articles_reich_who-we-are_1_The_Deep_History_of_Our_Species_1_How_the_Genome_Explains_Who_We_Are_md" role="button" aria-expanded="false" aria-controls="section-tree-item-body-sapiens_branches_articles_reich_who-we-are_1_The_Deep_History_of_Our_Species_1_How_the_Genome_Explains_Who_We_Are_md" >…<i class="fas fa-caret-down" class="collapsed"></i> </a>
        </div>
        <nav id="section-tree-item-body-sapiens_branches_articles_reich_who-we-are_1_The_Deep_History_of_Our_Species_1_How_the_Genome_Explains_Who_We_Are_md" class="card-body p-0 collapse">
            
            <li>draft: false</li>
            
            <li>iscjklanguage: false</li>
            
            <li>title: 1 How the Genome Explains Who We Are</li>
            
        </nav>
    </aside>


      </main>
    </div>
    <footer class="bg-yellow  p-1 noPrint" role="contentinfo">
  

  <div id="div_foot_nav_bar" class="row">
    <a href="https://vishvAsa.github.io/notes/search"  class="btn btn-light btn-sm">🥽🔦</a>
    <div class="row col  d-flex justify-content-center">
      <div><a id="previousPageFoot" href="" class="btn btn-secondary">###<i class="fas fa-caret-left"></i></a></div>
      <div ><a id="nextPageFoot" href="" class="btn btn-secondary"><i class="fas fa-caret-right"></i> ### </a></div>
    </div>
      <ul id="footer-bar-right-custom" class="list-group list-group-horizontal">
      </ul>
  </div>
  <div id="div_footer_bar" class="container-fluid d-flex justify-content-between">
    <a class="btn btn-secondary" href="https://github.com/vishvAsa/notes/issues/new" >
      ✉️Contact
    </a>
    <a class="btn btn-secondary" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/80x15.png" /></a>
    <div class="btn small border">
      
      <a href="https://github.com/vishvAsa/notes/actions/workflows/build.yml"><img src="https://github.com/vishvAsa/notes/actions/workflows/build.yml/badge.svg"> Built </a> on 2025 Oct 31 15:25:51 UTC. (<a href="http://google.com/search?q=15%3a25%3a51%20UTC to IST">IST</a>)
    </div>
  </div>
</footer>



<script async src="https://www.googletagmanager.com/gtag/js?id=G-33H08JSLQR"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-33H08JSLQR');
</script>

  </body>
  <script type='text/javascript'>
    
    
    
    module_uiLib.onDocumentReadyTasks();
  </script>
  
</html>
