<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
    
    <title>Vishvas&#39;s notes  | 10 The Genomics of Inequality</title>
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">

    <meta name="viewport" content="width=device-width,minimum-scale=1">
    <meta name="generator" content="Hugo 0.143.0">
    
    
      <META NAME="ROBOTS" CONTENT="INDEX, FOLLOW">
    

    <link href="/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/" rel="alternate" type="application/rss+xml" title="Vishvas&#39;s notes" />
    <link href="/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/" rel="feed" type="application/rss+xml" title="Vishvas&#39;s notes" />

    <meta property="og:url" content="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/">
  <meta property="og:site_name" content="Vishvas&#39;s notes">
  <meta property="og:title" content="10 The Genomics of Inequality">
  <meta property="og:description" content="The Great Mixing
The American melting pot began to swirl almost as soon as Christopher Columbus arrived in 1492. European colonists, their African slaves, and the indigenous Americans were from populations whose ancestors had been isolated from each other for tens of thousands of years. Within a few years of meeting they began mixing, founding new populations that today number in the hundreds of millions.
Mart√≠n Cort√©s ‚Äúel Mestizo‚Äù belonged to one of the first of those populations. He was born within four years of the start of the 1519 military campaign in which his father, Hern√°n Cort√©s, led just five hundred soldiers to overthrow the Aztec Empire that dominated Mexico. His mother, ‚ÄúLa Malinche,‚Äù was one of twenty female captives given over to the Spanish after a battle and she first served as an interpreter before becoming the mistress of Hern√°n Cort√©s. The Spanish quickly invented a term for the people of combined European and Native American ancestry who emerged from unions like this. ‚ÄúMestizo‚Äù comes from the Spanish word mestizaje, which in English means miscegenation‚Äîthe mixing of different ‚Äúracial‚Äù types. To maintain their status in the social hierarchy, the Spanish and Portuguese set up a casta system in which people of entirely European ancestry (especially those born in Europe) had the highest status, while people who had even some non-European ancestry had lower status.&#43;&#43;&#43;(4)&#43;&#43;&#43; This system collapsed under the demographic inevitability of admixture; within a few centuries people of entirely European ancestry were either an extreme minority or gone, and it was no longer feasible to limit power to those with entirely European ancestry.&#43;&#43;&#43;(4)&#43;&#43;&#43; Following the independence movements of the nineteenth and early twentieth centuries, mixed ancestry became a source of pride in South and Central America. In Mexico, it defines national identity.1">
  <meta property="og:locale" content="en_us">
  <meta property="og:type" content="article">
    <meta property="article:section" content="sapiens">

  <meta itemprop="name" content="10 The Genomics of Inequality">
  <meta itemprop="description" content="The Great Mixing
The American melting pot began to swirl almost as soon as Christopher Columbus arrived in 1492. European colonists, their African slaves, and the indigenous Americans were from populations whose ancestors had been isolated from each other for tens of thousands of years. Within a few years of meeting they began mixing, founding new populations that today number in the hundreds of millions.
Mart√≠n Cort√©s ‚Äúel Mestizo‚Äù belonged to one of the first of those populations. He was born within four years of the start of the 1519 military campaign in which his father, Hern√°n Cort√©s, led just five hundred soldiers to overthrow the Aztec Empire that dominated Mexico. His mother, ‚ÄúLa Malinche,‚Äù was one of twenty female captives given over to the Spanish after a battle and she first served as an interpreter before becoming the mistress of Hern√°n Cort√©s. The Spanish quickly invented a term for the people of combined European and Native American ancestry who emerged from unions like this. ‚ÄúMestizo‚Äù comes from the Spanish word mestizaje, which in English means miscegenation‚Äîthe mixing of different ‚Äúracial‚Äù types. To maintain their status in the social hierarchy, the Spanish and Portuguese set up a casta system in which people of entirely European ancestry (especially those born in Europe) had the highest status, while people who had even some non-European ancestry had lower status.&#43;&#43;&#43;(4)&#43;&#43;&#43; This system collapsed under the demographic inevitability of admixture; within a few centuries people of entirely European ancestry were either an extreme minority or gone, and it was no longer feasible to limit power to those with entirely European ancestry.&#43;&#43;&#43;(4)&#43;&#43;&#43; Following the independence movements of the nineteenth and early twentieth centuries, mixed ancestry became a source of pride in South and Central America. In Mexico, it defines national identity.1">
  <meta itemprop="wordCount" content="6750">
  <meta name="twitter:card" content="summary">
  <meta name="twitter:title" content="10 The Genomics of Inequality">
  <meta name="twitter:description" content="The Great Mixing
The American melting pot began to swirl almost as soon as Christopher Columbus arrived in 1492. European colonists, their African slaves, and the indigenous Americans were from populations whose ancestors had been isolated from each other for tens of thousands of years. Within a few years of meeting they began mixing, founding new populations that today number in the hundreds of millions.
Mart√≠n Cort√©s ‚Äúel Mestizo‚Äù belonged to one of the first of those populations. He was born within four years of the start of the 1519 military campaign in which his father, Hern√°n Cort√©s, led just five hundred soldiers to overthrow the Aztec Empire that dominated Mexico. His mother, ‚ÄúLa Malinche,‚Äù was one of twenty female captives given over to the Spanish after a battle and she first served as an interpreter before becoming the mistress of Hern√°n Cort√©s. The Spanish quickly invented a term for the people of combined European and Native American ancestry who emerged from unions like this. ‚ÄúMestizo‚Äù comes from the Spanish word mestizaje, which in English means miscegenation‚Äîthe mixing of different ‚Äúracial‚Äù types. To maintain their status in the social hierarchy, the Spanish and Portuguese set up a casta system in which people of entirely European ancestry (especially those born in Europe) had the highest status, while people who had even some non-European ancestry had lower status.&#43;&#43;&#43;(4)&#43;&#43;&#43; This system collapsed under the demographic inevitability of admixture; within a few centuries people of entirely European ancestry were either an extreme minority or gone, and it was no longer feasible to limit power to those with entirely European ancestry.&#43;&#43;&#43;(4)&#43;&#43;&#43; Following the independence movements of the nineteenth and early twentieth centuries, mixed ancestry became a source of pride in South and Central America. In Mexico, it defines national identity.1">
<script type="text/javascript">
    
    let baseURL = "https:\/\/vishvAsa.github.io\/notes\/";
    let basePath = "/" + baseURL.split("/").slice(3).join("/");
    let siteParams = JSON.parse("{\u0022build_img\u0022:\u0022https:\/\/github.com\/vishvAsa\/notes\/actions\/workflows\/build.yml\/badge.svg\u0022,\u0022build_url\u0022:\u0022https:\/\/github.com\/vishvAsa\/notes\/actions\/workflows\/build.yml\u0022,\u0022contactlink\u0022:\u0022https:\/\/github.com\/vishvAsa\/notes\/issues\/new\u0022,\u0022env\u0022:\u0022production\u0022,\u0022githubeditmepathbase\u0022:\u0022https:\/\/github.com\/vishvAsa\/notes\/edit\/content\/\u0022,\u0022js_extras\u0022:[\u0022mathjax\u0022]}");
    let pageDefaultsList = JSON.parse("[{\u0022scope\u0022:{\u0022pathPrefix\u0022:\u0022\u0022},\u0022values\u0022:{\u0022comments\u0022:true,\u0022layout\u0022:\u0022page\u0022,\u0022search\u0022:true,\u0022sidebar\u0022:\u0022home_sidebar\u0022}}]");
    let sidebarsData = JSON.parse("{\u0022home_sidebar\u0022:{\u0022contents\u0022:[{\u0022url\u0022:\u0022recdir:\/\/\u0022},{\u0022contents\u0022:[{\u0022title\u0022:\u0022‡§ú‡•ç‡§Ø‡•å‡§§‡§ø‡§∑‡§Æ‡•ç\u0022,\u0022url\u0022:\u0022..\/jyotiSham\/\u0022},{\u0022title\u0022:\u0022‡§∏‡§Ç‡§∏‡•ç‡§ï‡•É‡§§‡§Æ‡•ç\u0022,\u0022url\u0022:\u0022..\/sanskrit\/\u0022},{\u0022title\u0022:\u0022‡§¶‡•á‡§µ‡§É\u0022,\u0022url\u0022:\u0022..\/devaH\/\u0022},{\u0022title\u0022:\u0022‡§Ü‡§ó‡§Æ‡§É\u0022,\u0022url\u0022:\u0022..\/AgamaH\/\u0022},{\u0022title\u0022:\u0022‡§Ü‡§ó‡§Æ‡§É ‡§∂‡•à‡§µ‡§É\u0022,\u0022url\u0022:\u0022..\/AgamaH_shaivaH\/\u0022},{\u0022title\u0022:\u0022‡§Ü‡§ó‡§Æ‡§É ‡§µ‡•à‡§∑‡•ç‡§£‡§µ‡§É\u0022,\u0022url\u0022:\u0022..\/AgamaH_vaiShNavaH\/\u0022},{\u0022title\u0022:\u0022‡§∞‡§æ‡§Æ‡§æ‡§®‡•Å‡§ú‡•Ä‡§Ø‡§Æ‡•ç\u0022,\u0022url\u0022:\u0022..\/rAmAnujIyam\/\u0022},{\u0022title\u0022:\u0022‡§Æ‡§æ‡§ß‡•ç‡§µ‡§Æ‡•ç\u0022,\u0022url\u0022:\u0022..\/mAdhvam\/\u0022},{\u0022title\u0022:\u0022‡§Ü‡§ó‡§Æ‡§É ‡§¨‡•ç‡§∞‡§æ‡§π‡•ç‡§Æ‡§É\u0022,\u0022url\u0022:\u0022..\/AgamaH_brAhmaH\/\u0022},{\u0022title\u0022:\u0022‡§ï‡§≤‡•ç‡§™‡§æ‡§®‡•ç‡§§‡§∞‡§Æ‡•ç\u0022,\u0022url\u0022:\u0022..\/kalpAntaram\/\u0022},{\u0022title\u0022:\u0022Notes Home\u0022,\u0022url\u0022:\u0022..\/notes\/\u0022},{\u0022title\u0022:\u0022‡§µ‡•á‡§¶‡§æ‡§É\u0022,\u0022url\u0022:\u0022..\/vedAH\/\u0022},{\u0022title\u0022:\u0022‡§µ‡•á‡§¶‡§æ‡§É - ‡§ã‡§ï‡•ç\u0022,\u0022url\u0022:\u0022..\/vedAH_Rk\/\u0022},{\u0022title\u0022:\u0022‡§µ‡•á‡§¶‡§æ‡§É - ‡§Ø‡§ú‡•Å‡§É\u0022,\u0022url\u0022:\u0022..\/vedAH_yajuH\/\u0022},{\u0022title\u0022:\u0022‡§µ‡•á‡§¶‡§æ‡§É - ‡§∏‡§æ‡§Æ\u0022,\u0022url\u0022:\u0022..\/vedAH_sAma\/\u0022},{\u0022title\u0022:\u0022‡§™‡•Å‡§∞‡§æ‡§£‡§Æ‡•ç\u0022,\u0022url\u0022:\u0022..\/purANam\u0022},{\u0022title\u0022:\u0022‡§™‡•Å‡§∞‡§æ‡§£‡§Æ‡•ç ‡§µ‡•à‡§∑‡•ç‡§£‡§µ‡§Æ‡•ç\u0022,\u0022url\u0022:\u0022..\/purANam_vaiShNavam\/\u0022},{\u0022title\u0022:\u0022‡§Æ‡§π‡§æ‡§≠‡§æ‡§∞‡§§‡§Æ‡•ç\u0022,\u0022url\u0022:\u0022..\/mahAbhAratam\/\u0022},{\u0022title\u0022:\u0022‡§∞‡§æ‡§Æ‡§æ‡§Ø‡§£‡§Æ‡•ç\u0022,\u0022url\u0022:\u0022..\/rAmAyaNam\/\u0022},{\u0022title\u0022:\u0022‡§ï‡§æ‡§µ‡•ç‡§Ø‡§Æ‡•ç\u0022,\u0022url\u0022:\u0022..\/kAvyam\/\u0022},{\u0022title\u0022:\u0022‡§Æ‡•Ä‡§Æ‡§æ‡§Ç‡§∏‡§æ\u0022,\u0022url\u0022:\u0022..\/mImAMsA\/\u0022},{\u0022title\u0022:\u0022‡≤ï‡≤®‡≥ç‡≤®‡≤°\u0022,\u0022url\u0022:\u0022..\/kannaDa\/\u0022},{\u0022title\u0022:\u0022‡§§‡•ç‡§∞‡§ø‡§™‡§ø‡§ü‡§ï‡§Æ‡•ç\u0022,\u0022url\u0022:\u0022..\/tipiTaka\/\u0022},{\u0022title\u0022:\u0022‡§™‡§æ‡§≥‡§Ø‡§É\u0022,\u0022url\u0022:\u0022..\/pALi\/\u0022},{\u0022title\u0022:\u0022‡§≠‡§æ‡§∑‡§æ‡§®‡•ç‡§§‡§∞‡§Æ‡•ç\u0022,\u0022url\u0022:\u0022..\/bhAShAntaram\/\u0022},{\u0022title\u0022:\u0022English Lit\u0022,\u0022url\u0022:\u0022..\/english\/\u0022},{\u0022title\u0022:\u0022Book pub\u0022,\u0022url\u0022:\u0022..\/book-pub\u0022},{\u0022title\u0022:\u0022Vishvas\u0027s home page\u0022,\u0022url\u0022:\u0022..\/\u0022}],\u0022title\u0022:\u0022‡§∏‡§ô‡•ç‡§ó‡•ç‡§∞‡§π‡§æ‡§®‡•ç‡§§‡§∞‡§Æ‡•ç\u0022},{\u0022title\u0022:\u0022\\u003ci class=\\\u0022fas fa-hand-holding-heart\\\u0022\\u003e\\u003c\/i\\u003eDonate Via Vishvas\u0022,\u0022url\u0022:\u0022https:\/\/vishvAsa.github.io\/interests\/dharma-via-vishvas\/\u0022}],\u0022title\u0022:\u0022Parts\u0022}}");
    
    var pageRelUrlTree = {};
</script>

    
    <script type="text/javascript">
  let pageVars = {};
  pageVars.pageUrlMinusBasePath = "\/notes\/sapiens\/branches\/articles\/reich_who-we-are\/3_The_Disruptive_Genome\/10_The_Genomics_of_Inequality\/".replace(basePath, "/");
  pageVars.pageParams = {};
  pageVars.pageSource = "sapiens\/branches\/articles\/reich_who-we-are\/3_The_Disruptive_Genome\/10_The_Genomics_of_Inequality.md";
  console.log(pageVars.pageSource);
  var pageDefaults;
  for (let possiblePageDefaults of pageDefaultsList) {
    if (pageVars.pageSource.startsWith(possiblePageDefaults.scope.pathPrefix)) {
      pageDefaults = possiblePageDefaults.values
    }
  }
    
</script>


    <script src="/notes/webpack_dist/uiLib-bundle.js"></script>


<script>


MathJax = {
  tex: {
    inlineMath: [ ["@#", "#@"], ["\\(", "\\)"] ],
    displayMath: [ ["$$","$$"], ["\\[", "\\]"] ],
    processEscapes: true,  
  },
  svg: {
    fontCache: 'global'
  },
  options: {
    ignoreHtmlClass: "tex2jax_ignore|dno"
  }
};
</script>
<script type="text/javascript" id="MathJax-script" async
        src="/notes/non_webpack_js/mathjax/tex-svg.js">
</script>






    <link rel="stylesheet" href="/notes/webpack_dist/uiLib.css">
<link rel="stylesheet" href="/notes/css/uiLibPrint.css" media="print" >
<link rel="stylesheet" href="/notes/css/fonts.css" >
<link rel="stylesheet" href="/notes/css/@fortawesome/fontawesome-free/css/solid.min.css">

<link rel="stylesheet" href="/notes/css/@fortawesome/fontawesome-free/css/fontawesome.min.css">


<link rel="alternate" hreflang="sa-Deva" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/" />
<link rel="alternate" hreflang="sa-Deva" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=devanagari" />

<link rel="alternate" hreflang="sa-Knda" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=kannada" />
<link rel="alternate" hreflang="sa-Mlym" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=malayalam" />
<link rel="alternate" hreflang="sa-Telu" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=telugu" />
<link rel="alternate" hreflang="sa-Taml-t-sa-Taml-m0-superscript" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=tamil_superscripted" />
<link rel="alternate" hreflang="sa-Taml" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=tamil" />
<link rel="alternate" hreflang="sa-Gran" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=grantha" />

<link rel="alternate" hreflang="sa-Gujr" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=gujarati" />
<link rel="alternate" hreflang="sa-Guru" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=gurmukhi" />
<link rel="alternate" hreflang="sa-Shar" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=sharada" />
<link rel="alternate" hreflang="sa-Dogr" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=dogra" />
<link rel="alternate" hreflang="sa-Phag" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=phags_pa" />
<link rel="alternate" hreflang="sa-Tibt" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=tibetan" />
<link rel="alternate" hreflang="sa-Sidd" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=siddham" />
<link rel="alternate" hreflang="sa-Newa" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=newa" />
<link rel="alternate" hreflang="sa-Modi" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=modi" />
<link rel="alternate" hreflang="sa-Tirh" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=tirhuta_maithili" />
<link rel="alternate" hreflang="sa-Gong" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=gondi_gujala" />
<link rel="alternate" hreflang="sa-Gonm" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=gondi_masaram" />


<link rel="alternate" hreflang="sa-Orya" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=oriya" />
<link rel="alternate" hreflang="sa-Beng-t-sa-Beng-m0-assamese" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=assamese" />
<link rel="alternate" hreflang="sa-Beng" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=bengali" />
<link rel="alternate" hreflang="sa-Mtei" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=manipuri" />
<link rel="alternate" hreflang="sa-Sinh" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=sinhala" />
<link rel="alternate" hreflang="sa-Limb" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=limbu" />
<link rel="alternate" hreflang="sa-Lepc" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=lepcha" />


<link rel="alternate" hreflang="sa-Cyrl" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=cyrillic" />
<link rel="alternate" hreflang="sa-Latn-t-sa-Zyyy-m0-iso" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=iso" />
<link rel="alternate" hreflang="sa-Latn-t-sa-Zyyy-m0-iast" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=iast" />
<link rel="alternate" hreflang="sa-Aran" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=urdu" />
<link rel="alternate" hreflang="sa-Avst" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=avestan" />

<link rel="alternate" hreflang="sa-Brah" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=brahmi" />
<link rel="alternate" hreflang="sa-Khar" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=kharoshthi" />
<link rel="alternate" hreflang="sa-Zanb" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=zanbazar_square" />
<link rel="alternate" hreflang="sa-Bhks" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=bhaiksuki" />

<link rel="alternate" hreflang="sa-Bali" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=balinese" />
<link rel="alternate" hreflang="sa-Java" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=javanese" />
<link rel="alternate" hreflang="sa-Laoo" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=lao" />
<link rel="alternate" hreflang="sa-Laoo-t-sa-Laoo-m0-Laopali" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=lao_pali" />
<link rel="alternate" hreflang="sa-Mymr" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=burmese" />
<link rel="alternate" hreflang="sa-Mymr-t-sa-Mymr-m0-Mon" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=mon" />
<link rel="alternate" hreflang="sa-Cham" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=cham" />
<link rel="alternate" hreflang="sa-Khmr" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=khmer" />
<link rel="alternate" hreflang="sa-Thai" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/?transliteration_target=thai" />

  </head>

  <body class="ma0 bg-near-white">
    
\(
% groupings of objects.
\newcommand{\set}[1]{\left\{ #1 \right\}}
\newcommand{\seq}[1]{\left(#1\right)}
\newcommand{\ang}[1]{\langle#1\rangle}
\newcommand{\tuple}[1]{\left(#1\right)}
\newcommand{\size}[1]{\left| #1\right|}

\newcommand{\comp}{\circ}


% numerical shortcuts.
\newcommand{\abs}[1]{\left| #1\right|}
\newcommand{\floor}[1]{\left\lfloor #1 \right\rfloor}
\newcommand{\ceil}[1]{\left\lceil #1 \right\rceil}

% linear algebra shortcuts.
\newcommand{\change}{\Delta}
\newcommand{\norm}[1]{\left\| #1\right\|}
\newcommand{\dprod}[1]{\langle#1\rangle}
\newcommand{\linspan}[1]{\langle#1\rangle}
\newcommand{\conj}[1]{\overline{#1}}
\newcommand{\der}[1]{\frac{d#1}{dx}}
\newcommand{\lap}{\Delta}
\newcommand{\kron}{\otimes}
\newcommand{\nperp}{\nvdash}

\newcommand{\mat}[1]{\left[ \begin{smallmatrix}#1 \end{smallmatrix} \right]}

% derivatives and limits
\newcommand{\partder}[2]{\frac{\partial #1}{\partial #2}}
\newcommand{\partdern}[3]{\frac{\partial^{#3 #1}}{\partial #2^{#3}}}
\newcommand{\gradient}{\nabla}
\newcommand{\subdifferential}{\partial}

% Arrows
\newcommand{\diverge}{\nearrow}
\newcommand{\notto}{\nrightarrow}
\newcommand{\up}{\uparrow}
\newcommand{\down}{\downarrow}
% gets and gives are defined!

% ordering operators
\newcommand{\oleq}{\preceq}
\newcommand{\ogeq}{\succeq}

% programming and logic operators
\newcommand{\dfn}{:=}
\newcommand{\assign}{:=}
\newcommand{\co}{\ co\ }
\newcommand{\en}{\ en\ }


% logic operators
\newcommand{\xor}{\oplus}
\newcommand{\Land}{\bigwedge}
\newcommand{\Lor}{\bigvee}
\newcommand{\finish}{\Box}
\newcommand{\contra}{\Rightarrow \Leftarrow}
\newcommand{\iseq}{\stackrel{_?{=}}}


% Set theory
\newcommand{\symdiff}{\Delta}
\newcommand{\setdiff}{\backslash}
\newcommand{\union}{\cup}
\newcommand{\inters}{\cap}
\newcommand{\Union}{\bigcup}
\newcommand{\Inters}{\bigcap}
\newcommand{\nullSet}{\phi}


% graph theory
\newcommand{\nbd}{\Gamma}

% Script alphabets
% For reals, use \Re

% greek letters
\newcommand{\eps}{\epsilon}
\newcommand{\del}{\delta}
\newcommand{\ga}{\alpha}
\newcommand{\gb}{\beta}
\newcommand{\gd}{\del}
\newcommand{\gp}{\pi}
\newcommand{\gf}{\phi}
\newcommand{\gh}{\eta}
\newcommand{\gF}{\Phi}
\newcommand{\gl}{\lambda}
\newcommand{\gm}{\mu}
\newcommand{\gn}{\nu}
\newcommand{\gr}{\rho}
\newcommand{\gs}{\sigma}
\newcommand{\gth}{\theta}
\newcommand{\gx}{\xi}
\newcommand{\gw}{\omega}

\newcommand{\sw}{\sigma}
\newcommand{\SW}{\Sigma}
\newcommand{\ew}{\lambda}
\newcommand{\EW}{\Lambda}

\newcommand{\Del}{\Delta}
\newcommand{\gD}{\Delta}
\newcommand{\gG}{\Gamma}
\newcommand{\gW}{\Omega}
\newcommand{\gS}{\Sigma}
\newcommand{\gTh}{\Theta}

% Bold english letters.
\newcommand{\bA}{\mathbf{A}}
\newcommand{\bB}{\mathbf{B}}
\newcommand{\bC}{\mathbf{C}}
\newcommand{\bD}{\mathbf{D}}
\newcommand{\bE}{\mathbf{E}}
\newcommand{\bF}{\mathbf{F}}
\newcommand{\bG}{\mathbf{G}}
\newcommand{\bH}{\mathbf{H}}
\newcommand{\bI}{\mathbf{I}}
\newcommand{\bJ}{\mathbf{J}}
\newcommand{\bK}{\mathbf{K}}
\newcommand{\bL}{\mathbf{L}}
\newcommand{\bM}{\mathbf{M}}
\newcommand{\bN}{\mathbf{N}}
\newcommand{\bO}{\mathbf{O}}
\newcommand{\bP}{\mathbf{P}}
\newcommand{\bQ}{\mathbf{Q}}
\newcommand{\bR}{\mathbf{R}}
\newcommand{\bS}{\mathbf{S}}
\newcommand{\bT}{\mathbf{T}}
\newcommand{\bU}{\mathbf{U}}
\newcommand{\bV}{\mathbf{V}}
\newcommand{\bW}{\mathbf{W}}
\newcommand{\bX}{\mathbf{X}}
\newcommand{\bY}{\mathbf{Y}}
\newcommand{\bZ}{\mathbf{Z}}

\newcommand{\bba}{\mathbf{a}}
\newcommand{\bbb}{\mathbf{b}}
\newcommand{\bbc}{\mathbf{c}}
\newcommand{\bbd}{\mathbf{d}}
\newcommand{\bbe}{\mathbf{e}}
\newcommand{\bbf}{\mathbf{f}}
\newcommand{\bbg}{\mathbf{g}}
\newcommand{\bbh}{\mathbf{h}}
\newcommand{\bbk}{\mathbf{k}}
\newcommand{\bbl}{\mathbf{l}}
\newcommand{\bbm}{\mathbf{m}}
\newcommand{\bbn}{\mathbf{n}}
\newcommand{\bbp}{\mathbf{p}}
\newcommand{\bbq}{\mathbf{q}}
\newcommand{\bbr}{\mathbf{r}}
\newcommand{\bbs}{\mathbf{s}}
\newcommand{\bbt}{\mathbf{t}}
\newcommand{\bbu}{\mathbf{u}}
\newcommand{\bbv}{\mathbf{v}}
\newcommand{\bbw}{\mathbf{w}}
\newcommand{\bbx}{\mathbf{x}}
\newcommand{\bby}{\mathbf{y}}
\newcommand{\bbz}{\mathbf{z}}
\newcommand{\0}{\mathbf{0}}
\newcommand{\1}{\mathbf{1}}

% Caligraphic english alphabet
\newcommand{\cA}{\mathcal{A}}
\newcommand{\cB}{\mathcal{B}}
\newcommand{\cC}{\mathcal{C}}
\newcommand{\cD}{\mathcal{D}}
\newcommand{\cE}{\mathcal{E}}
\newcommand{\cF}{\mathcal{F}}
\newcommand{\cG}{\mathcal{G}}
\newcommand{\cH}{\mathcal{H}}
\newcommand{\cI}{\mathcal{I}}
\newcommand{\cJ}{\mathcal{J}}
\newcommand{\cK}{\mathcal{K}}
\newcommand{\cL}{\mathcal{L}}
\newcommand{\cM}{\mathcal{M}}
\newcommand{\cN}{\mathcal{N}}
\newcommand{\cO}{\mathcal{O}}
\newcommand{\cP}{\mathcal{P}}
\newcommand{\cQ}{\mathcal{Q}}
\newcommand{\cR}{\mathcal{R}}
\newcommand{\cS}{\mathcal{S}}
\newcommand{\cT}{\mathcal{T}}
\newcommand{\cU}{\mathcal{U}}
\newcommand{\cV}{\mathcal{V}}
\newcommand{\cW}{\mathcal{W}}
\newcommand{\cX}{\mathcal{X}}
\newcommand{\cY}{\mathcal{Y}}
\newcommand{\cZ}{\mathcal{Z}}


% Formatting shortcuts
\newcommand{\red}[1]{\textcolor{red}{#1}}
\newcommand{\blue}[1]{\textcolor{blue}{#1}}
\newcommand{\htext}[2]{\texorpdfstring{#1}{#2}}

% Statistics
\newcommand{\distr}{\sim}
\newcommand{\stddev}{\sigma}
\newcommand{\covmatrix}{\Sigma}
\newcommand{\mean}{\mu}
\newcommand{\param}{\theta}
\newcommand{\gthEst}{\hat{\theta}}
\newcommand{\ftr}{\phi}
\newcommand{\est}[1]{\hat{#1}}

% General utility
\newcommand{\todo}[1]{\textbf{[TODO]}] \footnote{TODO: #1}}
\newcommand{\tbc}{[\textbf{Incomplete}]}
\newcommand{\chk}{[\textbf{Check}]}
\newcommand{\why}{[\textbf{Find proof}]}
\newcommand{\opt}[1]{\textit{#1}}

\newcommand{\experience}[1]{[\textbf{Personal Experience}]: #1 \blacktriangle}
\newcommand{\pf}[1]{[\textbf{Proof}]: #1 \Box}
\newcommand{\core}[1]{\textbf{Core Idea}: #1 \Arrowvert}
\newcommand{\example}[1]{\textbf{Example}: #1 \blacktriangle}
\newcommand{\error}[1]{\textbf{Error alert}: #1 \triangle}
\newcommand{\oprob}{[\textbf{OP}]: }


\renewcommand{\~}{\htext{$\sim$}{~}}

\DeclareMathOperator*{\argmin}{arg\,min}
\DeclareMathOperator*{\argmax}{arg\,max}


\)



    
<header class="p-1 bg-yellow noPrint">
    <nav role="navigation" class="col">
      <div id="div_top_bar" class="row">
        <div class="col-md-3 justify-content-start">
          <a  href="/" class="btn col border">
            <i class="fas fa-gopuram ">Vishvas&#39;s notes <br/> 10 The Genomics of Inequality</i>
          </a>
        </div>
        <div id="div_top_bar_right" class="col-md-auto d-flex justify-content-center">
          <button class="btn btn-light"  href="#transliterationDropdown" data-toggle="collapse">
            ‚úç
          </button>
          <select id="transliterationDropdown" size="1" class="collapse" onchange="module_uiLib.default.content.updateTransliteration()">
            <optgroup label="üåç">
              <option value="iso">ƒÅISO</option>
              <option value="iso_vedic">ƒÅi-ƒÅu</option>
              <option value="cyrillic">–ø—É</option>
              <option value="urdu">ŸÜÿ≥</option>
              <option value="??">üåç??</option>
            </optgroup>
            <optgroup label="üõïüß≠S">
              <option value="kannada">‡≤Ö</option>
              <option value="malayalam">‡¥Ö</option>
              <option value="telugu">‡∞ï</option>
              <option value="tamil_superscripted">‡Æï¬≤</option>
              <option value="tamil_extended">‡Æï</option>
              <option value="grantha">ëåÖ</option>
            </optgroup>
            <optgroup label="üõïüß≠N">
              <option value="devanagari" selected="">‡§∏</option>
              <option value="gujarati">‡™Ö</option>
              <option value="gurmukhi">‡®Ö</option>
            </optgroup>
            <optgroup label="üõïüß≠E">
              <option value="oriya">‡¨Ö</option>
              <option value="assamese">‡¶Ö‡¶∏</option>
              <option value="bengali">‡¶Ö</option>
            </optgroup>
            <optgroup label="üõïüè∫üìú">
              <option value="sharada">ëÜëëáÄëÜ∞</option>
              <option value="brahmi">ëÄÖ</option>
            </optgroup>
            <optgroup label="üõïüåã">
              <option value="balinese">·¨©·¨Æ·¨∂</option>
            </optgroup>
          </select>
          <div>
          <button class="btn btn-outline-secondary btn-sm"  href="#transliterationDropdownAlt" data-toggle="collapse">
          (‚úç)
          </button>
          <select id="transliterationDropdownAlt" size="1"  style="font-size:12px" class="bg-light-yellow collapse" onchange="module_uiLib.default.content.updateTransliteration()">
            <optgroup label="üåç">
              <option value="iso">ƒÅISO</option>
              <option value="iso_vedic">ƒÅi-ƒÅu</option>
              <option value="cyrillic">–ø—É</option>
              <option value="urdu">ŸÜÿ≥</option>
              <option value="??">üåç??</option>
            </optgroup>
            <optgroup label="üõïüß≠S">
              <option value="kannada">‡≤Ö</option>
              <option value="malayalam">‡¥Ö</option>
              <option value="telugu">‡∞ï</option>
              <option value="tamil_superscripted">‡Æï¬≤</option>
              <option value="tamil_extended">‡Æï</option>
              <option value="grantha">ëåÖ</option>
            </optgroup>
            <optgroup label="üõïüß≠N">
              <option value="devanagari" selected="">‡§∏</option>
              <option value="gujarati">‡™Ö</option>
              <option value="gurmukhi">‡®Ö</option>
            </optgroup>
            <optgroup label="üõïüß≠E">
              <option value="oriya">‡¨Ö</option>
              <option value="assamese">‡¶Ö‡¶∏</option>
              <option value="bengali">‡¶Ö</option>
            </optgroup>
            <optgroup label="üõïüè∫üìú">
              <option value="sharada">ëÜëëáÄëÜ∞</option>
              <option value="brahmi">ëÄÖ</option>
            </optgroup>
            <optgroup label="üõïüåã">
              <option value="balinese">·¨©·¨Æ·¨∂</option>
            </optgroup>
          </select>
          </div>
          <div class="row col d-flex  dropdown">
            <button class="btn btn-secondary-outline btn-sm dropdown-toggle" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
              A¬±
            </button>
            <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
            <a class="btn btn-secondary btn-sm" onclick="module_uiLib.changeTextSize(2)">A+</a>
            <a class="btn btn-secondary btn-sm" onclick="module_uiLib.changeTextSize(-2)">A-</a>
            </div>
          </div>
          <button class="btn btn-light btn-sm"  href="#commentStyleDropdown" data-toggle="collapse">
            üó®
          </button>
          <select id="commentStyleDropdown" class="collapse" size="1" onchange="module_uiLib.default.content.updateCommentStyleFromDropdown()">
            <option value="on">‚úî</option>
            <option value="hidden">‚úó</option>
          </select>
          <div><a id="expandAllButton" class="btn btn-secondary" onclick="module_uiLib.default.query.setParamsAndGo({'expandDetails': '.*'})">‚ÜïÔ∏èÔ∏è</a></div>
          <div id="printStyleDiv" class="row col d-flex  dropdown">
          <button class="btn btn-light btn-sm dropdown-toggle"  href="#printStyleDropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
          üñ®
          </button>
            <div class="dropdown-menu col" aria-labelledby="printStyleDropdown">
              <div class="row justify-content-center">
                <p>Cols:</p>
                <input type="number" min="1" max="10" value="2" step="1" id="tbColumns" style="width: 3ch;"  accesskey="c"/>
              </div>
              <div class="row justify-content-center">
                <p>A¬±:</p> <input type="text" id="tbBodyFontSize" value="0.4cm" style="width: 6ch;"  accesskey="f"></input>
              </div>
              <div class="row justify-content-center">
                <p>Incl:</p> <input checked type="checkbox" id="tbIncludeStyle" value="on"></input>
                <button id="btnPrintStyle" onclick="module_uiLib.default.content.updatePrintStyle()">üñ®
                </button>
              </div>
            </div>
        </div>
        </div>
        <div id="audioDiv" class="row col d-flex  dropdown">
          <button class="btn btn-secondary-outline btn-sm dropdown-toggle" type="button" id="audioDropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
            üéß
          </button>
          <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
          <a id="speakerButton" class="btn btn-outline-secondary btn-sm" onclick="module_uiLib.default.content.handleSpeakToggle()">‚èØ</a>
          <button id="speakerPauseButton" class="btn btn-outline-secondary btn-sm">‚è∏</button>
            <div id="audioOptionsDiv" class="row justify-content-center" >
              <div>
                <p class="lead">Vol</p>
                <input type="range" min="0" max="1" value="1" step="0.1" id="volume" />
                <span id="volume-label" class="ms-2">1</span>
              </div>
              <div class="mx-5">
                <p class="lead">Rate</p>
                <input type="range" min="0.1" max="10" value="1" id="rate" step="0.1" />
                <span id="rate-label" class="ms-2">1</span>
              </div>
              <div>
                <p class="lead">Pitch</p>
                <input type="range" min="0" max="2" value="1" step="0.1" id="pitch" />
                <span id="pitch-label" class="ms-2">1</span>
              </div>
            </div>
          </div>
        </div>
        <a href="https://vishvAsa.github.io/notes/search"  class="btn btn-light btn-sm">ü•Ωüî¶</a>
        <div class="row col  d-flex justify-content-center">
          <div><a id="previousPage" href="" class="btn btn-secondary">###<i class="fas fa-caret-left"></i></a></div>
          <div ><a id="nextPage" href="" class="btn btn-secondary"><i class="fas fa-caret-right"></i> ### </a></div>
        </div>
        <ul id="top-bar-right-custom" class="list-group list-group-horizontal">
        </ul>
      </div>
    </nav>
</header>

    
    <div class="row" name="contentRow">
      
      <aside class="col-md-3 card border noPrint " id="sidebar">
          <div id="fastSearch">
              <datalist id="pageDataList"></datalist>
              <input id="searchInput" placeholder="‡§∂‡•Ä‡§∞‡•ç‡§∑‡§ø‡§ï‡§æ‡§®‡•ç‡§µ‡§ø‡§∑‡•ç‡§Ø‡§§‡§æ‡§Æ‡•ç" list="pageDataList" onchange="module_uiLib.default.navigation.pageLoader()">
          </div>
          <details open  class="bg-light-gray" >
              <summary id="sidebarToggleLink" onclick="module_uiLib.default.navigation.sidebarToggleHandler()">
            Menu
              </summary>
              <ul id="sidebarBody" class="list pl2 p-0 bg-yellow">
              </ul>
          </details>
      </aside>
      <main class="col p-3" role="main" data-pagefind-body>
        
<header class='border d-flex justify-content-between'>
    <h1 id="10 The Genomics of Inequality">10 The Genomics of Inequality</h1>
    
    <a id="editLink" class="btn btn-primary noPrint"  href="https://github.com/vishvAsa/notes/edit/content/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality.md"><i class="fas fa-edit"></i></a>
    
</header>
<article>
  <aside id="toc_card" class="card border noPrint ">
    <details id="toc_header">
        <summary>What's in this page?</summary>
      <div id="toc_body" class="p-0">
        
        <ul id="toc_ul" class="list p-0">
        </ul>
      </div>
    </details>
    <div id="progressBarDiv" class="row d-flex justify-content-center">
      üêéüí™<progress id="progressLoading" value="0" max="100"> 32% </progress>
    </div>
  </aside>
  <div id="post_content">
  <p>The Great Mixing</p>
<p>The American melting pot began to swirl almost as soon as Christopher Columbus arrived in 1492. European colonists, their African slaves, and the indigenous Americans were from populations whose ancestors had been isolated from each other for tens of thousands of years. Within a few years of meeting they began mixing, founding new populations that today number in the hundreds of millions.</p>
<p>Mart√≠n Cort√©s ‚Äúel Mestizo‚Äù belonged to one of the first of those populations. He was born within four years of the start of the 1519 military campaign in which his father, Hern√°n Cort√©s, led just five hundred soldiers to overthrow the Aztec Empire that dominated Mexico. His mother, ‚ÄúLa Malinche,‚Äù was one of twenty female captives given over to the Spanish after a battle and she first served as an interpreter before becoming the mistress of Hern√°n Cort√©s. The Spanish quickly invented a term for the people of combined European and Native American ancestry who emerged from unions like this. ‚ÄúMestizo‚Äù comes from the Spanish word <em>mestizaje,</em> which in English means miscegenation‚Äîthe mixing of different ‚Äúracial‚Äù types. To maintain their status in the social hierarchy, the Spanish and Portuguese set up a <em>casta</em> system in which people of entirely European ancestry (especially those born in Europe) had the highest status, while people who had even some non-European ancestry had lower status.+++(4)+++ This system collapsed under the demographic inevitability of admixture; within a few centuries people of entirely European ancestry were either an extreme minority or gone, and it was no longer feasible to limit power to those with entirely European ancestry.+++(4)+++ Following the independence movements of the nineteenth and early twentieth centuries, mixed ancestry became a source of pride in South and Central America. In Mexico, it defines national identity.<sup id="fnref:1"><a href="#fn:1" class="footnote-ref" role="doc-noteref">1</a></sup></p>
<p>Migration of Africans to the Americas after 1492 occurred on a similar scale as migration of Europeans. All told, an estimated twelve million enslaved Africans were forced to make the journey, jammed into the holds of ships before being sold at auction.<sup id="fnref:2"><a href="#fn:2" class="footnote-ref" role="doc-noteref">2</a></sup> Slave traders from Spain, Portugal, France, Britain, and the United States made great fortunes by satisfying the colonialists‚Äô need for manual labor. African slaves worked in the silver mines of Peru and Mexico and raised crops such as sugarcane and eventually tobacco and cotton. Africans were less affected by Old World diseases than Native Americans and easier to exploit than indigenous people, as they were far from home and scattered among a population that did not speak their languages.+++(4)+++ Deprived of their cultural points of reference, slaves had little ability to organize or resist. Most were sold in South America or the Caribbean, where they were often worked to death. Around 5 to 10 percent were brought to what became the United States.+++(4)+++ Following the first recorded sale of slaves by Portuguese traders in 1526, the rate of importation into the New World increased, plateauing at around seventy-five thousand per year until the trans-Atlantic slave trade was outlawed‚Äîin the British colonies in 1807, in the United States in 1808, and in Brazil in 1850.+++(4)+++</p>
<p>Today there are hundreds of millions of people in the Americas with African ancestry, the largest numbers in Brazil, the Caribbean, and the United States. The mixing of three highly divergent populations in the Americas‚ÄîEuropeans, indigenous people, and sub-Saharan Africans‚Äîthat began almost five hundred years ago continues to this day. Even in the United States, where European Americans are still in the majority, African Americans and Latinos comprise around a third of the population.+++(4)+++ Nearly all individuals from these mixed populations derive large stretches of their genomes from ancestors who lived on different continents fewer than twenty generations ago. A small percentage of European Americans have large stretches of African or Native American DNA as well, the legacy of people who successfully ‚Äúpassed‚Äù themselves off into the white majority.<sup id="fnref:3"><a href="#fn:3" class="footnote-ref" role="doc-noteref">3</a></sup>+++(4)+++</p>
<p>A 1973 science-fiction novel, Piers Anthony‚Äôs <em>Race Against Time,</em> envisions a future in which the mixing of populations initiated by European colonialism reaches its inevitable conclusion, and by the year 2300 nearly all humans belong to a ‚ÄúStandard‚Äù population.<sup id="fnref:4"><a href="#fn:4" class="footnote-ref" role="doc-noteref">4</a></sup> In that year, only six unmixed people are left: one pair of ‚Äúpurebred Caucasians,‚Äù one pair of ‚Äúpurebred Africans,‚Äù and one pair of ‚Äúpurebred Chinese.‚Äù These ‚Äúpurebreds‚Äù are being raised in human zoos by foster parents and are being groomed to breed with the only remaining individual of similar ancestry to sustain humanity‚Äôs diversity, a diversity that is viewed by the ‚ÄúStandard‚Äù population as a resource of irreplaceable biological value on the verge of being lost. The premise of the novel is that the centuries after 1492 were a uniquely homogenizing time in the history of our species, a period of unprecedented mixing of previously separated populations enabled by transoceanic travel, which brought together groups whose ancestors had not been in contact with one another for tens or hundreds of thousands of years.</p>
<p>But this premise is mistaken. The genome revolution has shown that we are not living in particularly special times when viewed from the perspective of the great sweep of the human past. Mixtures of highly divergent groups have happened time and again, homogenizing populations just as divergent from one another as Europeans, Africans, and Native Americans.+++(4)+++ And in many of these great admixtures, a central theme has been the coupling of men with social power in one population and women from the other.+++(5)+++</p>
<h2 id="founding-fathers">Founding Fathers</h2>
<p>Not long after the Constitutional Convention of 1787, the man who would become the United States‚Äô third president, Thomas Jefferson, began a sexual relationship with his slave Sally Hemings. Jefferson owned a large plantation in the state of Virginia, where some 40 percent of the population was enslaved.<sup id="fnref:5"><a href="#fn:5" class="footnote-ref" role="doc-noteref">5</a></sup>+++(4)+++ Sally Hemings was an African American slave with three European grandparents. But her mother‚Äôs mother was a slave of African descent, and under Virginia law the status of a slave was maternally inherited. Jefferson and Hemings had six children together.<sup id="fnref:6"><a href="#fn:6" class="footnote-ref" role="doc-noteref">6</a></sup>+++(4)+++</p>
<p>The Jefferson-Hemings relationship has been disputed by some who have suggested that Jefferson‚Äîwho is America‚Äôs greatest Enlightenment thinker and the author of the U.S. Declaration of Independence‚Äîwould not have maintained an illegitimate family. However, a genetic study published in 1998 revealed a Y-chromosome match between the male-line descendants of Eston Hemings Jefferson, the youngest son of Sally Hemings, and the male-line descendants of Jefferson‚Äôs paternal uncle.<sup id="fnref:7"><a href="#fn:7" class="footnote-ref" role="doc-noteref">7</a></sup> The genetic findings could in theory be explained if a male relative of Jefferson was the father rather than Jefferson himself. But there is no historical evidence for this possibility, and there is a credible nineteenth-century account of the Hemings-Jefferson relationship from Madison Hemings, another son of Hemings. A study by the Thomas Jefferson Memorial Foundation in 2000 concluded that, with high probability, the story was true.<sup id="fnref:8"><a href="#fn:8" class="footnote-ref" role="doc-noteref">8</a></sup>+++(4)+++</p>
<p>According to the account of Madison Hemings, his mother had a chance at freedom because she joined Jefferson in France, where slavery was illegal, but she agreed to return as a slave to the United States with Jefferson under the condition that their children would eventually be set free. Hemings was thirty years younger than Jefferson, and in France, where she began her relationship with him between the ages of fourteen and sixteen, she was dependent on him. She was also the half sister of Jefferson‚Äôs wife, Martha Randolph, who had died of complications of childbirth several years earlier and whose father had a secret relationship with the mother of Sally Hemings.<sup id="fnref:9"><a href="#fn:9" class="footnote-ref" role="doc-noteref">9</a></sup></p>
<p>Historians have attempted to quantify how widespread families like these were in the United States. Mixed-ancestry unions were often unrecorded, and when they were, children were categorized in different ways by different states. Genetics can help here. Although so far no one has analyzed DNA from African American graveyards to chart the emergence of a mixed-ancestry community in the United States, genetic studies of the present-day African American population are already enriching our understanding. Mark Shriver led a 2001 study that analyzed mutations that are extremely different in frequency between present-day Europeans and West Africans in order to study the African American populations of South Carolina. Shriver and his colleagues used these results to estimate the proportion of ancestors who lived in Europe a few dozen generations earlier.<sup id="fnref:10"><a href="#fn:10" class="footnote-ref" role="doc-noteref">10</a></sup> The highest proportion, around 18 percent, is found in the inland state capital, Columbia, a percentage at the low end of the range of cities in other U.S. states. They estimated about 12 percent European ancestry along the South Carolina coast, including in the slave port of Charleston, which they thought might reflect waves of slave importation keeping the African ancestry high. They estimated the lowest proportion of European ancestry, around 4 percent, on the Sea Islands off the coast, reflecting the history of isolation of the slaves who settled there, an isolation attested to by the fact that the Sea Islanders are the only African Americans still speaking a language, Gullah, with an African-derived grammar.+++(4)+++ Comparison of Y-chromosome and mitochondrial DNA types that are highly different in frequency between African Americans and Europeans also shows that by far the majority of the European ancestry in these populations comes from males, the result of social inequality in which mixed-race couplings were primarily between free males and female slaves.<sup id="fnref:11"><a href="#fn:11" class="footnote-ref" role="doc-noteref">11</a></sup>+++(4)+++</p>
<p>The patterns in South Carolina are a microcosm of those in the United States as a whole. Katarzyna Bryc, at the personal ancestry testing company 23andMe, worked with me to analyze more than five thousand self-described African Americans in the company database, and found that the average European ancestry proportion was 27 percent in most of the genome but only 23 percent on chromosome X.<sup id="fnref:12"><a href="#fn:12" class="footnote-ref" role="doc-noteref">12</a></sup>+++(4)+++ Comparing proportions of ancestry on chromosome X and the other chromosomes can provide information about differences in male and female behavior during population mixture, because two-thirds of X chromosomes in the world are carried in females compared to only about half of all other chromosomes, so the X chromosome is relatively more influenced by female history. By computing the proportion of European male and female ancestors that would be necessary to produce the observed difference in European ancestry between chromosome X and the autosomes, Bryc was able to estimate the separate male (38 percent) and female (10 percent) proportion of European ancestors in African Americans.+++(4)+++ These numbers imply that the contribution of European American men to the genetic makeup of the present-day African American population is about four times that of European American women. When I discussed these findings with the sociologist Orlando Patterson, he pointed out that the fraction of the European ancestry in African Americans that came from males‚Äîwhich if different from half is called ‚Äúsex bias‚Äù‚Äîmust have been far greater during the time of slavery. Since the civil rights movement in the United States in the mid-twentieth century, cultural changes have caused the sex bias to reverse, with more coupling between black men and white women. If we carried out DNA studies of African American skeletons from a hundred years ago, there is every reason to expect an even greater sex bias.+++(4)+++</p>
<p>The genetic patterns suggest that the Thomas Jefferson‚ÄìSally Hemings model was replicated countless times by other couples. While this story is one we know about because it is close to us in time and involved famous people, there is every reason to think that sex bias has been central to the history of our species. The genome revolution makes it possible to measure sex bias dating to periods for which we have no records, and thus to begin to understand how inequality may have shaped humanity in deep time.</p>
<h2 id="the-genomic-signature-of-inequality">The Genomic Signature of Inequality</h2>
<p>In humans, the profound biological differences that exist between the sexes mean that a single male is physically capable of having far more children than is a single female. Women carry unborn children for nine months and often nurse them for several years prior to having additional children.<sup id="fnref:13"><a href="#fn:13" class="footnote-ref" role="doc-noteref">13</a></sup> Men, meanwhile, are able to procreate while investing far less time in the bearing and early rearing of each child, a biological difference whose effects are amplified by social factors such as the fact that in many societies, men are expected to spend little time with their children. So it is that, as measured by the contribution to the next generation, powerful men have the potential to have a far greater impact than powerful women, and we can see this in genetic data.+++(4)+++</p>
<p>The great variability among males in the number of offspring produced means that by searching for genomic signatures of past variability in the number of children men have had, we can obtain genetic insights into the degree of social inequality in society as a whole, and not just between males and females. An extraordinary example of this is provided by the inequality in the number of male offspring that seems to have characterized the empire established by Genghis Khan, who ruled lands stretching from China to the Caspian Sea. After his death in 1227, his successors, including several of his sons and grandsons, extended the Mongol Empire even farther‚Äîto Korea in the east, to central Europe in the west, and to Tibet in the south. The Mongols maintained rested horses at strategically spaced posts, allowing rapid communication across their more than eight-thousand-kilometer span of territory. The united Mongol Empire was short-lived‚Äîfor example, the Y√ºan dynasty they established in China fell in 1368‚Äîbut their rise to power nevertheless allowed them to leave an extraordinary genetic impact on Eurasia.<sup id="fnref:14"><a href="#fn:14" class="footnote-ref" role="doc-noteref">14</a></sup></p>
<p>A 2003 study led by Christopher Tyler-Smith showed how a relatively small number of powerful males living during the Mongol period succeeded in having an outsize impact on the billions of people living in East Eurasia today.<sup id="fnref:15"><a href="#fn:15" class="footnote-ref" role="doc-noteref">15</a></sup> His study of Y chromosomes suggested that one single male who lived around the time of the Mongols left many millions of direct male-line descendants across the territory that the Mongols occupied. The evidence is that about 8 percent of males in the lands that the Mongol Empire once occupied share a characteristic Y-chromosome sequence or one differing from it by just a few mutations.+++(5)+++ Tyler-Smith and his colleagues called this a ‚ÄúStar Cluster‚Äù to reflect the idea of a single ancestor with many descendants, and estimated the date of the founder of this lineage to be thirteen hundred to seven hundred years ago based on the estimated rate of accumulation of mutations on the Y chromosome. The date coincides with that of Genghis Khan, suggesting that this single successful Y chromosome may have been his.</p>
<p>Star Clusters are not limited to Asia. The geneticist Daniel Bradley and his colleagues identified a Y-chromosome type that is present in two to three million people today and derives from an ancestor who lived around fifteen hundred years ago.<sup id="fnref:16"><a href="#fn:16" class="footnote-ref" role="doc-noteref">16</a></sup> It is especially common in people with the last name O‚ÄôDonnell, who descend from one of the most powerful royal families of medieval Ireland, the ‚ÄúDescendants of Niall‚Äù‚Äîreferring to Niall of the Nine Hostages, a legendary warlord from the earliest period of medieval Irish history.+++(4)+++ If Niall actually existed, he would have lived at about the right time to match the Y-chromosome ancestor.</p>
<p>Star Clusters capture the imagination because they can be tied, albeit speculatively, to historical figures. But the more important point is that Star Cluster analysis provides insights about shifts in social structure that occurred in the deep past that are difficult to get information about in other ways. This is therefore one area in which Y-chromosome and mitochondrial DNA analysis can be instructive, even without whole-genome data. For example, a perennial debate among historians is the extent to which the human past is shaped by single individuals whose actions leave a disproportionate impact on subsequent generations. Star Cluster analysis provides objective information about the importance of extreme inequalities in power at different points in the past.</p>
<p>Two studies, one led by Toomas Kivisild and the other led by Mark Stoneking, have compared the results of Star Cluster analysis on Y-chromosome sequences and on mitochondrial DNA sequences and arrived at an extraordinary result.<sup id="fnref:17"><a href="#fn:17" class="footnote-ref" role="doc-noteref">17</a></sup> By counting the number of differences per DNA letter between pairs of sequences, which reflects mutations that accumulated in a clocklike way over time, these studies estimated the time since different pairs of individuals shared common ancestors on the entirely male (Y-chromosome) and entirely female (mitochondrial DNA) lineages.</p>
<p>In mitochondrial DNA data, all the studies found that most couples living in a population today have a very low probability of sharing a common ancestor along their entirely female line in the last ten thousand years, a period postdating the transition to agriculture in many parts of the world.+++(4)+++ This is exactly as expected if population sizes were large throughout this period. But on the Y chromosome, the studies found a pattern that was strikingly different. In East Asians, Europeans, Near Easterners, and North Africans, the authors found many Star Clusters with common male ancestors living roughly around five thousand years ago.<sup id="fnref:18"><a href="#fn:18" class="footnote-ref" role="doc-noteref">18</a></sup>+++(4)+++</p>
<p>The time around five thousand years ago coincides with the period in Eurasia that the archaeologist Andrew Sherratt called the ‚ÄúSecondary Products Revolution,‚Äù in which people began to find many uses for domesticated animals beyond meat production, including employing them to pull carts and plows and to produce dairy products and clothing such as wool.<sup id="fnref:19"><a href="#fn:19" class="footnote-ref" role="doc-noteref">19</a></sup> This was also around the time of the onset of the Bronze Age, a period of greatly increased human mobility and wealth accumulation, facilitated by the domestication of the horse, the invention of the wheel and wheeled vehicles, and the accumulation of rare metals like copper and tin, which are the ingredients of bronze and had to be imported from hundreds or even thousands of kilometers away. The Y-chromosome patterns reveal that this was also a time of greatly increased inequality, a genetic reflection of the unprecedented concentration of power in tiny fractions of the population that began to be possible during this time due to the new economy. Powerful males in this period left an extraordinary impact on the populations that followed them‚Äîmore than in any previous period‚Äîwith some bequeathing DNA to more descendants today than Genghis Khan.</p>
<p>From ancient DNA combined with archaeology, we are beginning to build a picture of what this inequality might have meant. The period around five thousand years ago north of the Black and Caspian seas corresponds to the rise of the Yamnaya, who, as discussed in part II, took advantage of horses and wheels to exploit the resources of the open steppe for the first time.<sup id="fnref:20"><a href="#fn:20" class="footnote-ref" role="doc-noteref">20</a></sup> The genetic data show that the Yamnaya and their descendants were extraordinarily successful, largely displacing the farmers of northern Europe in the west and the hunter-gatherers of central Asia in the east.<sup id="fnref:21"><a href="#fn:21" class="footnote-ref" role="doc-noteref">21</a></sup></p>
<p>The archaeologist Marija Gimbutas has argued that Yamnaya society was unprecedentedly sex-biased and stratified. The Yamnaya left behind great mounds, about 80 percent of which had male skeletons at the center, often with evidence of violent injuries and buried amidst fearsome metal daggers and axes.<sup id="fnref:22"><a href="#fn:22" class="footnote-ref" role="doc-noteref">22</a></sup>+++(4)+++ Gimbutas argued that the arrival of the Yamnaya in Europe heralded a shift in the power relationships between the sexes. It coincided with the decline of ‚ÄúOld Europe,‚Äù which according to Gimbutas was a society with little evidence of violence, and in which females played a central social role as is apparent in the ubiquitous Venus figurines. In her reconstruction, ‚ÄúOld Europe‚Äù was replaced by a male-centered society, evident not only in the archaeology but also in the male-centered Greek, Norse, and Hindu mythologies of the Indo-European cultures plausibly spread by the Yamnaya.<sup id="fnref:23"><a href="#fn:23" class="footnote-ref" role="doc-noteref">23</a></sup></p>
<figure class="">

    <a href="../images/10_28a_mtDNA-history.jpg" target="_blank" rel="noopener">
        <img src="../images/10_28a_mtDNA-history.jpg" alt="Figure 28a" title="">
    </a>

    

</figure>
<p>Figure 28a. Human populations have expanded dramatically in the last fifty thousand years. We can see this in trees of relationships constructed based on mitochondrial DNA, where the rarity of recent shared ancestors in this period reflects the large size.</p>
<p>Any attempt to paint a vivid picture of what a human culture was like before the period of written texts needs to be viewed with caution. Nevertheless, ancient DNA data have provided evidence that the Yamnaya were indeed a society in which power was concentrated among a small number of elite males. The Y chromosomes that the Yamnaya carried were nearly all of a few types, which shows that a limited number of males must have been extraordinarily successful in spreading their genes.+++(4)+++ In contrast, in their mitochondrial DNA, the Yamnaya had more diverse sequences.<sup id="fnref:24"><a href="#fn:24" class="footnote-ref" role="doc-noteref">24</a></sup>+++(4)+++ The descendants of the Yamnaya or their close relatives spread their Y chromosomes into Europe and India, and the demographic impact of this expansion was profound, as the Y-chromosome types they carried were absent in Europe and India before the Bronze Age but are predominant in both places today.<sup id="fnref:25"><a href="#fn:25" class="footnote-ref" role="doc-noteref">25</a></sup>+++(4)+++</p>
<figure class="">

    <a href="../images/10_28b_Y-history.jpg" target="_blank" rel="noopener">
        <img src="../images/10_28b_Y-history.jpg" alt="Figure 28b" title="">
    </a>

    

</figure>
<p>Figure 28b. On the Y chromosome, many people shared ancestors around five thousand years ago. This corresponds to the dawn of the Bronze Age‚Äîa period of the first highly socially stratified societies‚Äîwhen some males succeeded in accumulating wealth and making an extraordinary contribution to the next generation.</p>
<p>This Yamnaya expansion also cannot have been entirely friendly, as is clear from the fact that the proportion of Y chromosomes of steppe origin in both western Europe<sup id="fnref:26"><a href="#fn:26" class="footnote-ref" role="doc-noteref">26</a></sup> and in India<sup id="fnref:27"><a href="#fn:27" class="footnote-ref" role="doc-noteref">27</a></sup> today is much larger than the proportion of steppe ancestry in the rest of the genome. This preponderance of male ancestry coming from the steppe implies that male descendants of the Yamnaya with political or social power were more successful at competing for local mates than men from the local groups. The most striking example I know of is from Iberia in far southwestern Europe, where Yamnaya-derived ancestry arrived at the onset of the Bronze Age between forty-five hundred and four thousand years ago. Daniel Bradley‚Äôs laboratory and my laboratory independently produced ancient DNA from individuals of this period.<sup id="fnref:28"><a href="#fn:28" class="footnote-ref" role="doc-noteref">28</a></sup> We found that approximately 30 percent of the Iberian population was replaced along with the arrival of steppe ancestry. However, the replacement of Y chromosomes was much more dramatic: in our data around 90 percent of males who carry Yamnaya ancestry have a Y-chromosome type of steppe origin that was absent in Iberia prior to that time.+++(5)+++ It is clear that there were extraordinary hierarchies and imbalances in power at work in the expansions from the steppe.</p>
<p>The Star Cluster work rests on Y chromosomes and mitochondrial DNA. What can whole-genome analysis add? When whole-genome data are used to reconstruct the size of the ancestral population of most agricultural groups in the last ten thousand years, they document population growth throughout this period, with no evidence of the Bronze Age population bottlenecks detected from Y chromosomes.<sup id="fnref:29"><a href="#fn:29" class="footnote-ref" role="doc-noteref">29</a></sup> This is not what one would expect from averaging the mitochondrial DNA and Y chromosomes. Instead, it is clear that the Y chromosome was a nonrepresentative part of the genome where certain genetic types were more successful at being passed down to later generations than others. In principle, one possible explanation for this is natural selection, whereby some Y chromosomes gave a biological advantage to those who carried them, such as increased fertility. But the fact that this genetic pattern manifested itself around the same time in multiple places around the world‚Äîin a period coinciding with the rise of socially stratified societies‚Äîis too striking a pattern to be explained by natural selection at multiple independently occurring advantageous mutations. I think a more plausible explanation is that in this period, it began to be possible for single males to accumulate so much power that they could not only gain access to large numbers of females, but they could also pass on their social prestige to subsequent generations and ensure that their male descendants were similarly successful.+++(5)+++ This process caused the Y chromosomes these males carried to increase in frequency generation after generation, leaving a genetic scar that speaks volumes about past societies.</p>
<p>It is also possible that in this period, individual women began to accumulate more power than they ever had before. Yet because it is biologically impossible for a woman, even a very powerful one, to have an extremely large number of children, the genetic effects of social inequality are much easier to detect on the male line.+++(4)+++</p>
<h2 id="sex-bias-in-population-mixture">Sex Bias in Population Mixture</h2>
<p>There are many ways that populations come together‚Äîfor example through invasions, migrations into each other‚Äôs homelands, demographic expansion into the same territory, and trade and cultural exchange. Potentially, populations could mix as equals‚Äîfor example through the overlapping of two equally resourced populations moving peaceably into the same area. But much more often there is asymmetry in the relationship, as reflected in mixture involving males from one group and females from the other, as occurred in the history of African Americans and in the history of the Yamnaya. The different histories of men and women recorded in different parts of the genome make it possible to study this mixture, and thereby to obtain clues about cultural interactions that occurred long ago.</p>
<p>Some of the examples of sex bias evident from genetic data are truly ancient. Take for example the founding of the ancestral population of non-Africans. Any genetic analysis of non-Africans reveals evidence of a population bottleneck dating to some time before fifty thousand years ago‚Äîthat is, a small number of individuals giving rise to many descendants today.+++(4)+++ In 2009, I worked with Alon Keinan, a postdoctoral scientist in my laboratory, to compare genetic variation on the X chromosome, the larger of our two sex chromosomes, to the rest of the genome. To our surprise, we found much less genetic variation in non-Africans on chromosome X than would be expected from the level of variation in the rest of the genome, assuming that males and females participated equally in the founding of the ancestral population of non-Africans. The pattern was too extreme to be explained by a simple scenario of more men than women participating in the founding of the ancestral population of non-Africans. But we discovered that one scenario that could explain the pattern is that after the initial budding off of the ancestral population of non-Africans, there was genetic input into this population from males of other groups. Since males carry one copy of chromosome X for every two copies of other chromosomes, a process of repeated waves of male immigration would decrease X-chromosome diversity (meaning that there would be less genetic variation in the population) compared to the rest of the genome, producing the pattern we observed.<sup id="fnref:30"><a href="#fn:30" class="footnote-ref" role="doc-noteref">30</a></sup>+++(5)+++</p>
<p>This hypothesis gains some plausibility from what we know of the interaction of central African Pygmy hunter-gatherer populations with the Bantu-speaking agriculturalist populations that surround them. When the Bantu first expanded out of west-central Africa several thousand years ago, they had a profound influence on the indigenous rainforest hunter-gatherer populations they encountered, as is evident from the fact that today no Pygmies speak a non-Bantu language and all harbor substantial Bantu-related ancestry.+++(4)+++ Even today, the overwhelming pattern is that Bantu men mix with Pygmy women and the children are raised in Pygmy communities.<sup id="fnref:31"><a href="#fn:31" class="footnote-ref" role="doc-noteref">31</a></sup>+++(4)+++ The waves of Bantu-related gene flow into the Pygmy population are similar to the scenario that Keinan and I had suggested for the ancestral population of non-Africans. The genetic consequence of this anthropological pattern is reflected in Pygmies having a substantially reduced degree of genetic diversity on chromosome X compared to the expectation from the rest of the genome.<sup id="fnref:32"><a href="#fn:32" class="footnote-ref" role="doc-noteref">32</a></sup> Perhaps similar processes were at work in the shared history of non-Africans, explaining the reduced X-chromosome diversity relative to the rest of the genome in that case too.</p>
<p>Evidence of sex bias in the mixture of human populations is becoming commonplace. The male-biased European contribution to admixed populations in the Americas is stark in African Americans, but it is truly extraordinary in populations in South and Central America, reflecting stories like that of Hern√°n Cort√©s and La Malinche. Andr√©s Ruiz-Linares and colleagues have documented how in the Antioquia region of Colombia, which was relatively isolated between the sixteenth and nineteenth centuries, about 94 percent of the Y chromosomes are European in origin, whereas about 90 percent of the mitochondrial DNA sequences are of Native American origin.<sup id="fnref:33"><a href="#fn:33" class="footnote-ref" role="doc-noteref">33</a></sup> This reflects social selection against Native American men.+++(4)+++ Because nearly all the male ancestry comes from Europeans and nearly all the female ancestry comes from Native Americans, one might naively expect that the people of Antioquia today would derive about half their genome-wide ancestry from Europeans and half from Native Americans, but this is not the case. Instead, about 80 percent of Antioquian ancestry comes from Europeans.<sup id="fnref:34"><a href="#fn:34" class="footnote-ref" role="doc-noteref">34</a></sup>+++(5)+++ The explanation is that Antioquia was flooded by male migrants over many generations.+++(5)+++ The first European men to arrive mixed with Native American women. Additional European male migrants came later. Through repeated waves of male European migration, the proportion of European ancestry kept increasing everywhere in the genome except for mitochondrial DNA, because mitochondrial DNA is passed to the next generation entirely by females.</p>
<p>Massive sex bias in population mixture also occurred between four thousand and two thousand years ago during the formation of the present-day populations of India.<sup id="fnref:35"><a href="#fn:35" class="footnote-ref" role="doc-noteref">35</a></sup> As discussed in part II, endogamous groups in India with traditionally higher social status tend to have more West Eurasian‚Äìrelated ancestry than groups with traditionally lower social status,<sup id="fnref:36"><a href="#fn:36" class="footnote-ref" role="doc-noteref">36</a></sup> and the effect is highly sex-biased, as mitochondrial DNA tends to be largely of local origin, whereas a much higher proportion of Y-chromosome types have affinity to West Eurasians.<sup id="fnref:37"><a href="#fn:37" class="footnote-ref" role="doc-noteref">37</a></sup>+++(4)+++ This pattern plausibly reflects a history in which males of West Eurasian‚Äìrelated ancestry were more highly placed in the caste system and sometimes married lower-ranking females. It speaks to a dramatic coming together of socially unequal populations to form the present genetic structure of India.</p>
<p>DNA has the power to overturn expectations from other fields, though, and in this case it has also revealed a surprise about sex-biased mixture. Today, almost every Pacific island population harbors some of its ancestry from people of mainland East Asian origin. As described in part II, this ancestry derives from people whose ancestors originated on Taiwan island and who invented long-distance seafaring and used it to disperse their people, language, and genes. But almost every Pacific island population also harbors Papuan ancestry related to the indigenous hunter-gatherers of the island of New Guinea. Surprisingly, in light of the theme that males from an expanding population tend to mix with local females, initial studies of mitochondrial DNA and Y chromosomes showed that the mixed populations of the Pacific today derive most of their East Asian origin DNA not from male but from female ancestors.<sup id="fnref:38"><a href="#fn:38" class="footnote-ref" role="doc-noteref">38</a></sup>+++(4)+++</p>
<p>One explanation that has been suggested for this pattern is that in early Pacific island societies, property usually passed down the female line and males were the primary people who moved across islands.<sup id="fnref:39"><a href="#fn:39" class="footnote-ref" role="doc-noteref">39</a></sup> But there is another process that may also have contributed. As described in part II, my laboratory showed that the first people of the open Pacific had little Papuan-related ancestry.<sup id="fnref:40"><a href="#fn:40" class="footnote-ref" role="doc-noteref">40</a></sup> We showed that later west-to-east waves of migration of people of mixed Papuan and mainland East Asian ancestry explain the ubiquity of Papuan ancestry in the remote Pacific today. If males from this later-arriving population had social advantages relative to the previously resident population, this could have resulted in newly arriving males of primarily Papuan ancestry mixing with previously established females of primarily East Asian‚Äìrelated ancestry.</p>
<p>The Pacific islander example highlights the importance of not simply assuming that genetic analyses of sex-biased events will fulfill expectations from anthropology. Now that the genome revolution has arrived, with its power to reject long-standing theories, we need to abandon the practice of approaching questions about the human past with strong expectations. To understand who we are, we need to approach the past with humility and with an open mind, and to be ready to change our minds out of respect for the power of hard data.</p>
<h2 id="the-future-of-genetic-studies-of-inequality">The Future of Genetic Studies of Inequality</h2>
<p>At the present time, our methods for using genetic data to study sex bias in human history are frustratingly primitive. Many of the most interesting findings about sex bias so far have been based on just two locations in the genome, the Y chromosome and mitochondrial DNA, which reflect only tiny fractions of our family trees. Studies of sex-biased population dynamics using these sections of the genome become nearly useless for understanding events that occurred more than around ten thousand years ago, because at that time depth, everyone in the world descends from only a small handful of male and female ancestors who are too few in number to support a statistically precise measurement of sex bias.</p>
<p>Future studies of sex-biased mixture, though, will take full advantage of the power of the whole genome. Whole-genome studies can compare the thousands of independent genealogies recorded on chromosome X to the tens of thousands of independent genealogies carried in the rest of the genome. Comparison of genetic variation on the X chromosome to genetic variation elsewhere in the genome should in theory give far more statistical resolution, but while some studies of this sort have been revealing, the accuracy of their estimates has so far been disappointing, which may be because of intense bouts of natural selection that have affected chromosome X more than other chromosomes and that make interpretations of its patterns more difficult. So while many major mixture events‚Äîsuch as those of steppe pastoralists and farmers in Europe, or of Neanderthals and Denisovans with modern humans further back in human prehistory‚Äîmay well have been sex-biased, detecting them by comparing fractions of ancestry on chromosome X to the rest of the genome is currently challenging.<sup id="fnref:41"><a href="#fn:41" class="footnote-ref" role="doc-noteref">41</a></sup> But our present problems with making precise estimates of sex bias based on the X chromosome are to a large extent technical, reflecting the limitations of the statistical techniques currently available. New methods that will be developed over the coming years will unleash the full power of comparison of the X chromosome to the rest of the genome. I hope that these improved methods, along with direct ancient DNA data from people who lived at the times mixture was happening, will enable new genomic insights into the nature of inequality in the deep human past.</p>
<p>The genomic evidence of the antiquity of inequality‚Äîbetween men and women, and between people of the same sex but with greater and lesser power‚Äîis sobering in light of the undeniable persistence of inequality today.+++(4)+++ One possible response might be to conclude that inequality is part of human nature and that we should just accept it. But I think the lesson is just the opposite. Constant effort to struggle against our demons‚Äîagainst the social and behavioral habits that are built into our biology‚Äîis one of the ennobling behaviors of which we humans as a species are capable, and which has been critical to many of our triumphs and achievements. Evidence of the antiquity of inequality should motivate us to deal in a more sophisticated way with it today, and to behave a little better in our own time.</p>
<div class="footnotes" role="doc-endnotes">
<hr>
<ol>
<li id="fn:1">
<p>Peter Wade, <em>Race and Ethnicity in Latin America</em> (London and New York: Pluto Press, 2010).&#160;<a href="#fnref:1" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:2">
<p>Trans-Atlantic Slave Trade Database, <a href="http://www.slavevoyages.org/assessment/estimates">www.slavevoyages.org/assessment/estimates</a>.&#160;<a href="#fnref:2" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:3">
<p>K. Bryc et al., ‚ÄúThe Genetic Ancestry of African Americans, Latinos, and European Americans Across the United States,‚Äù <em>American Journal of Human Genetics</em> 96 (2015): 37‚Äì53.&#160;<a href="#fnref:3" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:4">
<p>Piers Anthony, <em>Race Against Time</em> (New York: Hawthorn Books, 1973).&#160;<a href="#fnref:4" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:5">
<p>The first federal census in 1790 recorded 292,627 male slaves in Virginia out of a total male population of 747,610; available online at <a href="http://www.nationalgeographic.org/media/us-census-1790/">www.nationalgeographic.org/media/us-census-1790/</a>.&#160;<a href="#fnref:5" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:6">
<p>Joshua D. Rothman, <em>Notorious in the Neighborhood: Sex and Families Across the Color Line in Virginia, 1787‚Äì1861</em> (Chapel Hill: University of North Carolina Press, 2003).&#160;<a href="#fnref:6" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:7">
<p>E. A. Foster et al., ‚ÄúJefferson Fathered Slave‚Äôs Last Child,‚Äù <em>Nature</em> 396 (1998): 27‚Äì28.&#160;<a href="#fnref:7" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:8">
<p>‚ÄúStatement on the TJMF Research Committee Report on Thomas Jefferson and Sally Hemings,‚Äù January 26, 2000, available online at <a href="https://www.monticello.org/sites/default/files/inline-pdfs/jefferson-hemings_report.pdf">https://www.monticello.org/sites/default/files/inline-pdfs/jefferson-hemings_report.pdf</a>.&#160;<a href="#fnref:8" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:9">
<p>M. Hemings, ‚ÄúLife Among the Lowly, No. 1,‚Äù <em>Pike County (Ohio) Republican,</em> March 13, 1873.&#160;<a href="#fnref:9" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:10">
<p>E. J. Parra et al., ‚ÄúAncestral Proportions and Admixture Dynamics in Geographically Defined African Americans Living in South Carolina,‚Äù <em>American Journal of Physical Anthropology</em> 114 (2001): 18‚Äì29.&#160;<a href="#fnref:10" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:11">
<p>Ibid.&#160;<a href="#fnref:11" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:12">
<p>Bryc et al., ‚ÄúGenetic Ancestry.‚Äù&#160;<a href="#fnref:12" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:13">
<p>J. N. Fenner, ‚ÄúCross-Cultural Estimation of the Human Generation Interval for Use in Genetics-Based Population Divergence Studies,‚Äù <em>American Journal of Physical Anthropology</em> 128 (2005): 415‚Äì23.&#160;<a href="#fnref:13" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:14">
<p>David Morgan, <em>The Mongols</em> (Malden, MA, and Oxford: Blackwell, 2007).&#160;<a href="#fnref:14" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:15">
<p>T. Zerjal et al., ‚ÄúThe Genetic Legacy of the Mongols,‚Äù <em>American Journal of Human Genetics</em> 72 (2003): 717‚Äì21.&#160;<a href="#fnref:15" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:16">
<p>L. T. Moore et al., ‚ÄúA Y-Chromosome Signature of Hegemony in Gaelic Ireland,‚Äù <em>American Journal of Human Genetics</em> 78 (2006): 334‚Äì38.&#160;<a href="#fnref:16" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:17">
<p>S. Lippold et al., ‚ÄúHuman Paternal and Maternal Demographic Histories: Insights from High-Resolution Y Chromosome and mtDNA Sequences,‚Äù <em>Investigative Genetics</em> 5 (2014): 13; M. Karmin et al., ‚ÄúA Recent Bottleneck of Y Chromosome Diversity Coincides with a Global Change in Culture,‚Äù <em>Genome Research</em> 25 (2015): 459‚Äì66.&#160;<a href="#fnref:17" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:18">
<p>Ibid.&#160;<a href="#fnref:18" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:19">
<p>A. Sherratt, ‚ÄúPlough and Pastoralism: Aspects of the Secondary Products Revolution,‚Äù in <em>Pattern of the Past: Studies in Honour of David Clarke,</em> ed. Ian Hodder, Glynn Isaac, and Norman Hammond (Cambridge: Cambridge University Press, 1981), 261‚Äì306.&#160;<a href="#fnref:19" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:20">
<p>David W. Anthony, <em>The Horse, the Wheel, and Language: How Bronze-Age Riders from the Eurasian Steppes Shaped the Modern World</em> (Princeton, NJ: Princeton University Press, 2007).&#160;<a href="#fnref:20" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:21">
<p>W. Haak et al., ‚ÄúMassive Migration from the Steppe Was a Source for Indo-European Languages in Europe,‚Äù <em>Nature</em> 522 (2015): 207‚Äì11; M. E. Allentoft et al., ‚ÄúPopulation Genomics of Bronze Age Eurasia,‚Äù <em>Nature</em> 522 (2015): 167‚Äì72.&#160;<a href="#fnref:21" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:22">
<p>E. Murphy and A. Khokhlov, ‚ÄúA Bioarchaeological Study of Prehistoric Populations from the Volga Region,‚Äù in <em>A Bronze Age Landscape in the Russian Steppes: The Samara Valley Project, Monumenta Archaeologica 37,</em> ed. David W. Anthony, Dorcas R. Brown, Aleksandr A. Khokhlov, Pavel V. Kuznetsov, and Oleg D. Mochalov (Los Angeles: Cotsen Institute of Archaeology Press, 2016), 149‚Äì216.&#160;<a href="#fnref:22" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:23">
<p>Marija Gimbutas, <em>The Prehistory of Eastern Europe, Part I: Mesolithic, Neolithic and Copper Age Cultures in Russia and the Baltic Area</em> (American School of Prehistoric Research, Harvard University, Bulletin No. 20) (Cambridge, MA: Peabody Museum, 1956).&#160;<a href="#fnref:23" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:24">
<p>Haak et al., ‚ÄúMassive Migration.‚Äù&#160;<a href="#fnref:24" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:25">
<p>R. S. Wells et al., ‚ÄúThe Eurasian Heartland: A Continental Perspective on Y-Chromosome Diversity,‚Äù <em>Proceedings of the National Academy of Sciences of the U.S.A.</em> 98 (2001): 10244‚Äì49.&#160;<a href="#fnref:25" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:26">
<p>R. Martiniano et al., ‚ÄúThe Population Genomics of Archaeological Transition in West Iberia: Investigation of Ancient Substructure Using Imputation and Haplotype-Based Methods,‚Äù <em>PLoS Genetics</em> 13 (2017): e1006852.&#160;<a href="#fnref:26" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:27">
<p>M. Silva et al., ‚ÄúA Genetic Chronology for the Indian Subcontinent Points to Heavily Sex-Biased Dispersals,‚Äù <em>BMC Evolutionary Biology</em> 17 (2017): 88.&#160;<a href="#fnref:27" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:28">
<p>Martiniano et al., ‚ÄúWest Iberia‚Äù; unpublished results from David Reich‚Äôs laboratory.&#160;<a href="#fnref:28" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:29">
<p>J. A. Tennessen et al., ‚ÄúEvolution and Functional Impact of Rare Coding Variation from Deep Sequencing of Human Exomes,‚Äù <em>Science</em> 337 (2012): 64‚Äì69.&#160;<a href="#fnref:29" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:30">
<p>A. Keinan, J. C. Mullikin, N. Patterson, and D. Reich, ‚ÄúAccelerated Genetic Drift on Chromosome X During the Human Dispersal out of Africa,‚Äù <em>Nature Genetics</em> 41 (2009): 66‚Äì70; A. Keinan and D. Reich, ‚ÄúCan a Sex-Biased Human Demography Account for the Reduced Effective Population Size of Chromosome X in Non-Africans?,‚Äù <em>Molecular Biology and Evolution</em> 27 (2010): 2312‚Äì21.&#160;<a href="#fnref:30" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:31">
<p>P. Verdu et al., ‚ÄúSociocultural Behavior, Sex-Biased Admixture, and Effective Population Sizes in Central African Pygmies and Non-Pygmies,‚Äù <em>Molecular Biology and Evolution</em> 30 (2013): 918‚Äì37.&#160;<a href="#fnref:31" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:32">
<p>S. Mallick et al., ‚ÄúThe Simons Genome Diversity Project: 300 Genomes from 142 Diverse Populations,‚Äù <em>Nature</em> 538 (2016): 201‚Äì6.&#160;<a href="#fnref:32" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:33">
<p>L. G. Carvajal -Carmona et al., ‚ÄúStrong Amerind/White Sex Bias and a Possible Sephardic Contribution Among the Founders of a Population in Northwest Colombia,‚Äù <em>American Journal of Human Genetics</em> 67 (2000): 1287‚Äì95.&#160;<a href="#fnref:33" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:34">
<p>Bedoya et al., ‚ÄúAdmixture Dynamics in Hispanics: A Shift in the Nuclear Genetic Ancestry of a South American Population Isolate,‚Äù <em>Proceedings of the National Academy of Sciences of the U.S.A.</em> 103 (2006): 7234‚Äì39.&#160;<a href="#fnref:34" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:35">
<p>P. Moorjani et al., ‚ÄúGenetic Evidence for Recent Population Mixture in India,‚Äù <em>American Journal of Human Genetics</em> 93 (2013): 422‚Äì38.&#160;<a href="#fnref:35" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:36">
<p>M. Bamshad et al., ‚ÄúGenetic Evidence on the Origins of Indian Caste Populations,‚Äù <em>Genome Research</em> 11 (2001): 994‚Äì1004; D. Reich et al., ‚ÄúReconstructing Indian Population History,‚Äù <em>Nature</em> 461 (2009): 489‚Äì94.&#160;<a href="#fnref:36" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:37">
<p>Bamshad et al., ‚ÄúGenetic Evidence‚Äù; I. Thanseem et al., ‚ÄúGenetic Affinities Among the Lower Castes and Tribal Groups of India: Inference from Y Chromosome and Mitochondrial DNA,‚Äù <em>BMC Genetics</em> 7 (2006): 42.&#160;<a href="#fnref:37" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:38">
<p>M. Kayser, ‚ÄúThe Human Genetic History of Oceania: Near and Remote Views of Dispersal,‚Äù <em>Current Biology</em> 20 (2010): R194‚Äì201; P. Skoglund et al., ‚ÄúGenomic Insights into the Peopling of the Southwest Pacific,‚Äù <em>Nature</em> 538 (2016): 510‚Äì13.&#160;<a href="#fnref:38" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:39">
<p>F. M. Jordan, R. D. Gray, S. J. Greenhill, and R. Mace, ‚ÄúMatrilocal Residence Is Ancestral in Austronesian Societies,‚Äù <em>Proceedings of the Royal Society B‚ÄîBiological Sciences</em> 276 (2009): 1957‚Äì64.&#160;<a href="#fnref:39" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:40">
<p>Skoglund et al., ‚ÄúGenomic Insights.‚Äù&#160;<a href="#fnref:40" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:41">
<p>I. Lazaridis and D. Reich, ‚ÄúFailure to Replicate a Genetic Signal for Sex Bias in the Steppe Migration into Central Europe,‚Äù <em>Proceedings of the National Academy of Sciences of the U.S.A.</em> 114 (2017): E3873‚Äì74.&#160;<a href="#fnref:41" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
</ol>
</div>

  </div>
</article>


        

        



    <aside class="card border noPrint" id="section-tree-item-sapiens_branches_articles_reich_who-we-are_3_The_Disruptive_Genome_10_The_Genomics_of_Inequality_md">
        <div class="card-title bg-light-gray border d-flex justify-content-between">
            <a href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/10_The_Genomics_of_Inequality/">10 The Genomics of Inequality </a>
            <a data-toggle="collapse" href="#section-tree-item-body-sapiens_branches_articles_reich_who-we-are_3_The_Disruptive_Genome_10_The_Genomics_of_Inequality_md" role="button" aria-expanded="false" aria-controls="section-tree-item-body-sapiens_branches_articles_reich_who-we-are_3_The_Disruptive_Genome_10_The_Genomics_of_Inequality_md" >‚Ä¶<i class="fas fa-caret-down" class="collapsed"></i> </a>
        </div>
        <nav id="section-tree-item-body-sapiens_branches_articles_reich_who-we-are_3_The_Disruptive_Genome_10_The_Genomics_of_Inequality_md" class="card-body p-0 collapse">
            
            <li>draft: false</li>
            
            <li>iscjklanguage: false</li>
            
            <li>title: 10 The Genomics of Inequality</li>
            
        </nav>
    </aside>


      </main>
    </div>
    <footer class="bg-yellow  p-1 noPrint" role="contentinfo">
  

  <div id="div_foot_nav_bar" class="row">
    <a href="https://vishvAsa.github.io/notes/search"  class="btn btn-light btn-sm">ü•Ωüî¶</a>
    <div class="row col  d-flex justify-content-center">
      <div><a id="previousPageFoot" href="" class="btn btn-secondary">###<i class="fas fa-caret-left"></i></a></div>
      <div ><a id="nextPageFoot" href="" class="btn btn-secondary"><i class="fas fa-caret-right"></i> ### </a></div>
    </div>
      <ul id="footer-bar-right-custom" class="list-group list-group-horizontal">
      </ul>
  </div>
  <div id="div_footer_bar" class="container-fluid d-flex justify-content-between">
    <a class="btn btn-secondary" href="https://github.com/vishvAsa/notes/issues/new" >
      ‚úâÔ∏èContact
    </a>
    <a class="btn btn-secondary" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/80x15.png" /></a>
    <div class="btn small border">
      
      <a href="https://github.com/vishvAsa/notes/actions/workflows/build.yml"><img src="https://github.com/vishvAsa/notes/actions/workflows/build.yml/badge.svg"> Built </a> on 2026 Feb 12 01:34:45 UTC. (<a href="http://google.com/search?q=01%3a34%3a45%20UTC to IST">IST</a>)
    </div>
  </div>
</footer>



<script async src="https://www.googletagmanager.com/gtag/js?id=G-33H08JSLQR"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-33H08JSLQR');
</script>

  </body>
  <script type='text/javascript'>
    
    
    
    module_uiLib.onDocumentReadyTasks();
  </script>
  
</html>
