<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
    
    <title>Vishvas&#39;s notes  | 11 The Genomics of Race and Identity</title>
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">

    <meta name="viewport" content="width=device-width,minimum-scale=1">
    <meta name="generator" content="Hugo 0.143.0">
    
    
      <META NAME="ROBOTS" CONTENT="INDEX, FOLLOW">
    

    <link href="/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/" rel="alternate" type="application/rss+xml" title="Vishvas&#39;s notes" />
    <link href="/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/" rel="feed" type="application/rss+xml" title="Vishvas&#39;s notes" />

    <meta property="og:url" content="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/">
  <meta property="og:site_name" content="Vishvas&#39;s notes">
  <meta property="og:title" content="11 The Genomics of Race and Identity">
  <meta property="og:description" content="Fear of Biological Difference When I started my first academic job in 2003, I bet my career on the idea that the history of mixture of West Africans and Europeans in the Americas would make it possible to find risk factors that contribute to health disparities for diseases like prostate cancer, which occurs at about a rate 1.7 times higher in African Americans than in European Americans.1 This particular disparity had not been possible to explain based on dietary and environmental differences across populations, suggesting that genetic factors might play a role.">
  <meta property="og:locale" content="en_us">
  <meta property="og:type" content="article">
    <meta property="article:section" content="sapiens">

  <meta itemprop="name" content="11 The Genomics of Race and Identity">
  <meta itemprop="description" content="Fear of Biological Difference When I started my first academic job in 2003, I bet my career on the idea that the history of mixture of West Africans and Europeans in the Americas would make it possible to find risk factors that contribute to health disparities for diseases like prostate cancer, which occurs at about a rate 1.7 times higher in African Americans than in European Americans.1 This particular disparity had not been possible to explain based on dietary and environmental differences across populations, suggesting that genetic factors might play a role.">
  <meta itemprop="wordCount" content="11753">
  <meta name="twitter:card" content="summary">
  <meta name="twitter:title" content="11 The Genomics of Race and Identity">
  <meta name="twitter:description" content="Fear of Biological Difference When I started my first academic job in 2003, I bet my career on the idea that the history of mixture of West Africans and Europeans in the Americas would make it possible to find risk factors that contribute to health disparities for diseases like prostate cancer, which occurs at about a rate 1.7 times higher in African Americans than in European Americans.1 This particular disparity had not been possible to explain based on dietary and environmental differences across populations, suggesting that genetic factors might play a role.">
<script type="text/javascript">
    
    let baseURL = "https:\/\/vishvAsa.github.io\/notes\/";
    let basePath = "/" + baseURL.split("/").slice(3).join("/");
    let siteParams = JSON.parse("{\u0022build_img\u0022:\u0022https:\/\/github.com\/vishvAsa\/notes\/actions\/workflows\/build.yml\/badge.svg\u0022,\u0022build_url\u0022:\u0022https:\/\/github.com\/vishvAsa\/notes\/actions\/workflows\/build.yml\u0022,\u0022contactlink\u0022:\u0022https:\/\/github.com\/vishvAsa\/notes\/issues\/new\u0022,\u0022env\u0022:\u0022production\u0022,\u0022githubeditmepathbase\u0022:\u0022https:\/\/github.com\/vishvAsa\/notes\/edit\/content\/\u0022,\u0022js_extras\u0022:[\u0022mathjax\u0022]}");
    let pageDefaultsList = JSON.parse("[{\u0022scope\u0022:{\u0022pathPrefix\u0022:\u0022\u0022},\u0022values\u0022:{\u0022comments\u0022:true,\u0022layout\u0022:\u0022page\u0022,\u0022search\u0022:true,\u0022sidebar\u0022:\u0022home_sidebar\u0022}}]");
    let sidebarsData = JSON.parse("{\u0022home_sidebar\u0022:{\u0022contents\u0022:[{\u0022url\u0022:\u0022recdir:\/\/\u0022},{\u0022contents\u0022:[{\u0022title\u0022:\u0022ज्यौतिषम्\u0022,\u0022url\u0022:\u0022..\/jyotiSham\/\u0022},{\u0022title\u0022:\u0022संस्कृतम्\u0022,\u0022url\u0022:\u0022..\/sanskrit\/\u0022},{\u0022title\u0022:\u0022देवः\u0022,\u0022url\u0022:\u0022..\/devaH\/\u0022},{\u0022title\u0022:\u0022आगमः\u0022,\u0022url\u0022:\u0022..\/AgamaH\/\u0022},{\u0022title\u0022:\u0022आगमः शैवः\u0022,\u0022url\u0022:\u0022..\/AgamaH_shaivaH\/\u0022},{\u0022title\u0022:\u0022आगमः वैष्णवः\u0022,\u0022url\u0022:\u0022..\/AgamaH_vaiShNavaH\/\u0022},{\u0022title\u0022:\u0022रामानुजीयम्\u0022,\u0022url\u0022:\u0022..\/rAmAnujIyam\/\u0022},{\u0022title\u0022:\u0022माध्वम्\u0022,\u0022url\u0022:\u0022..\/mAdhvam\/\u0022},{\u0022title\u0022:\u0022आगमः ब्राह्मः\u0022,\u0022url\u0022:\u0022..\/AgamaH_brAhmaH\/\u0022},{\u0022title\u0022:\u0022कल्पान्तरम्\u0022,\u0022url\u0022:\u0022..\/kalpAntaram\/\u0022},{\u0022title\u0022:\u0022Notes Home\u0022,\u0022url\u0022:\u0022..\/notes\/\u0022},{\u0022title\u0022:\u0022वेदाः\u0022,\u0022url\u0022:\u0022..\/vedAH\/\u0022},{\u0022title\u0022:\u0022वेदाः - ऋक्\u0022,\u0022url\u0022:\u0022..\/vedAH_Rk\/\u0022},{\u0022title\u0022:\u0022वेदाः - यजुः\u0022,\u0022url\u0022:\u0022..\/vedAH_yajuH\/\u0022},{\u0022title\u0022:\u0022वेदाः - साम\u0022,\u0022url\u0022:\u0022..\/vedAH_sAma\/\u0022},{\u0022title\u0022:\u0022पुराणम्\u0022,\u0022url\u0022:\u0022..\/purANam\u0022},{\u0022title\u0022:\u0022पुराणम् वैष्णवम्\u0022,\u0022url\u0022:\u0022..\/purANam_vaiShNavam\/\u0022},{\u0022title\u0022:\u0022महाभारतम्\u0022,\u0022url\u0022:\u0022..\/mahAbhAratam\/\u0022},{\u0022title\u0022:\u0022रामायणम्\u0022,\u0022url\u0022:\u0022..\/rAmAyaNam\/\u0022},{\u0022title\u0022:\u0022काव्यम्\u0022,\u0022url\u0022:\u0022..\/kAvyam\/\u0022},{\u0022title\u0022:\u0022मीमांसा\u0022,\u0022url\u0022:\u0022..\/mImAMsA\/\u0022},{\u0022title\u0022:\u0022ಕನ್ನಡ\u0022,\u0022url\u0022:\u0022..\/kannaDa\/\u0022},{\u0022title\u0022:\u0022त्रिपिटकम्\u0022,\u0022url\u0022:\u0022..\/tipiTaka\/\u0022},{\u0022title\u0022:\u0022पाळयः\u0022,\u0022url\u0022:\u0022..\/pALi\/\u0022},{\u0022title\u0022:\u0022भाषान्तरम्\u0022,\u0022url\u0022:\u0022..\/bhAShAntaram\/\u0022},{\u0022title\u0022:\u0022English Lit\u0022,\u0022url\u0022:\u0022..\/english\/\u0022},{\u0022title\u0022:\u0022Book pub\u0022,\u0022url\u0022:\u0022..\/book-pub\u0022},{\u0022title\u0022:\u0022Vishvas\u0027s home page\u0022,\u0022url\u0022:\u0022..\/\u0022}],\u0022title\u0022:\u0022सङ्ग्रहान्तरम्\u0022},{\u0022title\u0022:\u0022\\u003ci class=\\\u0022fas fa-hand-holding-heart\\\u0022\\u003e\\u003c\/i\\u003eDonate Via Vishvas\u0022,\u0022url\u0022:\u0022https:\/\/vishvAsa.github.io\/interests\/dharma-via-vishvas\/\u0022}],\u0022title\u0022:\u0022Parts\u0022}}");
    
    var pageRelUrlTree = {};
</script>

    
    <script type="text/javascript">
  let pageVars = {};
  pageVars.pageUrlMinusBasePath = "\/notes\/sapiens\/branches\/articles\/reich_who-we-are\/3_The_Disruptive_Genome\/11_The_Genomics_of_Race_and_Identity\/".replace(basePath, "/");
  pageVars.pageParams = {};
  pageVars.pageSource = "sapiens\/branches\/articles\/reich_who-we-are\/3_The_Disruptive_Genome\/11_The_Genomics_of_Race_and_Identity.md";
  console.log(pageVars.pageSource);
  var pageDefaults;
  for (let possiblePageDefaults of pageDefaultsList) {
    if (pageVars.pageSource.startsWith(possiblePageDefaults.scope.pathPrefix)) {
      pageDefaults = possiblePageDefaults.values
    }
  }
    
</script>


    <script src="/notes/webpack_dist/uiLib-bundle.js"></script>


<script>


MathJax = {
  tex: {
    inlineMath: [ ["@#", "#@"], ["\\(", "\\)"] ],
    displayMath: [ ["$$","$$"], ["\\[", "\\]"] ],
    processEscapes: true,  
  },
  svg: {
    fontCache: 'global'
  },
  options: {
    ignoreHtmlClass: "tex2jax_ignore|dno"
  }
};
</script>
<script type="text/javascript" id="MathJax-script" async
        src="/notes/non_webpack_js/mathjax/tex-svg.js">
</script>






    <link rel="stylesheet" href="/notes/webpack_dist/uiLib.css">
<link rel="stylesheet" href="/notes/css/uiLibPrint.css" media="print" >
<link rel="stylesheet" href="/notes/css/fonts.css" >
<link rel="stylesheet" href="/notes/css/@fortawesome/fontawesome-free/css/solid.min.css">

<link rel="stylesheet" href="/notes/css/@fortawesome/fontawesome-free/css/fontawesome.min.css">


<link rel="alternate" hreflang="sa-Deva" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/" />
<link rel="alternate" hreflang="sa-Deva" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=devanagari" />

<link rel="alternate" hreflang="sa-Knda" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=kannada" />
<link rel="alternate" hreflang="sa-Mlym" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=malayalam" />
<link rel="alternate" hreflang="sa-Telu" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=telugu" />
<link rel="alternate" hreflang="sa-Taml-t-sa-Taml-m0-superscript" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=tamil_superscripted" />
<link rel="alternate" hreflang="sa-Taml" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=tamil" />
<link rel="alternate" hreflang="sa-Gran" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=grantha" />

<link rel="alternate" hreflang="sa-Gujr" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=gujarati" />
<link rel="alternate" hreflang="sa-Guru" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=gurmukhi" />
<link rel="alternate" hreflang="sa-Shar" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=sharada" />
<link rel="alternate" hreflang="sa-Dogr" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=dogra" />
<link rel="alternate" hreflang="sa-Phag" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=phags_pa" />
<link rel="alternate" hreflang="sa-Tibt" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=tibetan" />
<link rel="alternate" hreflang="sa-Sidd" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=siddham" />
<link rel="alternate" hreflang="sa-Newa" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=newa" />
<link rel="alternate" hreflang="sa-Modi" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=modi" />
<link rel="alternate" hreflang="sa-Tirh" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=tirhuta_maithili" />
<link rel="alternate" hreflang="sa-Gong" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=gondi_gujala" />
<link rel="alternate" hreflang="sa-Gonm" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=gondi_masaram" />


<link rel="alternate" hreflang="sa-Orya" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=oriya" />
<link rel="alternate" hreflang="sa-Beng-t-sa-Beng-m0-assamese" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=assamese" />
<link rel="alternate" hreflang="sa-Beng" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=bengali" />
<link rel="alternate" hreflang="sa-Mtei" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=manipuri" />
<link rel="alternate" hreflang="sa-Sinh" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=sinhala" />
<link rel="alternate" hreflang="sa-Limb" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=limbu" />
<link rel="alternate" hreflang="sa-Lepc" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=lepcha" />


<link rel="alternate" hreflang="sa-Cyrl" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=cyrillic" />
<link rel="alternate" hreflang="sa-Latn-t-sa-Zyyy-m0-iso" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=iso" />
<link rel="alternate" hreflang="sa-Latn-t-sa-Zyyy-m0-iast" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=iast" />
<link rel="alternate" hreflang="sa-Aran" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=urdu" />
<link rel="alternate" hreflang="sa-Avst" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=avestan" />

<link rel="alternate" hreflang="sa-Brah" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=brahmi" />
<link rel="alternate" hreflang="sa-Khar" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=kharoshthi" />
<link rel="alternate" hreflang="sa-Zanb" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=zanbazar_square" />
<link rel="alternate" hreflang="sa-Bhks" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=bhaiksuki" />

<link rel="alternate" hreflang="sa-Bali" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=balinese" />
<link rel="alternate" hreflang="sa-Java" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=javanese" />
<link rel="alternate" hreflang="sa-Laoo" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=lao" />
<link rel="alternate" hreflang="sa-Laoo-t-sa-Laoo-m0-Laopali" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=lao_pali" />
<link rel="alternate" hreflang="sa-Mymr" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=burmese" />
<link rel="alternate" hreflang="sa-Mymr-t-sa-Mymr-m0-Mon" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=mon" />
<link rel="alternate" hreflang="sa-Cham" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=cham" />
<link rel="alternate" hreflang="sa-Khmr" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=khmer" />
<link rel="alternate" hreflang="sa-Thai" href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/?transliteration_target=thai" />

  </head>

  <body class="ma0 bg-near-white">
    
\(
% groupings of objects.
\newcommand{\set}[1]{\left\{ #1 \right\}}
\newcommand{\seq}[1]{\left(#1\right)}
\newcommand{\ang}[1]{\langle#1\rangle}
\newcommand{\tuple}[1]{\left(#1\right)}
\newcommand{\size}[1]{\left| #1\right|}

\newcommand{\comp}{\circ}


% numerical shortcuts.
\newcommand{\abs}[1]{\left| #1\right|}
\newcommand{\floor}[1]{\left\lfloor #1 \right\rfloor}
\newcommand{\ceil}[1]{\left\lceil #1 \right\rceil}

% linear algebra shortcuts.
\newcommand{\change}{\Delta}
\newcommand{\norm}[1]{\left\| #1\right\|}
\newcommand{\dprod}[1]{\langle#1\rangle}
\newcommand{\linspan}[1]{\langle#1\rangle}
\newcommand{\conj}[1]{\overline{#1}}
\newcommand{\der}[1]{\frac{d#1}{dx}}
\newcommand{\lap}{\Delta}
\newcommand{\kron}{\otimes}
\newcommand{\nperp}{\nvdash}

\newcommand{\mat}[1]{\left[ \begin{smallmatrix}#1 \end{smallmatrix} \right]}

% derivatives and limits
\newcommand{\partder}[2]{\frac{\partial #1}{\partial #2}}
\newcommand{\partdern}[3]{\frac{\partial^{#3 #1}}{\partial #2^{#3}}}
\newcommand{\gradient}{\nabla}
\newcommand{\subdifferential}{\partial}

% Arrows
\newcommand{\diverge}{\nearrow}
\newcommand{\notto}{\nrightarrow}
\newcommand{\up}{\uparrow}
\newcommand{\down}{\downarrow}
% gets and gives are defined!

% ordering operators
\newcommand{\oleq}{\preceq}
\newcommand{\ogeq}{\succeq}

% programming and logic operators
\newcommand{\dfn}{:=}
\newcommand{\assign}{:=}
\newcommand{\co}{\ co\ }
\newcommand{\en}{\ en\ }


% logic operators
\newcommand{\xor}{\oplus}
\newcommand{\Land}{\bigwedge}
\newcommand{\Lor}{\bigvee}
\newcommand{\finish}{\Box}
\newcommand{\contra}{\Rightarrow \Leftarrow}
\newcommand{\iseq}{\stackrel{_?{=}}}


% Set theory
\newcommand{\symdiff}{\Delta}
\newcommand{\setdiff}{\backslash}
\newcommand{\union}{\cup}
\newcommand{\inters}{\cap}
\newcommand{\Union}{\bigcup}
\newcommand{\Inters}{\bigcap}
\newcommand{\nullSet}{\phi}


% graph theory
\newcommand{\nbd}{\Gamma}

% Script alphabets
% For reals, use \Re

% greek letters
\newcommand{\eps}{\epsilon}
\newcommand{\del}{\delta}
\newcommand{\ga}{\alpha}
\newcommand{\gb}{\beta}
\newcommand{\gd}{\del}
\newcommand{\gp}{\pi}
\newcommand{\gf}{\phi}
\newcommand{\gh}{\eta}
\newcommand{\gF}{\Phi}
\newcommand{\gl}{\lambda}
\newcommand{\gm}{\mu}
\newcommand{\gn}{\nu}
\newcommand{\gr}{\rho}
\newcommand{\gs}{\sigma}
\newcommand{\gth}{\theta}
\newcommand{\gx}{\xi}
\newcommand{\gw}{\omega}

\newcommand{\sw}{\sigma}
\newcommand{\SW}{\Sigma}
\newcommand{\ew}{\lambda}
\newcommand{\EW}{\Lambda}

\newcommand{\Del}{\Delta}
\newcommand{\gD}{\Delta}
\newcommand{\gG}{\Gamma}
\newcommand{\gW}{\Omega}
\newcommand{\gS}{\Sigma}
\newcommand{\gTh}{\Theta}

% Bold english letters.
\newcommand{\bA}{\mathbf{A}}
\newcommand{\bB}{\mathbf{B}}
\newcommand{\bC}{\mathbf{C}}
\newcommand{\bD}{\mathbf{D}}
\newcommand{\bE}{\mathbf{E}}
\newcommand{\bF}{\mathbf{F}}
\newcommand{\bG}{\mathbf{G}}
\newcommand{\bH}{\mathbf{H}}
\newcommand{\bI}{\mathbf{I}}
\newcommand{\bJ}{\mathbf{J}}
\newcommand{\bK}{\mathbf{K}}
\newcommand{\bL}{\mathbf{L}}
\newcommand{\bM}{\mathbf{M}}
\newcommand{\bN}{\mathbf{N}}
\newcommand{\bO}{\mathbf{O}}
\newcommand{\bP}{\mathbf{P}}
\newcommand{\bQ}{\mathbf{Q}}
\newcommand{\bR}{\mathbf{R}}
\newcommand{\bS}{\mathbf{S}}
\newcommand{\bT}{\mathbf{T}}
\newcommand{\bU}{\mathbf{U}}
\newcommand{\bV}{\mathbf{V}}
\newcommand{\bW}{\mathbf{W}}
\newcommand{\bX}{\mathbf{X}}
\newcommand{\bY}{\mathbf{Y}}
\newcommand{\bZ}{\mathbf{Z}}

\newcommand{\bba}{\mathbf{a}}
\newcommand{\bbb}{\mathbf{b}}
\newcommand{\bbc}{\mathbf{c}}
\newcommand{\bbd}{\mathbf{d}}
\newcommand{\bbe}{\mathbf{e}}
\newcommand{\bbf}{\mathbf{f}}
\newcommand{\bbg}{\mathbf{g}}
\newcommand{\bbh}{\mathbf{h}}
\newcommand{\bbk}{\mathbf{k}}
\newcommand{\bbl}{\mathbf{l}}
\newcommand{\bbm}{\mathbf{m}}
\newcommand{\bbn}{\mathbf{n}}
\newcommand{\bbp}{\mathbf{p}}
\newcommand{\bbq}{\mathbf{q}}
\newcommand{\bbr}{\mathbf{r}}
\newcommand{\bbs}{\mathbf{s}}
\newcommand{\bbt}{\mathbf{t}}
\newcommand{\bbu}{\mathbf{u}}
\newcommand{\bbv}{\mathbf{v}}
\newcommand{\bbw}{\mathbf{w}}
\newcommand{\bbx}{\mathbf{x}}
\newcommand{\bby}{\mathbf{y}}
\newcommand{\bbz}{\mathbf{z}}
\newcommand{\0}{\mathbf{0}}
\newcommand{\1}{\mathbf{1}}

% Caligraphic english alphabet
\newcommand{\cA}{\mathcal{A}}
\newcommand{\cB}{\mathcal{B}}
\newcommand{\cC}{\mathcal{C}}
\newcommand{\cD}{\mathcal{D}}
\newcommand{\cE}{\mathcal{E}}
\newcommand{\cF}{\mathcal{F}}
\newcommand{\cG}{\mathcal{G}}
\newcommand{\cH}{\mathcal{H}}
\newcommand{\cI}{\mathcal{I}}
\newcommand{\cJ}{\mathcal{J}}
\newcommand{\cK}{\mathcal{K}}
\newcommand{\cL}{\mathcal{L}}
\newcommand{\cM}{\mathcal{M}}
\newcommand{\cN}{\mathcal{N}}
\newcommand{\cO}{\mathcal{O}}
\newcommand{\cP}{\mathcal{P}}
\newcommand{\cQ}{\mathcal{Q}}
\newcommand{\cR}{\mathcal{R}}
\newcommand{\cS}{\mathcal{S}}
\newcommand{\cT}{\mathcal{T}}
\newcommand{\cU}{\mathcal{U}}
\newcommand{\cV}{\mathcal{V}}
\newcommand{\cW}{\mathcal{W}}
\newcommand{\cX}{\mathcal{X}}
\newcommand{\cY}{\mathcal{Y}}
\newcommand{\cZ}{\mathcal{Z}}


% Formatting shortcuts
\newcommand{\red}[1]{\textcolor{red}{#1}}
\newcommand{\blue}[1]{\textcolor{blue}{#1}}
\newcommand{\htext}[2]{\texorpdfstring{#1}{#2}}

% Statistics
\newcommand{\distr}{\sim}
\newcommand{\stddev}{\sigma}
\newcommand{\covmatrix}{\Sigma}
\newcommand{\mean}{\mu}
\newcommand{\param}{\theta}
\newcommand{\gthEst}{\hat{\theta}}
\newcommand{\ftr}{\phi}
\newcommand{\est}[1]{\hat{#1}}

% General utility
\newcommand{\todo}[1]{\textbf{[TODO]}] \footnote{TODO: #1}}
\newcommand{\tbc}{[\textbf{Incomplete}]}
\newcommand{\chk}{[\textbf{Check}]}
\newcommand{\why}{[\textbf{Find proof}]}
\newcommand{\opt}[1]{\textit{#1}}

\newcommand{\experience}[1]{[\textbf{Personal Experience}]: #1 \blacktriangle}
\newcommand{\pf}[1]{[\textbf{Proof}]: #1 \Box}
\newcommand{\core}[1]{\textbf{Core Idea}: #1 \Arrowvert}
\newcommand{\example}[1]{\textbf{Example}: #1 \blacktriangle}
\newcommand{\error}[1]{\textbf{Error alert}: #1 \triangle}
\newcommand{\oprob}{[\textbf{OP}]: }


\renewcommand{\~}{\htext{$\sim$}{~}}

\DeclareMathOperator*{\argmin}{arg\,min}
\DeclareMathOperator*{\argmax}{arg\,max}


\)



    
<header class="p-1 bg-yellow noPrint">
    <nav role="navigation" class="col">
      <div id="div_top_bar" class="row">
        <div class="col-md-3 justify-content-start">
          <a  href="/" class="btn col border">
            <i class="fas fa-gopuram ">Vishvas&#39;s notes <br/> 11 The Genomics of Race and Identity</i>
          </a>
        </div>
        <div id="div_top_bar_right" class="col-md-auto d-flex justify-content-center">
          <button class="btn btn-light"  href="#transliterationDropdown" data-toggle="collapse">
            ✍
          </button>
          <select id="transliterationDropdown" size="1" class="collapse" onchange="module_uiLib.default.content.updateTransliteration()">
            <optgroup label="🌍">
              <option value="iso">āISO</option>
              <option value="iso_vedic">āi-āu</option>
              <option value="cyrillic">пу</option>
              <option value="urdu">نس</option>
              <option value="??">🌍??</option>
            </optgroup>
            <optgroup label="🛕🧭S">
              <option value="kannada">ಅ</option>
              <option value="malayalam">അ</option>
              <option value="telugu">క</option>
              <option value="tamil_superscripted">க²</option>
              <option value="tamil_extended">க</option>
              <option value="grantha">𑌅</option>
            </optgroup>
            <optgroup label="🛕🧭N">
              <option value="devanagari" selected="">स</option>
              <option value="gujarati">અ</option>
              <option value="gurmukhi">ਅ</option>
            </optgroup>
            <optgroup label="🛕🧭E">
              <option value="oriya">ଅ</option>
              <option value="assamese">অস</option>
              <option value="bengali">অ</option>
            </optgroup>
            <optgroup label="🛕🏺📜">
              <option value="sharada">𑆑𑇀𑆰</option>
              <option value="brahmi">𑀅</option>
            </optgroup>
            <optgroup label="🛕🌋">
              <option value="balinese">ᬩᬮᬶ</option>
            </optgroup>
          </select>
          <div>
          <button class="btn btn-outline-secondary btn-sm"  href="#transliterationDropdownAlt" data-toggle="collapse">
          (✍)
          </button>
          <select id="transliterationDropdownAlt" size="1"  style="font-size:12px" class="bg-light-yellow collapse" onchange="module_uiLib.default.content.updateTransliteration()">
            <optgroup label="🌍">
              <option value="iso">āISO</option>
              <option value="iso_vedic">āi-āu</option>
              <option value="cyrillic">пу</option>
              <option value="urdu">نس</option>
              <option value="??">🌍??</option>
            </optgroup>
            <optgroup label="🛕🧭S">
              <option value="kannada">ಅ</option>
              <option value="malayalam">അ</option>
              <option value="telugu">క</option>
              <option value="tamil_superscripted">க²</option>
              <option value="tamil_extended">க</option>
              <option value="grantha">𑌅</option>
            </optgroup>
            <optgroup label="🛕🧭N">
              <option value="devanagari" selected="">स</option>
              <option value="gujarati">અ</option>
              <option value="gurmukhi">ਅ</option>
            </optgroup>
            <optgroup label="🛕🧭E">
              <option value="oriya">ଅ</option>
              <option value="assamese">অস</option>
              <option value="bengali">অ</option>
            </optgroup>
            <optgroup label="🛕🏺📜">
              <option value="sharada">𑆑𑇀𑆰</option>
              <option value="brahmi">𑀅</option>
            </optgroup>
            <optgroup label="🛕🌋">
              <option value="balinese">ᬩᬮᬶ</option>
            </optgroup>
          </select>
          </div>
          <div class="row col d-flex  dropdown">
            <button class="btn btn-secondary-outline btn-sm dropdown-toggle" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
              A±
            </button>
            <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
            <a class="btn btn-secondary btn-sm" onclick="module_uiLib.changeTextSize(2)">A+</a>
            <a class="btn btn-secondary btn-sm" onclick="module_uiLib.changeTextSize(-2)">A-</a>
            </div>
          </div>
          <button class="btn btn-light btn-sm"  href="#commentStyleDropdown" data-toggle="collapse">
            🗨
          </button>
          <select id="commentStyleDropdown" class="collapse" size="1" onchange="module_uiLib.default.content.updateCommentStyleFromDropdown()">
            <option value="on">✔</option>
            <option value="hidden">✗</option>
          </select>
          <div><a id="expandAllButton" class="btn btn-secondary" onclick="module_uiLib.default.query.setParamsAndGo({'expandDetails': '.*'})">↕️️</a></div>
          <div id="printStyleDiv" class="row col d-flex  dropdown">
          <button class="btn btn-light btn-sm dropdown-toggle"  href="#printStyleDropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
          🖨
          </button>
            <div class="dropdown-menu col" aria-labelledby="printStyleDropdown">
              <div class="row justify-content-center">
                <p>Cols:</p>
                <input type="number" min="1" max="10" value="2" step="1" id="tbColumns" style="width: 3ch;"  accesskey="c"/>
              </div>
              <div class="row justify-content-center">
                <p>A±:</p> <input type="text" id="tbBodyFontSize" value="0.4cm" style="width: 6ch;"  accesskey="f"></input>
              </div>
              <div class="row justify-content-center">
                <p>Incl:</p> <input checked type="checkbox" id="tbIncludeStyle" value="on"></input>
                <button id="btnPrintStyle" onclick="module_uiLib.default.content.updatePrintStyle()">🖨
                </button>
              </div>
            </div>
        </div>
        </div>
        <div id="audioDiv" class="row col d-flex  dropdown">
          <button class="btn btn-secondary-outline btn-sm dropdown-toggle" type="button" id="audioDropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
            🎧
          </button>
          <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
          <a id="speakerButton" class="btn btn-outline-secondary btn-sm" onclick="module_uiLib.default.content.handleSpeakToggle()">⏯</a>
          <button id="speakerPauseButton" class="btn btn-outline-secondary btn-sm">⏸</button>
            <div id="audioOptionsDiv" class="row justify-content-center" >
              <div>
                <p class="lead">Vol</p>
                <input type="range" min="0" max="1" value="1" step="0.1" id="volume" />
                <span id="volume-label" class="ms-2">1</span>
              </div>
              <div class="mx-5">
                <p class="lead">Rate</p>
                <input type="range" min="0.1" max="10" value="1" id="rate" step="0.1" />
                <span id="rate-label" class="ms-2">1</span>
              </div>
              <div>
                <p class="lead">Pitch</p>
                <input type="range" min="0" max="2" value="1" step="0.1" id="pitch" />
                <span id="pitch-label" class="ms-2">1</span>
              </div>
            </div>
          </div>
        </div>
        <a href="https://vishvAsa.github.io/notes/search"  class="btn btn-light btn-sm">🥽🔦</a>
        <div class="row col  d-flex justify-content-center">
          <div><a id="previousPage" href="" class="btn btn-secondary">###<i class="fas fa-caret-left"></i></a></div>
          <div ><a id="nextPage" href="" class="btn btn-secondary"><i class="fas fa-caret-right"></i> ### </a></div>
        </div>
        <ul id="top-bar-right-custom" class="list-group list-group-horizontal">
        </ul>
      </div>
    </nav>
</header>

    
    <div class="row" name="contentRow">
      
      <aside class="col-md-3 card border noPrint " id="sidebar">
          <div id="fastSearch">
              <datalist id="pageDataList"></datalist>
              <input id="searchInput" placeholder="शीर्षिकान्विष्यताम्" list="pageDataList" onchange="module_uiLib.default.navigation.pageLoader()">
          </div>
          <details open  class="bg-light-gray" >
              <summary id="sidebarToggleLink" onclick="module_uiLib.default.navigation.sidebarToggleHandler()">
            Menu
              </summary>
              <ul id="sidebarBody" class="list pl2 p-0 bg-yellow">
              </ul>
          </details>
      </aside>
      <main class="col p-3" role="main" data-pagefind-body>
        
<header class='border d-flex justify-content-between'>
    <h1 id="11 The Genomics of Race and Identity">11 The Genomics of Race and Identity</h1>
    
    <a id="editLink" class="btn btn-primary noPrint"  href="https://github.com/vishvAsa/notes/edit/content/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity.md"><i class="fas fa-edit"></i></a>
    
</header>
<article>
  <aside id="toc_card" class="card border noPrint ">
    <details id="toc_header">
        <summary>What's in this page?</summary>
      <div id="toc_body" class="p-0">
        
        <ul id="toc_ul" class="list p-0">
        </ul>
      </div>
    </details>
    <div id="progressBarDiv" class="row d-flex justify-content-center">
      🐎💪<progress id="progressLoading" value="0" max="100"> 32% </progress>
    </div>
  </aside>
  <div id="post_content">
  <h2 id="fear-of-biological-difference">Fear of Biological Difference</h2>
<p>When I started my first academic job in 2003, I bet my career on the idea that the history of mixture of West Africans and Europeans in the Americas would make it possible to find risk factors that contribute to health disparities for diseases like prostate cancer, which occurs at about a rate 1.7 times higher in African Americans than in European Americans.<sup id="fnref:1"><a href="#fn:1" class="footnote-ref" role="doc-noteref">1</a></sup> This particular disparity had not been possible to explain based on dietary and environmental differences across populations, suggesting that genetic factors might play a role.</p>
<p>African Americans today derive about 80 percent of their ancestry from enslaved Africans brought to North America between the sixteenth and nineteenth centuries. In a large group of African Americans, the proportion of African ancestry at any one location in the genome is expected to be close to the average (defining the proportion of African ancestry as the fraction of ancestors that were in West Africa before around five hundred years ago). However, if there are risk factors for prostate cancer that occur at higher frequency in West Africans than in Europeans, then African Americans with prostate cancer are expected to have inherited more African ancestry than the average in the vicinity of these genetic variations. This idea can be used to pinpoint disease genes.</p>
<p>To make such studies possible, I set up a molecular biology laboratory to identify mutations that differed in frequency between West Africans and Europeans. My colleagues and I developed methods that used information from these mutations to identify where in the genome people harbor segments of DNA derived from their West African and European ancestors.<sup id="fnref:2"><a href="#fn:2" class="footnote-ref" role="doc-noteref">2</a></sup> To prove that these ideas worked in practice, we applied them to many traits, including prostate cancer, uterine fibroids, late-stage kidney disease, multiple sclerosis, low white blood cell count, and type 2 diabetes.</p>
<p>In 2006, my colleagues and I applied our methods to 1,597 African American men with prostate cancer, and found that in one region of the genome, they had about 2.8 percent more African ancestry than the average in the rest of their genomes.<sup id="fnref:3"><a href="#fn:3" class="footnote-ref" role="doc-noteref">3</a></sup> The odds of seeing a rise in African ancestry this large by accident were about ten million to one. When we looked in more detail, we found that this region contained at least seven independent risk factors for prostate cancer, all more common in West Africans than in Europeans.<sup id="fnref:4"><a href="#fn:4" class="footnote-ref" role="doc-noteref">4</a></sup> Our findings could account entirely for the higher rate of prostate cancer in African Americans than in European Americans. We could conclude this because African Americans who happen to have entirely European ancestry in this small section of their genomes had about the same risk for prostate cancer as random European Americans.<sup id="fnref:5"><a href="#fn:5" class="footnote-ref" role="doc-noteref">5</a></sup>+++(5)+++</p>
<p>In 2008, I gave a talk about my work on prostate cancer to a conference on health disparities across ethnic groups in the United States. In my talk, I tried to communicate my excitement about the scientific approach and my conviction that it could help to find genetic risk factors for other diseases. Afterward, though, I was angrily questioned by an anthropologist in the audience, who believed that by studying “West African” or “European” segments of DNA to understand biological differences between groups, I was flirting with racism. Her questions were seconded by several others, and I encountered similar responses at other meetings. A legal ethicist who heard me talk on a similar theme suggested that I might want to refer to the populations from which African Americans descend as “cluster A” and “cluster B.” But I replied that it would be dishonest to disguise the model of history that was driving this work. Every feature of the data I looked at suggested that this model was a scientifically meaningful one, providing accurate estimates of where in the genome people harbor segments of DNA from ancestors who lived in West Africa or in Europe in the last twenty generations, prior to the mixture caused by colonialism and the slave trade. It was also clear that the approach was identifying real risk factors for disease that differ in frequency across populations, leading to discoveries with the potential to improve health.</p>
<p>Far from being extremists, my questioners were articulating a mainstream view about the danger of work exploring biological differences among human populations. In 1942, the anthropologist Ashley Montagu wrote <em>Man’s Most Dangerous Myth: The Fallacy of Race,</em> arguing that race is a social concept and has no biological reality, and setting the tone for how anthropologists and many biologists have discussed this issue ever since.<sup id="fnref:6"><a href="#fn:6" class="footnote-ref" role="doc-noteref">6</a></sup>+++(4)+++ A classic example often cited is the inconsistent definition of “black.” In the United States, people tend to be called “black” if they have sub-Saharan African ancestry—even if it is a small fraction and even if their skin color is very light. In Great Britain, “black” tends to mean anyone with sub-Saharan African ancestry who also has dark skin. In Brazil, the definition is different yet again: a person is only “black” if he or she is entirely African in ancestry. If “black” has so many inconsistent definitions, how can there be any biological meaning to “race”?+++(4)+++</p>
<p>Beginning in 1972, genetic arguments began to be incorporated into the assertions that anthropologists were making about the lack of substantial biological differences among human populations. In that year, Richard Lewontin published a study of variation in protein types in blood.<sup id="fnref:7"><a href="#fn:7" class="footnote-ref" role="doc-noteref">7</a></sup> He grouped the populations he analyzed into seven “races”—West Eurasians, Africans, East Asians, South Asians, Native Americans, Oceanians, and indigenous Australians—and found that around 85 percent of variation in the protein types could be accounted for by variation <em>within</em> populations and “races,” and only 15 percent by variation <em>across</em> them. He concluded: “Races and populations are remarkably similar to each other, with the largest part by far of human variation being accounted for by the differences between individuals. Human racial classification is of no social value and is positively destructive of social and human relations. Since such racial classification is now seen to be of virtually no genetic or taxonomic significance either, no justification can be offered for its continuance.”</p>
<p>In this way, through the collaboration of anthropologists and geneticists, a consensus was established that there are no differences among human populations that are large enough to support the concept of “biological race.”+++(4)+++ Lewontin’s results made it clear that for the great majority of traits, human populations overlap to such a degree that it is impossible to identify a single biological trait that distinguishes people in any two groups, which is intuitively what some people think of when they conceive of “biological race.”</p>
<p>But this consensus view of many anthropologists and geneticists has morphed, seemingly without questioning, into an orthodoxy that the biological differences among human populations are so modest that they should in practice be ignored—and moreover, because the issues are so fraught, that study of biological differences among populations should be avoided if at all possible.+++(4)+++ It should come as no surprise, then, that some anthropologists and sociologists see genetic research into differences across populations, even if done in a well-intentioned way, as problematic. They are concerned that work on such differences will be used to validate concepts of race that should be considered discredited. They see this work as located on a slippery slope to the kinds of pseudoscientific arguments about biological difference that were used in the past to try to justify the slave trade, the eugenics movement to sterilize the disabled as biologically defective, and the Nazis’ murder of six million Jews.</p>
<p>The concern is so acute that the political scientist Jacqueline Stevens has even suggested that research and even emails discussing biological differences across populations should be banned, and that the United States “should issue a regulation prohibiting its staff or grantees…from publishing in any form—including internal documents and citations to other studies—claims about genetics associated with variables of race, ethnicity, nationality, or any other category of population that is observed or imagined as heritable unless statistically significant disparities between groups exist and description of these will yield clear benefits for public health, as deemed by a standing committee to which these claims must be submitted and authorized.”<sup id="fnref:8"><a href="#fn:8" class="footnote-ref" role="doc-noteref">8</a></sup>+++(4)+++</p>
<h2 id="the-language-of-ancestry">The Language of Ancestry</h2>
<p>But whether we like it or not, there is no stopping the genome revolution. The results that it is producing are making it impossible to maintain the orthodoxy established over the last half century, as they are revealing hard evidence of substantial differences across populations.</p>
<p>The first major engagement between the genome revolution and anthropological orthodoxy came in 2002, when Marc Feldman and his colleagues showed that by studying enough places in the genome—they analyzed 377 variable positions—it is possible to group most people in a worldwide population sample into clusters that correlate strongly to popular categories of race in the United States: “African,” “European,” “East Asian,” “Oceanian,” or “Native American.”<sup id="fnref:9"><a href="#fn:9" class="footnote-ref" role="doc-noteref">9</a></sup>+++(5)+++ While Feldman’s conclusions were broadly consistent with Lewontin’s in that his data also showed more variation within groups than among them, his study defined clusters in terms of combinations of mutations instead of looking at mutations individually as Lewontin had done.</p>
<p>Scientists were quick to respond. One was Svante Pääbo, who eight years later would go on to lead the work to sequence whole genomes of archaic Neanderthals and Denisovans. Pääbo came to the debate about the nature of human population structure as a founding director of the Max Planck Institute for Evolutionary Anthropology in Leipzig, which was set up in 1997 in an effort to return Germany to a field in which it had played a leading role before the Second World War but that it had largely abandoned due to anthropologists’ central contribution to developing Nazi race theory.</p>
<p>Pääbo took seriously his moral responsibility as head of an ambitious German institute of anthropology, and wondered whether the truth about human population structure could be more like the anthropologist Frank Livingston’s suggestion that “there are no races, there are only clines”—a view in which human genetic variation is characterized by gradual geographic gradients that reflect interbreeding among neighbors.<sup id="fnref:10"><a href="#fn:10" class="footnote-ref" role="doc-noteref">10</a></sup> To explore this possibility, Pääbo investigated whether the clusters the Feldman study found appeared sharply defined because the analyzed populations had been sampled in a nonrandom fashion across the world. To understand how nonrandom sampling could contribute to this result, consider the United States, which harbors extraordinary diversity, but where genetic discontinuities among groups such as African Americans, European Americans, and East Asians are sharper than in the places from which immigrant populations came because the United States has drawn its immigrants from a subset of world locations.+++(4)+++ For example, in the United States, most of the African ancestry is from a handful of groups in West Africa,<sup id="fnref:11"><a href="#fn:11" class="footnote-ref" role="doc-noteref">11</a></sup> most of the European ancestry is from northwest Europe, and most of the Asian ancestry is from Northeast Asia. Pääbo showed that such nonrandom sampling could account for some of the effects Feldman and colleagues observed. However, later work proved that nonrandom sampling could not account for most of the structure, as substantial clustering of human populations is observed even when repeating analyses on geographically more evenly distributed sets of samples.<sup id="fnref:12"><a href="#fn:12" class="footnote-ref" role="doc-noteref">12</a></sup>+++(4)+++</p>
<p>Another flurry of discussion followed a 2003 paper led by Neil Risch, who argued that racial grouping is useful in medical research, not just to adjust for socioeconomic and cultural differences, but also because it correlates with genetic differences that are important to know about when diagnosing and treating disease.<sup id="fnref:13"><a href="#fn:13" class="footnote-ref" role="doc-noteref">13</a></sup> Risch was convinced by examples like sickle cell disease, which occurs far more often in African Americans than in other populations in the United States. He argued that it was appropriate for doctors to be more likely to think of sickle cell disease if the patient is African American.+++(4)+++</p>
<p>In 2005, the U.S. Food and Drug Administration lent support to this way of thinking when it approved BiDil, a combination of two medications approved to treat heart failure in African Americans because data suggested it was more effective in African Americans than in European Americans. But on the other side of the argument, David Goldstein suggested that U.S. racial categories are so weakly predictive of most biological outcomes that they do not have long-term value.<sup id="fnref:14"><a href="#fn:14" class="footnote-ref" role="doc-noteref">14</a></sup> He and his colleagues showed that the frequencies of genetic variants that determine dangerous reactions to drugs are poorly predicted by U.S. census categories. He acknowledged that the reliance on racial and ethnic categories is useful given our poor present knowledge, but predicted that the future will involve testing individuals directly for what mutations they have, and doing away altogether with racial classification as a basis for making individualized decisions about care.+++(4)+++</p>
<p>Against this backdrop of controversy emerged work like mine, focusing on methods to determine population origin not just of our ancestors but also of individual segments of our genomes. The anthropologist Duana Fullwiley has written that the development of what she calls “admixture technology” and the language of “ancestry” that geneticists like me have adopted is a reversion to traditional ideas of biological race.<sup id="fnref:15"><a href="#fn:15" class="footnote-ref" role="doc-noteref">15</a></sup> She has pointed out that in the United States, the “ancestry” terms that we use map relatively closely to traditional racial categories, and her view is that the population genetics community has invented a set of euphemisms to discuss topics that had become taboo. The belief that we have embraced euphemisms is also shared by some on the other side of the political spectrum. At a 2010 meeting I attended at Cold Spring Harbor Laboratory, the journalist Nicholas Wade described his resentment of the population genetics community’s “ancestry” terminology, asserting that “race is a perfectly good English word.”</p>
<p>But “ancestry” is not a euphemism, nor is it synonymous with “race.”+++(4)+++ Instead, the term is born of an urgent need to come up with a precise language to discuss genetic differences among people at a time when scientific developments have finally provided the tools to detect them. It is now undeniable that there are nontrivial average genetic differences across populations in multiple traits, and the race vocabulary is too ill-defined and too loaded with historical baggage to be helpful. If we continue to use it we will not be able to escape the current debate, which is mired in an argument between two indefensible positions. On the one side there are beliefs about the nature of the differences that are grounded in bigotry and have little basis in reality. On the other side there is the idea that any biological differences among populations are so modest that as a matter of social policy they can be ignored and papered over. It is time to move on from this paralyzing false dichotomy and to figure out what the genome is actually telling us.</p>
<h2 id="real-biological-difference">Real Biological Difference</h2>
<p>I have deep sympathy for the concern that genetic discoveries about differences among populations may be misused to justify racism. But it is precisely because of this sympathy that I am worried that people who deny the possibility of substantial biological differences among populations across a range of traits are digging themselves into an indefensible position, one that will not survive the onslaught of science. In the last couple of decades, most population geneticists have sought to avoid contradicting the orthodoxy. When asked about the possibility of biological differences among human populations, we have tended to obfuscate, making mathematical statements in the spirit of Richard Lewontin about the average difference between individuals from within any one population being around six times greater than the average difference between populations. We point out that the mutations that underlie some traits that differ dramatically across populations—the classic example is skin color—are unusual, and that when we look across the genome it is clear that the typical differences in frequencies of mutations across populations are far less.<sup id="fnref:16"><a href="#fn:16" class="footnote-ref" role="doc-noteref">16</a></sup>+++(4)+++ But this carefully worded formulation is deliberately masking the possibility of substantial average differences in biological traits across populations.+++(4)+++</p>
<p>To understand why it is no longer an option for geneticists to lock arms with anthropologists and imply that any differences among human populations are so modest that they can be ignored, go no further than the “genome bloggers.” Since the genome revolution began, the Internet has been alive with discussion of the papers written about human variation, and some genome bloggers have even become skilled analysts of publicly available data. Compared to most academics, the politics of genome bloggers tend to the right—Razib Khan<sup id="fnref:17"><a href="#fn:17" class="footnote-ref" role="doc-noteref">17</a></sup> and Dienekes Pontikos<sup id="fnref:18"><a href="#fn:18" class="footnote-ref" role="doc-noteref">18</a></sup> post on findings of average differences across populations in traits including physical appearance and athletic ability.+++(4)+++ The <em>Eurogenes</em> blog spills over with sometimes as many as one thousand comments in response to postings on the charged topic of which ancient peoples spread Indo-European languages,<sup id="fnref:19"><a href="#fn:19" class="footnote-ref" role="doc-noteref">19</a></sup> a highly sensitive issue since as discussed in part II, narratives about the expansion of Indo-European speakers have been used as a basis for building national myths,<sup id="fnref:20"><a href="#fn:20" class="footnote-ref" role="doc-noteref">20</a></sup> and sometimes have been abused as happened in Nazi Germany.<sup id="fnref:21"><a href="#fn:21" class="footnote-ref" role="doc-noteref">21</a></sup> The genome bloggers’ political beliefs are fueled partly by the view that when it comes to discussion about biological differences across populations, the academics are not honoring the spirit of scientific truth-seeking. The genome bloggers take pleasure in pointing out contradictions between the politically correct messages academics often give about the indistinguishability of traits across populations and their papers showing that this is not the way the science is heading.</p>
<p>What real differences do we know about? We cannot deny the existence of substantial average genetic differences across populations, not just in traits such as skin color, but also in bodily dimensions, the ability to efficiently digest starch or milk sugar, the ability to breathe easily at high altitudes, and susceptibility to particular diseases.+++(4)+++ These differences are just the beginning. I expect that the reason we don’t know about a much larger number of differences among human populations is that studies with adequate statistical power to detect them have not yet been carried out. For the great majority of traits, there is, as Lewontin said, much more variation within populations than across populations. This means that individuals with extreme high or low values of the great majority of traits can occur in any population. But it does not preclude the existence of subtler, average differences in traits across populations.</p>
<p>The indefensibility of the orthodoxy is obvious at almost every turn. In 2016, I attended a lecture on race and genetics by the biologist Joseph L. Graves Jr. at the Peabody Museum of Archaeology and Ethnography at Harvard. At one point, Graves compared the approximately five mutations known to have large effects on skin pigmentation and that are obviously different in frequency across populations to the more than ten thousand genes known to be active in human brains. He argued that in contrast to pigmentation genes, the patterns at genes particularly active in the brain would surely average out over so many locations, with some mutations nudging cognitive and behavioral traits in one direction and some pushing in the other direction. But this argument doesn’t work, because in fact, if natural selection has exerted different pressures on two populations since they separated, traits influenced by many mutations are just as capable of achieving large average differences across populations as traits influenced by few mutations.+++(4)+++ And indeed, it is already known that traits shaped by many mutations (as is probably the case for behavior and cognition) are at least as important targets of natural selection as traits like skin color that are driven by a small number of mutations.<sup id="fnref:22"><a href="#fn:22" class="footnote-ref" role="doc-noteref">22</a></sup> The best example we currently have of a trait governed by many mutations is height.+++(5)+++ Studies in hundreds of thousands of people have shown that height is determined by thousands of variable positions across the genome. A 2012 analysis led by Joel Hirschhorn showed that natural selection on these is responsible for the shorter average height in southern Europeans compared to northern Europeans.<sup id="fnref:23"><a href="#fn:23" class="footnote-ref" role="doc-noteref">23</a></sup> Height isn’t the only example. Jonathan Pritchard led a study showing that in the last approximately two thousand years there has been selection for genetic variations that affect many other traits in Britain, including an increase in average infant head size and an increase in average female hip size (possibly to accommodate the increased higher average infant head size during childbirth).<sup id="fnref:24"><a href="#fn:24" class="footnote-ref" role="doc-noteref">24</a></sup>+++(4)+++</p>
<p>It is tempting to argue that genetic influence on bodily dimensions is one thing, but that cognitive and behavioral traits are another. But this line has already been crossed. Often when a person participates in a genetic study of a disease, he or she fills out a form providing information on height, weight, and number of years of education. By compiling the information on the number of years of education for over four hundred thousand people of European ancestry whose genomes have been surveyed in the course of various disease studies, Daniel Benjamin and colleagues identified seventy-four genetic variations each of which has overwhelming evidence of being more common in people with more years of education than in people with fewer years even after controlling for such possibly confounding factors as heterogeneity in the study population.<sup id="fnref:25"><a href="#fn:25" class="footnote-ref" role="doc-noteref">25</a></sup>+++(4)+++ Benjamin and colleagues also showed that the power of genetics to predict number of years of education is far from trivial, even though social influences surely have a greater average influence on this behavior than genetics. They showed that in the European ancestry population in which they carried out their study, it should be possible to build a genetic predictor in which the probability of completing twelve years of education is 96 percent for the twentieth of people with the highest prediction compared to 37 percent for the lowest.<sup id="fnref:26"><a href="#fn:26" class="footnote-ref" role="doc-noteref">26</a></sup>+++(4)+++</p>
<p>How do these genetic variations influence educational attainment? The obvious guess is that they have a direct effect on academic abilities, but that is probably wrong. A study of more than one hundred thousand Icelanders showed that the variations also increase the age at which a woman has her first child, and that this is a more powerful effect than the one on the number of years of education.+++(5)+++ It is possible that these variations exert their effect indirectly, by nudging people to defer having children, which makes it easier for them to complete their education.<sup id="fnref:27"><a href="#fn:27" class="footnote-ref" role="doc-noteref">27</a></sup> This shows that when we discover biological differences governing behavior, they may not be working in the way we naively assume.</p>
<p>Average differences across populations in the frequencies of the mutations that affect educational attainment have not yet been identified. But a sobering finding is that older people in Iceland are systematically different from younger people in having a higher genetically predicted number of years of education.<sup id="fnref:28"><a href="#fn:28" class="footnote-ref" role="doc-noteref">28</a></sup> Augustine Kong, the lead author of the Icelandic study, showed that this reflects natural selection over the last century against people with more predicted education, likely because of selection for people who began having children at a younger age.+++(4)+++ Given that the genetic underpinnings of the number of years of education a person achieves have measurably changed within a century in a single population under the pressure of natural selection, it seems highly likely that the trait differs across populations too.</p>
<p>No one knows how the genetic variations that influence educational attainment in people of European ancestry affect behavior in people of non-European ancestries, or in differently structured social systems. That said, it seems likely that if these mutations have an effect on behavior in one population they will do so in others, too, even if the effects differ by social context. And educational attainment as a trait is likely to be only the tip of an iceberg of behavioral traits affected by genetics. The Benjamin study has already been joined by others finding genetic predictors of behavioral traits,<sup id="fnref:29"><a href="#fn:29" class="footnote-ref" role="doc-noteref">29</a></sup> including one of more than seventy thousand people that found mutations in more than twenty genes that were significantly predictive of performance on intelligence tests.<sup id="fnref:30"><a href="#fn:30" class="footnote-ref" role="doc-noteref">30</a></sup>+++(5)+++</p>
<p>For those who wish to argue against the possibility of biological differences across populations that are substantial enough to make a difference in people’s abilities or propensities, the most natural refuge might be to make the case that even if such differences exist, they will be small. The argument would be that even if there are average differences across human populations in genetically determined traits affecting cognition or behavior, so little time has passed since the separation of populations that the quantitative differences across populations are likely to be trivially small, harkening back to Lewontin’s argument that the average genetic difference between populations is much less than the average difference between individuals. But this argument doesn’t hold up either. The average time separation between pairs of human populations since they diverged from common ancestral populations, which is up to around fifty thousand years for some pairs of non-African populations, and up to two hundred thousand years or more for some pairs of sub-Saharan African populations, is far from negligible on the time scale of human evolution.+++(4)+++ If selection on height and infant head circumference can occur within a couple of thousand years,<sup id="fnref:31"><a href="#fn:31" class="footnote-ref" role="doc-noteref">31</a></sup> it seems a bad bet to argue that there cannot be similar average differences in cognitive or behavioral traits.+++(4)+++ Even if we do not yet know what the differences are, we should prepare our science and our society to be able to deal with the reality of differences instead of sticking our heads in the sand and pretending that differences cannot be discovered. The approach of staying mum, of implying to the public and to colleagues that substantial differences in traits across populations are unlikely to exist, is a strategy that we scientists can no longer afford, and that in fact is positively harmful. If as scientists we willfully abstain from laying out a rational framework for discussing human differences, we will leave a vacuum that will be filled by pseudoscience, an outcome that is far worse than anything we could achieve by talking openly.+++(4)+++</p>
<h2 id="the-genome-revolutions-insight">The Genome Revolution’s Insight</h2>
<p>On the question of whether traditional social categories of race correspond to meaningful biological categories, the genome revolution has already provided us with new insights that go far beyond the information that was available to the first population geneticists and anthropologists who grappled with the issue. In this way, the data provided by the genome revolution are potentially liberating, providing an opportunity for intellectual progress beyond the current stale framing of the debate.</p>
<p>As recently as 2012, it still seemed reasonable to interpret human genetic data as pointing to immutable categories such as “East Asians,” “Caucasians,” “West Africans,” “Native Americans,” and “Australasians,” with each group having been separated and unmixed for tens of thousands of years. The 2002 study led by Marc Feldman produced clusters that corresponded relatively well to these categories, and the model seemed to be doing a good job of describing variation in many parts of the world (with some exceptions).<sup id="fnref:32"><a href="#fn:32" class="footnote-ref" role="doc-noteref">32</a></sup> In other papers, Feldman and his colleagues proposed a model for how this kind of structure could arise among human populations. Their proposal was that modern humans expanding out of Africa and the Near East after around fifty thousand years ago left descendant populations along the way, which in turn budded off their own descendant populations, with the present-day inhabitants of each region being descended directly from the modern humans who first arrived.<sup id="fnref:33"><a href="#fn:33" class="footnote-ref" role="doc-noteref">33</a></sup> Their “serial founder” model was more sophisticated than that imagined by biological race theorists in the seventeenth to twentieth centuries, but shared with it the prediction that after being established, human populations hardly mixed with each other.</p>
<p>But ancient DNA discoveries have rendered the serial founder model untenable.+++(5)+++ We now know that the present-day structure of populations does not reflect the one that existed many thousands of years ago.<sup id="fnref:34"><a href="#fn:34" class="footnote-ref" role="doc-noteref">34</a></sup> Instead, the current populations of the world are mixtures of highly divergent populations that no longer exist in unmixed form—for example, the Ancient North Eurasians, who contributed a large amount of the ancestry of present-day Europeans as well as of Native Americans,<sup id="fnref:35"><a href="#fn:35" class="footnote-ref" role="doc-noteref">35</a></sup> and multiple ancient populations of the Near East, each as differentiated from the other as Europeans and East Asians are differentiated from each other today.<sup id="fnref:36"><a href="#fn:36" class="footnote-ref" role="doc-noteref">36</a></sup> Most of today’s populations are not exclusive descendants of the populations that lived in the same locations ten thousand years ago.+++(4)+++</p>
<p>The findings that the nature of human population structure is not what we assumed should serve as a warning to those who think they know that the true nature of human population differences will correspond to racial stereotypes.+++(4)+++ Just as we had an inaccurate picture of early human origins before the ancient DNA revolution unleashed an avalanche of surprises, so we should distrust the instincts that we have about biological differences. We do not yet have sufficient sample sizes to carry out compelling studies of most cognitive and behavioral traits, but the technology is now available, and once high-quality studies are performed—which they will be somewhere in the world whether we like it or not—any genetic associations they find will be undeniable. We will need to deal with these studies and react responsibly to them when they are published, but we can already be sure that we will be surprised by some of the outcomes.</p>
<p>Unfortunately, today there is a new breed of writers and scholars who argue not only that there are average genetic differences, but that they can guess what they are based on traditional racial stereotypes.</p>
<p>The person who has most recently made a prominent argument that there is a genetic basis to stereotypes about differences across human populations is the <em>New York Times</em> journalist Nicholas Wade, who in 2014 published <em>A Troublesome Inheritance: Genes, Race and Human History</em>.<sup id="fnref:37"><a href="#fn:37" class="footnote-ref" role="doc-noteref">37</a></sup> The abiding theme of Wade’s reporting is the propensity of academics to band together to enforce orthodoxies and to be shown up by a band of rebels speaking the truth (he has written on scientific fraud, described the Human Genome Project as a monolith wastefully spending the public’s money, and attacked the value of genome-wide association studies for finding common genetic variations contributing to risk for diseases). Wade’s <em>Troublesome Inheritance</em> ran with the theme again, suggesting that a politically correct alliance of anthropologists and geneticists has banded together to suppress the truth that there are significant differences among human populations and that those differences correspond to classic stereotypes. One part of the argument has something to it—Wade correctly highlights the problem of an academic community trying to enforce an implausible orthodoxy. Yet the “truth” that he puts forward in opposition, the idea that not only are there substantial differences, but that they likely correspond to traditional racial stereotypes, has no merit. Wade’s book combines compelling content with parts that are entirely speculative, presenting everything with the same authority and in the same voice, so that naive readers who accept the parts of it that are well argued are tempted to accept the rest. Worse, when compared to Wade’s previous writing, in which the rebels speaking the truth were scholars of creativity and accomplishment, he does not identify any serious scholarship in genetics supporting his speculations.<sup id="fnref:38"><a href="#fn:38" class="footnote-ref" role="doc-noteref">38</a></sup> And yet by celebrating those who have opposed the flawed orthodoxy, he implies wrongly that their alternative theories must be right.</p>
<p>As an example of the speculations to which Wade gives pride of place, one of his chapters focuses on a 2006 essay by Gregory Cochran, Jason Hardy, and Henry Harpending suggesting that the high average intelligence quotient (IQ) of Ashkenazi Jews (more than one standard deviation above the world average), and their disproportionate share of Nobel Prizes (about one hundred times the world average), might reflect natural selection due to a millennium-long history in which Jewish populations practiced moneylending, a profession that required writing and calculation.<sup id="fnref:39"><a href="#fn:39" class="footnote-ref" role="doc-noteref">39</a></sup> They also pointed to the high rate in Ashkenazi Jews of Tay-Sachs disease and Gaucher disease, which are due to mutations that affect storage of fat in brain cells, and which they hypothesized rose in frequency under the pressure of selection for genetic variations contributing to intelligence (they argued that these mutations might be beneficial when they occur in one copy rather than the two needed to cause disease).+++(4)+++ This argument is contradicted by the evidence that these diseases almost certainly owe their origin to random bad luck—the fact that during the medieval population bottleneck that affected Ashkenazi Jews, the small number of individuals who had many descendants happened to carry these mutations<sup id="fnref:40"><a href="#fn:40" class="footnote-ref" role="doc-noteref">40</a></sup>—yet Wade highlights the work on the basis that it <em>might</em> be right. Harpending has a track record of speculating without evidence on the causes of behavioral differences among populations. In a talk he gave at a 2009 conference on “Preserving Western Civilization,” he asserted that people of sub-Saharan African ancestry have no propensity to work when they don’t have to—“I’ve never seen anyone with a hobby in Africa,” he said—because, he thought, sub-Saharan Africans have not gone through the type of natural selection for hard work in the last thousands of years that some Eurasians had.<sup id="fnref:41"><a href="#fn:41" class="footnote-ref" role="doc-noteref">41</a></sup>+++(4)+++</p>
<p>Wade also highlighted <em>A Farewell to Alms,</em> a book by the economist Gregory Clark suggesting that the reason the Industrial Revolution took off in Britain before it did elsewhere was the relatively high birth rate among wealthy people in Britain for the preceding five centuries compared to less wealthy people.+++(5)+++ Clark argued that this higher birth rate spread through the population the traits needed for a capitalist surge, including individualism, patience, and willingness to work long hours.<sup id="fnref:42"><a href="#fn:42" class="footnote-ref" role="doc-noteref">42</a></sup> Clark admits that he cannot distinguish between the transmission of genes and the transmission of culture across the generations, but Wade nevertheless takes his argument as evidence that genetics <em>might</em> have played a role.</p>
<p>I have spent some space discussing errors in Wade’s book because I feel it is important to explain that just because many academics have been engaged in trying to maintain an implausible orthodoxy, it does not mean that every unorthodox “heretic” is right. And yet Wade suggests precisely this. He writes, “Each of the major civilizations has developed the institutions appropriate for its circumstances and survival. But these institutions, though heavily imbued with cultural traditions, rest on a bedrock of genetically shaped human behavior. And when a civilization produces a distinctive set of institutions that endures for many generations, that is the sign of a supporting suite of variations in the genes that influence human social behavior.”<sup id="fnref:43"><a href="#fn:43" class="footnote-ref" role="doc-noteref">43</a></sup> In a written version of a nod and a wink, Wade is suggesting that popular racist ideas about the differences that exist among populations have something to them.</p>
<p>Wade is far from the only person who is convinced he knows the truth about the differences among populations. At the same 2010 meeting on “DNA, Genetics, and the History of Mankind” at which I first met Wade, I heard a rustling behind my shoulder and turned with a shock to see James Watson, who in 1953 codiscovered the structure of DNA. Watson had until a few years earlier been the director of the Cold Spring Harbor Laboratory at which the meeting was held. A century ago, the laboratory was the epicenter of the eugenics movement in the United States, keeping records on traits in many people to help guide selective breeding, and lobbying for legislation that was passed in many states to sterilize people considered to be defective and to combat a perceived degradation of the gene pool.+++(4)+++ It was ironic, then, that Watson was forced to retire as head of Cold Spring Harbor after being quoted in an interview with the British <em>Sunday Times</em> newspaper as having said that he was “inherently gloomy about the prospect of Africa,” adding that “[all] our social policies are based on the fact that their intelligence is the same as ours—whereas all the testing says not really.”<sup id="fnref:44"><a href="#fn:44" class="footnote-ref" role="doc-noteref">44</a></sup>+++(4)+++ (No genetic evidence for this claim exists.) When I saw Watson at Cold Spring Harbor, he leaned over and whispered to me and to the geneticist Beth Shapiro, who was sitting next to me, something to the effect of “When are you guys going to figure out why it is that you Jews are so much smarter than everyone else?” He then said that Jews and Indian Brahmins were both high achievers because of genetic advantages conferred by thousands of years of natural selection to be scholars.+++(5)+++ He went on to whisper that Indians in his experience were also servile, much like he thought they had been under British colonialism, and he speculated that this trait had come about because of selection under the caste system. He also talked about how East Asian students tended to be conformist, because of selection for conformity in ancient Chinese society.+++(4)+++</p>
<p>The pleasure Watson takes in challenging establishment views is legendary. His obstreperousness may have been important to his success as a scientist. But now as an eighty-two-year-old man, his intellectual rigor was gone, and what remained was a willingness to vent his gut impressions without subjecting them to any of the testing that characterized his scientific work on DNA.</p>
<p>Writing now, I shudder to think of Watson, or of Wade, or their forebears, behind my shoulder. The history of science has revealed, again and again, the danger of trusting one’s instincts or of being led astray by one’s biases—of being too convinced that one knows the truth. From the errors of thinking that the sun revolves around the earth, that the human lineage separated from the great ape lineage tens of millions of years ago, and that the present-day human population structure is fifty thousand years old whereas in fact we know that it was forged through population mixtures largely over the last five thousand years—from all of these errors and more, we should take the cautionary lesson not to trust our gut instincts or the stereotyped expectations we find around us. If we can be confident of anything, it is that whatever differences we think we perceive, our expectations are most likely wrong. What makes Watson’s and Wade’s and Harpending’s statements racist is the way they jump from the observation that the academic community is denying the possibility of differences that are plausible, to a claim with no scientific evidence<sup id="fnref:45"><a href="#fn:45" class="footnote-ref" role="doc-noteref">45</a></sup> that they know what those differences are and also that the differences correspond to long-standing popular stereotypes—a conviction that is essentially guaranteed to be wrong.</p>
<p>We truly have no idea right now what the nature or direction of genetically encoded differences among populations will be. An example is the extreme overrepresentation of people of West African ancestry among elite sprinters. All the male finalists in the Olympic hundred-meter race since 1980, even those from Europe and the Americas, had recent West African ancestry.<sup id="fnref:46"><a href="#fn:46" class="footnote-ref" role="doc-noteref">46</a></sup>+++(5)+++ The genetic hypothesis most often invoked to explain this is that there has been an upward shift in the average sprinting ability of people of West African ancestry due to natural selection. A small increase in the average might not sound like much, but it can make a big difference at the extremes of high ability—+++(5)+++ for example, a 0.8-standard-deviation increase in the average sprinting ability in West Africans would be expected to lead to a hundredfold enrichment in the proportion of people above the 99.9999999th percentile point in Europeans. But an alternative explanation that would predict the same magnitude of effect is that there is simply more variation in sprinting ability in people of West African ancestry—with more people of both very high and very low abilities.<sup id="fnref:47"><a href="#fn:47" class="footnote-ref" role="doc-noteref">47</a></sup>+++(5)+++ A wider spread of abilities around the same mean and a hundredfold enrichment in West Africans in the proportion of people above the 99.9999999th percentile point seen in Europeans is in fact exactly what is expected given the approximately 33 percent higher genetic diversity in West Africans than in Europeans.<sup id="fnref:48"><a href="#fn:48" class="footnote-ref" role="doc-noteref">48</a></sup> Whether or not this explains the dominance of West Africans in sprinting, for many biological traits—including cognitive ones—there is expected to be a higher proportion of sub-Saharan Africans with extreme genetically predicted abilities.+++(5)+++</p>
<p>So how should we prepare for the likelihood that in the coming years, genetic studies will show that behavioral or cognitive traits are influenced by genetic variation, and that these traits will differ on average across human populations, both with regard to their average and their variation within populations?+++(4)+++ Even if we do not yet know what those differences will be, we need to come up with a new way of thinking that can accommodate such differences, rather than deny categorically that differences can exist and so find ourselves caught without a strategy once they are found.</p>
<p>It would be tempting, in the wake of the genome revolution, to settle on a new comforting platitude, invoking the history of repeated admixture in the human past as an argument for population differences being meaningless. But such a statement is wrongheaded, as if we were to randomly pick two people living in the world today, we would find that many of the population lineages contributing to them have been isolated from each other for long enough that there has been ample opportunity for substantial average biological differences to arise between them. The right way to deal with the inevitable discovery of substantial differences across populations is to realize that their existence should not affect the way we conduct ourselves. As a society we should commit to according everyone equal rights despite the differences that exist among individuals. If we aspire to treat all individuals with respect regardless of the extraordinary differences that exist among individuals within a population, it should not be so much more of an effort to accommodate the smaller but still significant average differences across populations.</p>
<p>Beyond the imperative to give everyone equal respect, it is also important to keep in mind that there is a great diversity of human traits, including not just cognitive and behavioral traits, but also areas of athletic ability, skill with one’s hands, and capacity for social interaction and empathy. For most traits, the degree of variation among individuals is so large that any one person in any population can excel at any trait regardless of his or her population origin, even if particular populations have different average values due to a mixture of genetic and cultural influences. For most traits, hard work and the right environment are sufficient to allow someone with a lower genetically predicted performance at some task to excel compared to people with a higher genetically predicted performance. Because of the multidimensionality of human traits, the great variation that exists among individuals, and the extent to which hard work and upbringing can compensate for genetic endowment, the only sensible approach is to celebrate every person and every population as an extraordinary realization of our human genius and to give each person every chance to succeed, regardless of the particular average combination of genetic propensities he or she happens to display.</p>
<p>For me, the natural response to the challenge is to learn from the example of the biological differences that exist between males and females.+++(4)+++ The differences between the sexes are in fact more profound than those that exist among human populations, reflecting more than a hundred million years of evolution and adaptation. Males and females differ by huge tracts of genetic material—a Y chromosome that males have and that females don’t, and a second X chromosome that females have and males don’t. Most people accept that the biological differences between males and females are profound, and that they contribute to average differences in size and physical strength as well as in temperament and behavior, even if there are questions about the extent to which particular differences are also influenced by social expectations and upbringing (for example, many of the jobs in industry and the professions that women fill in great numbers today had few women in them a century ago).+++(4)+++ Today we aspire both to recognize that biological differences exist and to accord everyone the same freedoms and opportunities regardless of them. It is clear from the abiding average inequities that persist between women and men that fulfilling these aspirations is a challenge, and yet it is important to accommodate and even embrace the real differences that exist, while at the same time struggling to get to a better place.</p>
<p>The real offense of racism, in the end, is to judge individuals by a supposed stereotype of their group—to ignore the fact that when applied to specific individuals, stereotypes are almost always misleading. Statements such as “You are black, you must be musical” or “You are Jewish, you must be smart” are unquestionably very harmful. Everyone is his or her own person with unique strengths and weaknesses, and should be treated as such. Suppose you are the coach of a track-and-field team, and a young person walks on and asks to try out for the hundred-meter race, in which people of West African ancestry are statistically highly overrepresented, suggesting the possibility that genetics may play a role. For a good coach, race is irrelevant. Testing the young person’s sprinting speed is simple—take him or her out to the track to run against the stopwatch. Most situations are like this.</p>
<h2 id="a-new-basis-for-identity">A New Basis for Identity</h2>
<p>The genome revolution is actually a far more effective force for coming to a new understanding of human difference and identity—for understanding our own personal place in the world around us—than for promoting old beliefs that more often than not are mistaken.</p>
<p>To understand the power of the genome revolution for undermining old stereotypes about identity and building up a new basis for identity, consider how its finding of repeated mixture in human history has destroyed nearly every argument that used to be made for biologically based nationalism. The Nazi ideology of a “pure” Indo-European-speaking Aryan race with deep roots in Germany, traceable through artifacts of the Corded Ware culture, has been shattered by the finding that the people who used these artifacts came from a mass migration from the Russian steppe, a place that German nationalists would have despised as a source.<sup id="fnref:49"><a href="#fn:49" class="footnote-ref" role="doc-noteref">49</a></sup> The Hindutva +++(lemurian)+++ ideology that there was no major contribution to Indian culture from migrants from outside South Asia is undermined by the fact that approximately half of the ancestry of Indians today is derived from multiple waves of mass migration from Iran and the Eurasian steppe within the last five thousand years.<sup id="fnref:50"><a href="#fn:50" class="footnote-ref" role="doc-noteref">50</a></sup>+++(4)+++ Similarly, the idea that the Tutsis in Rwanda and Burundi have ancestry from West Eurasian farmers that Hutus do not—an idea that has been incorporated into arguments for genocide<sup id="fnref:51"><a href="#fn:51" class="footnote-ref" role="doc-noteref">51</a></sup>—is nonsense. We now know that nearly every group living today is the product of repeated population mixtures that have occurred over thousands and tens of thousands of years. Mixing is in human nature, and no one population is—or could be—“pure.”</p>
<p>Nonscientists have already realized the potential of the genome revolution for forming new narratives. African Americans have been at the forefront of this movement. During the slave trade, Africans were uprooted and forcibly deprived of their culture, with the effect that within a few generations much of their ancestors’ religion, language, and traditions were gone. In 1976, Alex Haley’s novel <em>Roots</em> used literature to begin to reclaim lost roots by recounting the odyssey of the slave Kunta Kinte and his descendants.<sup id="fnref:52"><a href="#fn:52" class="footnote-ref" role="doc-noteref">52</a></sup> Following in this tradition, Harvard professor of literature Henry Louis Gates Jr. has capitalized on the potential of genetic studies to recover lost roots for African Americans. In his <em>Faces of Americans</em> television series and the <em>Finding Your Roots</em> series that followed it, he declares to the cellist Yo-Yo Ma, who is able to trace his ancestry back to thirteenth-century China, that Gates, as an African American, will never know how that feels, but he shows that genetics can provide richly informative insights even for African Americans with limited genealogical records.<sup id="fnref:53"><a href="#fn:53" class="footnote-ref" role="doc-noteref">53</a></sup></p>
<p>A new industry, “personal ancestry testing,” has sprung up to capitalize on the potential of the genome revolution to form the basis for new narratives and to compare the genomes of consumers to others who have already been tested. The television programs that Gates has produced have been built around the idea of tracing the genealogies and DNA of celebrity guests, using the literary device of telling the personal stories of famous people to help viewers understand the power of genetic data to reveal features of their family’s past about which they could not otherwise have been aware. For example, the programs revealed unknown deep relationships between pairs of guests on the program (shared ancestors within the last few hundred years). They also used genetic tests to determine not only the continents on which people’s ancestors lived, but also the regions within continents.</p>
<p>As a white person in the United States with its history of forcible deprivation of peoples of their roots, I feel that everyone—African Americans and Native Americans especially—has the right to try to use genetic data to help fill in missing pieces in his or her family history. Nevertheless, for those who assume that personal ancestry testing results have the authority of science, it is important to keep in mind that many of the results are easily misinterpreted and rarely include the warnings that scientists attach to tentative findings.</p>
<p>Some of the best examples come from the industry that sprang up to provide genetic results to African Americans. One company is African Ancestry, which provides customers with information on the West African tribe and country in which their Y-chromosome or mitochondrial DNA type is most common. Such results are easy to overinterpret, as the frequencies of Y-chromosome and mitochondrial DNA types are too similar across West Africa to make exact determinations with confidence. As an example, consider a Y-chromosome type that is carried slightly more often in the Hausa ethnic group than in the neighboring Yoruba, Mende, Fulani, and Beni groups. When African Ancestry sends its report, it might state that an African American man has a Y-chromosome type that is most common in the Hausa.<sup id="fnref:54"><a href="#fn:54" class="footnote-ref" role="doc-noteref">54</a></sup> But it is quite possible and even likely that the true ancestor was not the Hausa, because there are many tribes in West Africa, and no one tribe contributed more than a modest fraction of the African ancestry of African Americans.<sup id="fnref:55"><a href="#fn:55" class="footnote-ref" role="doc-noteref">55</a></sup>+++(4)+++ And yet people who have taken these tests often return with the impression that they know their origin. The geneticist Rick Kittles, a population geneticist who is the cofounder of African Ancestry, described this feeling, asserting, “My female line goes back to northern Nigeria, the land of the Hausa tribe. I then went to Nigeria and talked to people and learned about the Hausa’s culture and tradition. That gave me a sense about who I am.”<sup id="fnref:56"><a href="#fn:56" class="footnote-ref" role="doc-noteref">56</a></sup> Whole-genome ancestry tests in theory have much more power than tests based on Y chromosomes and mitochondrial DNA. But at present, even whole-genome methods are not good enough to provide high-resolution information about where the ancestors of an African American person lived within Africa, in part because the databases of present-day populations in West Africa are not complete enough. Much more research needs to be done to make it possible to carry out studies like these with any reliability.</p>
<p>For African Americans, another frustration may be that the cultural upheaval that occurred after African slaves arrived in North America has been so enormous that today there are few differences among African Americans with respect to the places in Africa from which their ancestors came. Africans from one part of the continent were traded around and mixed with those from another, with the result that within a few generations the great cultural diversity and variation of ancestry that existed among the first slaves were blurred to the point of unrecognizability. The nearly complete homogenization of African ancestry that occurred was evident in an unpublished study I carried out in 2012 with Kasia Bryc, who analyzed genome-wide data from more than fifteen thousand African Americans from Chicago, New York, San Francisco, Mississippi, North Carolina, and the South Carolina Sea Islands, and tested if some African American populations were more closely related to particular West Africans than others, as might be expected based on the heterogeneous supply routes for U.S. slaves.<sup id="fnref:57"><a href="#fn:57" class="footnote-ref" role="doc-noteref">57</a></sup> It made sense to expect some differences. Of the four big slave ports, New Orleans was supplied mostly by French slave traders, whereas Baltimore, Savannah, and Charleston were supplied mostly by the British drawing from different points in Africa. But what we found is that the mixing of the West African ancestors of African Americans has been so thorough that we could not detect any differences in the African source populations for mainland populations.+++(5)+++ Only in the Sea Islands off South Carolina did we detect evidence of a particular connection to one place in Africa, in this case to people of the country of Sierra Leone, the place of origin of the language with an African grammar still spoken by Gullah Sea Islanders. It will take ancient DNA studies of first-generation enslaved Africans to actually trace roots to Africa.<sup id="fnref:58"><a href="#fn:58" class="footnote-ref" role="doc-noteref">58</a></sup>+++(4)+++</p>
<p>The problem with the results sometimes provided by personal ancestry testing companies is not limited to African Americans. It is a more general pitfall that stems from the financial incentive that such companies have to provide people with what feel like meaningful findings. This is a problem even for the most rigorous of the companies. Between 2011 and 2015, the genetic testing company 23andMe provided customers with an estimate of their proportion of Neanderthal ancestry, allowing them to make a personal connection to the research showing that non-Africans derive around 2 percent of their genomes from Neanderthals.<sup id="fnref:59"><a href="#fn:59" class="footnote-ref" role="doc-noteref">59</a></sup> The measurement made by the test was highly inaccurate, however, since the true variation in Neanderthal proportion within most populations is only a few tenths of a percent, and the test reports variation of a few percentage points.<sup id="fnref:60"><a href="#fn:60" class="footnote-ref" role="doc-noteref">60</a></sup> Several people have told me excitedly that their 23andMe Neanderthal testing result put them in the top few percent of people in the world in Neanderthal ancestry, but because of the test’s inaccuracies, the probability that people who got such a high 23andMe Neanderthal reading really do have more than the average proportion of Neanderthal ancestry is only slightly greater than 50/50.+++(4)+++ I raised this problem to members of the 23andMe team and even highlighted the problems in a 2014 scientific paper.<sup id="fnref:61"><a href="#fn:61" class="footnote-ref" role="doc-noteref">61</a></sup> Later, 23andMe changed its report to no longer provide these statements. However, the company continues to provide its customers with a ranking of the number of Neanderthal-derived mutations they carry.<sup id="fnref:62"><a href="#fn:62" class="footnote-ref" role="doc-noteref">62</a></sup> This ranking, too, does not provide strong evidence that customers have inherited more Neanderthal DNA than their population average.</p>
<p>Not all the findings reported by the personal ancestry companies are inaccurate, and many people have obtained what for them is satisfying information from such testing, especially when it comes to tracing genealogies where the paper trail runs cold. One example is adoptees seeking their biological parents. Another is tracking down extended families.</p>
<p>From my own perspective, though, I do not find this approach to be satisfying. In preparing to write this book, I considered whether I should send my DNA to a personal testing company or study it in my own lab, and then describe the results, in imitation of the approach taken by many journalists covering the field of personal ancestry testing. But honestly, I am not interested. My own group—Ashkenazi Jews—is already overstudied. I am confident that my genome will be much like that of anyone else from this population. I would much rather use any resources I have to sequence the genomes of people who are understudied. I am also worried about the intellectual pitfall of self-study. I am innately suspicious of scientists who are hyper-interested in their own family or culture.+++(4 Genetically? predisposed to be navyonmatta.)+++ They simply care too much. In my own laboratory, there are researchers from all over the world, and I encourage them, not always successfully, to choose projects on peoples not their own.+++(4)+++ For me, the approach of using the genome as a tool to connect myself to the world around me through personal links of family and tribe seems parochial and unfulfilling.</p>
<p>What the genome revolution has given us, though, is an even more important way to come to grips with who we are—a way to hold in our minds the extraordinary human diversity that exists today and has existed in our past. The problem of understanding the connections between self and the world is a central one for me, and has driven my lifelong interest in geography, history, and biology. Ironically for a person like myself, who is not at all religious, it is an example from the Bible that provides me with insight into how the genome revolution might be able to help solve this existential problem.</p>
<p>Every year on the holiday of Passover, Jews sit around the dinner table and recount the story of the Exodus from Egypt. The Passover holiday is important to Jews because it reminds them of their place in the world and encourages them to draw lessons about how they should behave. This narrative has been extraordinarily successful, as measured by the fact that it has sustained Jews in their identity for thousands of years as a minority living in foreign lands.</p>
<p>The Passover story begins with the myth of the patriarchs in ancient Israel: the first generation of Abraham and Sarah; the second of Isaac and Rebecca; the third of Jacob, Leah, Rachel, Bilhah, and Zilpah; and the fourth generation of twelve male children (the forefathers of the tribes of Israel) and a daughter, Dinah. These people are too removed from the huge populations of today to seem meaningfully connected to the present. The literary device that connects this ancient family to the multitudes that follow is Joseph, one of the sons of Jacob, who is sold by his brothers into slavery in Egypt, and who rises to a position of great power. When a famine strikes the land, the rest of the family also migrates to Egypt, where they are welcomed by Joseph despite the earlier crime they had committed against him.+++(4)+++ Four hundred years pass, and their descendants exponentially multiply into a nation numbering more than six hundred thousand military-age men and an even larger number of women and children. Under the leadership of Moses, they break their bonds of oppression, wander for dozens of years, and work out their code of laws. They then return to the Promised Land of their ancestors.</p>
<p>After reading the Passover story, Jews intuitively understand how within their population, numbering millions of people, they are related to each other and the past. The story allows Jews to think of those millions of coreligionists as direct relations—and to treat them with equal respect and seriousness even if they do not understand their exact relationships—to break out from the trap of thinking of the world from the perspective of the relatively small families we were raised in.</p>
<p>For me, the multitude of interconnected populations that have contributed to each of our genomes provide a similar narrative that helps me to understand my own place in the world and to avoid being daunted by the vast number of people in our species—the immensity of the human population numbering in the billions. The centrality of mixture in the history of our species, as revealed in just the last few years by the genome revolution, means that we are all interconnected and that we will all keep connecting with one another in the future. This narrative of connection allows me to feel Jewish even if I may not be descended from the matriarchs and patriarchs of the Bible. I feel American, even if I am not descended from indigenous Americans or the first European or African settlers. I speak English, a language not spoken by my ancestors a hundred years ago. I come from an intellectual tradition, the European Enlightenment, which is not that of my direct ancestors. I claim these as my own, even if they were not invented by my ancestors, even if I have no close genetic relationship to them. Our particular ancestors are not the point. The genome revolution provides us with a shared history that, if we pay proper attention, should give us an alternative to the evils of racism and nationalism, and make us realize that we are all entitled equally to our human heritage.+++(4 Bought into the idiocy.)+++</p>
<div class="footnotes" role="doc-endnotes">
<hr>
<ol>
<li id="fn:1">
<p>Centers for Disease Control and Prevention, “Prostate Cancer Rates by Race and Ethnicity,” <a href="https://www.cdc.gov/cancer/prostate/statistics/race.htm">https://www.cdc.gov/cancer/prostate/statistics/race.htm</a>.&#160;<a href="#fnref:1" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:2">
<p>N. Patterson et al., “Methods for High-Density Admixture Mapping of Disease Genes,” <em>American Journal of Human Genetics</em> 74 (2004): 979–1000; M. W. Smith et al., “A High-Density Admixture Map for Disease Gene Discovery in African Americans,” <em>American Journal of Human Genetics</em> 74 (2004): 1001–13.&#160;<a href="#fnref:2" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:3">
<p>M. L. Freedman et al., “Admixture Mapping Identifies 8q24 as a Prostate Cancer Risk Locus in African-American Men,” <em>Proceedings of the National Academy of Sciences of the U.S.A.</em> 103 (2006): 14068–73.&#160;<a href="#fnref:3" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:4">
<p>C. A. Haiman et al., “Multiple Regions within 8q24 Independently Affect Risk for Prostate Cancer,” <em>Nature Genetics</em> 39 (2007): 638–44.&#160;<a href="#fnref:4" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:5">
<p>Freedman et al., “Admixture Mapping Identifies 8q24.”&#160;<a href="#fnref:5" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:6">
<p>M. F. Ashley Montagu, <em>Man’s Most Dangerous Myth: The Fallacy of Race</em> (New York: Columbia University Press, 1942).&#160;<a href="#fnref:6" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:7">
<p>R. C. Lewontin, “The Apportionment of Human Diversity,” <em>Evolutionary Biology</em> 6 (1972): 381–98.&#160;<a href="#fnref:7" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:8">
<p>J. M. Stevens, “The Feasibility of Government Oversight for NIH-Funded Population Genetics Research,” in <em>Revisiting Race in a Genomic Age</em> (Studies in Medical Anthropology), ed. Barbara A. Koenig, Sandra Soo-Jin Lee, and Sarah S. Richardson (New Brunswick, NJ: Rutgers University Press, 2008), 320–41; J. Stevens, “Racial Meanings and Scientific Methods: Policy Changes for NIH-Sponsored Publications Reporting Human Variation,” <em>Journal of Health Policy, Politics and Law</em> 28 (2003): 1033–87.&#160;<a href="#fnref:8" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:9">
<p>N. A. Rosenberg et al., “Genetic Structure of Human Populations,” <em>Science</em> 298 (2002): 2381–85.&#160;<a href="#fnref:9" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:10">
<p>D. Serre and S. Pääbo, “Evidence for Gradients of Human Genetic Diversity Within and Among Continents,” <em>Genome Research</em> 14 (2004): 1679–85; F. B. Livingstone, “On the Non-Existence of Human Races,” <em>Current Anthropology</em> 3 (1962): 279.&#160;<a href="#fnref:10" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:11">
<p>J. Dreyfuss, “Getting Closer to Our African Origins,” <em>The Root</em>, October 17, 2011, <a href="http://www.theroot.com/getting-closer-to-our-african-origins-1790866394">www.theroot.com/getting-closer-to-our-african-origins-1790866394</a>.&#160;<a href="#fnref:11" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:12">
<p>N. A. Rosenberg et al., “Clines, Clusters, and the Effect of Study Design on the Inference of Human Population Structure,” <em>PLoS Genetics</em> 1 (2005): e70.&#160;<a href="#fnref:12" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:13">
<p>E. G. Burchard et al., “The Importance of Race and Ethnic Background in Biomedical Research and Clinical Practice,” <em>New England Journal of Medicine</em> 348 (2003): 1170–75.&#160;<a href="#fnref:13" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:14">
<p>J. F. Wilson et al., “Population Genetic Structure of Variable Drug Response,” <em>Nature Genetics</em> 29 (2001): 265–69.&#160;<a href="#fnref:14" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:15">
<p>D. Fullwiley, “The Biologistical Construction of Race: ‘Admixture’ Technology and the New Genetic Medicine,” <em>Social Studies of Science</em> 38 (2008): 695–735.&#160;<a href="#fnref:15" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:16">
<p>Lewontin, “The Apportionment of Human Diversity”; A. R. Templeton, “Biological Races in Humans,” <em>Studies in History and Philosophy of Biological and Biomedical Science</em> 44 (2013): 262–71.&#160;<a href="#fnref:16" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:17">
<p><em>Razib Khan,</em> <a href="http://www.razib.com/wordpress">www.razib.com/wordpress</a>.&#160;<a href="#fnref:17" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:18">
<p><em>Dienekes’ Anthropology Blog,</em> <a href="http://dienekes.blogspot.com">dienekes.blogspot.com</a>.&#160;<a href="#fnref:18" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:19">
<p><em>Eurogenes Blog,</em> <a href="http://eurogenes.blogspot.com">http://eurogenes.blogspot.com</a>.&#160;<a href="#fnref:19" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:20">
<p>Léon Poliakov, <em>The Aryan Myth: A History of Racist and Nationalist Ideas in Europe</em> (New York: Basic Books, 1974).&#160;<a href="#fnref:20" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:21">
<p>B. Arnold, “The Past as Propaganda: Totalitarian Archaeology in Nazi Germany,” <em>Antiquity</em> 64 (1990): 464–78.&#160;<a href="#fnref:21" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:22">
<p>J. K. Pritchard, J. K. Pickrell, and G. Coop, “The Genetics of Human Adaptation: Hard Sweeps, Soft Sweeps, and Polygenic Adaptation,” <em>Current Biology</em> 20 (2010): R208–15; R. D. Hernandez et al., “Classic Selective Sweeps Were Rare in Recent Human Evolution,” <em>Science</em> 331 (2011): 920–24.&#160;<a href="#fnref:22" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:23">
<p>M. C. Turchin et al., “Evidence of Widespread Selection on Standing Variation in Europe at Height-Associated SNPs,” <em>Nature Genetics</em> 44 (2012): 1015–19.&#160;<a href="#fnref:23" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:24">
<p>Y. Field et al., “Detection of Human Adaptation During the Past 2000 Years,” <em>Science</em> 354 (2016): 760–64.&#160;<a href="#fnref:24" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:25">
<p>A. Okbay et al., “Genome-Wide Association Study Identifies 74 Loci Associated with Educational Attainment,” <em>Nature</em> 533 (2016): 539–42.&#160;<a href="#fnref:25" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:26">
<p>To compute the expected difference in number of years of education between the highest 5 percent and lowest 5 percent of genetically predicted educational attainment based on the numbers in the 2016 study by Benjamin and colleagues, I performed the following computation: (1) The number of years of education in the cohort analyzed by Benjamin and colleagues is quoted as 14.3 ± 3.7. I estimated the standard deviation of 3.7 years from the fact that the study estimates the effect size in weeks to be “0.014 to 0.048 standard deviations per allele (2.7 to 9.0 weeks of schooling).” These numbers translate to 188 ( = 9.0 / 0.048) to 193 ( = 2.7 / 0.014) weeks. Dividing by 52 weeks per year gives 3.7. (2) Benjamin and colleagues also report a genetic predictor of number of years of education that explains 3.2 percent of the variance of the trait. Therefore, the correlation between the predicted value and the actual value is √0.032 = 0.18. We can model this mathematically using a two-dimensional normal distribution. (3) The probability that a person who is in the bottom 5% of the predicted distribution (more than 1.64 standard deviations below the average) has more than 12 years of education is then given by the proportion of people who are in the bottom 5 percent of the predicted distribution and also have more than 12 years of education (which can be calculated by measuring the area of the two-dimensional normal distribution that matches these criteria), divided by 0.05. This gives a probability of 60 percent. A similar calculation for the proportion of people in the top 5 percent of the predicted distribution gives a probability of 84 percent. (4) The Benjamin study also suggests that with enough samples it would be possible to build a reliable genetic predictor that accounts for 20 percent of the variance. Redoing the calculation using 20 percent instead of 3.2 percent leads to a prediction that 37 percent of people in the bottom 5 percent of the predicted distribution would complete twelve years of education compared to 96 percent of the top 5 percent.&#160;<a href="#fnref:26" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:27">
<p>A. Kong et al., “Selection Against Variants in the Genome Associated with Educational Attainment,” <em>Proceedings of the National Academy of Sciences of the U.S.A.</em> 114 (2017): E727–32.&#160;<a href="#fnref:27" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:28">
<p>Kong et al., “Selection Against Variants,” estimate that the genetically predicted number of years of education has decreased by an estimated 0.1 standard deviations over the last century under the pressure of natural selection.&#160;<a href="#fnref:28" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:29">
<p>G. Davies et al., “Genome-Wide Association Study of Cognitive Functions and Educational Attainment in UK Biobank (N=112 151),” <em>Molecular Psychiatry</em> 21 (2016): 758–67; M. T. Lo et al., “Genome-Wide Analyses for Personality Traits Identify Six Genomic Loci and Show Correlations with Psychiatric Disorders,” <em>Nature Genetics</em> 49 (2017): 152–56.&#160;<a href="#fnref:29" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:30">
<p>S. Sniekers et al., “Genome-Wide Association Meta-Analysis of 78,308 Individuals Identifies New Loci and Genes Influencing Human Intelligence,” <em>Nature Genetics</em> 49 (2017): 1107–12.&#160;<a href="#fnref:30" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:31">
<p>I. Mathieson et al., “Genome-wide Patterns of Selection in 230 Ancient Eurasians,” <em>Nature</em> 528 (2015): 499–503; Field et al., “Detection of Human Adaptation.”&#160;<a href="#fnref:31" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:32">
<p>N. A. Rosenberg et al., “Genetic Structure of Human Populations,” <em>Science</em> 298 (2002): 2381–85.&#160;<a href="#fnref:32" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:33">
<p>S. Ramachandran et al., “Support from the Relationship of Genetic and Geographic Distance in Human Populations for a Serial Founder Effect Originating in Africa,” <em>Proceedings of the National Academy of Sciences of the U.S.A.</em> 102 (2005): 15942–47; B. M. Henn, L. L. Cavalli-Sforza, and M. W. Feldman, “The Great Human Expansion,” <em>Proceedings of the National Academy of Sciences of the U.S.A.</em> 109 (2012): 17758–64.&#160;<a href="#fnref:33" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:34">
<p>J. K. Pickrell and D. Reich, “Toward a New History and Geography of Human Genes Informed by Ancient DNA,” <em>Trends in Genetics</em> 30 (2014): 377–89.&#160;<a href="#fnref:34" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:35">
<p>M. Raghavan et al., “Upper Palaeolithic Siberian Genome Reveals Dual Ancestry of Native Americans,” <em>Nature</em> (2013): doi: 10.1038/nature 12736.&#160;<a href="#fnref:35" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:36">
<p>I. Lazaridis et al., “Genomic Insights into the Origin of Farming in the Ancient Near East,” <em>Nature</em> 536 (2016): 419–24.&#160;<a href="#fnref:36" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:37">
<p>Nicholas Wade, <em>A Troublesome Inheritance: Genes, Race and Human History</em> (New York: Penguin Press, 2014).&#160;<a href="#fnref:37" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:38">
<p>G. Coop et al., “A Troublesome Inheritance” (letters to the editor), <em>New York Times,</em> August 8, 2014.&#160;<a href="#fnref:38" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:39">
<p>G. Cochran, J. Hardy, and H. Harpending, “Natural History of Ashkenazi Intelligence,” <em>Journal of Biosocial Science</em> 38 (2006): 659–93.&#160;<a href="#fnref:39" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:40">
<p>P. F. Palamara, T. Lencz, A. Darvasi, and I. Pe’er, “Length Distributions of Identity by Descent Reveal Fine-Scale Demographic History,” <em>American Journal of Human Genetics</em> 91 (2012): 809–22; M. Slatkin, “A Population-Genetic Test of Founder Effects and Implications for Ashkenazi Jewish Diseases,” <em>American Journal of Human Genetics</em> 75 (2004): 282–93.&#160;<a href="#fnref:40" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:41">
<p>H. Harpending, “The Biology of Families and the Future of Civilization” (minute 38), Preserving Western Civilization, 2009 Conference, audio available at <a href="http://www.preservingwesternciv.com/audio/07%20Prof._Henry_Harpending--The_Biology_of_Families_and_the_Future_of_Civilization.mp3">www.preservingwesternciv.com/audio/07%20Prof._Henry_Harpending&ndash;The_Biology_of_Families_and_the_Future_of_Civilization.mp3</a> (2009).&#160;<a href="#fnref:41" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:42">
<p>G. Clark, “Genetically Capitalist? The Malthusian Era, Institutions and the Formation of Modern Preferences” (2007), <a href="http://www.econ.ucdavis.edu/faculty/gclark/papers/Capitalism%20Genes.pdf">www.econ.ucdavis.edu/faculty/gclark/papers/Capitalism%20Genes.pdf</a>; Gregory Clark, <em>A Farewell to Alms: A Brief Economic History of the World</em> (Princeton, NJ: Princeton University Press, 2007).&#160;<a href="#fnref:42" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:43">
<p>Wade, <em>A Troublesome Inheritance</em>.&#160;<a href="#fnref:43" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:44">
<p>C. Hunt-Grubbe, “The Elementary DNA of Dr. Watson,” <em>The Sunday Times,</em> October 14, 2017.&#160;<a href="#fnref:44" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:45">
<p>Coop et al. letters, <em>New York Times</em>.&#160;<a href="#fnref:45" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:46">
<p>David Epstein, <em>The Sports Gene: Inside the Science of Extraordinary Athletic Performance</em> (New York: Current, 2013).&#160;<a href="#fnref:46" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:47">
<p>Ibid.&#160;<a href="#fnref:47" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:48">
<p>I performed this computation as follows. (1) The 99.9999999th percentile of a trait corresponds to 6.0 standard deviations from the mean, whereas the 99.99999th percentile corresponds to 5.2 standard deviations. Thus a 0.8-standard-deviation shift corresponds to a hundredfold enrichment of individuals. (2) I assumed that the 1.33-fold higher genetic variation in sub-Saharan Africans applies not just to random mutations in the genome, but also to mutations modulating biological traits. The standard deviation is thus expected to be 1.15 = √1.33-fold higher in sub-Saharan Africans based on a formula in J. J. Berg and G. Coop, “A Population Genetic Signal of Polygenic Adaptation,” <em>PLoS Genetics</em> 10 (2014): e1004412, so the 6.0-standard-deviation cutoff in non-Africans corresponds to 5.2 = 6.0 / 1.15 of that in sub-Saharan Africans, leading to the same predicted hundredfold enrichment above the 99.9999999th percentile.&#160;<a href="#fnref:48" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:49">
<p>W. Haak et al., “Massive Migration from the Steppe Was a Source for Indo-European Languages in Europe,” <em>Nature</em> 522 (2015): 207–11; M. E. Allentoft et al., “Population Genomics of Bronze Age Eurasia,” <em>Nature</em> 522 (2015): 167–72.&#160;<a href="#fnref:49" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:50">
<p>D. Reich et al., “Reconstructing Indian Population History,” <em>Nature</em> 461 (2009): 489–94; Lazaridis et al., “Genomic Insights.”&#160;<a href="#fnref:50" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:51">
<p>Michael F. Robinson, <em>The Lost White Tribe: Explorers, Scientists, and the Theory That Changed a Continent</em> (New York: Oxford University Press, 2016).&#160;<a href="#fnref:51" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:52">
<p>Alex Haley, <em>Roots: The Saga of an American Family</em> (New York: Doubleday, 1976).&#160;<a href="#fnref:52" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:53">
<p>“Episode 4: (2010) Know Thyself” (minute 17) in <em>Faces of America with Henry Louis Gates Jr.</em>, <a href="http://www.pbs.org/wnet/facesofamerica/video/episode-4-know-thyself/237/">http://www.pbs.org/wnet/facesofamerica/video/episode-4-know-thyself/237/</a>.&#160;<a href="#fnref:53" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:54">
<p>African Ancestry, “Frequently Asked Questions,” “About the Results,” question 3 (2016), <a href="http://www.africanancestry.com/faq/">http://www.africanancestry.com/faq/</a>.&#160;<a href="#fnref:54" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:55">
<p>Dreyfuss, “Getting Closer to Our African Origins.”&#160;<a href="#fnref:55" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:56">
<p>S. Sailer, “African Ancestry Inc. Traces DNA Roots,” United Press International, April 28, 2003, <a href="http://www.upi.com/inc/view.php?StoryID=20030428-074922-7714r">www.upi.com/inc/view.php?StoryID=20030428-074922-7714r</a>.&#160;<a href="#fnref:56" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:57">
<p>Unpublished results from David Reich’s laboratory.&#160;<a href="#fnref:57" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:58">
<p>H. Schroeder et al., “Genome-Wide Ancestry of 17th-Century Enslaved Africans from the Caribbean,” <em>Proceedings of the National Academy of Sciences of the U.S.A.</em> 112 (2015): 3669–73.&#160;<a href="#fnref:58" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:59">
<p>R. E. Green et al., “A Draft Sequence of the Neanderthal Genome,” <em>Science</em> 328 (2010): 710–22.&#160;<a href="#fnref:59" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:60">
<p>E. Durand, 23andMe: “White Paper 23-05: Neanderthal Ancestry Estimator” (2011), <a href="https://web.stanford.edu/class/gene210/files/readings/23andme_Neanderthal_Ancestry.pdf">https://web.stanford.edu/class/gene210/files/readings/23andme_Neanderthal_Ancestry.pdf</a>; S. Sankararaman et al., “The Genomic Landscape of Neanderthal Ancestry in Present-Day Humans,” <em>Nature</em> 507 (2014): 354–57.&#160;<a href="#fnref:60" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:61">
<p>Sankararaman et al., “Genomic Landscape.”&#160;<a href="#fnref:61" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:62">
<p><a href="https://customercare.23andme.com/hc/en-us/articles/212873707-Neanderthal-Report-Basics">https://customercare.23andme.com/hc/en-us/articles/212873707-Neanderthal-Report-Basics</a>, #13514.&#160;<a href="#fnref:62" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
</ol>
</div>

  </div>
</article>


        

        



    <aside class="card border noPrint" id="section-tree-item-sapiens_branches_articles_reich_who-we-are_3_The_Disruptive_Genome_11_The_Genomics_of_Race_and_Identity_md">
        <div class="card-title bg-light-gray border d-flex justify-content-between">
            <a href="https://vishvAsa.github.io/notes/sapiens/branches/articles/reich_who-we-are/3_The_Disruptive_Genome/11_The_Genomics_of_Race_and_Identity/">11 The Genomics of Race and Identity </a>
            <a data-toggle="collapse" href="#section-tree-item-body-sapiens_branches_articles_reich_who-we-are_3_The_Disruptive_Genome_11_The_Genomics_of_Race_and_Identity_md" role="button" aria-expanded="false" aria-controls="section-tree-item-body-sapiens_branches_articles_reich_who-we-are_3_The_Disruptive_Genome_11_The_Genomics_of_Race_and_Identity_md" >…<i class="fas fa-caret-down" class="collapsed"></i> </a>
        </div>
        <nav id="section-tree-item-body-sapiens_branches_articles_reich_who-we-are_3_The_Disruptive_Genome_11_The_Genomics_of_Race_and_Identity_md" class="card-body p-0 collapse">
            
            <li>draft: false</li>
            
            <li>iscjklanguage: false</li>
            
            <li>title: 11 The Genomics of Race and Identity</li>
            
        </nav>
    </aside>


      </main>
    </div>
    <footer class="bg-yellow  p-1 noPrint" role="contentinfo">
  

  <div id="div_foot_nav_bar" class="row">
    <a href="https://vishvAsa.github.io/notes/search"  class="btn btn-light btn-sm">🥽🔦</a>
    <div class="row col  d-flex justify-content-center">
      <div><a id="previousPageFoot" href="" class="btn btn-secondary">###<i class="fas fa-caret-left"></i></a></div>
      <div ><a id="nextPageFoot" href="" class="btn btn-secondary"><i class="fas fa-caret-right"></i> ### </a></div>
    </div>
      <ul id="footer-bar-right-custom" class="list-group list-group-horizontal">
      </ul>
  </div>
  <div id="div_footer_bar" class="container-fluid d-flex justify-content-between">
    <a class="btn btn-secondary" href="https://github.com/vishvAsa/notes/issues/new" >
      ✉️Contact
    </a>
    <a class="btn btn-secondary" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/80x15.png" /></a>
    <div class="btn small border">
      
      <a href="https://github.com/vishvAsa/notes/actions/workflows/build.yml"><img src="https://github.com/vishvAsa/notes/actions/workflows/build.yml/badge.svg"> Built </a> on 2025 Oct 31 15:25:51 UTC. (<a href="http://google.com/search?q=15%3a25%3a51%20UTC to IST">IST</a>)
    </div>
  </div>
</footer>



<script async src="https://www.googletagmanager.com/gtag/js?id=G-33H08JSLQR"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-33H08JSLQR');
</script>

  </body>
  <script type='text/javascript'>
    
    
    
    module_uiLib.onDocumentReadyTasks();
  </script>
  
</html>
